<template class="task-template" id="iSanXoT-Help">

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:Roboto;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:center;
	line-height:107%;
	page-break-after:avoid;
	border:none;
	padding:0cm;
	font-size:26.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:normal;
	font-style:italic;}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-link:"Footnote Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:10.0pt;
	margin-left:0cm;
	text-align:center;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoFootnoteReference
	{vertical-align:super;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpFirst, li.MsoTitleCxSpFirst, div.MsoTitleCxSpFirst
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpMiddle, li.MsoTitleCxSpMiddle, div.MsoTitleCxSpMiddle
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpLast, li.MsoTitleCxSpLast, div.MsoTitleCxSpLast
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
a:link, span.MsoHyperlink
	{font-family:"Calibri",sans-serif;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
p
	{margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoQuote, li.MsoQuote, div.MsoQuote
	{mso-style-link:"Quote Char";
	margin-top:10.0pt;
	margin-right:43.2pt;
	margin-bottom:8.0pt;
	margin-left:43.2pt;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#404040;
	font-style:italic;}
span.MsoIntenseReference
	{font-variant:small-caps;
	color:#4472C4;
	letter-spacing:.25pt;
	font-weight:bold;}
span.MsoBookTitle
	{letter-spacing:.25pt;
	font-weight:bold;
	font-style:italic;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-link:Title;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-link:"Footnote Text";}
p.lead, li.lead, div.lead
	{mso-style-name:lead;
	margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.QuoteChar
	{mso-style-name:"Quote Char";
	mso-style-link:Quote;
	color:#404040;
	font-style:italic;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:595.3pt 841.9pt;
	margin:70.85pt 3.0cm 70.85pt 3.0cm;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>

</head>

<body lang=EN-GB link="#000000" vlink="#954F72">

<div class=WordSection1>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitle align=center style='text-align:center'><b><i><span
style='font-size:36.0pt;letter-spacing:.25pt'><img width=310 height=355
id="Picture 16" src="iSanXoT-Help_files/image001.png" alt=Molecule></span></i></b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitleCxSpFirst align=center style='text-align:center'><span
class=MsoBookTitle><span style='font-size:36.0pt'>Help and documentation</span></span></p>

<p class=MsoTitleCxSpLast align=center style='text-align:center'><span
class=MsoBookTitle>(0.4.1)</span></p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1>Table of contents</h1>

</div>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span>Introduction</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Symbol;letter-spacing:
0pt;font-weight:normal;font-style:normal'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Installation">Installation</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoHyperlink><span style='font-family:"Courier New";text-decoration:none'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
class=MsoBookTitle><span style='line-height:107%;letter-spacing:0pt;font-weight:
normal;font-style:normal'><a href="#_Windows_distribution">Windows distribution</a></span></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_MacOS_distribution">MacOS distribution</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Linux_distribution">Linux distribution</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Symbol;letter-spacing:
0pt;font-weight:normal;font-style:normal'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Get_Started">Get Started</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Symbol;letter-spacing:
0pt;font-weight:normal;font-style:normal'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Modules">Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Basic_modules">Basic Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_RELS_CREATOR">RELS CREATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_LEVEL_CREATOR">LEVEL CREATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_LEVEL_CALIBRATOR">LEVEL CALIBRATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_INTEGRATE">INTEGRATE</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_NORCOMBINE">NORCOMBINE</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_RATIOS">RATIOS</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a href="#_SBT">SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Report_modules">Report Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_REPORT">REPORT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_SANSON">SANSON</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Complete_Modules">Complete Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WSPP-SBT">WSPP-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WSPPG-SBT">WSPPG-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WPP-SBT">WPP-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoHyperlink><span style='font-family:Wingdings;
text-decoration:none'>§<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WPPG-SBT">WPPG-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Special_parameters">Special Parameters</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Keywords_for_the">Keywords for the Task-Tables</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Multiple_samples">Multiple samples</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Asterisk_is_our">Asterisk is our jack of all trades</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_More_params">More params</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span style='line-height:107%;font-family:
Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Filter_param_(in">Filter (in REPORT module)</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Symbol;letter-spacing:
0pt;font-weight:normal;font-style:normal'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Adaptors">Adaptors</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Main_Inputs">Main-Input</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoHyperlink><span style='font-family:"Courier New";text-decoration:none'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Inputs_from_PD">Close search: Inputs from PD</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Inputs_from_MSFragger">Close search: Inputs from MSFragger</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Inputs_from_MaxQuant">Close search: Inputs from MaxQuant</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Symbol;letter-spacing:
0pt;font-weight:normal;font-style:normal'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Workflows">Workflows</a></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_References">References</a></p>

<span class=MsoBookTitle><span style='font-size:11.0pt;line-height:107%;
font-family:"Calibri",sans-serif;letter-spacing:0pt;font-weight:normal;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></span>

<p class=MsoNormal align=left style='text-align:left'><span class=MsoBookTitle><span
style='line-height:107%;letter-spacing:0pt;font-weight:normal;font-style:normal'>&nbsp;</span></span></p>

<p class=MsoNormal><span class=MsoBookTitle><span style='line-height:107%;
letter-spacing:0pt;font-weight:normal;font-style:normal'>&nbsp;</span></span></p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Introduction"></a>Introduction</h1>

</div>

<p class=MsoNormal>iSanXoT is a Graphical User Interface application that
executes workflows for quantitative high-throughput proteomics, systems biology
and the statistical analysis, integration and comparison of experiments, based
on SanXoT (Trevisan-Herraz <i>et al</i>., 2019) integrations.</p>

<p class=MsoNormal style='background:white'><span style='font-size:9.0pt;
line-height:107%;font-family:"Roboto",serif;color:#212529'>&nbsp;</span></p>

<h3>Workflows composed by multiple modules</h3>

<p class=MsoNormal>The workflows developed with iSanXoT follow a modular
structure, where the main modules are: INTEGRATE, NORCOMBINE, SBT, RATIOS,
LEVEL CREATOR, and LEVEL CALIBRATOR. The data of each level are analysed
independently and sequentially.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;background:white'><span
style='font-size:9.0pt;line-height:107%;font-family:"Roboto",serif;color:#212529'><img
border=0 width=582 height=226 id="Picture 24"
src="iSanXoT-Help_files/image002.png"></span></p>

<p class=MsoNormal style='background:white'><span style='font-size:9.0pt;
line-height:107%;font-family:"Roboto",serif;color:#212529'>&nbsp;</span></p>

<h3>The modules are described by task-tables</h3>

<p class=MsoNormal>The parameters of each module are described by means a
task-table.</p>

<p class=MsoNormal align=center style='text-align:center;background:white'><span
style='font-size:9.0pt;line-height:107%;font-family:"Roboto",serif;color:#212529'><img
border=0 width=577 height=158 id="Picture 23"
src="iSanXoT-Help_files/image003.png"></span></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Installation"></a>Installation</h1>

</div>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3><a name=windows-distribution></a><a name=installation-on-windows></a><a
name="_Installation_on_Windows"></a><a name="_Windows_distribution"></a>Windows
distribution</h3>

<p class=MsoNormal>The iSanXoT application for Windows distribution is packaged
in a NSIS Launcher (exe file).</p>

<p class=MsoNormal>Download the Launcher in the assets part from the&nbsp;release&nbsp;link:</p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal>Double-click the Launcher file to open it, and you'll see an
Installer window.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 28" src="iSanXoT-Help_files/image004.png"
alt="Installer window"></span></p>

<p class=MsoNormal>Then, you can choose the folder in which to install iSanXoT</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=397 height=309 id="Picture 25" src="iSanXoT-Help_files/image005.png"
alt="Choose Install Location"></span></p>

<p class=MsoNormal>Wait while iSanXoT is being installed</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 22" src="iSanXoT-Help_files/image006.png"
alt="Installing window"></span></p>

<p class=MsoNormal>Finally, you'll see a window telling you completed setup.
You can run iSanXoT immediately if you want.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 21" src="iSanXoT-Help_files/image007.png"
alt="Completing setup"></span></p>

<p class=MsoNormal><span style='font-family:"Segoe UI",sans-serif;color:#24292E'>&nbsp;</span></p>

<h3><a name=macos-distribution></a><a name="_MacOS_distribution"></a>MacOS
distribution</h3>

<p class=MsoNormal>The iSanXoT application for MacOS distribution is packaged
in a DMG container.</p>

<p class=MsoNormal>Download the DMG file in the assets part from
the&nbsp;release&nbsp;link:</p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal>Double-click the DMG file to open it, and you'll see a
Finder window. Often these will include the application itself, some form of
arrow, and a shortcut to the Applications folder.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=416 height=309 id="Picture 19" src="iSanXoT-Help_files/image008.png"
alt="Installer window"></p>

<p class=MsoNormal>Simply drag the iSanXoT icon to your Applications folder...</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=439 height=315 id="Picture 18" src="iSanXoT-Help_files/image009.png"
alt="Installing window"></span></p>

<p class=MsoNormal>and you're done: the iSanXoT app is now installed.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name=linux-distribution></a><a name="_Linux_distribution"></a>Linux
distribution</h3>

<p class=MsoNormal>The iSanXoT application for Linux distribution is packaged
in an <i>AppImage</i>.</p>

<p class=MsoNormal>Download the AppImage in the assets part from
the&nbsp;release&nbsp;link:</p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=581 height=294 id="Picture 15" src="iSanXoT-Help_files/image010.png"
alt="Installer window"></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:0cm'>The AppImage file is
just the application's compressed image. When it runs, the file is mounted in
the unique temporal folder everything the application is opened. However, for the
correct behaviour of iSanXoT, the application has to be extracted into the
directory &quot;squashfs-root&quot; in the current working directory doing the
following sentence:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm'>Then you can now
run the iSanXoT application doing:</p>

<p class=MsoListParagraphCxSpLast style='margin-left:0cm;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<em><span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><span
style='font-family:"Calibri",sans-serif;font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<em><span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00;font-style:normal'>&nbsp;</span></b></em></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Get_Started"></a><em><span style='font-family:"Calibri Light",sans-serif;
font-style:normal'>Get Started</span></em></h1>

</div>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>This chapter describes the graphical interface of the
iSanXoT desktop application and how to set up an analysis in iSanXoT.</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3>Opening the iSanXoT application</h3>

<p class=MsoNormal>To open the iSanXoT application:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Windows: from the Start menu choose Programs &gt; iSanXoT, or
by clicking the desktop icon.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In MacOS: from the Application folder and clicking on iSanXoT
icon.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Linux: from the <i>AppImage</i> file.</p>

<p class=MsoListParagraphCxSpMiddle>The contents are extracted into the
directory &quot;squashfs-root&quot; in the current working directory doing the
following sentence:</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_0.4.1.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle>Then you can now run the iSanXoT application:</p>

<p class=MsoListParagraphCxSpLast style='text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun [...]</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3>Installing required packages</h3>

<p class=MsoNormal>The first thing you see when iSanXoT run is a frame window.
This window displays a progress-bar that informs you about the percentage of
packages that have been installed. These packages are the libraries that are
required by the backend of iSanXoT, and they are installed in the first time
you open the application.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=482 height=317 id="Picture 2" src="iSanXoT-Help_files/image011.png"></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3>Closing the iSanXoT application</h3>

<p class=MsoNormal>Save your project before you exit the iSanXoT application,
because it does not prompt you.</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>To
close the iSanXoT application:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:53.4pt;text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Windows and Linux: choose Menu &gt; Exit, or click the X in
the upper right corner of the main iSanXoT window.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:53.4pt;text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Mac: choose iSanXoT (menu) &gt; Exit, or click the red X in
the upper left corner of the main iSanXoT window.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>An
informative dialog box asks you: “Do you really want to close the
application?”. Click Yes (if that you want)</p>

<p class=MsoNormal>&nbsp;</p>

<h3>iSanXoT Start Page</h3>

<p class=MsoNormal>The iSanXoT Start page is a preliminary front page before the
main view of iSanXoT.</p>

<p class=MsoNormal>&nbsp;</p>

<h3>Projects in the iSanXoT application</h3>

<p class=MsoNormal>The first step in conducting an analysis in the iSanXoT application
is to create a project. A project is primarily a container that you use to
structure your input file(s) adaptor and your workflow data. The input file
adaptors transform the identification files of multiple search engines (see <a
href="#_Adaptors">Adaptors</a> section). The workflows are based on SanXoT
software packages that include quantitative proteomic analysis, systems
biology, and comparison and merging of experimental data from technical or
biological replicates (see <a href="#_Workflows">Workflows</a> section).</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=239 height=226 id="Picture 14" src="iSanXoT-Help_files/image012.png"></p>

<h4>Create a new project</h4>

<p class=MsoNormal>It is one of available menu items initially, <i>New Project</i>.
This menu opens a window which allows the definition of name of project and the
folder that the project will be saved.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=567 height=202 id="Picture 13" src="iSanXoT-Help_files/image013.png"></p>

<h4>Open a project</h4>

<p class=MsoNormal>This menu opens a folder dialog that allows to attach the folder
of archived project and then, open itself in iSanXoT.</p>

<p class=MsoNormal>&nbsp;</p>

<h3>iSanXoT main view</h3>

<p class=MsoNormal>The main view of iSanXoT application is composed by a menu,
several tabs, and the content panel and execution panel.</p>

<h4>Menu</h4>

<p class=MsoNormal>The Menu contains the following items:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Menu: houses the operations for the project and workflow.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Adaptors: contains the several adaptors (see Adaptors section).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Processes: link to the iSanXoT view of processes executing (see
the Processes executing).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Help: houses the several sections of iSanXoT help.</p>

<h4>Tabs bar</h4>

<p class=MsoNormal>There are some tabs in iSanXoT. Each tab is used for
different adaptor or modules.</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>The
first tab is used for the selected Adaptors (see Adaptors section).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>The
second tab is used for the Relation Table module (see RELS CREATOR module).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>The
third tab contains the Basic modules (see Basic modules section).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>This
tab contains the Complete modules (see Complete modules section).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'>5.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>The
last tab contains the Report modules (see Report modules).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=344 id="Picture 31"
src="iSanXoT-Help_files/image014.png"></p>

<p class=MsoCaption><b>Figure </b><b>1</b><b>. iSanXoT main view.</b></p>

<h4>Content panel</h4>

<p class=MsoNormal>This panel houses the content of the adaptors and the
modules that it is different in each case. However, there are some common
patterns. At the left side of panel, there is a sidebar menu that shows the
multiple modules that compose the tab-section. For example, the sidebar menu of
“Basic modules” tab contains the list of basic modules, the sidebar menu of
“Complete modules” tab contains the complete modules, and so on. Other common
pattern is a help icon for the specific module or adaptor you are seen in the Content
panel. In addition, there is a title and a briefly description for that
specific module.</p>

<h4>Execution panel</h4>

<p class=MsoNormal>The execution panel is composed by a field which you can
indicate how many processors you want to use in your execution, and “Start”
button to execute your workflow.</p>

<p class=MsoNormal>&nbsp;</p>

<h2>Importing and Exporting Workflows</h2>

<p class=MsoNormal>A project contains, as well as the input file(s), the
workflow data that are based on SanXoT software packages. There is a way to
import and export just the workflow structure by means of two menu items.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=237 height=225 id="Picture 35" src="iSanXoT-Help_files/image015.png"></p>

<h4>Import workflow</h4>

<p class=MsoNormal>This option allows to import the task-tables of a workflow.
For that, you have to provide the folder where the workflow is saved.</p>

<h4>Export workflow</h4>

<p class=MsoNormal>The export workflow saves the task-table data of a workflow
in the given folder.</p>

<p class=MsoNormal>&nbsp;</p>

<h2>Processes executing</h2>

<p class=MsoNormal>In the moment to execute a project work, new window is
opened. This window shows the processes executing through which is composed by
two tables:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Project logs table: shows the status of a project execution. It
is possible to see several projects executions and you have to click to the row
to see their own workflow logs.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Workflow logs table: displays the status for each job from the
multiple modules. If you click on row you will see the trace log of the
implicated jobs (except in the case of “cached” status).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=378 height=262 id="Picture 98"
src="iSanXoT-Help_files/image016.png"></p>

<p class=MsoCaption><b>Figure </b><b>2</b><b>. View of processes executing.</b></p>

<p class=MsoNormal align=left style='text-align:left'>You can see also this
window of processes executing from the “Processes &gt; Main page” menu.</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Modules"></a>Modules</h1>

</div>

<p class=MsoNormal>The iSanXoT desktop application houses several modules based
on the SanXoT software package [1]. The execution of each module is done and is
described by means of a Task-Table (TT).  These TT represent the parameters
that are needed to the setting up and the execution of module. </p>

<p class=MsoNormal>There are three types of modules:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Basic modules: These modules are inherently based on the SanXoT
packages [1]. They represent the minimal expression of integration in SanXoT.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Report modules: These modules are used to extract the information
from the integrations and to generate the similarity graph of a set of
categories.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Complete modules: These modules conform the fundamental
statistical models where the WSPP model [3] was described for quantitative
proteomics analysis and the SBT model [2] for system biology analysis.</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Basic_modules"></a>Basic
modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_RELS_CREATOR"></a><span style='font-family:"Calibri",sans-serif'>RELS
CREATOR</span></h3>

<p class=MsoNormal>This module parse data from a given tabular file, extract
several columns creating a file with tab separated table, Relation Table (RT).
These RT created, are used by the integrations in the rest modules. For this
reason, the name of RT indicated by the parameter/column of its task-table
(“Relation Table to be created”), has to coincide with the “lower-level” and
the ”higher-level” of the integration; e.g. whether an integration is between
the “peptide” to “protein” levels, the RT applied has to be called
“peptide2protein”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Relation Table to be created: indicates the file name of relation
table. This name must be formed with the lower level to higher level for one
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Column name of Lower level: column name from the given tabular
file that will use for the lower level in the resulted RT.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Column name of Higher level: column name from the given tabular
file that will use for the higher level in the resulted RT.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Column name of 3<sup>rd</sup> column: column name from the given
tabular file that will use for the third column in the resulted RT.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Table from which RT is extracted: file name with path which the
levels are extracted. This file has to be in tabular format. In this parameter
the keywords can be used. For more details read “Keywords for the Task-Tables”
in the “Special parameters” section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=566 height=118 id="Picture 37"
src="iSanXoT-Help_files/image017.png"></p>

<p class=MsoCaption><b>Figure </b><b>3</b><b>. Task-Table example of RELS
CREATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_LEVEL_CREATOR"></a><span style='font-family:"Calibri",sans-serif'>LEVEL
CREATOR</span></h3>

<p class=MsoNormal>The LEVEL CREATOR module creates a level that it the
quantitative data. This data file is text file that contains three columns:
identifier (a text string that is used to identify the element), quantitative
value (X: 2-base logarithm of the ratio of the two quantitative measurements)
and prior weight (V: a parameter that measures the accuracy of the quantitative
value before performing the integration).</p>

<p class=MsoNormal>This module takes the preliminary data from a Quantification
table (ID-q file) that is provided by the user applying the “Main Input”
adaptor, or it is created by one of adaptors. For more details, read the
“Adaptors” section.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Experiment: is the name of experiment. This value has to be in a
column called ‘Experiment’ within the Quantification data (ID-q file). Only the
data with the value specified in 'Experiment' column will be extracted.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio numerator column: specifies which sample quantification
values make up the numerator for the calculation of log2-ratio values.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio denominator column(s): specifies which sample
quantification make up the denominator for the calculation of log2-ratio
values.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Level to be created: designates the level name.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
data file of level will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=152 id="Picture 38"
src="iSanXoT-Help_files/image018.png"></p>

<p class=MsoCaption><b>Figure </b><b>4</b><b>. Task-Table example of LEVEL
CREATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_LEVEL_CALIBRATOR"></a><span style='font-family:"Calibri",sans-serif'>LEVEL
CALIBRATOR</span></h3>

<p class=MsoNormal>This module calibrates the V values of a level by performing
the specified integration. This calibration is done using the “klibrate”
program developed in the SanXoT software package [1].</p>

<p class=MsoNormal>To perform the calibration two parameters have to be
calculated: the k (weight constant), and the variance. They are calculated
iteratively using the Levenberg-Marquardt algorithm. More detailed information
may be found in “klibrate” integration from SanXoT software package [1].</p>

<p class=MsoNormal>The calibrated level contains the same information as the
original level from LEVEL CREATOR module, but changing the values of the third
column (containing the weights) to adapt the information to the calibrated
weights that can be used as input in the INTEGRATE module.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=353 height=283 id="Picture 57"
src="iSanXoT-Help_files/image019.png"></p>

<p class=MsoCaption><b>Figure </b><b>5</b><b>. Schema of LEVEL CALIBRATOR
module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Sample folder(s): indicates the names of the folders where the
uncalibrated data is located. These data files have been created by LEVEL
CREATOR module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Lower level for integration: indicates the name of lower-level
for the integration during the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Higher level for integration: indicates the name of higher-level
for the integration during the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Name of calibrated level: designates the level name.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
data file of level will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=169 id="Picture 4"
src="iSanXoT-Help_files/image020.png"></p>

<p class=MsoCaption><b>Figure </b><b>6</b><b>. Task-Table of LEVEL CALIBRATOR
module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_INTEGRATE"></a><span style='font-family:"Calibri",sans-serif'>INTEGRATE</span></h3>

<p class=MsoNormal>The INTEGRATE module performs WSPP-statistics by applying
iteratively the Generic Integration Algorithm (GIA) [2] on the data. Each GIA
integration performs an independent statistic and its output may be used as
input for the following integration step.</p>

<p class=MsoNormal>This module is applied to integrate lower-level data to higher-level
data. For example, to integrate from peptide-level data to protein-level, or to
integrate from protein-level data to gene-level.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=423 height=414 id="Picture 58" src="iSanXoT-Help_files/image021.png"></p>

<p class=MsoCaption><b>Figure </b><b>7</b><b>. Schema of INTEGRATE module</b>. Integration
from any lower-level to any higher-level, using the Generic Integration
Algorithm (GIA) which is composed by the programs: “sanxot” and “sanxotsieve”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>More in detail</h4>

<p class=MsoNormal>Each INTEGRATE needs as input tab-separated text tables:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Data
file, that contains the quantitative data of the lower level. This file
contains three columns: identifier (a text string that is used to identify the
element), quantitative value (2-base logarithm of the ratio of the two
quantitative measurements) and prior weight (a parameter that measures the
accuracy of the quantitative value before performing the integration).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Relation
table (RT), which establishes the correspondence of the identifiers in the
lower level with those in the higher level. This file contains two columns, the
first one has the identifiers of the higher level and the second one the
identifiers of the lower level.</p>

<p class=MsoNormal style='margin-left:18.0pt'>The INTEGRATE generates an output
data file containing the quantitative data of the higher level. This file can
be used as input for the others modules.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=582 height=524 id="Picture 41"
src="iSanXoT-Help_files/image022.png"></p>

<p class=MsoCaption><b>Figure </b><b>8</b><b>. Flowchart of INTEGRATE module.</b>
A first integration is done with “sanxot” that calculates the variance;
“sanxotsieve” removes the outliers tagging them in a new relation table; a
second integration is done with “sanxot” using the fixed variance calculated in
the first step and removing the outliers tagged in the relations table.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Each integration takes the input data file from the lower-level
and the RT, calculates the general variance of the integration (using a robust
iterative method) and generates as output a data file containing the integrated
quantifications of the higher-level. The integration is done using the “sanxot”
program.</p>

<p class=MsoNormal>In addition to the quantitative data, each integration may
generate several additional files which contain information about the
integration. More detailed information may be found in exploring SanXoT program
[1].</p>

<p class=MsoNormal>One of the advantages of this integration is that it
provides a straightforward method to eliminate outliers [3] in each integration
step. This is done by calculating standardized log2-ratios (z-values) allowing to
estimate the probability for each element of the lower level is a significant
outlier of the z distribution (e.g., N(0,1)) and also to obtain the associated
FDR. The most extreme outliers may be removed sequentially and the integration
repeated, until all outliers below a user-defined FDR-level are removed.</p>

<p class=MsoNormal>Outlier removal is performed by the “sanxotsieve” program by
tagging the outlier elements in a new relation table. Then, a second
integration is done by “sanxot” program using the fixed variance calculated in
the first integration (“sanxot” program) and discarding the outliers tagged in
the relations table.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Sample folder(s): indicates the names of the folders where the
lower-level data is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Lower level: indicates the name of lower-level. The module will
use the data file with the same name and it contains the three columns with the
identifier, quantitative value and prior weight.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Higher level: indicates the name of higher-level to integrate.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=373 height=151 id="Picture 36"
src="iSanXoT-Help_files/image023.png"></p>

<p class=MsoCaption><b>Figure </b><b>9</b><b>. Task-Table example for INTEGRATE
module.</b></p>

<p class=MsoNormal>The Task-table is the way to execute the module. Each row
represents one or more executions.</p>

<p class=MsoNormal>This module performs an integration from an existing lower-level
to a higher-level using a previously created Relation Table whose name matches
the higher and lower levels (e.g. if the user wants to integrate from peptide
to protein, the module will use the Relation Table called scan2peptide.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Tag: label that below to accept or discard elements is the
calculation of variance. By default, the “outliers” are discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>FDR: limit of False Discovery Rate different than 0.01 (1%). If
FDR = 0, then the “outliers” are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Var(x): force a variance. By default, the variance calculated by
first “sanxot” is applied.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>More params: allows to add more parameters to the programs inside
the program. For more detail read “More params” in “Special Parameters”
section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=182 id="Picture 1"
src="iSanXoT-Help_files/image024.png"></p>

<p class=MsoCaption><b>Figure </b><b>10</b><b>. Task-Table example with
advanced parameters for INTEGRATE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_NORCOMBINE"></a><span style='font-family:"Calibri",sans-serif'>NORCOMBINE</span></h3>

<p class=MsoNormal>NORCOMBINE is a module to integrate technical or biological
replicates. For example, there are multiple proteins obtained from integrations
separately for each replicate and then the protein-level data are integrated to
obtain protein averages (grouped-level data).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=451 height=388 id="Picture 114"
src="iSanXoT-Help_files/image025.png"></p>

<p class=MsoCaption><b>Figure </b><b>11</b><b>. Schema of NORCOMBINE module.</b>
Integration of technical or biological replicates.</p>

<p class=MsoNormal>The Merging of experiments is done within NORCOMBINE module
using the “cardenio” program from the SanXoT software package [1]. “Cardenio”
is used to generate suitable relation tables to make averages from technical or
biological replicates. Then, NORCOMBINE normalizes (&quot;center&quot;) the
data integrating the grouped-level to take into account the systematic
quantitative error of each experiment.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>More in detail</h4>

<p class=MsoNormal>This module needs a set of files, produced by INTEGRATE
module:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>“lowerNormV”: this file contains three columns: </p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>the identifiers of the lower level,</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>the second column contains the Xinf - Xsup (i.e. the ratios of
the lower level, but centred for each element they belong to), and </p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>the third column is the former untouched V(inferior) weight.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>“lowerNormW”: this file contains three columns. The first two are
the same than “lowerNormV” but the third is the new weight W(inferior)
containing the variance of the integration.</p>

<p class=MsoNormal>More detailed information may be found in “sanxot”
integration from SanXoT software package [1].</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=452 height=522 id="Picture 115"
src="iSanXoT-Help_files/image026.png"></p>

<p class=MsoCaption><b>Figure </b><b>12</b><b>. Flowchart of NORCOMBINE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Sample folder(s): indicates the names of the folders where the
lower-level data is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Level: indicates the name of level. The module will use the data
file with the same name of level.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Norm: specifies the normalization scheme (“proteinall” here) to
be used with level.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>lowerNorm: selects the type of lower normalization to use: V (the
untouched V weight) or W (the new weight containing the variance of the
integration).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: the name of the folder where the resulting be
saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=562 height=104 id="Picture 6"
src="iSanXoT-Help_files/image027.png"></p>

<p class=MsoCaption><b>Figure </b><b>13</b><b>. Task-Table example for
NORCOMBINE module.</b> In this case, the asterisk wildcard has been used to
select multiple sample folders.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters belonging to the
integration part:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Tag: label that below to accept or discard elements is the
calculation of variance. By default, the “outliers” are discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>FDR: limit of False Discovery Rate different than 0.01 (1%). If
FDR = 0, then the “outliers” are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Var(x): force a variance. By default, the variance calculated by
first “sanxot” is applied.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>More params: allows to add more parameters to the programs inside
the program. For more detail read “More params” in “Special Parameters”
section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=73 id="Picture 7"
src="iSanXoT-Help_files/image028.png"></p>

<p class=MsoCaption><b>Figure </b><b>14</b><b>. Task-Table with advanced
parameters for NORCOMBINE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_RATIOS"></a><span style='font-family:"Calibri",sans-serif'>RATIOS</span></h3>

<p class=MsoNormal>This module calculates the statistical weight of the newly
calculated log2-ratios from two comparison samples. This module takes (X,V) of
given level from the numerator and denominator samples. It calculates the difference
for the new X value (i.e. log2 of the indicated ratios), and combines the V values
using the indicated method. Then, the result (X,V) is stores as the same level
in the output sample folder.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>More in detail</h4>

<p class=MsoNormal>This module takes calculates the new (X,V) values from the
numerator and denominator samples. The new X value is the difference between
both X’s values. The “V method” determines how to calculate the new V value. These
methods are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>the maximum of both V’s.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>the applied form V = 1/(1/V1+1/V2).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>and the average of both V’s.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=581
height=199 id="Picture 17" src="iSanXoT-Help_files/image029.png"></p>

<p class=MsoCaption><b>Figure </b><b>15</b><b>. Flowchart of RATIOS module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio numerator: indicates which sample contains the level required
to make up the numerator of the ratio to be calculated (“KO” in this case).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio denominator: indicates which sample contains the level file
required to make up the denominator of the ratio to be calculated (“WT” in this
case).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Level: specifies the level from where the ratio calculation
should be made.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: the name of the folder where the resulting
log2-ratio and statistical weight values will be saved (“KO_vs_WT” in our
example).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=68 id="Picture 8"
src="iSanXoT-Help_files/image030.png"></p>

<p class=MsoCaption><b>Figure </b><b>16</b><b>. Task-Table of RATIOS module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_SBT"></a><span style='font-family:"Calibri",sans-serif'>SBT</span></h3>

<p class=MsoNormal>This module is based on The Systems Biology Triangle model
[2]. The SBT model performs a triangular integration using lower level,
intermediate level and higher level. It is common the levels would be protein,
category as lower and intermediate level, and the higher level would be the
grand mean. From this way, the SBT module performs a Systems biology analysis. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=436 height=435 id="Picture 131"
src="iSanXoT-Help_files/image031.png"></p>

<p class=MsoCaption><b>Figure </b><b>17</b><b>. Schema of SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Sample folder(s): indicates the names of the folders where the
levels data are located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Lower level: indicates the name of lower-level (“protein” in this
case).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Intermediate level: indicates the name of intermediate-level
(“category”).</p>

<p class=MsoNormal>By default, the higher-level is the grand mean of each
level.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=380 height=78 id="Picture 5"
src="iSanXoT-Help_files/image032.png"></p>

<p class=MsoCaption><b>Figure </b><b>18</b><b>. Task-Table of SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters belonging to the
integration part:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: the name of the folder where the resulting
log2-ratio and statistical weight values will be saved (“KO_vs_WT” in our
example).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Lower-Higher level and Int(ermediate)-Higher level: in the case
you don’t want to use the grand mean for the higher level, you can the level
for the respective integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Low(er) &gt; Int(ermediate) Tag and Int(ermediate) &gt; Hig(her)
Tag: labels from lower level to intermediate level and from intermediate level
to higher level respectively that below to accept or discard elements is the
calculation of variance. By default, the “outliers” are discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Low(er) &gt; Int(ermediate) FDR and Int(ermediate) &gt; Hig(her)
FDR: limit of False Discovery Rate different than 0.01 (1%) for the from lower
level to intermediate level and from intermediate level to higher level
respectively. If FDR = 0, then the “outliers” are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Low(er) &gt; Int(ermediate) Var(x) and Int(ermediate) &gt;
Hig(her) Var(x): force a variance for the from lower level to intermediate
level and from intermediate level to higher level respectively. By default, the
variance calculated by first “sanxot” is applied.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>More params: allows to add more parameters to the programs inside
the program. For more detail read “More params” in “Special Parameters”
section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=554 height=168 id="Picture 10"
src="iSanXoT-Help_files/image033.png"></p>

<p class=MsoCaption><b>Figure </b><b>19</b><b>. Task-Table with advanced
parameters for SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Report_modules"></a>Report modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_REPORT"></a><span style='font-family:"Calibri",sans-serif'>REPORT</span></h3>

<p class=MsoNormal>The REPORT module allows the collection of the integration
data into a result table. It compiles the statistical results from the given
integration and stores the values in a single table.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Sample folder(s): indicates the names of the folders where the statistical
data for each integration is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Lower level: indicates the starting level (peptide, protein,
category) for the integration whose data are to be reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Higher level: indicates the ending level for the integration
whose data are to be reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Reported vars: specifies which statistical variables will be
reported. The available variables are “n”, “tags”, “Z”, “FDR”, “Xsup”, “Vsup”,
“Xinf”, “Vinf”.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output report: is the report filename (without extension).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=302 id="Picture 11"
src="iSanXoT-Help_files/image034.png"></p>

<p class=MsoCaption><b>Figure </b><b>20</b><b>. Task-Table of REPORT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The first row of above task-table takes the “n” variable from
the statistical data of “scan-peptide” integration (scan2peptide_outStats.tsv
file). This means, from the indicated lower level and higher level. Due to asterisk
wildcard, the report of this row will contain all samples of the experiments.
The output of this report will be saved into “Nscan_pep” filename in the
“reports” folder of your project.</p>

<p class=MsoNormal>The second row takes the “Xinf”,”Z”, and “FDR” variables
from the statistical data of “peptide-protein” integration. Also, this row will
create a report file for all samples of the experiments, called “Nscan_Normpep_prot_XZ”.</p>

<p class=MsoNormal>The description of the rest of rows are similar.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Level names to show: if you don’t want to show one of the level
data in the report. By default, all the levels are shown.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Merge with report: Join with designates the file whose Reported
vars will be incorporated into Output after intersection with the latter file.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add columns from relation table: insert into the current report
the columns from the relation table (RT) at the end. The RT columns are added
based on the values of the “lower level”. It is possible to use multiple RT
separated by comma.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter: the data of report could be filtered based on the
reported variables (n, Z, FDR, etc). For more detail read “Filter” in “Special
Parameters” section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=278 id="Picture 20"
src="iSanXoT-Help_files/image035.png"></p>

<p class=MsoCaption><b>Figure </b><b>21</b><b>. Task-Table with advanced
parameters for REPORT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The way to merge one report with another is through the
column names “Output report” and “Merge with report”. Each row represents one report
and the “output report” is the file name of this report. The “merge with
report” houses the name of previous report name that will be used to join its
own data with the current report.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=588 height=144 id="Picture 26"
src="iSanXoT-Help_files/image036.png"></p>

<p class=MsoCaption><b>Figure </b><b>22</b><b>. Example of join of two reports.</b>
The first row creates a report with the (n)umber of peptides per scan, and this
data is saved in “Nscan_pep” file. The second row creates a report file called
“Nscan_Normpep_protXZ” with the variables “Xinf”, “Z”, and “FDR” for the
integration peptide to protein. In addition, the (n)umber of peptides per scan
that are saved in the “Nscan_pep” report file, will join to this report based
on the values of “peptide” (lower level).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>In addition, it is possible to add more data into the report
from one or more relation tables (RTs). The condition for the adding is that
the name of lower level is within the column names of RT. If this is the case,
the columns from RT are added based on the values of lower level. If not, the
REPORT module checks whether the name of higher level is with in the column
names of RT. If this is the case, the columns are added based on the values of
higher level. Otherwise, the module does nothing. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=597 height=94 id="Picture 27"
src="iSanXoT-Help_files/image037.png"></p>

<p class=MsoCaption><b>Figure </b><b>23</b><b>. Add data into report from
relation tables. </b>The RT “protein2gene” contains one column with the protein
identifiers, named “protein”, and another column with the gene of each protein.
The RT “protein2description” contains also the protein identifiers with a
column name “protein” and another column with the description of each protein.
The gene names and the protein descriptions contained in these RTs will be
included in the “Npep_Quanprot_cat” report file based on the protein
identifiers (“lower level”).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Moreover, the data of report file could be filtered from the
“Filter” parameter based on logical conditions over the reported variables. The
filtering could be as in these examples:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category == 5, retrieves the data report which the
(n)umber of categories per protein is equal to 5.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &lt;= 100, retrieves the data which the
(n)umber of categories per protein is less than or equal to 100.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &gt;= 5 &amp; n_protein2category &lt;= 100,
retrieves the data which the (n)umber of categories per protein is greater or
equal than 5 and less or equal than 100.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>KO_vs_WT@FDR_category2categoryall &lt; 0.05, retrieves the data
which the FDR of category to categoryall integration from “KO_vs_WT” samples is
less than 5%.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=590 height=89 id="Picture 32"
src="iSanXoT-Help_files/image038.png"></p>

<p class=MsoCaption><b>Figure </b><b>24</b><b>. Filtering the report data.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The variables in the filtering have a standard composition.
In the following example, “n_protein2category”, the variable is composed with
the reported variable (n) and the integration (protein2category). This filter
is applied for the protein to category integration of all samples.</p>

<p class=MsoNormal>However, in the example of
“KO_vs_WT@FDR_category2categoryall”, the filter is applied in the “FDR”
variable of category to categoryall integration but only for the “KO_vs_WT”
sample folder. Moreover, in the example “WT1,WT2@FDR_category2categoryall” the
filter is applied for the “WT1” and “WT2” samples.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_SANSON"></a><span style='font-family:"Calibri",sans-serif'>SANSON</span></h3>

<p class=MsoNormal>The SANSON module detects the categories that enclose a similar
set of proteins, and then, it shows the changing proteins within each category
generating a similarity graph. </p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Sample folder(s): indicates the names of the folders where the
lower-level data is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Lower level: indicates the name of lower-level (“protein” in this
case).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Higher level: indicates the name of higher-level to integrate
(“category” in this case).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: the name of the folder where the resulting
be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=423 height=55 id="Picture 33"
src="iSanXoT-Help_files/image039.png"></p>

<p class=MsoCaption><b>Figure </b><b>25</b><b>. Task-Table of SANSON module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Lower norm: specifies the normalization scheme to be used with
the lower level.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Higher norm: specifies the normalization scheme to be used with
the higher level.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Tag: filter the label of tags. By default, the “out” tag coming
from the outliers is filtered.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter: the data of report could be filtered based on the
reported variables (n, Z, FDR, etc). For more detail read “Filter” in “Special
Parameters” section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=46 id="Picture 34"
src="iSanXoT-Help_files/image040.png"></p>

<p class=MsoCaption><b>Figure </b><b>26</b><b>. Task-Table with advanced
parameters for SANSON module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Complete_Modules"></a>Complete modules</h2>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>SanXoT [1], our previous publicly available implementation
of the weighted spectrum, peptide, and protein (WSPP) statistical model [2]
using the Generic Integration Algorithm (GIA) [3]. The WSPP model is unique in
that it considers the error structure of the raw mass data and the steps of
sample preparation, addressing the corresponding error sources separately, by
analysing the spectrum, peptide, and protein levels independently and
sequentially. The model resolves most of the issues related to quantitative
proteomics analysis, including undersampling, the need for a null hypothesis,
the proteome dynamic range and the non-normality of protein abundance change
distributions [2]. In addition, the Systems Biology Triangle (SBT) [2] is a new
algorithm that allows the study of protein coordination by pairwise
quantitative proteomics and detects statistically significant coordination in
diverse biological models of very different nature and subjected to different
kinds of perturbations. Here we represent multiple modules that allow the
execution of combination of WSPP and SBT accompanied by more complete modules.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_WSPP-SBT"></a><span style='font-family:"Calibri",sans-serif'>WSPP-SBT</span></h3>

<p class=MsoNormal>This module performs the integrations scan to peptide
(scan2peptide), peptide to protein (peptide2protein), and protein to category (protein2category).
In addition, there are the integrations peptide to peptideall
(peptide2peptideall), protein to proteinall (protein2proteinall),  and category
to categoryall (category2categoryall). After all these integrations, the SBT
algorithm follows to analyze coordinated protein changes.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Experiment: is the name of experiment. Only the data with the
value specified in 'Experiment' column will be extracted.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Identifier column header: is the name of column with the
identifier that describes unequivocally a scan.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio numerator column: specifies which sample quantification
values make up the numerator for the calculation of log2-ratio values.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio denominator column(s): specifies which sample
quantification make up the denominator for the calculation of log2-ratio
values.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
data file of level will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=204 id="Picture 40"
src="iSanXoT-Help_files/image041.png"></p>

<p class=MsoCaption><b>Figure </b><b>27</b><b>. Task-table example for WSPP-SBT,
WSPPG-SBT, WPP-SBT, WPPG-SBT modules.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q Tag, p&gt;a Tag, c&gt;a Tag: labels that below to accept
or discard elements is the calculation of variance for the integrations:
peptide2protein, peptide2peptideall, and category2categoryall, respectively. By
default, the “outliers” are discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>s&gt;p FDR, p&gt;q FDR, q&gt;c FDR: limit of False Discovery Rate
different than 0.01 (1%) for the following integrations: scan2peptide, peptide2protein,
and protein2category. If FDR = 0, then the “outliers” are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>s&gt;p Var(x), p&gt;q Var(x), q&gt;c Var(x): force a variance for
the integrations: scan2peptide, peptide2protein, and protein2category. By
default, the variance calculated by first “sanxot” is applied.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>More params: allows to add more parameters to the programs inside
the program. For more detail read “More params” in “Special Parameters”
section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=99 id="Picture 42"
src="iSanXoT-Help_files/image042.png"></p>

<p class=MsoCaption><b>Figure </b><b>28</b><b>. Task-Table example with
advanced parameters for WSPP-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_WSPPG-SBT"></a><span style='font-family:"Calibri",sans-serif'>WSPPG-SBT</span></h3>

<p class=MsoNormal>This module performs the integrations scan2peptide,
peptide2protein, protein to gene (protein2gene), and gene to category (gene2category).
In addition, there are the integrations peptide to gene (peptide2gene), peptide2peptideall,
protein2proteinall, gene to geneall (gene2geneall),  and category2categoryall.
After all these integrations, the SBT algorithm follows to analyze coordinated gene
changes.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Experiment: is the name of experiment. Only the data with the
value specified in 'Experiment' column will be extracted.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Identifier column header: is the name of column with the
identifier that describes unequivocally a scan.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio numerator column: specifies which sample quantification
values make up the numerator for the calculation of log2-ratio values.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio denominator column(s): specifies which sample
quantification make up the denominator for the calculation of log2-ratio
values.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
data file of level will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q Tag, q&gt;g Tag, p&gt;a Tag, c&gt;a Tag: labels that below
to accept or discard elements is the calculation of variance for the
integrations: peptide2protein, protein2gene, peptide2peptideall, and
category2categoryall, respectively. By default, the “outliers” are discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>s&gt;p FDR, p&gt;q FDR, q&gt;g FDR, g&gt;c FDR: limit of False
Discovery Rate different than 0.01 (1%) for the following integrations:
scan2peptide, peptide2protein, protein2gene and gene2category. If FDR = 0, then
the “outliers” are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>s&gt;p Var(x), p&gt;q Var(x), q&gt;g Var(x), g&gt;c Var(x): force
a variance for the integrations: scan2peptide, peptide2protein, protein2gene and
gene2category. By default, the variance calculated by first “sanxot” is
applied.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>More params: allows to add more parameters to the programs inside
the program. For more detail read “More params” in “Special Parameters”
section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=54 id="Picture 44"
src="iSanXoT-Help_files/image043.png"></p>

<p class=MsoCaption><b>Figure </b><b>29</b><b>. Task-table example with
advanced parameters for WSPPG-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_WPP-SBT"></a><span style='font-family:"Calibri",sans-serif'>WPP-SBT</span></h3>

<p class=MsoNormal>This module performs the integrations peptide2protein,
peptide2peptideall, protein2cagegory and category2cagtegoryall. The SBT
algorithm is used to analyze coordinated protein changes.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Experiment: is the name of experiment. Only the data with the
value specified in 'Experiment' column will be extracted.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Identifier column header: is the name of column with the
identifier that describes unequivocally a scan.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio numerator column: specifies which sample quantification
values make up the numerator for the calculation of log2-ratio values.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio denominator column(s): specifies which sample
quantification make up the denominator for the calculation of log2-ratio
values.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
data file of level will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q Tag, p&gt;a Tag, c&gt;a Tag: labels that below to accept
or discard elements is the calculation of variance for the integrations:
peptide2protein, peptide2peptideall, and category2categoryall, respectively. By
default, the “outliers” are discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q FDR, q&gt;c FDR: limit of False Discovery Rate different
than 0.01 (1%) for the following integrations: peptide2protein, and protein2category.
If FDR = 0, then the “outliers” are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q Var(x), q&gt;c Var(x): force a variance for the
integrations: peptide2protein, and protein2category. By default, the variance
calculated by first “sanxot” is applied.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>More params: allows to add more parameters to the programs inside
the program. For more detail read “More params” in “Special Parameters”
section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=482 height=78 id="Picture 45"
src="iSanXoT-Help_files/image044.png"></p>

<p class=MsoCaption><b>Figure </b><b>30</b><b>. Task-table example with
advanced parameters for WPP-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_WPPG-SBT"></a><span style='font-family:"Calibri",sans-serif'>WPPG-SBT</span></h3>

<p class=MsoNormal>This module performs the integrations peptide2protein,
protein2gene, and gene2category. In addition, there are the integrations
peptide2gene, peptide2peptideall, protein2proteinall, gene2geneall,  and
category2categoryall. After all these integrations, the SBT algorithm follows to
analyze coordinated gene changes.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Simple parameters</h4>

<p class=MsoNormal>The parameters/columns of this module are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Experiment: is the name of experiment. Only the data with the
value specified in 'Experiment' column will be extracted.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Identifier column header: is the name of column with the
identifier that describes unequivocally a scan.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio numerator column: specifies which sample quantification
values make up the numerator for the calculation of log2-ratio values.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ratio denominator column(s): specifies which sample
quantification make up the denominator for the calculation of log2-ratio
values.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
data file of level will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts advanced parameters:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q Tag, q&gt;g Tag, p&gt;a Tag, c&gt;a Tag: labels that below
to accept or discard elements is the calculation of variance for the
integrations: peptide2protein, protein2gene, peptide2peptideall, and
category2categoryall, respectively. By default, the “outliers” are discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q FDR, q&gt;g FDR, g&gt;c FDR: limit of False Discovery Rate
different than 0.01 (1%) for the following integrations: peptide2protein,
protein2gene and gene2category. If FDR = 0, then the “outliers” are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>p&gt;q Var(x), q&gt;g Var(x), g&gt;c Var(x): force a variance for
the integrations: peptide2protein, protein2gene and gene2category. By default,
the variance calculated by first “sanxot” is applied.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>More params: allows to add more parameters to the programs inside
the program. For more detail read “More params” in “Special Parameters”
section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=63 id="Picture 46"
src="iSanXoT-Help_files/image045.png"></p>

<p class=MsoCaption><b>Figure </b><b>31</b><b>. Task-table with advanced
parameters for WPPG-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Special_parameters"></a>Special parameters</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Keywords_for_the"></a>Keywords for the Task-Tables</h3>

<p class=MsoNormal>The task-tables accept some keywords that represent values
mostly related to the working project. The keywords are the following:</p>

<p class=MsoListParagraph style='text-indent:-18.0pt'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>__IDQFIL__   represents the Identification-Quantification file
(ID-q file). This keyword is used mainly by the task-table of RELS CREATOR
module. When the workflow is using an adaptor and we don’t know “a priori” the
ID-q created by the adaptor, we use this __IDQFIL__ keyword in the “Table from
which RT is extracted” parameter.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Multiple_samples"></a>Multiple samples</h3>

<p class=MsoNormal>It is possible to add multiple samples within the “Sample
folder(s)” parameter in the task-tables. For example, having the following
task-table for LEVEL CREATOR module:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=567 height=162 id="Picture 64" src="iSanXoT-Help_files/image046.png"></p>

<p class=MsoNormal>We can add multiple samples separated by comma. </p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=567 height=227 id="Picture 65" src="iSanXoT-Help_files/image047.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Asterisk_is_our"></a>Asterisk is our jack of all trades</h3>

<p class=MsoNormal>Apart of include multiple samples separated by comma, there
is the asterisk is used as a wildcard. This asterisk means “zero or more
characters” for the “Sample folder(s)” parameter in the task-tables. If we have
the following task-table for LEVEL CREATOR module:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=567 height=162 id="Picture 59" src="iSanXoT-Help_files/image048.png"></p>

<p class=MsoNormal>Each row is the ratio saved in its corresponding “Output
Sample folder”. E.g. the ratio of 113 with the mean of 113,114,115,116 is saved
in “Junkat_WT/WT_1”, the ratio of 114 is saved in “Junkat_WT/WT_2”, and so on.
One way to create the task-table of INTEGRATE module could be as illustrate the
below  table where each row represents a sample and its integration.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=213 height=296 id="Picture 62" src="iSanXoT-Help_files/image049.png"></p>

<p class=MsoNormal>However, it is an easy way to do applying the asterisk
wildcard. The task-table below allows to select the multiple sample folders.
E.g. “Junkat_WT/*” selects all the sample folders within the “Junkat_WT”, and
“Junkat_KO/*” selects all the sample folders within the “Junkat_KO”.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=263 height=262 id="Picture 60" src="iSanXoT-Help_files/image050.png"></p>

<p class=MsoNormal>We can reduce this expression even more using only asterisk.
The first row of the following task-table takes the “peptide” level for all
samples folders of this project, the second row takes the “protein” levels for
all sample folders, and so on.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=272 height=147 id="Picture 63" src="iSanXoT-Help_files/image051.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_More_params"></a>More params</h3>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Filter_param_(in"></a>Filter (in REPORT module)</h3>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Adaptors"></a>Adaptors</h1>

</div>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Main_Inputs"></a><a name="_Main_Input"></a><span
style='font-family:"Calibri",sans-serif'>Main Input</span></h3>

<p class=MsoNormal>The Main-Input adaptor provides the following parameters to
your project:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Project folder: describes the path to the folder where iSanXoT
output files will be stored.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Identification file: specifies the location of the file that
contains the identification and quantification data.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=110 id="Picture 48"
src="iSanXoT-Help_files/image052.png"></p>

<p class=MsoCaption><b>Figure </b><b>32</b><b>. Panel of Main-Input adaptor
with a worked-out example.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The unique requirements for the
Identification/quantification file are:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Experiment column that contains the multiple names of your
experiments.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>A column with identifiers that describe unequivocally a level.
For example, if you want to use a WSPP-SBT “Complete module”, your
identification file has to contain a column that describes the scans in a
unique way.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Close_Search_–"></a><a name="_Close_Search:_Inputs"></a><a
name="_Inputs_from_PD"></a>Inputs from PD (Close search)</h3>

<p class=MsoNormal>This adaptor allows you to transform the output of Proteome
Discoverer (PD) into an Identification-Quantification file appropriate for
iSanXoT workflows. iSanXoT has been tested for the outputs of PD 2.5.</p>

<p class=MsoNormal>The adaptor uses the following fields:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output folder: describes the path to the folder where iSanXoT
output files will be stored.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Default Input folder: specifies the location of the files
containing PSM identification and quantification data (the PSMs.txt files).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Input files: indicates which of the PSM.txt files stored in the
input folder must be considered by iSanXoT. PSMs.txt file names are listed
under Infile, while their experiment allocation is indicated in Experiment.</p>

<p class=MsoNormal align=left style='text-align:left'>The following figure
displays six PSMs.txt files originate from the same experiment (termed “Jurkat”),
but larger projects may encompass several experiments (e.g. “TMT1”, “TMT2” and
“TMT3”; or “Exp1”, “Exp2” and “Exp3”).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=528 height=292 id="Picture 49"
src="iSanXoT-Help_files/image053.png"></p>

<p class=MsoCaption><b>Figure </b><b>33</b><b>. Panel of &quot;Input from
PD&quot; adaptor with a worked-out example.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=left style='text-align:left'>The PSM.txt files are
plain text files that reveal information for every PSM obtained upon database
searching; however, only a reduced subset of these data will be accessed by
iSanXoT, as shown in the following Table.</p>

<div align=center>

<table class=MsoTableGrid border=0 cellspacing=0 cellpadding=0 width="91%"
 style='width:91.5%;border-collapse:collapse;border:none'>
 <tr style='height:15.0pt'>
  <td width="100%" colspan=2 valign=top style='width:100.0%;border:none;
  border-bottom:solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;
  height:15.0pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal'><b><span style='font-size:10.0pt'>Data
  accessed by iSanXoT from the Proteome Discoverer results (PSMs.txt files)</span></b></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;border:none;border-bottom:
  solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><b><span style='font-size:10.0pt'>Header</span></b></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;border:none;border-bottom:
  solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;height:14.25pt'>
  <p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal'><b><span style='font-size:10.0pt'>Description</span></b></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;border:none;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Spectrum File</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;border:none;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Name of the raw LC-MS/MS file</span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>First Scan</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Scan number identifier</span></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Sequence</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Peptide amino acid sequence</span></p>
  </td>
 </tr>
 <tr style='height:27.8pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Modifications</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Unimod<a href="#_ftn1" name="_ftnref1" title=""><span
  class=MsoFootnoteReference><span class=MsoFootnoteReference><span
  style='font-size:10.0pt;line-height:107%;font-family:"Calibri",sans-serif'>[1]</span></span></span></a>
  chemical or posttranslational modifications to peptide sequence</span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Charge</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Peptide charge</span></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Search Engine Rank</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Rank of search engine</span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>XCorr</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Cross-correlation value as provided by SEQUEST<a
  href="#_ftn2" name="_ftnref2" title=""><span class=MsoFootnoteReference><span
  class=MsoFootnoteReference><span style='font-size:10.0pt;line-height:107%;
  font-family:"Calibri",sans-serif'>[2]</span></span></span></a> algorithm </span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>MHplus in Da</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Measured monoisotopic protonated peptide mass in Da</span></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Theo MHplus in Da</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Theoretical monoisotopic protonated peptide mass in
  Da</span></p>
  </td>
 </tr>
 <tr style='height:27.8pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Delta M in ppm</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Difference between measured and theoretical
  monoisotopic mass in ppm</span></p>
  </td>
 </tr>
 <tr style='height:27.05pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.05pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Protein Accessions</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.05pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Accession codes for the proteins to which the
  peptide sequence is ascribed </span></p>
  </td>
 </tr>
 <tr style='height:16.4pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:16.4pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Protein Descriptions</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:16.4pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>List of protein descriptions separated by comma </span></p>
  </td>
 </tr>
 <tr style='height:27.8pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>113-121, 126-131, etc</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Intensity of reporter ions</span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoCaption><b>&nbsp;</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Validating peptide identification with FDR</h4>

<p class=MsoNormal>iSanXoT relies on the probability ratio (pRatio) method, an
algorithm that calculates the probability of random peptide matching and
provides the corresponding FDR for peptide identification. The FDR parameters
displays the following fields:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Forced execution: checkbox that determines to force the execution
or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Experiment: the aforementioned experiment allocation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Threshold (ppm): is the postscoring mass filtering cut-off to be
applied after using wide mass windows in the database search, as was the case
with these sample data. Threshold is actually equivalent to the relative
deviation experimentally observed for precursor ions in a particular LC-MS/MS
run.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Isotopologue recovery: allows pRatio to recover precursor m/z
values matching some 13C isotopologue when using wide mass windows in the
LC-MS/MS acquisition that otherwise would remain unnoticed. For that, it must
be indicated whether precursor m/z values should be tracked only around their
experimental m/z (value = 1) or also ± 1 Th (value = 3) and ± 2 Th (value = 5,
the one used here) away.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>FDR cutoff: establishes the FDR cut-off for PSM validation. The
value used here (0.01, i.e. 1% FDR) implies that one in every 100 validated
PSMs is incorrect.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Score: determines the score used to calculate the FDR. These scores
are “XCorr” (SEQUEST5 cross correlation score), or “cXCorr” (the corrected
XCorr) will be used by pRatio for FDR calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Decoy label: is the tag attached to decoy protein identifiers in
the concatenated protein database previously used for peptide identification
(“INV_” in this example). </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=108 id="Picture 9"
src="iSanXoT-Help_files/image054.png"></p>

<p class=MsoCaption><b>Figure </b><b>34</b><b>. Task-Table of FDR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Close_Search_–_1"></a><a name="_Inputs_from_MSFragger"></a>Inputs
from MSFragger (Close search)</h3>

<p class=MsoNormal>This adaptor allows you to transform the output of MSFragger
[4] into an Identification-Quantification file appropriate for iSanXoT
workflows. iSanXoT has been tested for the outputs of MSFragger version 3.1.1
(with FragPipe version 14.0).</p>

<p class=MsoNormal>The adaptor uses the following fields:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output folder: describes the path to the folder where iSanXoT
output files will be stored.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Default Input folder: specifies the location of the files
containing results files.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Input files: indicates which of the text plain files stored in
the input folder must be considered by iSanXoT. The file names are listed under
Identification file column, while their experiment is indicated in Experiment
column.</p>

<p class=MsoNormal>So far, the MSFragger version used did not contain the
quantification values. Thus, this adaptor allows you to extract the
quantification values from the “mzML” files and then, pair them with the
identifications. For that, the “Input files” table has the “advanced options”:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Quantification file (in mzML format): indicating the mzML names
if the files are located in the “Default Input folder”, or giving the absolute
path of mzML files.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Quan method: table files that describes the ion isotopic.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=175 id="Picture 50"
src="iSanXoT-Help_files/image055.png"></p>

<p class=MsoCaption><b>Figure </b><b>35</b><b>. Panel of &quot;Input from
MSF&quot; adaptor with a worked-out example.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The results files are plain text files that reveal information
for every PSM; however, only a reduced subset of these data will be accessed by
iSanXoT, as shown in the following Table.</p>

<div align=center>

<table class=MsoTableGrid border=0 cellspacing=0 cellpadding=0 width="91%"
 style='width:91.5%;border-collapse:collapse;border:none'>
 <tr style='height:15.0pt'>
  <td width="100%" colspan=2 valign=top style='width:100.0%;border:none;
  border-bottom:solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;
  height:15.0pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal'><span style='font-size:10.0pt'>Table.<b>
  Data accessed by iSanXoT in the results from MSFragger</b></span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;border:none;border-bottom:
  solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><b><span style='font-size:10.0pt'>Header</span></b></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;border:none;border-bottom:
  solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;height:14.25pt'>
  <p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal'><b><span style='font-size:10.0pt'>Description</span></b></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>scannum</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Scan number identifier</span></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>peptide</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Peptide amino acid sequence</span></p>
  </td>
 </tr>
 <tr style='height:27.8pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>modification_info</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Unimod chemical or posttranslational modifications
  to peptide sequence</span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>charge</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Peptide charge</span></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>hit_rank</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Rank of search engine</span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>calc_neutral_pep_mass</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Theoretical mass of the identified peptide ion in Da</span></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>massdiff</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Difference between measured and theoretical
  precursor neutral mass</span></p>
  </td>
 </tr>
 <tr style='height:27.05pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.05pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>protein</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.05pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Accession codes for the proteins to which the
  peptide sequence is ascribed </span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoNormal>&nbsp;</p>

<h4>Validating peptide identification with FDR</h4>

<p class=MsoNormal>This adaptor also contains the probability ratio method that
calculates the probability of random peptide matching and provides the
corresponding FDR for peptide identification. This module contains the same
parameters than the “Input from PD” adaptor. It should be pointed out that
score to use has to be the “hyperscore” from MSFragger.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=149 id="Picture 51"
src="iSanXoT-Help_files/image056.png"></p>

<p class=MsoCaption><b>Figure </b><b>36</b><b>. Task-Table of FDR module for a
work-out example with MSF input.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Close_Search_–_2"></a><a name="_Close_Search_–_3"></a><a
name="_Inputs_from_MaxQuant"></a>Inputs from MaxQuant (close search)</h3>

<p class=MsoNormal>This adaptor allows you to transform the output of MaxQuant
into an Identification-Quantification file appropriate for iSanXoT workflows.
iSanXoT has been tested for the outputs of MaxQuant 1.6.5.0.</p>

<p class=MsoNormal>The adaptor uses the following fields:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output folder: describes the path to the folder where iSanXoT
output files will be stored.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Default Input folder: specifies the location of the files
containing results files.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Input files: indicates which of the text plain files stored in
the input folder. The “modificationSpecificPeptides.txt” is the file considered
by iSanXoT. The file names are listed under Identification file column, while
their experiment is indicated in Experiment column.</p>

<p class=MsoNormal>Identifier of the associated modification summary stored in
the file “modificationSpecificPeptides.txt”. The following figure displays originate
file from the experiment (called “PME12”), but larger projects may encompass
several experiments.</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=567
height=225 id="Picture 3" src="iSanXoT-Help_files/image057.png"></p>

<p class=MsoCaption style='text-align:justify'><b>Figure </b><b>37</b><b>. Panel of &quot;Input from MaxQuant&quot; adaptor with a worked-out example.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The results files are plain text files that reveal information
for every PSM; however, only a reduced subset of these data will be accessed by
iSanXoT, as shown in the following Table.</p>

<div align=center>

<table class=MsoTableGrid border=0 cellspacing=0 cellpadding=0 width="91%"
 style='width:91.5%;border-collapse:collapse;border:none'>
 <tr style='height:15.0pt'>
  <td width="100%" colspan=2 valign=top style='width:100.0%;border:none;
  border-bottom:solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;
  height:15.0pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal'><b><span style='font-size:10.0pt'>Data
  accessed by iSanXoT from the MaxQuant (modifiedSpecificPeptides.txt)</span></b></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;border:none;border-bottom:
  solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><b><span style='font-size:10.0pt'>Header</span></b></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;border:none;border-bottom:
  solid windowtext 1.0pt;padding:0cm 5.75pt 0cm 5.75pt;height:14.25pt'>
  <p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal'><b><span style='font-size:10.0pt'>Description</span></b></p>
  </td>
 </tr>
 <tr style='height:13.5pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Sequence</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:13.5pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Peptide amino acid sequence</span></p>
  </td>
 </tr>
 <tr style='height:4.0pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:4.0pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Modifications</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:4.0pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Post-translational modifications contained within
  the sequence</span></p>
  </td>
 </tr>
 <tr style='height:14.25pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Score</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:14.25pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>Andromeda identification score for the MS/MS
  spectrum </span></p>
  </td>
 </tr>
 <tr style='height:27.05pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.05pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Proteins </span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.05pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>The IPI identifiers of the proteins the identified
  peptide is associated with. </span></p>
  </td>
 </tr>
 <tr style='height:16.4pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:16.4pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Protein Names</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:16.4pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>List of protein descriptions separated by comma </span></p>
  </td>
 </tr>
 <tr style='height:27.8pt'>
  <td width="28%" valign=top style='width:28.04%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal align=right style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:right;line-height:normal'><span style='font-size:10.0pt;
  position:relative;top:11.0pt'>Intensities …</span></p>
  </td>
  <td width="71%" valign=top style='width:71.96%;padding:0cm 5.75pt 0cm 5.75pt;
  height:27.8pt'>
  <p class=MsoNormal style='margin-top:.5pt;margin-right:0cm;margin-bottom:
  0cm;margin-left:0cm;margin-bottom:.0001pt;line-height:normal'><span
  style='font-size:10.0pt'>The intensities of the peaks in the fragmentation
  spectrum after top-N filtering</span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The analysis of Label-Free starts with the quantification
data from the peptide level and therefore, the WPP-SBT and WPPG-SBT “Complete
modules” that start the integration at peptide level have been thought to these
statistical analyses.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><a name="_Open_Search_–_1"></a>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><a name="_Workflows"></a>&nbsp;</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><a name="_References"></a><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><span lang=ES>References</span></h1>

</div>

<p class=MsoNormal><span lang=ES>[1] Trevisan-Herraz M. et al. </span>(2019)
SanXoT: a modular and versatile package for the quantitative analysis of
high-throughput proteomics experiments. Bioinformatics. 35, 1594-96.</p>

<p class=MsoNormal>[2] Garcia-Marques, F., et al., A Novel Systems-Biology
Algorithm for the Analysis of Coordinated Protein Responses Using Quantitative
Proteomics. Mol Cell Proteomics, 2016. 15(5): p. 1740-60.</p>

<p class=MsoNormal>[3] Navarro, P., et al., General statistical framework for
quantitative proteomics by stable isotope labeling. J Proteome Res, 2014.
13(3): p. 1234-47.</p>

<p class=MsoNormal>[4] Kong, A. T., Leprevost, F. V., Avtonomov, D. M.,
Mellacheruvu, D., &amp; Nesvizhskii, A. I. (2017). MSFragger: ultrafast and
comprehensive peptide identification in mass spectrometry–based proteomics.
Nature Methods, 14(5), 513-520.</p>

</div>

<div><br clear=all>

<hr align=left size=1 width="33%">

<div id=ftn1>

<p class=MsoFootnoteText style='text-align:justify'><a href="#_ftnref1"
name="_ftn1" title=""><span class=MsoFootnoteReference><span lang=EN-US><span
class=MsoFootnoteReference><span lang=EN-US style='font-size:10.0pt;line-height:
107%;font-family:"Calibri",sans-serif'>[1]</span></span></span></span></a><span
lang=EN-US> https://www.unimod.org/</span></p>

</div>

<div id=ftn2>

<p class=MsoFootnoteText style='text-align:justify'><a href="#_ftnref2"
name="_ftn2" title=""><span class=MsoFootnoteReference><span lang=EN-US><span
class=MsoFootnoteReference><span lang=EN-US style='font-size:10.0pt;line-height:
107%;font-family:"Calibri",sans-serif'>[2]</span></span></span></span></a><span
lang=EN-US> Eng J.K. <i>et al.</i> (1994) An approach to correlate tandem mass
spectral data of peptides with amino acid sequences in a protein database. <i>J.
Am. Soc. Mass Spectrom</i>. <b>5</b>, 976–89.</span></p>

</div>

</div>

</body>

</template>
