<template class="task-template" id="User_Guide">

<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:center;
	line-height:107%;
	page-break-after:avoid;
	border:none;
	padding:0cm;
	font-size:26.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:normal;
	font-style:italic;}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-link:"Footnote Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-link:"Comment Text Char";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:10.0pt;
	margin-left:0cm;
	text-align:center;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoFootnoteReference
	{vertical-align:super;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpFirst, li.MsoTitleCxSpFirst, div.MsoTitleCxSpFirst
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpMiddle, li.MsoTitleCxSpMiddle, div.MsoTitleCxSpMiddle
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpLast, li.MsoTitleCxSpLast, div.MsoTitleCxSpLast
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
a:link, span.MsoHyperlink
	{font-family:"Calibri",sans-serif;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
p
	{margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-link:"Comment Subject Char";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:9.0pt;
	font-family:"Segoe UI",sans-serif;}
p.MsoRMPane, li.MsoRMPane, div.MsoRMPane
	{margin:0cm;
	margin-bottom:.0001pt;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoQuote, li.MsoQuote, div.MsoQuote
	{mso-style-link:"Quote Char";
	margin-top:10.0pt;
	margin-right:43.2pt;
	margin-bottom:8.0pt;
	margin-left:43.2pt;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#404040;
	font-style:italic;}
span.MsoIntenseReference
	{font-variant:small-caps;
	color:#4472C4;
	letter-spacing:.25pt;
	font-weight:bold;}
span.MsoBookTitle
	{letter-spacing:.25pt;
	font-weight:bold;
	font-style:italic;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-link:Title;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-link:"Footnote Text";}
p.lead, li.lead, div.lead
	{mso-style-name:lead;
	margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.QuoteChar
	{mso-style-name:"Quote Char";
	mso-style-link:Quote;
	color:#404040;
	font-style:italic;}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-link:"Comment Text";}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-link:"Comment Subject";
	font-weight:bold;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-link:"Balloon Text";
	font-family:"Segoe UI",sans-serif;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:595.3pt 841.9pt;
	margin:70.85pt 3.0cm 70.85pt 3.0cm;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>

</head>

<body lang=EN-GB link="#000000" vlink="#954F72">

<div class=WordSection1>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoTitle align=center style='text-align:center'><b><i><span
lang=EN-US style='font-size:36.0pt;letter-spacing:.25pt'><img width=310
height=355 id="Picture 16" src="User_Guide_files/image001.png" alt=Molecule></span></i></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoTitleCxSpFirst align=center style='text-align:center'><span
class=MsoBookTitle><span lang=EN-US style='font-size:36.0pt'>User Guide:</span></span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
class=MsoBookTitle><span lang=EN-US style='font-size:36.0pt'>Help and
documentation</span></span></p>

<p class=MsoTitleCxSpLast align=center style='text-align:center'><span
class=MsoBookTitle><span lang=EN-US>(1.0.x)</span></span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><span lang=EN-US>Table of contents</span></h1>

</div>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Introduction">Introduction</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_License">License</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:Symbol;text-decoration:
none'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_Installation">Installation</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Download">Download</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Available_operating_systems">Available operating systems</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:"Courier New";
text-decoration:none'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;letter-spacing:
0pt;font-weight:normal;font-style:normal'><a href="#_Windows_distribution">Windows
distribution</a></span></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_MacOS_distribution">MacOS distribution</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Linux_distribution">Linux distribution</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:Symbol;
letter-spacing:0pt;font-weight:normal;font-style:normal'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_Get_Started">Getting Started</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:Symbol;text-decoration:
none'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_Modules">Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Basic_modules">Relation Tables Module</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_RELS_CREATOR">RELS CREATOR</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Basic_modules">Basic Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_LEVEL_CREATOR">LEVEL CREATOR</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_LEVEL_CALIBRATOR">LEVEL CALIBRATOR</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_INTEGRATE">INTEGRATE</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_NORCOMBINE">NORCOMBINE</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_RATIOS">RATIOS</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_SBT">SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Compound_modules">Compound Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WSPP-SBT">WSPP-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WSPPG-SBT">WSPPG-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WPP-SBT">WPP-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoHyperlink><span lang=EN-US style='font-family:Wingdings;
text-decoration:none'>§<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WPPG-SBT">WPPG-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Report_modules">Report Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_REPORT">REPORT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoHyperlink><span lang=EN-US style='font-family:Wingdings;
text-decoration:none'>§<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_SANSON">SANSON</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Special_parameters">Special Parameters</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Keywords_for_the">Keywords for the Task-Tables</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Multiple_samples">Multiple samples</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Asterisk_is_our">Asterisk is our jack of all trades</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_More_params">More params</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Filter_param_(in">Filter (for REPORT and SANSON)</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:Symbol;
letter-spacing:0pt;font-weight:normal;font-style:normal'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_Adaptors">Input Adaptor</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
class=MsoHyperlink><span lang=EN-US>Input provided by user</span></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Inputs_from_PD">Inputs adapted from Proteomic Pipelines
results</a> </span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:Symbol;text-decoration:
none'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_References">References</a></span></p>

<span class=MsoBookTitle><span lang=EN-US style='font-size:11.0pt;line-height:
107%;font-family:"Calibri",sans-serif;letter-spacing:0pt;font-weight:normal;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Introduction"></a><span lang=EN-US>Introduction</span></h1>

</div>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><b>iSanXoT</b> is a standalone application for statistical
analysis of mass spectrometry-based quantitative proteomics data.  iSanXoT
builds upon SanXoT[1], our previous publicly available implementation of the
weighted spectrum, peptide, and protein (WSPP) statistical model[3] using the
Generic Integration Algorithm (GIA)[2].</p>

<p class=MsoNormal>iSanXoT executes several kind of workflows for quantitative
high-throughput proteomics, <b>systems biology </b>and the <b>statistical
analysis</b>, <b>integration and comparison of experiments</b>.</p>

<p class=MsoNormal>iSanXoT was developed by the <b><i>Cardiovascular Proteomics
Lab</i></b>/<b><i>Proteomic Unit</i></b> at <b>The National Centre for
Cardiovascular Research</b> (CNIC, <span lang=EN-US><a
href="https://www.cnic.es"><span lang=EN-GB style='color:#4472C4'>https://www.cnic.es</span></a></span>).</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_License"></a>License</h2>

<p class=MsoNormal>This application is licensed under a <b>Creative Commons
Attribution-NonCommercial-NoDerivs 4.0 Unported License</b>.</p>

<p class=MsoNormal><span lang=EN-US><a
href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><span lang=EN-GB
style='color:#4472C4'>https://creativecommons.org/licenses/by-nc-nd/4.0/</span></a></span></p>

<h4><span lang=EN-US>You are free to:</span></h4>

<p class=MsoNormal><span lang=EN-US>Share - copy and redistribute the material
in any medium or format.</span></p>

<p class=MsoNormal><span lang=EN-US>The licensor cannot revoke these freedoms
as long as you follow the license terms.</span></p>

<h3><span lang=EN-US>Under the following terms</span></h3>

<h4><span lang=EN-US>Attribution:</span></h4>

<p class=MsoNormal><span lang=EN-US>You must give appropriate credit, provide a
link to the license, and indicate if changes were made. You may do so in any
reasonable manner, but not in any way that suggests the licensor endorses you
or your use.</span></p>

<h4><span lang=EN-US>NonCommercial:</span></h4>

<p class=MsoNormal><span lang=EN-US>You may not use the material for commercial
purposes.</span></p>

<h4><span lang=EN-US>NoDerivatives:</span></h4>

<p class=MsoNormal><span lang=EN-US>If you remix, transform, or build upon the
material, you may not distribute the modified material.</span></p>

<h4><span lang=EN-US>No additional restrictions:</span></h4>

<p class=MsoNormal><span lang=EN-US>You may not apply legal terms or
technological measures that legally restrict others from doing anything the
license permits.</span></p>

<h4><span lang=EN-US>Notices</span></h4>

<p class=MsoNormal><span lang=EN-US>You do not have to comply with the license
for elements of the material in the public domain or where your use is
permitted by an applicable exception or limitation. No warranties are given.
The license may not give you all of the permissions necessary for your intended
use. For example, other rights such as publicity, privacy, or moral rights may
limit how you use the material.</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Installation"></a><span lang=EN-US>Installation</span></h1>

</div>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Download"></a><span lang=EN-US>Download</span></h3>

<p class=MsoNormal>The multiple releases are available in the
&quot;release&quot; section, located in the following link:</p>

<p class=MsoNormal style='text-indent:35.4pt'><span lang=EN-US><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span lang=EN-GB
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></span></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3><a name="_Available_operating_systems"></a>Available operating systems</h3>

<p class=MsoNormal>iSanXoT maintains the following operating systems and
architectures and may add additional ones in the future:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Windows 10 Pro (x64)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>MacOs High Sierra (10.13.6)</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ubuntu 20.04 (x64)</p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<h3><a name=windows-distribution></a><a name=installation-on-windows></a><a
name="_Installation_on_Windows"></a><a name="_Windows_distribution"></a><span
lang=EN-US>Windows distribution</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT Windows distribution is
packaged in a NSIS Launcher (exe file).</span></p>

<p class=MsoNormal><span lang=EN-US>Download the exe Launcher: iSanXoT_Launcher_1.X.X.win32-x64.exe</span></p>

<p class=MsoNormal><span lang=EN-US>Double-click the Launcher file; the Installer
window will show up:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 28" src="User_Guide_files/image002.jpg"></span></p>

<p class=MsoNormal><span lang=EN-US>WARNING: Windows Defender SmartScreen might
show a prompt suggesting that you cancel the installation; in such case click
“More info” and then select the “Run anyway” option.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=332 height=140 id="Picture 43"
src="User_Guide_files/image003.png"></span></p>

<p class=MsoNormal><span lang=EN-US>Then, you can choose the iSanXoT
installation folder:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 25" src="User_Guide_files/image004.jpg"></span></p>

<p class=MsoNormal><span lang=EN-US>Wait while iSanXoT is being installed.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 22" src="User_Guide_files/image005.png"
alt="Installing window"></span></p>

<p class=MsoNormal><span lang=EN-US>Once the installation has been completed,
you are ready to run .</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 21" src="User_Guide_files/image006.png"
alt="Completing setup"></span></p>

<p class=MsoNormal><span lang=EN-US style='font-family:"Segoe UI",sans-serif;
color:#24292E'>&nbsp;</span></p>

<h3><a name=macos-distribution></a><a name="_MacOS_distribution"></a><span
lang=EN-US>MacOS distribution</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT MacOS distribution is packaged
in a DMG container.</span></p>

<p class=MsoNormal><span lang=EN-US>Download the DMG file: iSanXoT_Launcher_1.X.X.darwin-x64.dmg</span></p>

<p class=MsoNormal><span lang=EN-US>Double-click the DMG file, then a Finder
window will show up. This window will usually display iSanXoT’s installer icon
and a shortcut to the Applications folder, together with some sort of linking arrow:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=416 height=309 id="Picture 19"
src="User_Guide_files/image007.png" alt="Installer window"></span></p>

<p class=MsoNormal><span lang=EN-US>Simply drag the iSanXoT icon to your
Applications folder...</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=439 height=315 id="Picture 18" src="User_Guide_files/image008.png"
alt="Installing window"></span></p>

<p class=MsoNormal><span lang=EN-US>and you're done: the iSanXoT application is
now installed.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name=linux-distribution></a><a name="_Linux_distribution"></a><span
lang=EN-US>Linux distribution</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT Linux distribution is packaged
in an <i>AppImage</i>.</span></p>

<p class=MsoNormal><span lang=EN-US>Download the AppImage: iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=567 height=241 src="User_Guide_files/image009.png"></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:0cm'><span lang=EN-US>The
AppImage file is just the application's compressed image. When executed, the application
is mounted in a temporal folder. However, for the correct behaviour, the application
has to be extracted to the &quot;squashfs-root&quot; folder in the current
working directory using:</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>&nbsp;</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm'><span lang=EN-US>Then,
launch the iSanXoT application using:</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:0cm;text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun</span></em></p>

<em><span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><span lang=EN-US
style='font-family:"Calibri",sans-serif;font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<em><span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><b><span lang=EN-US
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00;font-style:normal'>&nbsp;</span></b></em></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Get_Started"></a><em><span lang=EN-US style='font-family:"Calibri Light",sans-serif;
font-style:normal'>Getting Started</span></em></h1>

</div>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>This chapter describes iSanXoT’s graphical user interface and
how to set up an analysis with iSanXoT.</span></em></p>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3><span lang=EN-US>Opening the iSanXoT application</span></h3>

<p class=MsoNormal><span lang=EN-US>To open the iSanXoT application:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>In Windows: from the Start menu choose <i>Programs
&gt; iSanXoT</i>; or double-click the iSanXoT desktop icon.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>In MacOS: double-click the iSanXoT icon from the
Applications folder.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>In Linux: from the <i>AppImage</i> file.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>The contents are extracted
to the &quot;squashfs-root&quot; directory in the current working directory using:</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.x.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>Now you can launch the iSanXoT
application:</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun [...]</span></em></p>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3><span lang=EN-US>Installing required packages</span></h3>

<p class=MsoNormal><span lang=EN-US>The first time iSanXoT is run a window will
show up displaying a progress bar to inform you about the percentage of
packages that have been installed. These packages contain the libraries
required by iSanXoT’s backend, and are installed the first time you open the
application.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=482 height=317
src="User_Guide_files/image010.png"></span></p>

<p class=MsoCaption><b><span lang=EN-US>Figure </span></b><b><span
lang=EN-US>1</span></b><b><span lang=EN-US>. <a name="_Ref103093720">Installation
window.</a></span></b></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>Closing the iSanXoT application</span></h3>

<p class=MsoNormal><span lang=EN-US>WARNING: If valid changes were made to your
project, make sure to save it before quitting iSanXoT, as any changes will be
lost otherwise (you won’t be prompted for saving upon closing).  </span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>To close the
iSanXoT application:</span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>In Windows and
Linux: choose <i>Project &gt; Exit</i>, or click the X in the upper right
corner of the main iSanXoT window.</span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>In Mac: choose <i>iSanXoT
(menu) &gt; Exit</i>, or click the red X in the upper left corner of the main
iSanXoT window.</span></p>

<p class=MsoNormal><span lang=EN-US>A dialog window will show up asking you to
confirm the application closing. Click “Yes” if you really want to quit iSanXoT.
</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>iSanXoT Projects</span></h3>

<p class=MsoNormal><span lang=EN-US>An iSanXoT project is primarily a container
used to structure the data coming from your input file(s) and your workflow.
The input file contains the identification and quantification data (for further
details see the <a href="#_Input_Adaptor">Input Adaptor</a> Section). These fully-customisable
workflows can perform quantitative proteomics analysis, systems biology
analysis, and comparison and merging of experimental data from technical or
biological replicates.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=251 height=240
src="User_Guide_files/image011.png"></span></p>

<p class=MsoCaption><a name="_Ref103114194"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>2</span></b><b><span lang=EN-US>. Project menu.</span></b></p>

<h4><span lang=EN-US>Creating a new project</span></h4>

<p class=MsoNormal><span lang=EN-US>Creating a project is the first step when
conducting an analysis with iSanXoT. By selecting <i>Project  &gt; New Project</i>
a window will show up where you can provide a name for the project as well as
select a project folder where iSanXoT output files will be stored.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=536 height=199
src="User_Guide_files/image012.png"></span></p>

<p class=MsoCaption><b><span lang=EN-US>Figure </span></b><b><span
lang=EN-US>3</span></b><b><span lang=EN-US>. Window that creates a new project.</span></b></p>

<h4><span lang=EN-US>&nbsp;</span></h4>

<h4><span lang=EN-US>Opening a project</span></h4>

<p class=MsoNormal><span lang=EN-US>By selecting <i>Project &gt; Open Project</i>
as a folder selection dialog box shows up that allows the user to indicate the
location of an already existing project folder to be opened by iSanXoT.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>iSanXoT Main Window</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT main window consists of an
overhead menu, adaptor and module tabs, and content and execution panels (</span><span lang=EN-US><b>Figure 4</b></span><span lang=EN-US>).</span></p>

<h4><span lang=EN-US>Menu</span></h4>

<p class=MsoNormal><span lang=EN-US>The Menu contains the following items:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Project: Allows operations related to  projects
and workflows.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Processes: Links to real-time display of the processes
currently in execution by iSanXoT (see <a href="#_Running_Processes">Running Processes</a>
Section).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Help: Houses the different Sections of iSanXoT
help.</span></p>

<h4><span lang=EN-US>Content tabs</span></h4>

<p class=MsoListParagraph style='text-indent:-18.0pt'><span lang=EN-US>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US>Five tabs are displayed in iSanXoT’s project page. The Input tab
displays the Project folder, where iSanXoT output files are stored, as well as
the Identification file used in the project (see <a href="#_Adaptors">Input Adaptor</a>
Section). The remaining four tabs give access to iSanXoT modules: <a
href="#_Relation_tables_module">Relation Tables</a>, <a href="#_Basic_modules_1">Basic
Modules</a>, <a href="#_Compound_modules">Compound Modules</a>, and <a
href="#_Reports_modules">Report Modules</a>.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=570 height=337 id="Picture 132"
src="User_Guide_files/image013.png"></span></p>

<p class=MsoCaption><a name="_Ref103114143"></a><a name="_Ref103114150"><b><span
lang=EN-US>Figure </span></b></a><b><span lang=EN-US>4</span></b><b><span
lang=EN-US>. Main View of iSanXoT.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Content panel</span></h4>

<p class=MsoNormal><span lang=EN-US>This panel houses the elements of the Input
and Modules tabs, which can be accessed through the sidebar menu showing on the
left side of the panel. A title and a brief description of the Input/Module
element is provided, as well as help icon linking to additional information on
the specific Input element or Module selected.</span></p>

<h4><a name="_Execution_panel"></a><span lang=EN-US>Execution panel</span></h4>

<p class=MsoNormal><span lang=EN-US>The execution panel, located on the bottom
of the main window, allows the user to indicate the number of processors to be
used by iSanXoT, 4 by default. The Start button launches the execution of the
workflow shaped by the Input elements and Modules.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><span lang=EN-US>Importing and Exporting Workflows</span></h2>

<p class=MsoNormal><span lang=EN-US>A project is shaped by a workflow that
instructs iSanXoT how to process the data provided by the input file(s) . While
the whole project, including workflow and data, can be saved as indicated below
(see the <a href="#_Saving_a_project">Saving a project</a> section), there is a
way to import and export just the workflow structure using iSanXoT menu (</span><span lang=EN-US><b>Figure 2</b></span><span lang=EN-US>).</span></p>

<h4><span lang=EN-US>Import Workflow</span></h4>

<p class=MsoNormal><span lang=EN-US>This option allows to import the
task-tables of a workflow. For that, you have to provide the folder where the
workflow is saved.</span></p>

<h4><span lang=EN-US>Export Workflow</span></h4>

<p class=MsoNormal><span lang=EN-US>The export workflow saves the task-tables
of a workflow in the folder indicated by the user.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><span lang=EN-US>Executing a Project</span></h2>

<p class=MsoNormal><span lang=EN-US>Once your project contains all the
necessary input data and workflow elements, you can execute the workflow  by
clicking “Save and Run” in the “Execution panel” after indicating the number of
processors to be allocated for iSanXoT (see  <a href="#_Execution_panel">Execution
panel</a> above). </span></p>

<p class=MsoNormal><span lang=EN-US>Bear in mind that every time you click the
“Save and Run” button to execute a workflow, the project is first validated for
consistency and saved. To save a project without executing it, you must use the
appropriate menu item as explained in the next Section.</span></p>

<h4><a name="_Saving_a_project"></a><span lang=EN-US>Saving a project</span></h4>

<p class=MsoNormal><span lang=EN-US>The <i>Project &gt; Save Project</i> option
saves your project, which contains the input data and the workflow elements, to
the “Project folder”. The project files are saved in the “.isanxot” folder. WARNING:
Do not manipulate or delete the information stored in the “.isanxot” folder;
you risk losing your project.</span></p>

<p class=MsoNormal><span lang=EN-US>Whenever iSanXoT is prompted to save a project,
the corresponding workflow is first validated for consistency and won’t be
saved when failed. Neither will the workflow execute if it has not been validated
previously.</span></p>

<p class=MsoNormal><span lang=EN-US>As well as the “.isanxot” folder, the
following folders are necessary to shape your project:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Exps,   to store the files created by the Input
Data adapters.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Jobs, to store  the <i>sample folders</i> of
your workflow.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Rels, to store  the Relation Tables created by
the RELS CREATOR module (see below).</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Reports, to store the Report files created by
the REPORT module (see below).</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Stats, to store statistical data.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Logs, to store the workflow execution log files.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><a name="_Running_Processes"></a><span lang=EN-US>Running Processes</span></h2>

<p class=MsoNormal><span lang=EN-US>When workflow execution successfully starts,
a new window shows up displaying information about the processes currently
running:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The Project logs table shows project execution status.
Several project executions can be monitored here, and the user must click a row
to have the corresponding workflow logs displayed (see below).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Workflow logs table displays status for the jobs
set up in the workflow modules. If you click on a row you will see the trace
log of the involved</span><span class=MsoCommentReference><span lang=EN-US
style='font-size:8.0pt;line-height:107%'> j</span></span><span lang=EN-US>obs (unless 
the job status is “cached”).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=378 height=262 id="Picture 98"
src="User_Guide_files/image014.png"></span></p>

<p class=MsoCaption><b><span lang=EN-US>Figure </span></b><b><span
lang=EN-US>5</span></b><b><span lang=EN-US>. View of running processes.</span></b></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>The
running processes window can be also reached from the menu by selecting the “Processes
&gt; Main page” option.</span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span lang=EN-US
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Modules"></a><span lang=EN-US>Modules</span></h1>

</div>

<p class=MsoNormal><span lang=EN-US>The iSanXoT desktop application houses a
number of modules based on the SanXoT software package [1]. The information
required to setup and execute every module  is provided in a task-table.   </span></p>

<p class=MsoNormal><span lang=EN-US>There are four types of modules:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Relation tables is a module that creates the
relation tables used by the iSanXoT modules.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The Basic modules call the individual scripts
included in the SanXoT software package [1].</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The Compound modules perform a sequence of
consecutive integrations based on the weighted spectrum, peptide and protein (WSPP)
statistical model [3] and the systems-biology triangle (SBT) algorithm [2].</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Finally, there are two Reports: REPORT generates
report files displaying the quantitative results produced by the above Basic
and Composite modules when a workflow is executed; SANSON generates a
similarity graph showing relationships between functional categories on the
basis of the protein elements they share.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Basic_modules"></a><a
name="_Relation_tables_module"></a><span lang=EN-US>Relation tables module</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_RELS_CREATOR"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>RELS
CREATOR</span></h3>

<p class=MsoNormal><span lang=EN-US>This module generates relation tables (RT) from
tab-separated values (TSV) files. Relation tables, which are TSV files relating
lower-level identifiers (e.g. peptides) to the corresponding higher-level
elements (e.g. proteins),  are required for module execution. For this reason,
the naming convention for the file indicated under “Relation Table to be
created” in the RELS CREATOR task table is <i>lower level + “2” + higher level</i>;
e.g. whether an integration is from the “peptide” level to the “protein” level,
a relation table called “peptide2protein” will be necessary.</span></p>

<p class=MsoNormal><span lang=EN-US>The fields showing in the RELS CREATOR task
table are (</span><span lang=EN-US><b>Figure 6</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Relation Table to be created</span></i><span
lang=EN-US> specifies relation table filenames. As commented above, the naming convention
for these files is <i>lower level + “2” + higher level</i>; e.g. whether an
integration is from the “peptide” level to the “protein” level, a relation
table called “peptide2protein” will be necessary.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Column name of Lower level</span></i><span
lang=EN-US> is the column header that designates which elements from the
indicated file (see below) will be taken as lower level elements in the
resulting relation table.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Column name of Higher level</span></i><span
lang=EN-US> is the column header that designates which elements from the
indicated file (see below) will be taken as higher level elements in the
resulting relation table.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Column name of 3<sup>rd</sup> column</span></i><span
lang=EN-US> is the column header that designates which elements from the
indicated file (see below) will be taken as third column elements in the
resulting relation table.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Table from which RT is extracted is the full
path name for the TSV file to be used to build the relation tables. If the cell
is empty, the Input file (ID-q.tsv) is applied for iSanXoT workflow.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=163
src="User_Guide_files/image015.png"></span></p>

<p class=MsoCaption><a name="_Ref103114314"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>6</span></b><b><span lang=EN-US>. A sample Task-Table in the example
of RELS CREATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Basic_modules_1"></a><span
lang=EN-US>Basic modules</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_LEVEL_CREATOR"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>LEVEL
CREATOR</span></h3>

<p class=MsoNormal><span lang=EN-US>This module creates levels, which are TSV
files containing identifiers, log<sub>2</sub>-ratio values and statistical
weight values. The data are extracted from the Identification/Quantification
file (see <a href="#_Adaptors">Input Adaptor</a> section).</span></p>

<p class=MsoNormal><span lang=EN-US>The following fields are displayed in the
LEVEL CREATOR task table (</span><span lang=EN-US><b>Figure 7</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used to create the level indicated. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator column</span></i><span
lang=EN-US> specifies which column header from the identification file
designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator column(s)</span></i><span
lang=EN-US> specifies which column header from the identification file
designates the quantitative values to be used as a denominator for the log<sub>2</sub>-ratio
calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level to be created</span></i><span
lang=EN-US> designates the level name.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> indicates the name of the folder where the level data file will be
saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=581 height=174
src="User_Guide_files/image016.png"></span></p>

<p class=MsoCaption><a name="_Ref103114332"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>7</span></b><b><span lang=EN-US>. A sample task-table in the LEVEL
CREATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_LEVEL_CALIBRATOR"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>LEVEL
CALIBRATOR</span></h3>

<p class=MsoNormal><span lang=EN-US>This module calibrates the above-described
levels using the “Klibrate” program included in the SanXoT software package
[1].To perform the calibration, two parameters (weight constant and variance) are
iteratively calculated using the Levenberg-Marquardt algorithm (for more details
see the information about “Klibrate” in the SanXoT software package [1]).</span></p>

<p class=MsoNormal><span lang=EN-US>The output calibrated level contains new
statistic weight values for the identifier and log<sub>2</sub>-ratio elements displayed
in the uncalibrated level data file. This is necessary for the levels to be
used as inputs to the INTEGRATE module.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The fields to be completed in this module
are (</span><span lang=EN-US><b>Figure 8</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the name(s) of the folder(s) containing the uncalibrated data file(s)
that were previously generated by the LEVEL CREATOR module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level for integration</span></i><span
lang=EN-US> indicates which lower level elements are to be used in the integration
carried out for the calibration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level for integration</span></i><span
lang=EN-US> indicates which higher level elements are to be used in the
integration carried out for the calibration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Name of calibrated level</span></i><span
lang=EN-US> is the name for the output data file containing the new, calibrated
statistical weight values.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> specifies the name of the folder where the output data file
containing the new, calibrated statistical weight values will be saved. If the
cell is empty, the output sample folder is the given “Sample folder” (second
column).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=193 id="Picture 5"
src="User_Guide_files/image017.png"></span></p>

<p class=MsoCaption><a name="_Ref103114351"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>8</span></b><b><span lang=EN-US>. A sample task table in the LEVEL
CALIBRATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The LEVEL_CALIBRATOR module accepts the
following additional parameters (</span><span lang=EN-US><b>Figure 9</b></span><span
lang=EN-US>): </span></p>

<p class=MsoNormal><span lang=EN-US>To perform the calibration two parameters,
have to be calculated: the k (weight constant), and the variance.</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>K-constant </span></i><span lang=EN-US>sets a
forced value for the k-constant. Using this parameter, the introduced value is
forced as K-constant.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Var(x)</span></i><span lang=EN-US> sets a forced
value for the variance. Using this parameter, the introduced value is forced as
the variance.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US><img
border=0 width=579 height=145 src="User_Guide_files/image018.png"></span></p>

<p class=MsoCaption style='text-align:justify'><a name="_Ref103114398"><b><span
lang=EN-US>Figure </span></b></a><b><span lang=EN-US>9</span></b><b><span
lang=EN-US>. A task-table displaying advanced parameters for the LEVEL
CALIBRATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_INTEGRATE"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>INTEGRATE</span></h3>

<p class=MsoNormal><span lang=EN-US>The INTEGRATE module performs statistical
calculations based on the WSPP model by iteratively applying the generic
integration algorithm (GIA) [2] on calibrated data files (</span><span lang=EN-US><b>Figure 10</b></span><span lang=EN-US>). </span></p>

<p class=MsoNormal><span lang=EN-US>Integrations are carried out from lower level
data to higher level data (e.g. from the peptide level to the protein level and
from the protein level to the gene level).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=423 height=414 id="Picture 58"
src="User_Guide_files/image019.png"></span></p>

<p class=MsoCaption><a name="_Ref103096863"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>10</span></b><b><span lang=EN-US>. Schematic representation of the
INTEGRATE module.</span></b><span lang=EN-US> The integration is carried out
from any lower level to any higher level using the programs “SanXoT” and “SanXoTSieve”
and the generic integration algorithm (GIA).</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>More in detail, the INTEGRATE module needs two
TSV files as inputs:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US>1.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>A data file containing three data columns:
identifier (a text string that is used to unambiguously identify the low level elements),
quantitative value (log<sub>2</sub>-ratio of the two measurements to be
compared) and statistical weight (a parameter that measures the accuracy of the
quantitative value).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US>A relation table, which links the lower level identifiers to those
in the higher level. This file contains two columns: higher level identifiers
on the left and lower level identifiers on the right.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=582 height=524 id="Picture 41"
src="User_Guide_files/image020.png"></span></p>

<p class=MsoCaption><a name="_Ref103114442"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>11</span></b><b><span lang=EN-US>. The INTEGRATE module flowchart.</span></b><span
lang=EN-US> A first integration is done with “SanXoT” that calculates the
variance; then “SanXoTSieve” removes outliers tagging them in a new relation
table; finally, a second integration is done with “SanXoT” using the variance
calculated.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>For every integration, the SanXoT program calculates
the general variance using a robust iterative method. Then SanXoTSieve is used
to tag outlier elements [3] by assessing the probability that a lower level element
be a significant outlier of the standardized (i.e. N(0,1)) log<sub>2</sub>-ratio
distribution. The most extreme outliers are thus removed sequentially and the
integration repeated until all outliers below a user-defined false discovery
rate (FDR) threshold have been removed. Finally, a second integration is carried
out by SanXoT using the variance calculated in the first integration and
discarding the outliers tagged in the new relation table (</span><span lang=EN-US><b>Figure 11</b></span><span lang=EN-US>).</span></p>

<p class=MsoNormal><span lang=EN-US>The output data files generated by
INTEGRATE contain the quantitative data for the higher level and can be used as
inputs to other modules; in addition, each integration generates several
additional files which contain information about the integration. For further
details see SanXoT documentation [1].</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The parameters to be provided in the
INTEGRATE module task table are (</span><span lang=EN-US><b>Figure 12</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the names of the folder(s) where the lower level data file is
located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US> indicates
the name of the lower level data file to be used. This file contains three data
columns: identifier, quantitative value and statistical weight.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level</span></i><span lang=EN-US>
indicates the name of the higher level to which the lower level elements will
be integrated.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=282 height=159 id="Picture 6"
src="User_Guide_files/image021.png"></span></p>

<p class=MsoCaption><a name="_Ref103114454"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>12</span></b><b><span lang=EN-US>. A sample task-table in the
INTEGRATE module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><a name="_Advanced_parameters"></a><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The INTEGRATE module accepts the following additional
parameters (</span><span lang=EN-US><b>Figure 13</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> is the name of the folder where the level data and statistics are 
saved.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Tag</span></i><span lang=EN-US> is a text
label that indicates which elements from the lower level are integrated into
the higher level. The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label ”marked” is used as a Tag, only the lower level elements containing
the label marked  in the third column of the lower_level2higher_level Relation Table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><i><span lang=EN-US>Tag</span></i><span
lang=EN-US> is a parameter to distinguish groups to perform the integration. For
instance, if the user specifies “marked” the elements containing the label
“marked” in the third column of the “lower_level2higher_level” Relation Table
will be included in (or discarded from) the integration.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>The tag can be used by
inclusion, such as &quot;mod&quot; or by exclusion, putting first the
&quot;!&quot; symbol, such as &quot;!mod&quot;.  Tags should be included in a
third column of the relations file.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>Different tags can be
combined using logical operators &quot;and&quot; (&amp;), &quot;or&quot; (|),
and &quot;not&quot; (!), and parentheses. Some examples:</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>!out&amp;mod</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>!out&amp;(dig0|dig1)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>(!dig0&amp;!dig1)|mod1</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>mod1|mod2|mod3</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><b><i><span lang=EN-US>Warning</span></i></b><span
lang=EN-US>: Unless specified otherwise by the user, by default iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>iSanXoT automatically adds
the tag “out” in the third column of the relation table to label outliers, so
that they are not integrated. It is not thus recommended to use this tag for
other purposes.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><b><i><span lang=EN-US>Note</span></i></b><span
lang=EN-US> that although the discarded elements will not be included in
calculations, the parameter Z will be calculated and tabulated in the
corresponding output (outStats) file.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details see
SanXoT wiki [6].</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>FDR</span></i><span lang=EN-US> is an FDR
threshold other than the default value (0.01, i.e. 1%) for outlier removal. If “0”
is specified as the FDR value, then no outliers will be discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Var(x)</span></i><span lang=EN-US> sets a
fixed value for the variance. The default value (blank) means that the variance
will be iteratively calculated based on the Levenberg-Marquardt algorithm in
the first <i>Lower level </i>&gt;<i> Higher level</i> integration.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=136
src="User_Guide_files/image022.png"></span></p>

<p class=MsoCaption><a name="_Ref103114466"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>13</span></b><b><span lang=EN-US>. A sample task-table displaying
advanced parameters for the INTEGRATE module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_NORCOMBINE"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>NORCOMBINE</span></h3>

<p class=MsoNormal><span lang=EN-US>The NORCOMBINE module combines technical or
biological replicates (</span><span lang=EN-US><b>Figure 14</b></span><span
lang=EN-US>). For example, NORCOMBINE can be used to merge the protein-level
data from 4 individual patients and 4 individual controls into a patient- and a
control-level protein dataset, respectively.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=451 height=388 id="Picture 114"
src="User_Guide_files/image023.png"></span></p>

<p class=MsoCaption><a name="_Ref103114484"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>14</span></b><b><span lang=EN-US>. Schematic representation of the
NORCOMBINE module used to combine technical or biological replicates.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>Experiment merging relies on the “Cardenio”
program from the SanXoT software package [1], which is used to generate merged
data files and relation tables that are later integrated to the grouped level
using “SanXoT” and “SanXoTSieve” (</span><span
lang=EN-US><b>Figure 14</b></span><span lang=EN-US>).</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>NORCOMBINE requires the user to specify
which <i>lowerNorm</i> files contain the necessary data for the samples to be
combined. These <i>lowerNorm</i> files, previously generated by the INTEGRATE
module, display the lower level identifiers on the left, followed by the
corresponding centred log<sub>2</sub>-ratio values (i.e. the values obtained
after subtracting the high level value) in the second column, and either the integration
statistical weight (in the case of <i>lowerNormV</i>) or the variance (for <i>lowerNormW).</i>
The SanXoT program “Cardenio” [1] is then used to generate merged data files
and relation tables that are later integrated to the grouped level using
“SanXoT” and “SanXoTSieve” (</span><span lang=EN-US><b>Figure 15</b></span><span
lang=EN-US>).  </span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=452 height=522 id="Picture 115"
src="User_Guide_files/image024.png"></span></p>

<p class=MsoCaption><a name="_Ref103114585"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>15</span></b><b><span lang=EN-US>. The NORCOMBINE module flowchart.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The default NORCOMBINE module task table
shows the following fields (</span><span lang=EN-US><b>Figure 16</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folders</span></i><span lang=EN-US>
indicates the names of the folder(s) containing the lower level data (samples)
to be combined.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level</span></i><span lang=EN-US> indicates
the type of elements to be combined (e.g. peptides or proteins). </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Norm</span></i><span lang=EN-US> specifies
the normalization scheme to be used in the integrations.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>lowerNorm</span></i><span lang=EN-US> specifies
the type of <i>lowerNorm</i> file (see above) to be used.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> is the name of the folder where the grouped-level data and
statistics are  saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=467 height=89 src="User_Guide_files/image025.png"></span></p>

<p class=MsoCaption><a name="_Ref103114601"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>16</span></b><b><span lang=EN-US>. A sample task-table in the
NORCOMBINE module.</span></b><span lang=EN-US> In this case, the asterisk
wildcard has been used to select multiple sample folders.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The NORCOMBINE module accepts the following
additional parameters (</span><span lang=EN-US><b>Figure 17</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Tag</span></i><span lang=EN-US> is a text
label that indicates which elements from the lower level are integrated into
the higher level. The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label ”marked” is used as a Tag, only the lower level elements containing
the label marked  in the third column of the lower_level2higher_level Relation table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>FDR</span></i><span lang=EN-US> is an FDR
threshold other than the default value (0.01, i.e. 1%) for outlier removal. If
“0” is specified as the FDR value, then no outliers will be discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Var(x) sets a fixed value for the variance. The
default value (blank) means that the variance will be iteratively calculated
based on the Levenberg-Marquardt algorithm.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=72 id="Picture 9"
src="User_Guide_files/image026.png"></span></p>

<p class=MsoCaption><a name="_Ref103114615"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>17</span></b><b><span lang=EN-US>. A sample task table with advanced
parameters in the NORCOMBINE module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_RATIOS"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>RATIOS</span></h3>

<p class=MsoNormal><span lang=EN-US>This module prepares the data file and
relation table required as a first step in the calculation of a ratio defined
by the user (e.g KO vs WT). For that, the new log<sub>2</sub>-ratio is
calculated as the difference between numerator and denominator values, whereas
the corresponding statistical weight is assessed according to the method indicated
by the user in the V Method filed of the RATIOS task table (</span><span lang=EN-US><b>Figure 18</b></span><span lang=EN-US> and </span><span
lang=EN-US><b>Figure 19</b></span><span lang=EN-US>): </span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>max</span></i><span lang=EN-US> uses the
maximum value between the numerator and denominator statistical weight value. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>form</span></i><span lang=EN-US> uses the value
resulting from 1/(1/Vn + 1/Vd), where Vn and Vd are the statistical weight
value for the numerator and the denominator, respectively.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>avg</span></i><span lang=EN-US> uses the
average value between the numerator and denominator statistical weight value.</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US><img
border=0 width=581 height=199 id="Picture 17"
src="User_Guide_files/image027.png"></span></p>

<p class=MsoCaption><a name="_Ref103114679"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>18</span></b><b><span lang=EN-US>. The RATIOS module flowchart.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The RATIOS module task table displays the
following parameter fields (</span><span lang=EN-US><b>Figure 19</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator column</span></i><span
lang=EN-US> specifies the name of the folder containing the quantitative value
to be used as a numerator for the new log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator column(s)</span></i><span
lang=EN-US> specifies the name of the folder(s) containing the quantitative
values to be used as a denominator for the new log<sub>2</sub>-ratio
calculation. The sample folders have to be separated by comma.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level </span></i><span lang=EN-US>designates
the level (i.e. peptide, protein, gene or category) at which the ratio is to be
calculated.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> indicates the name of the folder where the resulting log<sub>2</sub>-ratio
and statistical weight values will be saved (e.g. KO_vs_WT).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=574 height=65 src="User_Guide_files/image028.png"></span></p>

<p class=MsoCaption><a name="_Ref103114687"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>19</span></b><b><span lang=EN-US>. A sample task-table in the RATIOS
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_SBT"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>This module is based on the Systems Biology
Triangle (SBT) algorithm [2], which performs an integration between the lower
and the higher levels using the variance previously obtained in an integration
between the lower and an intermediate level (</span><span
lang=EN-US><b>Figure 20</b></span><span lang=EN-US>). Usually the SBT module is
applied to carry out the protein &gt; grand mean integration using the variance
associated with the protein &gt; category integration.   </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=436 height=435 id="Picture 131"
src="User_Guide_files/image029.png"></span></p>

<p class=MsoCaption><a name="_Ref103093737"></a><a name="_Ref103093681"><b><span
lang=EN-US>Figure </span></b></a><b><span lang=EN-US>20</span></b><b><span
lang=EN-US>. <a name="_Ref103093692">Schematic representation of the SBT
module.</a></span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters to be entered in
the SBT module task table are (</span><span lang=EN-US><b>Figure 21</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the name of the folder(s) where the lower level data are located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US>
indicates the name of the lower level (e.g. protein).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Intermediate level</span></i><span
lang=EN-US> indicates the name of the intermediate level (e.g. category).</span></p>

<p class=MsoNormal><span lang=EN-US>By default, the higher level is the grand
mean of the lower level elements.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=292 height=66 id="Picture 13"
src="User_Guide_files/image030.png"></span></p>

<p class=MsoCaption><a name="_Ref103094105"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>21</span></b><b><span lang=EN-US>. A sample task table in the SBT
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>This module accepts the following additional
parameters (</span><span lang=EN-US><b>Figure 22</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> indicates an alternative folder to store the resulting log<sub>2</sub>-ratio
and statistical weight values other than “Sample folder(s)”.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower-Higher level</span></i><span
lang=EN-US> and <i>Int(ermediate)-Higher level </i>specify an alternative
higher level other than the grand mean of the lower level elements.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Low(er) &gt; Int(ermediate)</span></i><span
lang=EN-US> Tag and <i>Int(ermediate) &gt; Hig(her)</i> are the text label that
indicates which elements from the lower level are integrated into the intermediate
level, and the intermediate level are integrated into higher level. The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label ”marked” is used as
a Tag, only the lower level elements containing the label marked  in the third
column of the lower_level2intermediate_level and
intermediate_level2higher_level Relation Tables will be integrated. Logical
operators can also be used in the Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Low(er) &gt; Int(ermediate) FDR </span></i><span
lang=EN-US>and<i> Int(ermediate) &gt; Hig(her) FDR</i> determine an FDR
threshold other than the default value (0.01, i.e. 1%) for outlier removal in
the lower level &gt; intermediate level and intermediate level &gt; higher
level integration, respectively. If “0” is specified as the FDR value, then no outliers
will be discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Low(er) &gt; Int(ermediate) Var(x)</span></i><span
lang=EN-US> and <i>Int(ermediate) &gt; Hig(her) Var(x)</i> indicate the
variance to be used in the lower level &gt; intermediate level and intermediate
level &gt; higher level integration, respectively, as an alternative to the
variance calculated in the lower level &gt; intermediate level integration.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=554 height=168 id="Picture 10"
src="User_Guide_files/image031.png"></span></p>

<p class=MsoCaption><a name="_Ref103094086"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>22</span></b><b><span lang=EN-US>. A sample task table with advanced
parameters in the SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Compound_modules"></a><span
lang=EN-US>Compound modules</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The Compound modules perform a sequence of
consecutive integrations based on the WSPP statistical model [3] and the SBT
algorithm [2]. In addition, each module creates the initial level and calibrate
this initial level. The WSPP-SBT and WSPPG-SBT modules create and calibrate the
“scan” level; and the WPP-SBT and WPPG-SBT create and calibrate the “peptide”
level.</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WSPP-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WSPP-SBT module performs the following
integrations: scan &gt; peptide, peptide &gt; protein, protein &gt; category,
peptide &gt; peptideall, protein &gt; proteinall, and category &gt;
categoryall. In addition, the SBT algorithm is used to calculate the variance
associated to the protein &gt; category integration, which is applied to the
protein &gt; proteinall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WSPP-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan &gt; peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=564 height=227
src="User_Guide_files/image032.png"></span></p>

<p class=MsoCaption><a name="_Ref103096252"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>23</span></b><b><span lang=EN-US>. A sample task table in the
WSPP-SBT, WSPPG-SBT, WPP-SBT, and WPPG-SBT modules.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WSPP-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 24</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>p&gt;a
Tag</i>, <i>c&gt;a Tag</i> are the text label that indicates which elements
from the lower level (“p” and “c”) are integrated into the higher level (“q”
and “a”). The tags must be specified in the third column of the corresponding Relation
Table. This allows the user to discard elements for integration without needing
to eliminate them from the Relation Table. Thus, if the label ”marked” is used
as a Tag, only the lower level elements containing the label marked  in the
third column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span><span class=MsoCommentReference><span lang=EN-US
style='font-size:8.0pt;line-height:107%'> </span></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p FDR</span></i><span lang=EN-US>, <i>p&gt;q
FDR</i>, <i>q&gt;c FDR</i> establish an FDR threshold other than 0.01 (1%) for
outlier removal in the integrations scan&gt;peptide, peptide&gt;protein, and
protein&gt;category, respectively. If FDR = 0 is selected, then the outliers
are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p Var(x)</span></i><span lang=EN-US>, <i>p&gt;q
Var(x)</i>, <i>q&gt;c Var(x)</i> set a fixed value for the variance in the
integrations scan&gt;peptide, peptide&gt;protein, and protein&gt;category,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level &gt; higher level
integration (</span><span lang=EN-US><b>Figure 10</b></span><span lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=567 height=99 id="Picture 23"
src="User_Guide_files/image033.png"></span></p>

<p class=MsoCaption><a name="_Ref103096290"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>24</span></b><b><span lang=EN-US>. Task-Table with advanced
parameters in the WSPP-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WSPPG-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WSPPG-SBT module performs the following
integrations: scan &gt; peptide, peptide &gt; protein, protein &gt; gene, gene
&gt; category, peptide &gt; gene, peptide &gt; peptideall, protein &gt;
proteinall, gene &gt; geneall, and category &gt; categoryall. In addition, the
SBT algorithm is used to calculate the variance associated to the gen &gt;
category integration, which is applied to the gen &gt; geneall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WSPPG-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan &gt; peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WSPPG-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 25</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>q&gt;g
Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text label that indicates
which elements from the lower level (“p”, “q” and “c”) are integrated into the higher
level (“q”, “g” and “a”). The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label ”marked” is used as a Tag, only the lower level elements containing
the label marked  in the third column of the lower_level2higher_level Relation table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p FDR</span></i><span lang=EN-US>, <i>p&gt;q
FDR</i>, <i>q&gt;g FDR</i>, <i>g&gt;c FDR</i> establish an FDR threshold other
than 0.01 (1%) for outlier removal in the following integrations:
scan&gt;2peptide, peptide&gt;2protein, protein&gt;gene, and gene&gt;category,
respectively. If FDR = 0 is selected, then the outliers are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p Var(x)</span></i><span lang=EN-US>, <i>p&gt;q
Var(x)</i>, <i>q&gt;g Var(x)</i>, <i>g&gt;c Var(x)</i> set a fixed value for
the variance in the integrations: scan&gt;peptide, peptide&gt;protein,
protein&gt;gene, and gene&gt;category, respectively. By default, the variance
will be iteratively calculated based on the Levenberg-Marquardt algorithm in
the first lower level &gt; higher level integration (</span><span
lang=EN-US><b>Figure 10</b></span><span lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=567 height=54 id="Picture 24"
src="User_Guide_files/image034.png"></span></p>

<p class=MsoCaption><a name="_Ref103096474"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>25</span></b><b><span lang=EN-US>. Task-table with advanced
parameters in the WSPPG-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WPP-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WPP-SBT module performs the
integrations peptide&gt;protein, peptide&gt;peptideall, protein&gt;category,
and category&gt;categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the protein &gt; category integration,
which is applied to the protein &gt; proteinall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WPP-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan &gt; peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WPP-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 26</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>p&gt;a
Tag</i>, <i>c&gt;a Tag</i> are the text label that indicates which elements
from the lower level (“p” and “c”) are integrated into the higher level (“q”
and “a”). The tags must be specified in the third column of the corresponding Relation
Table. This allows the user to discard elements for integration without needing
to eliminate them from the Relation Table. Thus, if the label ”marked” is used
as a Tag, only the lower level elements containing the label marked  in the
third column of the lower_level2higher_level Relation Table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q FDR</span></i><span lang=EN-US>, <i>q&gt;c
FDR</i> establish an FDR threshold other than 0.01 (1%) for outlier removal in
the peptide&gt;protein and protein&gt;category integrations, respectively. If
FDR = 0 is selected, then the outliers are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Var(x)</span></i><span lang=EN-US>, <i>q&gt;c
Var(x)</i> set a fixed value for the variance in the peptide&gt;protein and
protein&gt;category integrations, respectively. By default, the variance will
be iteratively calculated based on the Levenberg-Marquardt algorithm in the
first lower level &gt; higher level integration (</span><span
lang=EN-US><b>Figure 10</b></span><span lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=482 height=78 id="Picture 29"
src="User_Guide_files/image035.png"></span></p>

<p class=MsoCaption><a name="_Ref103096698"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>26</span></b><b><span lang=EN-US>. Task-table with advanced
parameters in the WPP-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WPPG-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WPPG-SBT module performs the
integrations peptide&gt;protein, protein&gt;gene, gene&gt;category,
peptide&gt;gene, peptide&gt;peptideall, protein&gt;proteinall, gene&gt;geneall,
and category&gt;categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the gene &gt; category integration, which
is applied to the gene &gt; geneall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WPPG-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan &gt; peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WPPG-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 27</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>q&gt;g
Tag, p&gt;a Tag, q&gt;a Tag,</i> <i>c&gt;a Tag</i> are the text label that
indicates which elements from the lower level (“p”, “q” and “c”) are integrated
into the higher level (“q”, “g” and “a”). The tags must be specified in the
third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label ”marked” is used as a Tag, only the lower level
elements containing the label marked  in the third column of the lower_level2higher_level
Relation table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q FDR, q&gt;g FDR</span></i><span
lang=EN-US>, <i>g&gt;c FDR</i> establish an FDR threshold other than 0.01 (1%)
for outlier removal in the integrations peptide&gt;protein, protein&gt;gene,
and gene&gt;category, respectively. If FDR = 0 is selected, then the outliers
are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Var(x)</span></i><span lang=EN-US>, <i>q&gt;g
Var(x), g&gt;c Var(x)</i> set a fixed value for the variance in the
integrations peptide&gt;protein, protein&gt;gene, and gene&gt;category,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level &gt; higher level
integration (</span><span lang=EN-US><b>Figure 10</b></span><span lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=567 height=63 id="Picture 47"
src="User_Guide_files/image036.png"></span></p>

<p class=MsoCaption><a name="_Ref103097509"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>27</span></b><b><span lang=EN-US>. Task-table with advanced
parameters in the WPPG-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Report_modules"></a><a
name="_Reports_modules"></a><span lang=EN-US>Reports modules</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_REPORT"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>REPORT</span></h3>

<p class=MsoNormal><span lang=EN-US>The REPORT module allows the collection of the
statistical variables (n, tags, Xinf, Vinf, Xsup, Vsup, Z, and FDR) from the
different integrations performed into result tables. </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters showing in the
REPORT task table are (</span><span lang=EN-US><b>Figure 28</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US> indicates
the name(s) of the folder(s) where the values of the statistical variables to
be retrieved are located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US> indicates
the starting level (i.e. peptide, protein, or category) for the integration
whose statistical variables are to be reported.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level</span></i><span lang=EN-US> indicates
the ending level for the integration whose statistical variables are to be
reported.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Reported vars</span></i><span lang=EN-US>
specifies which statistical variables will be reported. The available variables
are n, tags, Xinf, Vinf, Xsup, Vsup, Z, FDR, X’inf and Winf.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output report</span></i><span lang=EN-US> is
the report filename (without extension).</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=535 height=318 id="Picture 3"
src="User_Guide_files/image037.png"></span></p>

<p class=MsoCaption><a name="_Ref103115700"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>28</span></b><b><span lang=EN-US>. A sample task table in the REPORT
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>For instance, the first row of the task table
shown in </span><span lang=EN-US><b>Figure 28</b></span><span lang=EN-US>
prompts the REPORT module to read the variable “n” from the <i>scan2peptide_outStats.tsv</i>
file that contains the statistical outcome from the scan &gt; peptide integration
  (the asterisk wildcard character in <i>Sample folder(s)</i> causes REPORT to
retrieve the “n” variable from every sample). These “n” values are written to a
report file named “Nscan_pep” that is stored in the project “reports” folder.</span></p>

<p class=MsoNormal><span lang=EN-US>The second row instructs the module to read
the variables “Xinf”, “Z” and “FDR” from the statistical outcome of the peptide
&gt; protein integration (once again for every sample). These values are
written to a report file named “Nscan_Normpep_prot_XZ”.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The REPORT module accepts the following advanced
parameters (</span><span lang=EN-US><b>Figure 29</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level names to show</span></i><span
lang=EN-US> allows the user to restrict the elements to be written to the <i>Output
report</i> to those from the <i>Lower level</i> or the <i>Higher level</i>. Both
levels are used by default.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Merge with report</span></i><span lang=EN-US>
designates the file whose <i>Reported vars</i> will be incorporated into the <i>Output
report</i> after intersection with the latter file.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Add columns from relation table</span></i><span
lang=EN-US> appends <i>Lower level</i> elements, extracted from the relation
table designated, to the <i>Output report</i>. It is possible to indicate multiple
relation tables separated by a comma.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Filter</span></i><span lang=EN-US> allows to filter
the data to be transferred to the <i>Output report</i> based on the <i>Reported
vars</i> (n, Z, FDR, etc.). For more details, see <a href="#_Filter_(for_REPORT">Filter</a>
in the “Special Parameters” Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=261 id="Picture 4"
src="User_Guide_files/image038.png"></span></p>

<p class=MsoCaption><a name="_Ref103116151"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>29</span></b><b><span lang=EN-US>. A sample task-table with advanced
parameters in the REPORT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The reports indicated under <i>Output
report</i> and <i>Merge with report</i> are merged according to the column
header that they share. Thus, the REPORT task table shown in </span><span lang=EN-US><b>Figure 30</b></span><span lang=EN-US> will cause the module to incorporate
the number of scans per peptide, displayed in the report “Nscan_pep”, to the
report “Nscan_Normpep_prot_XZ”, as these two reports share the lower level
elements showing under the “peptide” header.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=588 height=144 id="Picture 26"
src="User_Guide_files/image039.png"></span></p>

<p class=MsoCaption><a name="_Ref103116369"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>30</span></b><b><span lang=EN-US>. Report merging in the REPORT
module.</span></b><span lang=EN-US> The first task table row creates a report file
(“Nscan_pep”) with the (n)umber of scans per peptide. The second row creates a
report file called “Nscan_Normpep_prot_XZ” that contains, apart from the
variables “Xinf”, “Z”, and “FDR” coming from the peptide &gt; protein
integration, the (n)umber of scans per peptide previously stored in the “Nscan_pep”
report, as these two reports share the lower level elements showing under the
“peptide” header.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>In addition, it is possible to incorporate
additional data from one or more relation tables into the reports  (</span><span lang=EN-US><b>Figure 31</b></span><span lang=EN-US>). When a given relation
table is indicated under <i>Add columns from relation table</i>, the REPORT
module will first attempt to incorporate to the <i>Output report</i>
(“Npep_Quanprot_cat” in the example) the elements related to the <i>Lower level</i>
(“protein” in this case) elements in the relation table. If the <i>Lower level</i>
elements are missing in the relation table, then REPORT will try to incorporate
the elements related to the Higher level (“category” in this case) instead. If
neither the <i>Lower level</i> nor the <i>Higher level</i> can be found in the
relation table, then no action is performed. </span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=597 height=94 id="Picture 27"
src="User_Guide_files/image040.png"></span></p>

<p class=MsoCaption><a name="_Ref103116417"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>31</span></b><b><span lang=EN-US>. Adding data from relation tables
into the report files. </span></b><span lang=EN-US>The relation table “protein2gene”
contains a column with protein identifiers under the “protein” header and
another column with the corresponding gene name under the “gene” header,
whereas the relation table “protein2description” contains, apart from the
“protein” elements, a column with the corresponding protein description. The
first task table row will prompt REPORT to incorporate the gene names and
protein descriptions contained in the  relation tables to the report file “Npep_Quanprot_cat”
report file, as “protein” is the </span><span lang=EN-US style='font-style:
normal'>Lower level</span><span lang=EN-US> they all three share.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>Finally, the report data can be filtered performing
logical operations with the <i>Reported vars</i> in the <i>Filter</i> field. For
instance, in the report task table displayed in </span><span
lang=EN-US><b>Figure 32</b></span><span lang=EN-US>:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>n_protein2category &lt;= 100, filters out from
the report the variables Z and FDR for the category &gt; categoryall
integration when the (n)umber of proteins per category is greater than 100.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>n_protein2category &gt;= 5 &amp;
n_protein2category &lt;= 100 retrieves the variables Z and FDR for the category
&gt; categoryall integration when the (n)umber of proteins per category is  in
the [5, 100] range.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>KO_vs_WT@FDR_category2categoryall &lt; 0.05,
retrieves the variables Z and FDR for the category &gt; categoryall integration
provided that the FDR corresponding to the “KO_vs_WT” samples is less than 0.05.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=590 height=89 id="Picture 32"
src="User_Guide_files/image041.png"></span></p>

<p class=MsoCaption><a name="_Ref103116468"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>32</span></b><b><span lang=EN-US>. Filtering the report data.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The compound variables shown in the <i>Filter</i>
field on </span><span lang=EN-US><b>Figure 32</b></span><span lang=EN-US> follow
the structure <i>Reported var</i>_<i>integration</i>, like “n_protein2category”.
Such filter applies to all samples. </span></p>

<p class=MsoNormal><span lang=EN-US>However, the filter “KO_vs_WT@FDR_category2categoryall”
is applied based on the variable “FDR” from the category &gt; categoryall
integration, but only to the “KO_vs_WT” sample. Moreover, the filter “WT1,WT2@FDR_category2categoryall”
is applied to the “WT1” and “WT2” samples.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_SANSON"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>SANSON</span></h3>

<p class=MsoNormal><span lang=EN-US>The SANSON module generates a similarity
graph showing the relationship between functional categories based on their
protein components. </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters to be provided for this
module are (</span><span lang=EN-US><b>Figure 33</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the names of the folder(s) where the lower level data file is
located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US>
indicates the name of the lower level elements (“protein” in this case) to be
used.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level</span></i><span lang=EN-US>
indicates the name of the higher level (“category” in this case) to which the
lower level elements will be integrated.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder </span></i><span
lang=EN-US>designates a folder other than <i>Sample folder(s)</i> where the
results will be saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=383 height=65 src="User_Guide_files/image042.png"></span></p>

<p class=MsoCaption><a name="_Ref103116548"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>33</span></b><b><span lang=EN-US>. A sample task table in the SANSON
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The SANSON module accepts the following additional
parameters (</span><span lang=EN-US><b>Figure 34</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower norm</span></i><span lang=EN-US>
specifies the normalization scheme to be used with the lower level elements.
The default value is included the normalization of lower level to all.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher norm</span></i><span lang=EN-US>
specifies the normalization scheme to be used with the higher level elements.
The default value is included the normalization of higher level to all.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Tag</span></i><span lang=EN-US> is a text
label that indicates which elements from the lower level are integrated into
the higher level. The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label ”marked” is used as a Tag, only the lower level elements containing
the label marked  in the third column of the lower_level2higher_level Relation table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.<i> </i></span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Filter</span></i><span lang=EN-US> allows to
filter the data based on the FDR and number of proteins. For more details, see <a
href="#_Filter_param_(in">Filter</a> in the “Special Parameters” Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=57 id="Picture 2"
src="User_Guide_files/image043.png"></span></p>

<p class=MsoCaption><a name="_Ref103116564"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>34</span></b><b><span lang=EN-US>. A sample task-table with advanced
parameters in the SANSON module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><a name="_Complete_Modules"></a><a name="_WSPP-SBT"></a><a
name="_WSPPG-SBT"></a><a name="_WPP-SBT"></a><a name="_WPPG-SBT"></a><span
lang=EN-US>&nbsp;</span></p>

<h2><a name="_Special_parameters"></a><span lang=EN-US>Special parameters</span></h2>

<p class=MsoNormal><a name="_Keywords_for_the"></a><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Multiple_samples"></a><span lang=EN-US>Multiple samples</span></h3>

<p class=MsoNormal><span lang=EN-US>The “Sample folder(s)” field of the
different module task tables admit multiple samples. For instance, let's consider
the samples created with the following LEVEL CREATOR task table:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=567 height=162 id="Picture 64"
src="User_Guide_files/image044.png"></span></p>

<p class=MsoNormal><span lang=EN-US>We can include multiple samples separated
by a comma, for example, in the INTEGRATE module task table: </span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=567 height=215 id="Picture 7" src="User_Guide_files/image045.png"></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Asterisk_is_our"></a><span lang=EN-US>Asterisk is our jack of all
trades</span></h3>

<p class=MsoNormal><span lang=EN-US>The module task tables admit the usage of
the asterisk symbol as a wildcard character. Let’s once more consider the samples
created with the following LEVEL CREATOR task table:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=567 height=162 id="Picture 59"
src="User_Guide_files/image046.png"></span></p>

<p class=MsoNormal><span lang=EN-US>Each row calculates a ratio that is saved to
the corresponding <i>Output Sample folder</i>. Thus, the ratio of 113 to the
mean of 113, 114, 115, and 116 is saved to the “Jurkat_WT/WT_1” folder; the 114
to the mean of 113, 114, 115 ratio is saved to “Jurkat_WT/WT_2”, and so on. One
way to create the task table of the INTEGRATE module could be the following,
where each row represents an integration for a given sample:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=213 height=296 id="Picture 62"
src="User_Guide_files/image047.png"></span></p>

<p class=MsoNormal><span lang=EN-US>However, this task table can be simplified applying
the asterisk wildcard. For instance, the task-table below allows to indicate multiple
sample folders, namely every folder starting with “Jurkat_WT/” or “Jurkat_KO/”.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=263 height=262 id="Picture 60"
src="User_Guide_files/image048.png"></span></p>

<p class=MsoNormal><span lang=EN-US>We can reduce this expression even more
using just an asterisk: the first row of the following task table performs the
integrations peptide &gt; protein, protein &gt; category, peptide &gt;
peptideall, protein &gt; proteinall, and category &gt; categoryall in every
sample folder defined with LEVEL CREATOR.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=272 height=147 id="Picture 63"
src="User_Guide_files/image049.png"></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_More_params"></a><span lang=EN-US>More params</span></h3>

<p class=MsoNormal><span lang=EN-US>Some modules accept a column parameter in
the Task-Table called “More params”. This column allows you to provide advanced
parameters for the SanXoT programs [1]. The program descriptions are in the
following wiki link:</span></p>

<p class=MsoNormal><span lang=EN-US><a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The iSanXoT module are composed by several
programs of SanXoT. For this reason, the “More params” of a module accepts the
advanced parameters of composed programs indicated by a name.</span></p>

<p class=MsoNormal><span lang=EN-US>For example, </span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-US>INTEGRATE:</span></p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'><span
lang=EN-US>&quot;sanxot1&quot;: &quot; -m 300 -g &quot;, &quot;sanxot2&quot;:
&quot;-s --sweepdecimals=2.5&quot;</span></p>

<p class=MsoNormal><span lang=EN-US>In above example the first “sanxot” program
that compose the INTEGRATE module, receives the “-m 300 -g” as parameter, and
the second “sanxot” receives also the parameter &quot;-s
--sweepdecimals=2.5&quot;.</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-US>WSPP-SBT:</span></p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'><span
lang=EN-US>&quot;p2q_sanxot2&quot;: &quot; -m 100 -s &quot;,
&quot;q2a_sanxot1&quot;: &quot;-m 100&quot;</span></p>

<p class=MsoNormal><span lang=EN-US>In this example, the WSPP-SBT module has
multiple integrations: scan &gt; peptide, peptide &gt; protein, etc. For more
information, see the WSPP-SBT section. Thus, the second “sanxot” program of
peptide &gt; protein (q) integration will receive the parameter “-m 100 -s”, and
then, the first “sanxot” of protein (q) &gt; proteinall (a) integration will
receive the “-m 100”.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>The program names for each Module </span></h4>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>INTEGRATE: sanxot1, sanxotsieve, sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>NORCOMBINE: create_exp_tags, cardenio,
sanxot1, sanxotsieve, sanxot2 </span></p>

<p class=MsoNormal><span lang=EN-US>SBT: l2i_sanxot1, l2i_sanxotsieve,
l2i_sanxot2, i2h_sanxot1, i2h_sanxotsieve, i2h_sanxot2, l2h_sanxot1,
l2h_sanxotsieve, l2h_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WSPP_SBT: level_creator, klibrate,
s2p_sanxot1, s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2,
c2a_sanxot1, c2a_sanxotsieve, c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WSPPG_SBT: level_creator, klibrate,
s2p_sanxot1, s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1,
q2g_sanxotsieve, q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2,
p2a_sanxot1, p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve,
q2a_sanxot2, g2a_sanxot1, g2a_sanxotsieve, g2a_sanxot2, c2a_sanxot1,
c2a_sanxotsieve, c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WPP_SBT: level_creator, klibrate, p2q_sanxot1,
p2q_sanxotsieve, p2q_sanxot2, q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1,
c2a_sanxotsieve, c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WPPG_SBT: level_creator, klibrate, p2q_sanxot1,
p2q_sanxotsieve, p2q_sanxot2, p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1,
q2g_sanxotsieve, q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, g2a_sanxot1,
g2a_sanxotsieve, g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve, c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Filter_param_(in"></a><a name="_Filter_(for_REPORT"></a><span
lang=EN-US>Filter (for REPORT and SANSON)</span></h3>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The REPORT module and SANSON module accept
a Filter parameter. This parameter filters the data based on some variables
depending on the module.</span></p>

<p class=MsoNormal><span lang=EN-US>In the case of REPORT module, the filtered
variables are the Reported vars: n, Z, FDR, etc. For instance:</span></p>

<p class=MsoNormal><span lang=EN-US>FDR_category2category &lt; 0.05) &amp;
(n_protein2category &gt;=5) &amp; (n_protein2category &lt;= 100)</span></p>

<p class=MsoNormal><span lang=EN-US>For the SANSON module, the filtered
variables are FDR and the related number (n_rel). For example:</span></p>

<p class=MsoNormal style='text-indent:35.4pt'><span lang=EN-US>([FDR] &lt;
0.05) &amp; ([n_rel] &gt;= 10) &amp; ([n_rel] &lt;= 100)</span></p>

<p class=MsoNormal><span lang=EN-US>Different variables can be combined using
the comparisons: &gt;=, &lt;=, !=, &lt;&gt;, ==, &gt;, &lt;; and using logical
operators “and” (&amp;), “or” (||), and “not” (!). </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span lang=EN-US
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Adaptors"></a><a name="_Input_Adaptor"></a><span lang=EN-US>Input
Adaptor</span></h1>

</div>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The <i>iSanXoT</i> <i>Input Adaptor</i>
menu offers users the possibility to either provide their own <i>Identification/Quantification
file</i>, which contains the identification and quantification data, or have the
<i>Input Adaptor</i> prepare this file from the results obtained using any of
the mainstream proteomics pipelines. Please indicate in the <i>Project folder</i>
field the path to the folder where <i>iSanXoT</i> output files will be stored,
then choose one of two options (</span><span lang=EN-US><b>Figure 35</b></span><span
lang=EN-US>):   </span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=485 height=418 id="Group 9"
src="User_Guide_files/image050.png"></span></p>

<p class=MsoCaption><a name="_Ref103468260"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>35</span></b><b><span lang=EN-US>. Main view of the </span></b><b><span
lang=EN-US style='font-style:normal'>Input Adaptor</span><span lang=EN-US> menu.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Main_Inputs"></a><a name="_Main_Input"></a><span lang=EN-US
style='font-family:"Calibri",sans-serif'>User-provided
Identification/Quantification file </span></h3>

<p class=MsoNormal><span lang=EN-US>Choose this option if you already have an
Identification/Quantification file that meets the following conditions (</span><span lang=EN-US><b>Figure 36</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The contents are alphanumeric tab-separated
items arranged in rows and columns.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>An “Experiment” column is included providing
items with an experiment name.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>There are at least two columns displaying numeric
intensity values to be integrated.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Every data row is unambiguously identified by
means an alphanumeric identifier.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=549 height=90 id="Picture 74"
src="User_Guide_files/image051.png"></span></p>

<p class=MsoCaption><a name="_Ref103468273"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>36</span></b><b><span lang=EN-US>. The </span></b><b><span
lang=EN-US style='font-style:normal'>User-provided
Identification/Quantification file</span><span lang=EN-US> option.</span></b></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>Prepare Identification/Quantification file from proteomics
pipelines </span></h3>

<p class=MsoNormal><span lang=EN-US>This option prepares the Identification/Quantification
file required by <i>iSanXoT</i> based on the output from any proteomics
pipeline (so far it has been tested with <i>Trans-Proteomic Pipeline</i>, <i>FragPipe</i>,
<i>MaxQuant</i>, and <i>Proteome Discoverer</i>) provided that quantitative
values (e.g. intensities) are given for the items to be integrated.</span></p>

<p class=MsoNormal><span lang=EN-US>First, click the <i>Choose folder + Add
annots</i> button to indicate the folder that contains the results (</span><span lang=EN-US><b>Figure 37</b></span><span lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=581 height=175 id="Picture 1"
src="User_Guide_files/image052.png"></span></p>

<p class=MsoCaption><a name="_Ref103468288"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>37</span></b><b><span lang=EN-US>. Preparing the
Identification/Quantification file from proteomics pipelines.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>Then a three-panel window will pop-up (</span><span lang=EN-US><b>Figure 38</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The top panel lists the files included in the
folder indicated by the user. Here you can select one or more result files to
be considered by the <i>Input Adaptor</i> for preparing the  Identification/Quantification
file. Please bear in mind that the result files selected must have the same
column headers.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The middle panel is used to set the distribution
of data items across experiments according to result filenames.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The bottom panel allows to create identifiers by
concatenating result file headers. It is composed of two interfaces:</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US>The headers found on the result files are listed
on the left. Header names will be added to the interface on the right as they
are selected.</span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US>The interface on the right displays the header
names selected to generate the identifier as well as an identifier name that
must be provided by the user.</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-US>Please note that
the alphanumeric text that unambiguously identifies the items to be integrated
is the only identifier that must be necessarily included in the Identification/Quantification
file.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=523 id="Picture 11"
src="User_Guide_files/image053.png"></span></p>

<p class=MsoCaption><a name="_Ref103468299"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>38</span></b><b><span lang=EN-US>. Adapting proteomics pipeline
results.</span></b><span lang=EN-US> The top panel shows output files from </span><span
lang=EN-US style='font-style:normal'>FragPipe</span><span lang=EN-US>. On the
middle panel, the </span><span lang=EN-US style='font-style:normal'>psm.tsv</span><span
lang=EN-US> file containing identification/quantification data for one
experiment (</span><span lang=EN-US style='font-style:normal'>Jurkat</span><span
lang=EN-US>) has been selected (quantitative data were provided by the </span><span
lang=EN-US style='font-style:normal'>TMT-Integrator</span><span lang=EN-US>
module included in </span><span lang=EN-US style='font-style:normal'>FragPipe</span><span
lang=EN-US>). The bottom panel is used to create identifiers by concatenating
result file headers. In this case, unambiguous scan identification is achieved
by concatenating </span><span lang=EN-US style='font-style:normal'>Spectrum
File</span><span lang=EN-US> and </span><span lang=EN-US style='font-style:
normal'>Charge</span><span lang=EN-US> items, whereas the </span><span
lang=EN-US style='font-style:normal'>Identification/Quantification</span><span
lang=EN-US> file will incorporate an additional </span><span lang=EN-US
style='font-style:normal'>PepID</span><span lang=EN-US> column displaying the
data items included in the </span><span lang=EN-US style='font-style:normal'>Modified
Peptide</span><span lang=EN-US> column from the </span><span lang=EN-US
style='font-style:normal'>psm.tsv</span><span lang=EN-US> file.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=left style='text-align:left'><a name="_Close_Search_–"></a><a
name="_Close_Search:_Inputs"></a><a name="_Inputs_from_PD"></a><a
name="_Close_Search_–_1"></a><a name="_Inputs_from_MSFragger"></a><a
name="_Close_Search_–_2"></a><a name="_Close_Search_–_3"></a><a
name="_Inputs_from_MaxQuant"></a><a name="_Open_Search_–_1"></a><a
name="_Workflows"></a><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><a name="_References"></a><b><span
lang=EN-US style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><span lang=ES>References</span></h1>

</div>

<p class=MsoNormal><span lang=ES>[1] Trevisan-Herraz M. et al. </span><span
lang=EN-US>(2019) SanXoT: a modular and versatile package for the quantitative
analysis of high-throughput proteomics experiments. Bioinformatics. 35,
1594-96.</span></p>

<p class=MsoNormal><span lang=EN-US>[2] Garcia-Marques, F., et al., A Novel
Systems-Biology Algorithm for the Analysis of Coordinated Protein Responses
Using Quantitative Proteomics. Mol Cell Proteomics, 2016. 15(5): p. 1740-60.</span></p>

<p class=MsoNormal><span lang=EN-US>[3] Navarro, P., et al., General
statistical framework for quantitative proteomics by stable isotope labeling. J
Proteome Res, 2014. 13(3): p. 1234-47.</span></p>

<p class=MsoNormal><span lang=EN-US>[4] Kong, A. T., Leprevost, F. V.,
Avtonomov, D. M., Mellacheruvu, D., &amp; Nesvizhskii, A. I. (2017). MSFragger:
ultrafast and comprehensive peptide identification in mass spectrometry–based
proteomics. Nature Methods, 14(5), 513-520.</span></p>

<p class=MsoNormal><span lang=EN-US>[5] Eng, J.K., et al., A Deeper Look into
Comet - Implementation and Features. J Am Soc Mass Spectrom. 2015 Jun 27. doi:
10.1007/s13361-015-1179-x</span></p>

<p class=MsoNormal><span lang=EN-US>[6] SanXoT wiki: <a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

</div>

</body>

</html>
</template>