<template class="task-template" id="help_basic">
    <h2><a class="anchor" href="#help_basic"></a>Basic Workflow</h2>
    <p><i>Basic Workflow</i> is a general-purpose approach to the statistical analysis of multiplexed, high-throughput proteomics experiments.
    This workflow includes validation of peptide-spectrum matches (PSMs) through pRatio, calculation of isotope channel ratios and statistical assessment of alterations to peptide, 
    protein and functional category abundance based on the weighted spectrum, peptide and protein (WSPP) model and the systems biology triangle (SBT) approach.</p>
    
    <p><strong>Getting started with the Basic Workflow</strong> In the Basic Workflow Task tab, select the species corresponding to your experiment.
    The input directory must contain the PSMs.txt files (where the PSM identity and quantitation can be found) to be considered. To avoid messing up with files, 
    it is recommended that you select an iSanXoT output directory other than the input directory. In the protein database field select The FASTA protein database file that you used for protein identification, 
    which can be one of the databases included in iSanXoT or any other FASTA protein database provided by the user.
    Likewise, in the protein-category relation file field you will have to indicate iSanXoT whether you want to use one of the protein-category relation files 
    (the files that relate proteins to their functional category annotations) included or any other protein-category relation file.</p>
    
    <p><strong>PSM validation by FDR</strong> A number of parameters must be provided in the Basic Workflow FDR tab for PSM validation based on FDR. First, 
    for PSM discrimination based on delta mass values, a delta mass threshold (in ppm) must be specified.
    It is also necessary to select the number of delta mass areas in which the above delta mass criterion must be applied:
    1 area, when the delta mass threshold applies to the precursor mass alone;
    3 areas, when ±1 Da around the precursor ion is also included;
    or 5 areas, when both ±1 Da and ±2 Da are also checked for delta mass threshold compliance. In addition, the FDR cut-off value for PSM validation must be stated.
    In the raw or corrected XCorr usage field the user must specify if the FDR calculation should rely on raw XCorr values or charge- and mass-corrected XCorr (cXCorr) values.
    Finally, the user has to provide the label with which decoy protein sequences are labelled in the corresponding protein database.</p>
    
    <p><strong>Preparing the Task Table The Task Table</strong> indicates iSanXoT which statistical comparisons are to be carried out based on scan to peptide (s>p),
    peptide to protein (p>q) and protein to category (q>c) integrations. When several multiplexed experiments need to be considered, these can be differentiated using an experiment label on the first column.
    Then a name must be given to the corresponding statistical comparison on the second column that will be used as a name for the folder where the statistics output files will be stored.
    The Ratio numerator and Ratio denominator columns define the reporter ion channel used as a numerator and denominator, respectively, for the calculation of log2 ratio across the validated PSMs.
    This log2 ratios, together with their corresponding statistical weight, are used for the kick-off scan to peptide (s>p) integration.
    Finally, the next columns are used to specify outlier FDR cut-off and variance values for the three integrations. In the case of variance (last three columns in the Task Table), 
    entering “false” will prompt iSanXoT to calculate variance in the corresponding integration instead of using a fixed value.</p>
    
</template>
  