<template class="task-template" id="help_wf-basic">

<h2><a class="anchor" href="#help_wf-basic"></a>Basic Workflow</h2>
<p><i>Basic Workflow</i> is a general-purpose approach to the statistical analysis of multiplexed, high-throughput proteomics experiments.
This workflow includes validation of peptide-spectrum matches (PSMs) through pRatio, calculation of isotope channel ratios and statistical assessment of alterations to peptide, 
protein and functional category abundance based on the weighted spectrum, peptide and protein (WSPP) model and the systems biology triangle (SBT) approach.</p>

<p><strong>Getting started with the Basic Workflow</strong> In the Basic Workflow Task tab, select the species corresponding to your experiment.
The input directory must contain the PSMs.txt files (where the PSM identity and quantitation can be found) to be considered. To avoid messing up with files, 
it is recommended that you select an iSanXoT output directory other than the input directory. In the protein database field select The FASTA protein database file that you used for protein identification, 
which can be one of the databases included in iSanXoT or any other FASTA protein database provided by the user.
Likewise, in the protein-category relation file field you will have to indicate iSanXoT whether you want to use one of the protein-category relation files 
(the files that relate proteins to their functional category annotations) included or any other protein-category relation file.</p>

<p><strong>PSM validation by FDR</strong> A number of parameters must be provided in the Basic Workflow FDR tab for PSM validation based on FDR. First, 
for PSM discrimination based on delta mass values, a delta mass threshold (in ppm) must be specified.
It is also necessary to select the number of delta mass areas in which the above delta mass criterion must be applied:
1 area, when the delta mass threshold applies to the precursor mass alone;
3 areas, when ±1 Da around the precursor ion is also included;
or 5 areas, when both ±1 Da and ±2 Da are also checked for delta mass threshold compliance. In addition, the FDR cut-off value for PSM validation must be stated.
In the raw or corrected XCorr usage field the user must specify if the FDR calculation should rely on raw XCorr values or charge- and mass-corrected XCorr (cXCorr) values.
Finally, the user has to provide the label with which decoy protein sequences are labelled in the corresponding protein database.</p>

<p><strong>Preparing the Task Table The Task Table</strong> indicates iSanXoT which statistical comparisons are to be carried out based on scan to peptide (s>p),
peptide to protein (p>q) and protein to category (q>c) integrations. When several multiplexed experiments need to be considered, these can be differentiated using an experiment label on the first column.
Then a name must be given to the corresponding statistical comparison on the second column that will be used as a name for the folder where the statistics output files will be stored.
The Ratio numerator and Ratio denominator columns define the reporter ion channel used as a numerator and denominator, respectively, for the calculation of log2 ratio across the validated PSMs.
This log2 ratios, together with their corresponding statistical weight, are used for the kick-off scan to peptide (s>p) integration.
Finally, the next columns are used to specify outlier FDR cut-off and variance values for the three integrations. In the case of variance (last three columns in the Task Table), 
entering “false” will prompt iSanXoT to calculate variance in the corresponding integration instead of using a fixed value.</p>

<h4>Aim</h4>
<p><i>A good example is the best sermon</i>, claimed American polymath Benjamin Franklin. This worked-out example is intended as a step-by-step guide to assessing differential protein abundance from scratch using iSanXoT. For that, raw data from a high-throughput, quantitative proteomics experiment based on liquid chromatography coupled to tandem mass spectrometry (LC-MS/MS) analysis will be used.
</p>

<h4>Project description and starting files</h4>
<figure class="figure rounded float-right">
    <img class="figure-img" src="assets/images/help_basic_table1.png" />
    <figcaption class="figure-caption text-center">Table 1. Isotopic sample labeling.</figcaption>
</figure>
<p>
We shall be working with a simple model consisting of four wild-type (WT) and four knock-out (KO) Jurkat cell preparations which we would like to compare for differential protein abundance. The protein extracts obtained from these eight samples were subjected to tryptic protein digestion, after which the resulting peptides were isotopically labelled using the iTRAQ Reagent 8 plex kit<sup>1</sup> depicted in Table 1. Then the labeled peptide samples were pooled and partly separated into five fractions using high-pH reversed-phase chromatography<sup>2</sup>. LC-MS/MS analysis of both the unfractionated material and the five peptide fractions resulted in the six raw LC-MS/MS files that can be found in <i>…\WOE\Raw_files</i>. These raw files were used as inputs to Protein Discoverer 2.1<sup>3</sup> to generate, among others, the six PSM.txt files stored in <i>…\WOE\PSM.txt_files</i> upon database searching against the FASTA human protein database that can be found in <i>…\WOE\Database_and_category_files</i>. Please note that this protein database contains both true protein sequences and the corresponding inverted (decoy) sequences to enable false discovery rate (FDR) estimation for peptide identification.</p>

<p>
    A close inspection of the PSM.txt files reveals that these plain text files hold 57 columns displaying information for every peptide-spectrum match (PSM) obtained upon database searching; however, only a reduced subset of these data will be accessed by iSanXoT, as shown in Table 2.
</p>
<figure class="text-center">
    <img class="figure-img rounded" src="assets/images/help_basic_table2.png" />
    <figcaption class="figure-caption text-center">Table 2. Data accessed by iSanXoT in the PSMs.txt files.</figcaption>
</figure>
    


<h4>Getting started: the Inputs menu</h4>
<p>
    Once the program has been successfully installed, executing the isanxot.bat script will bring you to iSanXoT’s main page, where a number of predefined workflows are displayed. The link WSPP-SBT sample with iSanXoT databases showing below the short description of the Basic Workflow will take you to the Basic Workflow Inputs window (Fig. 1), where the following information is provided:
</p>
<ul>
    <li>Input folder specifies the location of the files containing PSM identification and quantification data (the PSMs.txt files in this example);</li>
    <li>Output folder describes the path to the folder where iSanXoT output files will be stored. Selection of an output folder other than the input folder is strongly recommended;</li>
    <li>Select input files indicates which of the PSM.txt files stored in the input folder must be considered by iSanXoT (all six files in this case). PSMs.txt file names are listed under Infile, while their experiment allocation is indicated in Experiment. All six PSMs.txt files originate from the same experiment (termed “TMT”) in our example.</li>
</ul>

<h4>Validating peptide identification</h4>
<p>
    The next step is to validate the collection of PSMs that we have used as inputs in the previous Section. For that, iSanXoT relies on the probability ratio (pRatio) method<sup>6</sup>, 
    <figure class="text-center">
        <img class="figure-img rounded" src="assets/images/help_basic_figure1.png" />
        <figcaption class="figure-caption text-center">Figure 1. The Basic Workflow Inputs window after loading the sample data.</figcaption>
    </figure>
which calculates the probability of random peptide matching and the corresponding FDR for peptide identification. The FDR menu displays the following fields:
</p>
<ul>
    <li>Experiment, the aforementioned experiment allocation;</li>
    <li>Threshold (ppm) is the postscoring mass filtering cutoff to be applied after using wide mass windows in the database search<sup>7</sup>, as was the case with these sample data. Threshold is actually equivalent to the relative deviation experimentally observed for precursor ions in a particular LC-MS/MS run (15 ppm in this case);</li>
    <li>Jump areas is XXXXX. Possible values are 1, 3 and 5;</li>
    <li>FDR establishes the FDR cutoff for PSM validation. The value used here (0.01, i.e. 1% FDR) implies that one in every 100 validated PSMs is wrong;</li>
    <li>XCorr type determines whether XCorr (SEQUEST cross correlation score) or the corrected XCorr (cXCorr) will be used by pRatio for FDR calculation;</li>
    <li>Decoy label is the tag attached to decoy protein identifiers in the concatenated protein database used for peptide identification (INV_ in this example).</li>
</ul>

<h4>Relations matter: the Databases menu</h4>
<p>
    Two databases are necessary for iSanXoT to work: a protein sequence database and a protein-category relation database. The latter, a plain text file relating proteins to functional categories, is used by the Systems Biology Triangle (SBT) algorithm to estimate the experimental protein variance<sup>8</sup>.
    <figure class="text-center">
        <img class="figure-img rounded" src="assets/images/help_basic_figure2.png" />
        <figcaption class="figure-caption text-center">Figure 2. The Basic Workflow Databases window in the WSPP-SBT sample with iSanXoT databases option.</figcaption>
    </figure>
    But don’t panic (yet): since you arrived here through the WSPP-SBT sample with iSanXoT databases link, your Databases menu should look like the one on Fig. 2. This means that you are having iSanXoT use its own databases downloaded from Uniprot, and therefore all you have to do is to select a species (human in this case, since we are using Jurkat cell preparations) from the Select Species drop-down menu and a database version from the Select version of Uniprot category database menu.
</p>
<p>
    Nevertheless, if for some reason you want to use your own databases, this is indeed possible. Select Menu on the top left corner and afterwards Main Page from the drop-down menu. Then follow the link WSPP-SBT sample with user-provided databases showing in the Basic Workflow field. In the Databases menu, both the Select Species and the Select version of Uniprot category database fields are now empty; but if you activate the Rels. From Cat. File option on the left, the Protein database and Protein-category relation file fields will display the path to a protein database and a protein-category relation file, respectively, stored in your hard drive (Fig. 3).
</p> 
<figure class="text-center">
    <img class="figure-img rounded" src="assets/images/help_basic_figure3.png" />
    <figcaption class="figure-caption text-center">Figure 3. The Basic Workflow Databases window in the WSPP-SBT sample with user-provided databases option.</figcaption>
</figure>

<div class="footnote">
    <hr/>
    <div>(1) Ross P.L. et al. (2004) Multiplexed protein quantitation in Saccharomyces cerevisiae using amine-reactive isobaric tagging reagents. Mol. Cell. Proteomics 3, 1154–69.</div>
    <div>(2) Stein D.R. et al. (2013) High pH reversed-phase chromatography as a superior fractionation scheme compared to off-gel isoelectric focusing for complex proteome analysis. Proteomics 13, 2956-66.</div>
    <div>(3) https://www.thermofisher.com/order/catalog/product/OPTON-30810#/OPTON-30810.</div>
    <div>(4) https://www.unimod.org/</div>
    <div>(5) Eng J.K. et al. (1994) An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. J. Am. Soc. Mass Spectrom. 5, 976–89.</div>
    <div>(6) Martínez-Bartolomé S. et al. (2008) Properties of average score distributions of SEQUEST: the probability ratio method. Mol. Cell. Proteomics 7, 1135-45.</div>
    <div>(7) Bonzon-Kulichenko E. et al. (2015) Revisiting Peptide Identification by High-Accuracy Mass Spectrometry: Problems Associated with the Use of Narrow Mass Precursor Windows. J. Proteome Res. 14, 700–10.</div>
    <div>(8) García-Marqués F. et al. (2016) A Novel Systems-Biology Algorithm for the Analysis of Coordinated Protein Responses Using Quantitative Proteomics. Mol. Cell. Proteomics 15, 1740-60.</div>
</div>
    
    
    

</template>
