<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Helvetica;
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:Courier;
	panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face
	{font-family:"Tms Rmn";
	panose-1:2 2 6 3 4 5 5 2 3 4;}
@font-face
	{font-family:Helv;
	panose-1:2 11 6 4 2 2 2 3 2 4;}
@font-face
	{font-family:"New York";
	panose-1:2 4 5 3 6 5 6 2 3 4;}
@font-face
	{font-family:System;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:Batang;
	panose-1:2 3 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:PMingLiU;
	panose-1:2 2 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;}
@font-face
	{font-family:Dotum;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimHei;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:MingLiU;
	panose-1:2 2 5 9 0 0 0 0 0 0;}
@font-face
	{font-family:Mincho;
	panose-1:2 2 6 9 4 3 5 8 3 5;}
@font-face
	{font-family:Gulim;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:Century;
	panose-1:2 4 6 4 5 5 5 2 3 4;}
@font-face
	{font-family:"Angsana New";
	panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face
	{font-family:"Cordia New";
	panose-1:2 11 3 4 2 2 2 2 2 4;}
@font-face
	{font-family:Mangal;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Latha;
	panose-1:2 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sylfaen;
	panose-1:1 10 5 2 5 3 6 3 3 3;}
@font-face
	{font-family:Vrinda;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Raavi;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Shruti;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sendnya;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Gautami;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Tunga;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Estrangelo Edessa";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Yu Gothic";
	panose-1:2 11 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:DengXian;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Palatino Linotype";
	panose-1:2 4 5 2 5 5 5 3 3 4;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"Arial Unicode MS";
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:"Aptos Display";}
@font-face
	{font-family:Aptos;}
@font-face
	{font-family:"Segoe UI Emoji";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:ACaslon-Regular;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:center;
	line-height:107%;
	page-break-after:avoid;
	border:none;
	padding:0in;
	font-size:26.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:normal;
	font-style:italic;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:11.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:22.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-link:"Footnote Text Char";
	margin:0in;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-link:"Comment Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0in;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0in;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	text-align:center;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoFootnoteReference
	{vertical-align:super;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpFirst, li.MsoTitleCxSpFirst, div.MsoTitleCxSpFirst
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpMiddle, li.MsoTitleCxSpMiddle, div.MsoTitleCxSpMiddle
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpLast, li.MsoTitleCxSpLast, div.MsoTitleCxSpLast
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
a:link, span.MsoHyperlink
	{font-family:"Calibri",sans-serif;
	color:#2E74B5;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
strong
	{font-family:"Times New Roman",serif;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-link:"Comment Subject Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Balloon Text Char";
	margin:0in;
	text-align:justify;
	font-size:9.0pt;
	font-family:"Segoe UI",sans-serif;}
p.MsoRMPane, li.MsoRMPane, div.MsoRMPane
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoQuote, li.MsoQuote, div.MsoQuote
	{mso-style-link:"Quote Char";
	margin-top:10.0pt;
	margin-right:.6in;
	margin-bottom:8.0pt;
	margin-left:.6in;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#404040;
	font-style:italic;}
span.MsoIntenseEmphasis
	{font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoIntenseReference
	{font-variant:small-caps;
	color:#4472C4;
	letter-spacing:.25pt;
	font-weight:bold;}
span.MsoBookTitle
	{letter-spacing:.25pt;
	font-weight:bold;
	font-style:italic;}
p.MsoTocHeading, li.MsoTocHeading, div.MsoTocHeading
	{margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-link:Title;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-link:"Footnote Text";}
p.lead, li.lead, div.lead
	{mso-style-name:lead;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.QuoteChar
	{mso-style-name:"Quote Char";
	mso-style-link:Quote;
	color:#404040;
	font-style:italic;}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-link:"Comment Text";}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-link:"Comment Subject";
	font-weight:bold;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-link:"Balloon Text";
	font-family:"Segoe UI",sans-serif;}
p.EndNoteBibliographyTitle, li.EndNoteBibliographyTitle, div.EndNoteBibliographyTitle
	{mso-style-name:"EndNote Bibliography Title";
	mso-style-link:"EndNote Bibliography Title Char";
	margin:0in;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyTitleChar
	{mso-style-name:"EndNote Bibliography Title Char";
	mso-style-link:"EndNote Bibliography Title";
	font-family:"Calibri",sans-serif;}
p.EndNoteBibliography, li.EndNoteBibliography, div.EndNoteBibliography
	{mso-style-name:"EndNote Bibliography";
	mso-style-link:"EndNote Bibliography Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyChar
	{mso-style-name:"EndNote Bibliography Char";
	mso-style-link:"EndNote Bibliography";
	font-family:"Calibri",sans-serif;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;
	font-family:"Calibri",sans-serif;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;
	font-family:"Calibri",sans-serif;}
span.UnresolvedMention1
	{mso-style-name:"Unresolved Mention1";
	font-family:"Times New Roman",serif;
	color:#605E5C;
	background:#E1DFDD;}
span.msoIns
	{mso-style-name:"";
	text-decoration:underline;
	color:teal;}
span.msoDel
	{mso-style-name:"";
	text-decoration:line-through;
	color:red;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:595.3pt 841.9pt;
	margin:70.85pt 85.05pt 70.85pt 85.05pt;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#2E74B5" vlink="#954F72" style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitle align=center style='text-align:center'><b><i><span
style='font-size:36.0pt;letter-spacing:.25pt'><img width=310 height=355
id="Picture 16" src="index_files/image001.png" alt=Molecule></span></i></b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitleCxSpFirst align=center style='text-align:center'><span
class=MsoBookTitle><span style='font-size:36.0pt'>User Guide:</span></span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
class=MsoBookTitle><span style='font-size:36.0pt'>Help and documentation</span></span></p>

<p class=MsoTitleCxSpLast align=center style='text-align:center'><span
class=MsoBookTitle>(2.1.0)</span></p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1 align=left style='text-align:left'>                  <a name="_Toc187758849"></a><a
name="_Toc199847972">Table of contents</a></h1>

</div>

<p class=MsoToc1><a href="#_Toc199847972">Table of contents<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>2</span></a></p>

<p class=MsoToc1><a href="#_Toc199847973">Introduction<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc2><a href="#_Toc199847974"><span lang=EN-GB>Download</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc2><a href="#_Toc199847975"><span lang=EN-GB>Installation</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc3><a href="#_Toc199847976"><span lang=EN-GB>Available operating
systems</span><span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc2><a href="#_Toc199847977"><span lang=EN-GB>Getting Started</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc2><a href="#_Toc199847978"><span lang=EN-GB>Modules</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc2><a href="#_Toc199847979"><span lang=EN-GB>Input Adaptor</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>6</span></a></p>

<p class=MsoToc2><a href="#_Toc199847980"><span lang=EN-GB>Sample Workflows</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>6</span></a></p>

<p class=MsoToc2><a href="#_Toc199847981"><span lang=EN-GB>More Documents</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>6</span></a></p>

<p class=MsoToc2><a href="#_Toc199847982"><span lang=EN-GB>License</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>7</span></a></p>

<p class=MsoToc3><a href="#_Toc199847983">Under the following terms<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>7</span></a></p>

<p class=MsoToc1><a href="#_Toc199847984">Installation<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>8</span></a></p>

<p class=MsoToc2><a href="#_Toc199847985">Download<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>8</span></a></p>

<p class=MsoToc2><a href="#_Toc199847986"><span lang=EN-GB>Available operating
systems</span><span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>8</span></a></p>

<p class=MsoToc2><a href="#_Toc199847987">Windows distribution<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>8</span></a></p>

<p class=MsoToc2><a href="#_Toc199847988">MacOS distribution<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>10</span></a></p>

<p class=MsoToc2><a href="#_Toc199847989">Linux distribution<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>11</span></a></p>

<p class=MsoToc1><a href="#_Toc199847990">Getting Started<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>13</span></a></p>

<p class=MsoToc2><a href="#_Toc199847991">Opening the iSanXoT application<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>13</span></a></p>

<p class=MsoToc2><a href="#_Toc199847992">Installing required packages<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>13</span></a></p>

<p class=MsoToc2><a href="#_Toc199847993">Closing the iSanXoT application<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>14</span></a></p>

<p class=MsoToc2><a href="#_Toc199847994">iSanXoT Projects<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>14</span></a></p>

<p class=MsoToc2><a href="#_Toc199847995">iSanXoT Main Window<span
style='color:windowtext;display:none;text-decoration:none'>.. </span><span
style='color:windowtext;display:none;text-decoration:none'>15</span></a></p>

<p class=MsoToc2><a href="#_Toc199847996">Importing and Exporting Workflows<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>16</span></a></p>

<p class=MsoToc2><a href="#_Toc199847997">Executing a Project<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>17</span></a></p>

<p class=MsoToc2><a href="#_Toc199847998">Running Processes<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>17</span></a></p>

<p class=MsoToc1><a href="#_Toc199847999">Modules<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>20</span></a></p>

<p class=MsoToc2><a href="#_Toc199848000">Relation tables module<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>20</span></a></p>

<p class=MsoToc3><a href="#_Toc199848001">RELS CREATOR<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>20</span></a></p>

<p class=MsoToc2><a href="#_Toc199848002">Basic modules<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>21</span></a></p>

<p class=MsoToc3><a href="#_Toc199848003">LEVEL CREATOR<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>21</span></a></p>

<p class=MsoToc3><a href="#_Toc199848004">LEVEL CALIBRATOR<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>22</span></a></p>

<p class=MsoToc3><a href="#_Toc199848005">INTEGRATE<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>23</span></a></p>

<p class=MsoToc3><a href="#_Toc199848006">NORCOMBINE<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>27</span></a></p>

<p class=MsoToc3><a href="#_Toc199848007">RATIOS<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>30</span></a></p>

<p class=MsoToc3><a href="#_Toc199848008">SBT<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>31</span></a></p>

<p class=MsoToc2><a href="#_Toc199848009">Compound modules<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>33</span></a></p>

<p class=MsoToc3><a href="#_Toc199848010">WSPP-SBT<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>34</span></a></p>

<p class=MsoToc3><a href="#_Toc199848011">WSPPG-SBT<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>35</span></a></p>

<p class=MsoToc3><a href="#_Toc199848012">WPP-SBT<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>36</span></a></p>

<p class=MsoToc3><a href="#_Toc199848013">WPPG-SBT<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>37</span></a></p>

<p class=MsoToc2><a href="#_Toc199848014">Reports modules<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>39</span></a></p>

<p class=MsoToc3><a href="#_Toc199848015">REPORT<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>39</span></a></p>

<p class=MsoToc3><a href="#_Toc199848016">SANSON<span style='color:windowtext;
display:none;text-decoration:none'>.. </span><span
style='color:windowtext;display:none;text-decoration:none'>42</span></a></p>

<p class=MsoToc2><a href="#_Toc199848017">Special parameters<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>43</span></a></p>

<p class=MsoToc3><a href="#_Toc199848018">Multiple samples<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>43</span></a></p>

<p class=MsoToc3><a href="#_Toc199848019">Asterisk is our jack of all trades<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>44</span></a></p>

<p class=MsoToc3><a href="#_Toc199848020">Multiple samples in the inputs and
outputs<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>46</span></a></p>

<p class=MsoToc3><a href="#_Toc199848021">More params<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>47</span></a></p>

<p class=MsoToc3><a href="#_Toc199848022">Filter in REPORT module<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>48</span></a></p>

<p class=MsoToc3><a href="#_Toc199848023">Filter in SANSON module<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>48</span></a></p>

<p class=MsoToc1><a href="#_Toc199848024">Sample Workflows with Application to
Case Studies<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>49</span></a></p>

<p class=MsoToc2><a href="#_Toc199848025"><span style='font-family:"Calibri Light",sans-serif'>Workflow
1: One-step quantification in a labeled experiment</span><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>49</span></a></p>

<p class=MsoToc3><a href="#_Toc199848026"><b><span style='font-family:"Calibri Light",sans-serif'>Experimental</span></b><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>49</span></a></p>

<p class=MsoToc3><a href="#_Toc199848027"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
execution</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>49</span></a></p>

<p class=MsoToc3><a href="#_Toc199848028"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
operation</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>49</span></a></p>

<p class=MsoToc2><a href="#_Toc199848029"><span style='font-family:"Calibri Light",sans-serif'>Workflow
2: Step-by-step quantification and sample combination in a labeled experiment</span><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>56</span></a></p>

<p class=MsoToc3><a href="#_Toc199848030"><b><span style='font-family:"Calibri Light",sans-serif'>Experimental</span></b><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>56</span></a></p>

<p class=MsoToc3><a href="#_Toc199848031"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
execution</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>57</span></a></p>

<p class=MsoToc3><a href="#_Toc199848032"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
operation</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>57</span></a></p>

<p class=MsoToc2><a href="#_Toc199848033"><span style='font-family:"Calibri Light",sans-serif'>Workflow
3: Quantification of posttranslationally modified peptides in a labeled
experiment</span><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>65</span></a></p>

<p class=MsoToc3><a href="#_Toc199848034"><b><span style='font-family:"Calibri Light",sans-serif'>Experimental</span></b><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>65</span></a></p>

<p class=MsoToc3><a href="#_Toc199848035"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
execution</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>65</span></a></p>

<p class=MsoToc3><a href="#_Toc199848036"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
operation</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>65</span></a></p>

<p class=MsoToc2><a href="#_Toc199848037"><span style='font-family:"Calibri Light",sans-serif'>Workflow
4: Label-free quantification</span><span style='color:windowtext;display:none;
text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>70</span></a></p>

<p class=MsoToc3><a href="#_Toc199848038"><b><span style='font-family:"Calibri Light",sans-serif'>Experimental</span></b><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>70</span></a></p>

<p class=MsoToc3><a href="#_Toc199848039"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
execution</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>71</span></a></p>

<p class=MsoToc3><a href="#_Toc199848040"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
operation</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>71</span></a></p>

<p class=MsoToc2><a href="#_Toc199848041"><span lang=EN-GB>Workflow 5:
PTM-compass</span><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>82</span></a></p>

<p class=MsoToc3><a href="#_Toc199848042"><b><span style='font-family:"Calibri Light",sans-serif'>Experimental</span></b><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>82</span></a></p>

<p class=MsoToc3><a href="#_Toc199848043"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
execution</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>82</span></a></p>

<p class=MsoToc3><a href="#_Toc199848044"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
operation</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>82</span></a></p>

<p class=MsoToc2><a href="#_Toc199848045"><span lang=EN-GB>Workflow 6:
Single-Cell Proteomics</span><span style='color:windowtext;display:none;
text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>83</span></a></p>

<p class=MsoToc3><a href="#_Toc199848046"><b><span style='font-family:"Calibri Light",sans-serif'>Experimental</span></b><span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>83</span></a></p>

<p class=MsoToc3><a href="#_Toc199848047"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
execution</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>83</span></a></p>

<p class=MsoToc3><a href="#_Toc199848048"><b><span style='font-family:"Calibri Light",sans-serif'>Workflow
operation</span></b><span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>83</span></a></p>

<p class=MsoToc1><a href="#_Toc199848049">Importing a workflow template<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>84</span></a></p>

<p class=MsoToc1><a href="#_Toc199848050">Creating the
identification/quantification file from proteomics pipelines<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>87</span></a></p>

<p class=MsoToc2><a href="#_Toc199848051">Preparing the <i>ID-q</i> file from
Proteome Discoverer output<span style='color:windowtext;display:none;
text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>88</span></a></p>

<p class=MsoToc3><a href="#_Toc199848052">Label-free experiments<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>88</span></a></p>

<p class=MsoToc3><a href="#_Toc199848053">Labelled experiments<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>88</span></a></p>

<p class=MsoToc2><a href="#_Toc199848054">Preparing the <i>ID-q</i> file from
MaxQuant output<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>88</span></a></p>

<p class=MsoToc3><a href="#_Toc199848055">Label-free experiments<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>89</span></a></p>

<p class=MsoToc3><a href="#_Toc199848056">Labelled experiments<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>89</span></a></p>

<p class=MsoToc2><a href="#_Toc199848057">Preparing the <i>ID-q</i> file from
FragPipe output<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>89</span></a></p>

<p class=MsoToc3><a href="#_Toc199848058">Label-free experiments<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>89</span></a></p>

<p class=MsoToc3><a href="#_Toc199848059">Labelled experiments<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>90</span></a></p>

<p class=MsoToc2><a href="#_Toc199848060">Adapting the results from proteomics
pipelines for iSanXoT<span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>90</span></a></p>

<p class=MsoToc1><a href="#_Toc199848061"><span lang=ES>References</span><span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>94</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk118388234"></a><a name="_Hlk118304308"></a><a
name="_Toc199847973"></a><a name="_Introduction"></a>Introduction</h1>

</div>

<p class=MsoNormal><b><span lang=EN-GB>iSanXoT</span></b><span lang=EN-GB> is a
standalone application for statistical analysis of mass spectrometry-based
quantitative proteomics data.  iSanXoT builds upon SanXoT </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>, our previous publicly available
implementation of the Weighted Spectrum, Peptide, and Protein (WSPP)
statistical model </span><span
lang=EN-GB>[2]</span><span lang=EN-GB> using the Generic Integration Algorithm
(GIA) </span><span
lang=EN-GB>[3]</span><span lang=EN-GB>.</span></p>

<p class=MsoNormal><span lang=EN-GB>iSanXoT executes several kinds of workflows
for quantitative high-throughput proteomics, <b>systems biology </b>and the <b>statistical
analysis</b>, <b>integration and comparison of experiments</b>.</span></p>

<p class=MsoNormal><span lang=EN-GB>iSanXoT was developed by the <b><i>Cardiovascular
Proteomics Lab</i></b>/<b><i>Proteomic Unit</i></b> at <b>The National Centre
for Cardiovascular Research</b> (CNIC, </span><a href="https://www.cnic.es"><span
lang=EN-GB style='color:#4472C4'>https://www.cnic.es</span></a><span
lang=EN-GB>).</span></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847974"><span lang=EN-GB>Download</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>Multiple releases are available in the “release”
section, which can be found at the following link:</span></p>

<p class=MsoNormal><a href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847975"><span lang=EN-GB>Installation</span></a></h2>

<h3><a name="_Toc199847976"><span lang=EN-GB>Available operating systems</span></a></h3>

<p class=MsoNormal><span lang=EN-GB>iSanXoT supports the following operating
systems and architectures, with the possibility of adding additional ones in
the future:</span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:53.4pt;text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Windows 10 Pro (x64)</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:53.4pt;text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>MacOS High Sierra (10.13.6)</span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:53.4pt;text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Ubuntu 20.04 (x64)</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, please refer to the “Installation”
section in the iSanXoT web documentation:</span></p>

<p class=MsoNormal><a
href="https://cnic-proteomics.github.io/iSanXoT/#_Installation"><span
lang=EN-GB>https://cnic-proteomics.github.io/iSanXoT/#_Installation</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847977"><span lang=EN-GB>Getting Started</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>This chapter describes iSanXoT graphical
user interface and provides guidance on setting up an analysis with iSanXoT.</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, please read the
&quot;Getting Started&quot; section in the iSanXoT web documentation:</span></p>

<p class=MsoNormal><a
href="https://cnic-proteomics.github.io/iSanXoT/#_Get_Started"><span
lang=EN-GB>https://cnic-proteomics.github.io/iSanXoT/#_Get_Started</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847978"><span lang=EN-GB>Modules</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>The iSanXoT desktop application comprises
several modules based on the SanXoT software package </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>. Information necessary for setting up
and executing each module is provided in a task table. There are four types of
modules.   </span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span lang=EN-GB>Relation Tables</span></b><span lang=EN-GB>:
This module creates the relation tables used by the iSanXoT modules.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span lang=EN-GB>Basic Modules</span></b><span lang=EN-GB>:
These modules call individual scripts included in the SanXoT software package </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span lang=EN-GB>Compound Modules</span></b><span lang=EN-GB>:
These modules perform a sequence of consecutive integrations based on the
Weighted Spectrum, Peptide, and Protein (WSPP) statistical model </span><span lang=EN-GB>[2]</span><span lang=EN-GB> and the Systems-Biology Triangle (SBT)
algorithm </span><span
lang=EN-GB>[3]</span><span lang=EN-GB>.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span lang=EN-GB>Reports</span></b><span lang=EN-GB>: There
are two types of reports, REPORT generates report files displaying the
quantitative results produced by the Basic and Composite modules when a
workflow is executed, while SANSON generates a similarity graph showing
relationships between functional categories based on the protein elements they
share.</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, please refer to the
“Modules” section in the iSanXoT web documentation:</span></p>

<p class=MsoNormal><a href="https://cnic-proteomics.github.io/iSanXoT/#_Modules"><span
lang=EN-GB>https://cnic-proteomics.github.io/iSanXoT/#_Modules</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847979"><span lang=EN-GB>Input Adaptor</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>The iSanXoT Input Adaptor provides users
with the option to either supply their own Identification/Quantification file,
containing the identification and quantification data, or allow the Input
Adaptor to generate this file from the results obtained using any of the
mainstream proteomics pipelines.</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, please refer to the “Input
Adaptor” section in the iSanXoT web documentation:</span></p>

<p class=MsoNormal><a
href="https://cnic-proteomics.github.io/iSanXoT/#_Creating_the_identification/quantif"><span
lang=EN-GB>https://cnic-proteomics.github.io/iSanXoT/#_Creating_the_identification/quantif</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847980"><span lang=EN-GB>Sample Workflows</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>We provide detailed descriptions of four
sample workflows that demonstrate iSanXoT's capability to statistically
ascertain abundance changes in both multiplexed, isotopically labeled</span><span
lang=EN-GB> </span>[3-5] and label-free [6] proteomics experiments.</p>

<p class=MsoListParagraphCxSpFirst align=left style='text-align:left;
text-indent:-.25in'><span lang=EN-GB style='font-family:Symbol'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a
href="https://cnic-proteomics.github.io/iSanXoT/#_Workflow_1:_One-step"><span
lang=EN-GB>Workflow 1</span></a><span lang=EN-GB>: One-step quantification in a
labeled experiment.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a
href="https://cnic-proteomics.github.io/iSanXoT/#_Workflow_2:_Step-by-step"><span
lang=EN-GB>Workflow 2</span></a><span lang=EN-GB>: Step-by-step quantification
and sample combination in a labeled experiment.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a
href="https://cnic-proteomics.github.io/iSanXoT/#_Workflow_3:_Quantification"><span
lang=EN-GB>Workflow 3</span></a><span lang=EN-GB>: Quantification of
posttranslationally modified peptides in a labeled experiment.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a
href="https://cnic-proteomics.github.io/iSanXoT/#_Workflow_4:_Label-free"><span
lang=EN-GB>Workflow 4</span></a><span lang=EN-GB>: Label-free quantification.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Workflow 5: PTM-compass.</span></p>

<p class=MsoNormal><span lang=EN-GB>For further details, please refer to each
respective section in the iSanXoT web documentation.</span></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847981"><span lang=EN-GB>More Documents</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>User Guide (v2.0.0):</span></p>

<p class=MsoNormal><a name="_License"></a><a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/user_guides/User_Guide_iSanXoT-2.0.0.pdf"><span
lang=EN-GB style='color:#4472C4'>https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/user_guides/User_Guide_iSanXoT-2.0.0.pdf</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Hlk118382155"></a><a name="_Toc199847982"><span lang=EN-GB>License</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>This application is licensed under a <b>Creative
Commons Attribution-NonCommercial-NoDerivs 4.0 Unported License</b>.</span></p>

<p class=MsoNormal><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><span
lang=EN-GB style='color:#4472C4'>https://creativecommons.org/licenses/by-nc-nd/4.0/</span></a></p>

<h4>You are free to:</h4>

<p class=MsoNormal>Share - copy and redistribute the material in any medium or
format.</p>

<p class=MsoNormal>The licensor cannot revoke these freedoms as long as you
follow the license terms.</p>

<h3><a name="_Toc199847983">Under the following terms</a></h3>

<p class=MsoNormal><a href="https://cnic-proteomics.github.io/iSanXoT/#_License"><span
lang=EN-GB style='color:#4472C4'>https://cnic-proteomics.github.io/iSanXoT/#_License</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk118375025"></a><a name="_Toc199847984"></a><a
name="_Installation"></a>Installation</h1>

</div>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h2><a name="_Toc199847985"></a><a name="_Download"></a>Download</h2>

<p class=MsoNormal><span lang=EN-GB>The multiple releases are available in the “release”
section, which can be found at the following link:</span></p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span lang=EN-GB
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB>Note: We
recommend downloading the latest release.</span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Toc199847986"></a><a name="_Available_operating_systems"></a><span
lang=EN-GB>Available operating systems</span></h2>

<p class=MsoNormal><span lang=EN-GB>iSanXoT currently supports the following
operating systems and architectures, and additional ones may be added in the
future:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Windows 10 Pro (x64)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>MacOS High Sierra (10.13.6)</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Ubuntu 20.04 (x64)</span></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h2><a name="_Toc199847987"></a><a name=windows-distribution></a><a
name=installation-on-windows></a><a name="_Installation_on_Windows"></a><a
name="_Windows_distribution"></a>Windows distribution</h2>

<p class=MsoNormal>The iSanXoT Windows distribution is packaged in a NSIS
Launcher (exe file).</p>

<p class=MsoNormal>To download the exe Launcher, click on: iSanXoT_Launcher_X.X.X.win32-x64.exe</p>

<p class=MsoNormal>Once downloaded, double-click the Launcher file, and the
Installer window will appear. Follow the on-screen instructions to complete the
installation:</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 28" src="index_files/image002.jpg"></span></p>

<p class=MsoNormal>WARNING: For the time being, it is necessary to install for
the current user (“only for me” option).</p>

<p class=MsoNormal>WARNING 2: Windows Defender SmartScreen might display a
prompt suggesting that you cancel the installation. In such cases, click on “More
info” and then select the “Run anyway” option to proceed with the installation.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=332 height=140 id="Picture 43" src="index_files/image003.png"></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>During the installation process, you can choose the iSanXoT
installation folder. Select the desired folder where you want iSanXoT to be
installed.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 25" src="index_files/image004.jpg"></span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Please wait while iSanXoT is being installed. This process
may take a few moments.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 22" src="index_files/image005.png"
alt="Installing window"></span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Once the installation has been completed, you are ready to
run iSanXoT.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 21" src="index_files/image006.png"
alt="Completing setup"></span></p>

<p class=MsoNormal><span style='font-family:"Segoe UI",sans-serif;color:#24292E'>&nbsp;</span></p>

<h2><a name="_Toc199847988"></a><a name=macos-distribution></a><a
name="_MacOS_distribution"></a>MacOS distribution</h2>

<p class=MsoNormal>The iSanXoT MacOS distribution is packaged in a DMG
container.</p>

<p class=MsoNormal>To download the DMG file, click on: iSanXoT_Launcher_X.X.X.darwin-x64.dmg</p>

<p class=MsoNormal>After downloading, double-click the DMG file, and a Finder
window will appear. This window typically displays iSanXoT’s installer icon and
a shortcut to the Applications folder, along with a linking arrow.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=416 height=309 id="Picture 19" src="index_files/image007.png"
alt="Installer window"></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Simply drag the iSanXoT icon to your Applications folder…</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=439 height=315 id="Picture 18" src="index_files/image008.png"
alt="Installing window"></span></p>

<p class=MsoNormal>And you’re done! The iSanXoT application is now successfully
installed on your MacOS system.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc199847989"></a><a name=linux-distribution></a><a
name="_Linux_distribution"></a>Linux distribution</h2>

<p class=MsoNormal>The iSanXoT Linux distribution is packaged in an <i>AppImage</i>.</p>

<p class=MsoNormal>To download the AppImage, click on: iSanXoT_Launcher_X.X.X.linux-x86_64.AppImage
</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=567 height=241 src="index_files/image009.png"></p>

<p class=MsoNormal>The <i>AppImage</i> file is essentially the compressed image
of the application. To ensure correct behavior, follow these steps:</p>

<p class=MsoListParagraph style='text-indent:-.25in'>1.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Execute the following command to extract the application to the
“squashfs-root” folder in the current working directory:</p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_X.X.X.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Launch
the iSanXoT application using the following command:</p>

<p class=MsoListParagraphCxSpLast style='margin-left:35.4pt;text-indent:35.4pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun</span></em></p>

<em><span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><span
style='font-family:"Calibri",sans-serif;font-style:normal'>&nbsp;</span></em></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc199847990"></a><a name="_Get_Started"></a><em><span
style='font-family:"Calibri Light",sans-serif;font-style:normal'>Getting
Started</span></em></h1>

</div>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>This chapter describes iSanXoT’s graphical user interface
and provides guidance on setting up an analysis with iSanXoT.</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h2><a name="_Toc199847991">Opening the iSanXoT application</a></h2>

<p class=MsoNormal>To open the iSanXoT application, follow these instructions
based on your operating system:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Windows:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>from the Start menu choose <i>Programs &gt; iSanXoT</i>. Alternatively
double-click the iSanXoT desktop icon.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In MacOS:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>double-click the iSanXoT icon from the Applications folder.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Linux:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>from the <i>AppImage</i> file:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:85.05pt;text-indent:
-14.15pt'>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>The contents are extracted to the “squashfs-root” directory in the
current working directory using:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:71.4pt;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_X.X.X.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:85.05pt;text-indent:
-14.25pt'>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Then, you can launch the iSanXoT application:</p>

<p class=MsoListParagraphCxSpLast style='margin-left:71.4pt;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun </span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h2><a name="_Toc199847992">Installing required packages</a></h2>

<p class=MsoNormal>The first time iSanXoT is run, a window will appear
displaying a progress bar, informing you about the percentage of packages that
have been installed. These packages contain the libraries required by iSanXoT’s
backend and are installed during the initial launch of the application.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=372 height=244 src="index_files/image010.png"></p>

<p class=MsoCaption>Figure 1. <a name="_Ref103093720">Installation window.</a></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h2><a name="_Toc199847993">Closing the iSanXoT application</a></h2>

<p class=MsoNormal>WARNING: If valid changes were made to your project, make
sure to save it before quitting iSanXoT, as any changes will be lost otherwise
(you won’t be prompted for saving upon closing).  </p>

<p class=MsoNormal style='margin-left:.25in'>To close the iSanXoT application:</p>

<p class=MsoNormal style='margin-left:.25in'>In Windows and Linux: choose <i>Project
&gt; Exit</i>, or click the X in the upper right corner of the main iSanXoT window.</p>

<p class=MsoNormal style='margin-left:.25in'>In Mac: choose <i>iSanXoT (menu)
&gt; Exit</i>, or click the red X in the upper left corner of the main iSanXoT
window.</p>

<p class=MsoNormal>A dialog window will show up asking you to confirm the
application closing. Click “Yes” if you really want to quit iSanXoT.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc199847994">iSanXoT Projects</a></h2>

<p class=MsoNormal>An iSanXoT project is primarily a container used to
structure the data coming from your input file(s) and your workflow. The input
file contains the identification and quantification data (for further details see
the <a href="#_Creating_the_identification/quantif">Input Adaptor</a> Section).
These fully-customisable workflows can perform quantitative proteomics analysis,
systems biology analysis, and comparison and merging of experimental data from
technical or biological replicates.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=224 height=213 src="index_files/image011.png"></p>

<p class=MsoCaption><a name="_Ref103114194">Figure </a>2. Project menu.</p>

<h4>Creating a new project</h4>

<p class=MsoNormal>Creating a project is the first step when conducting an
analysis with iSanXoT. By selecting <i>Project &gt; New Project</i> a window
will show up where you can provide a name for the project as well as select a
project folder where iSanXoT output files will be stored.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=536 height=199 src="index_files/image012.png"></p>

<p class=MsoCaption>Figure 3. Window that creates a new project.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Opening a project</h4>

<p class=MsoNormal>By selecting <i>Project &gt; Open Project</i> as a folder
selection dialog box shows up that allows the user to indicate the location of an
already existing project folder to be opened by iSanXoT.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc199847995">iSanXoT Main Window</a></h2>

<p class=MsoNormal>The iSanXoT main window consists of an overhead Menu, Content
tabs, and Content and Execution panels (<i><span
style='color:#BF8F00'>Figure 4</span></i>).</p>

<h4>Menu</h4>

<p class=MsoNormal>The Menu contains the following items:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Project: Allows operations related to projects and workflows.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Processes: Links to real-time display of the processes currently
in execution by iSanXoT (see <a href="#_Running_Processes">Running Processes</a>
Section).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Help: Houses the different Sections of iSanXoT help.</p>

<h4>Content tabs</h4>

<p class=MsoNormal>Five tabs are displayed in iSanXoT’s project page. The <i>Input
File(s)</i> tab displays the Project folder, where iSanXoT output files are
stored, as well as the Identification file used in the project (see <a
href="#_Creating_the_identification/quantif">Input Adaptor</a> Section). The
remaining four tabs give access to iSanXoT modules: <a
href="#_Relation_tables_module">Relation Tables</a>, <a href="#_Basic_modules_1">Basic
Modules</a>, <a href="#_Compound_modules">Compound Modules</a>, and <a
href="#_Reports_modules">Report Modules</a>.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=637 height=361 id="Picture 132" src="index_files/image013.jpg"></p>

<p class=MsoCaption><a name="_Ref103114143"></a><a name="_Ref103114150">Figure </a>4.
Main View of iSanXoT.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Content panel</h4>

<p class=MsoNormal>This panel houses the elements of the Content Tabs, whose
modules can be accessed through the sidebar menu on the left side of the panel
(<b>Modules Menu</b>). A title and a brief description of the module are
provided, along with a help icon linking to additional information on the
specific module selected.</p>

<h4><a name="_Execution_panel"></a>Execution panel</h4>

<p class=MsoNormal>The execution panel, located at the bottom of the main
window, allows the user to indicate the number of processors to be used by
iSanXoT, with 4 set as the default. The “Save and Run” button saves the project
into the output folder and launches the execution of the workflow shaped by the
Input elements and Modules.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc199847996">Importing and Exporting Workflows</a></h2>

<p class=MsoNormal>A project is shaped by a workflow that instructs iSanXoT how
to process the data provided by the input file(s). While the whole project,
including workflow and data, can be saved as indicated below (see the <a
href="#_Saving_a_project">Saving a project</a> section), there is a way to
import and export just the workflow structure using iSanXoT menu (<i><span style='color:#BF8F00'>Figure 2</span></i>).</p>

<h4>Import Workflow</h4>

<p class=MsoNormal>This option allows you to import the task tables of a
workflow. To do that, you have to provide the folder where the workflow is
saved.</p>

<h4>Export Workflow</h4>

<p class=MsoNormal>The export workflow saves the task tables of a workflow in
the folder indicated by the user.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc199847997">Executing a Project</a></h2>

<p class=MsoNormal>Once your project contains all the necessary input data and
workflow elements, you can execute the workflow by clicking “Save and Run” in
the “Execution panel” after indicating the number of processors to be allocated
for iSanXoT (see  <a href="#_Execution_panel">Execution panel</a> above). </p>

<p class=MsoNormal>Bear in mind that every time you click the “Save and Run”
button to execute a workflow, the project is first validated for consistency and
saved. To save a project without executing it, you must use the appropriate
menu item as explained in the next Section.</p>

<h4><a name="_Saving_a_project"></a>Saving a project</h4>

<p class=MsoNormal>The <i>Project &gt; Save Project</i> option saves your
project, which contains the input data and the workflow elements, to the
“Project folder”. The project files are saved in the “.isanxot” folder. WARNING:
Do not manipulate or delete the information stored in the “.isanxot” folder;
you risk losing your project.</p>

<p class=MsoNormal>Whenever iSanXoT is prompted to save a project, the corresponding
workflow is first validated for consistency and won’t be saved when failed. Neither
will the workflow execute if it has not been validated previously.</p>

<p class=MsoNormal>As well as the “.isanxot” folder, the following folders are
necessary to shape your project:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Exps, to store the files created by the Input Data adapters.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Jobs, to store the <i>sample folders</i> of your workflow.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Rels, to store the Relation Tables created by the RELS CREATOR module
(see below).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Reports, to store the Report files created by the REPORT module (see below).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Stats, to store statistical data.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Logs, to store the workflow execution log files.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc199847998"></a><a name="_Running_Processes"></a>Running Processes</h2>

<p class=MsoNormal>When workflow execution successfully starts, a new window shows
up displaying information about the processes currently running:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The “<i>Project logs”</i> table shows project execution status.
Several project executions can be monitored here, and the user must click a row
to have the corresponding workflow logs displayed (see below).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>“Workflow logs”</i> table displays status for the jobs set up
in the workflow modules. If you click on a row you will see the trace log of
the involved<span class=MsoCommentReference><span style='font-size:8.0pt;
line-height:107%'> j</span></span>obs (unless the job status is “cached”).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The selected project execution in the Project logs table can be
stopped by clicking on the “Stop” button. The processed programs will be
terminated.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=510 height=353 id="Picture 98" src="index_files/image014.png"></p>

<p class=MsoCaption>Figure 5. View of running processes.</p>

<p class=MsoNormal align=left style='text-align:left'>The running processes window
can be also reached from the menu by selecting the “Processes &gt; Main page” option.</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<p class=MsoNormal align=left style='text-align:left'>The “Processes” menu
includes a submenu, “Open Processed Project”, which opens the processed project
that has been selected in the “Project logs” table (<i><span
style='color:#BF8F00'>Figure 6</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
style='color:red'><img border=0 width=183 height=102
src="index_files/image015.png"></span></p>

<p class=MsoCaption><a name="_Ref153695478"></a><a name="_Ref153695493">Figure </a>6. The “Open Processed Project” submenu in the “Processes” menu allows the opening of
a selected project that has been executed or is currently executing.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=left style='text-align:left'>This submenu is
accessible exclusively in the “running processes” view and is activated when a
row is selected in the “Project logs” table (refer to <i><span style='color:#BF8F00'>Figure 7</span></i>). As iSanXoT facilitates the execution
of various types of workflows, the “Project logs” table may include different
project and workflow types. Consequently, users can reopen their chosen
processed projects using this method.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=567 height=174 src="index_files/image016.png"></p>

<p class=MsoCaption><a name="_Ref153695700">Figure </a>7. Reopening a
previously executed project.</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc199847999"></a><a name="_Modules"></a>Modules</h1>

</div>

<p class=MsoNormal>The iSanXoT desktop application houses a number of modules
based on the SanXoT software package [1].
The information required to setup and execute each module is provided in a task-table. 
 </p>

<p class=MsoNormal>There are four types of modules:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b>Relation tables</b>: A module that creates the relation tables
used by the iSanXoT modules.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b>Basic modules</b>: These call the individual scripts included
in the SanXoT software package [1].</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b>Compound modules</b>: These modules perform a sequence of
consecutive integrations based on the weighted spectrum, peptide and protein (WSPP)
statistical model [2] and the systems-biology triangle (SBT) algorithm [3].</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b>Reports</b>: There are two types of reports, REPORT generates report
files displaying the quantitative results produced by the above Basic and
Composite modules when a workflow is executed; SANSON generates a similarity
graph showing relationships between functional categories based on the protein
elements they share.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Toc199848000"></a><a
name="_Basic_modules"></a><a name="_Relation_tables_module"></a>Relation tables
module</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848001"></a><a name="_RELS_CREATOR"></a><span
style='font-family:"Calibri",sans-serif'>RELS CREATOR</span></h3>

<p class=MsoNormal>This module generates relation tables (RT) from tab-separated
values (TSV) files. Relation tables, which are TSV files relating lower level
identifiers (e.g. peptides) to the corresponding higher level elements (e.g.
proteins), are required for module execution. For this reason, the naming convention
for the file indicated under “Relation Table to be created” in the RELS CREATOR
task table is <i>lower level + “2” + higher level</i>; e.g. whether an
integration is from the “peptide” level to the “protein” level, a relation
table called “peptide2protein” will be necessary.</p>

<p class=MsoNormal>The fields showing in the RELS CREATOR task table are (<i><span style='color:#BF8F00'>Figure 8</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Relation Table to be created</i> specifies relation table
filenames. As commented above, the naming convention for these files is <i>lower
level + “2” + higher level</i>; e.g. whether an integration is from the
“peptide” level to the “protein” level, a relation table called
“peptide2protein” will be necessary.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of Lower level</i> is the column header that
designates which elements from the indicated file (see below) will be taken as
lower level elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of Higher level</i> is the column header that
designates which elements from the indicated file (see below) will be taken as higher
level elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of 3<sup>rd</sup> column</i> is the column header
that designates which elements from the indicated file (see below) will be
taken as third column elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Table from which RT is extracted is the full path name for the
TSV file to be used to build the relation tables. If the cell is empty, the
Input file (ID-q.tsv) is applied for iSanXoT workflow.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=163 src="index_files/image017.png"></p>

<p class=MsoCaption><a name="_Ref103114314">Figure </a>8. A sample Task-Table
in the example of RELS CREATOR module.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Toc199848002"></a><a
name="_Basic_modules_1"></a>Basic modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848003"></a><a name="_LEVEL_CREATOR"></a><span
style='font-family:"Calibri",sans-serif'>LEVEL CREATOR</span></h3>

<p class=MsoNormal>This module creates levels, which are TSV files containing
identifiers, log<sub>2</sub>-ratio values and statistical weight values. The
data are extracted from the Identification/Quantification file (see <a
href="#_Creating_the_identification/quantif">Input Adaptor</a> section).</p>

<p class=MsoNormal>The following fields are displayed in the LEVEL CREATOR task
table (<i><span style='color:#BF8F00'>Figure 9</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Batch</i> is the column header that designates which elements
from the Identification file will be used to create the level indicated. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator column</i> specifies which column header from
the identification file designates the quantitative values to be used as a numerator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator column(s)</i> specifies which column header
from the identification file designates the quantitative values to be used as a
denominator for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level to be created</i> designates the level name.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates the name of the folder
where the level data file will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=562 height=174 src="index_files/image018.png"></p>

<p class=MsoCaption><a name="_Ref103114332">Figure </a>9. A sample task-table
in the LEVEL CREATOR module.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848004"></a><a name="_LEVEL_CALIBRATOR"></a><span
style='font-family:"Calibri",sans-serif'>LEVEL CALIBRATOR</span></h3>

<p class=MsoNormal>From this release, this module consists of two sub-modules
and their corresponding task tables: Combine Calibrator and Level Calibrator.</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Combine Calibrator: Combines the uncalibrated data (samples) and
calibrates the V values of a level. This sub-module is ideal for single-cell
sample workflows (see the <a href="#_Workflow_6:_Single-Cell">Sample Workflow
section of Single-Cell Proteomics</a> for further details).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Level Calibrator: This is the original module. It calibrates the
V values of a level either freely or using the K and V constants from a
previously calibrated sample.</p>

<p class=MsoNormal>These modules calibrate the above-described levels using the
“Klibrate” program included in the SanXoT software package [1].
To perform the calibration, two parameters (weight constant and variance) are iteratively
calculated using the Levenberg-Marquardt algorithm (for more details see the information
about “Klibrate” in the SanXoT software package [1]).</p>

<p class=MsoNormal>The output calibrated level contains new statistic weight
values for the identifier and log<sub>2</sub>-ratio elements displayed in the
uncalibrated level data file. This is necessary for the levels to be used as
inputs to the INTEGRATE module.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The fields to be completed in these sub-modules (Combine
Calibrator and Level Calibrator) are (<i><span
style='color:#BF8F00'>Figure 10</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name(s) of the folder(s) containing
the uncalibrated data file(s) that were previously generated by the LEVEL
CREATOR module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level for integration</i> indicates which lower level elements
are to be used in the integration carried out for the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level for integration</i> indicates which higher level
elements are to be used in the integration carried out for the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Name of calibrated level</i> is the name for the output data
file containing the new, calibrated statistical weight values.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> specifies the name of the folder
where the output data file containing the new, calibrated statistical weight
values will be saved. If the cell is empty, the output sample folder is the
given “Sample folder” (second column).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=193 id="Picture 5" src="index_files/image019.png"></p>

<p class=MsoCaption><a name="_Ref103114351">Figure </a>10. A sample task table
in the LEVEL CALIBRATOR module (specifically, in the Level Calibrator
sub-module).</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The Combine Calibrator and Level Calibrator sub-modules accept
the following additional parameters (<i><span
style='color:#BF8F00'>Figure 11</span></i>): </p>

<p class=MsoNormal>To perform the calibration two parameters, have to be
calculated: the k (weight constant), and the variance.</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>K-constant </i>sets a forced value for the k-constant. Using
this parameter, the introduced value is forced as K-constant.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Var(x)</i> sets a forced value for the variance. Using this
parameter, the introduced value is forced as the variance.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder with combined calibration files</i>: This is a
specific parameter for the Level Calibrator sub-module. It specifies the name
of the folder containing the K and V values used for calibration. For further
information on how to use this parameter, refer to the <a
href="#_Workflow_6:_Single-Cell">Sample Workflow section of Single-Cell
Proteomics</a>.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=638 height=146 id="Picture 11" src="index_files/image020.png"></p>

<p class=MsoCaption><a name="_Ref103114398">Figure </a>11. A task-table
displaying advanced parameters for the LEVEL CALIBRATOR module (in the Level
Calibrator sub-module).</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848005"></a><a name="_INTEGRATE"></a><span
style='font-family:"Calibri",sans-serif'>INTEGRATE</span></h3>

<p class=MsoNormal>The INTEGRATE module performs statistical calculations based
on the WSPP model by iteratively applying the generic integration algorithm
(GIA) [3] on calibrated data files (<i><span
style='color:#BF8F00'>Figure 12</span></i>). </p>

<p class=MsoNormal>Integrations are carried out from lower level data to higher
level data (e.g. from the peptide level to the protein level and from the protein
level to the gene level).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=423 height=414 id="Picture 58" src="index_files/image021.png"></p>

<p class=MsoCaption><a name="_Ref103096863">Figure </a>12. Schematic
representation of the INTEGRATE module. The integration is carried out from any
lower level to any higher level using the programs “SanXoT” and “SanXoTSieve”
and the generic integration algorithm (GIA).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>More in detail, the INTEGRATE module needs two TSV files as
inputs:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>A
data file containing three data columns: identifier (a text string that is used
to unambiguously identify the low level elements), quantitative value (log<sub>2</sub>-ratio
of the two measurements to be compared) and statistical weight (a parameter
that measures the accuracy of the quantitative value).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>A
relation table, which links the lower level identifiers to those in the higher
level. This file contains two columns: higher level identifiers on the left and
lower level identifiers on the right.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=582 height=524 id="Picture 41" src="index_files/image022.png"></p>

<p class=MsoCaption><a name="_Ref103114442">Figure </a>13. The INTEGRATE module
flowchart. A first integration is done with “SanXoT” that calculates the
variance; then “SanXoTSieve” removes outliers tagging them in a new relation
table; finally, a second integration is done with “SanXoT” using the variance
calculated.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For every integration, the SanXoT program calculates the general
variance using a robust iterative method. Then SanXoTSieve is used to tag
outlier elements [2] by assessing the probability that a lower level element be
a significant outlier of the standardized (i.e. N(0,1)) log<sub>2</sub>-ratio
distribution. The most extreme outliers are thus removed sequentially and the
integration repeated until all outliers below a user-defined false discovery
rate (FDR) threshold have been removed. Finally, a second integration is carried
out by SanXoT using the variance calculated in the first integration and
discarding the outliers tagged in the new relation table (<i><span style='color:#BF8F00'>Figure 13</span></i>).</p>

<p class=MsoNormal>The output data files generated by INTEGRATE contain the
quantitative data for the higher level and can be used as inputs to other
modules; in addition, each integration generates several additional files which
contain information about the integration. For further details see SanXoT documentation
[1].</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The parameters to be provided in the INTEGRATE module task
table are (<i><span
style='color:#BF8F00'>Figure 14</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the names of the folder(s)
where the lower level data file is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level data
file to be used. This file contains three data columns: identifier,
quantitative value and statistical weight.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the name of the higher level to which
the lower level elements will be integrated.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=282 height=159 id="Picture 6" src="index_files/image023.png"></p>

<p class=MsoCaption><a name="_Ref103114454">Figure </a>14. A sample task-table
in the INTEGRATE module.</p>

<p class=MsoNormal>&nbsp;</p>

<h4><a name="_Advanced_parameters"></a>Advanced parameters</h4>

<p class=MsoNormal>The INTEGRATE module accepts the following additional parameters
(<i><span style='color:#BF8F00'>Figure 15</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> is the name of the folder where the level
data and statistics are saved.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the
lower_level2higher_level Relation Table will be integrated. Logical operators
can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><i>Tag</i> is a parameter to distinguish
groups to perform the integration. For instance, if the user specifies “marked”
the elements containing the label “marked” in the third column of the “lower_level2higher_level”
Relation Table will be included in (or discarded from) the integration.</p>

<p class=MsoListParagraphCxSpMiddle>The tag can be used by inclusion, such as &quot;mod&quot;
or by exclusion, putting first the &quot;!&quot; symbol, such as &quot;!mod&quot;. 
Tags should be included in a third column of the relations file.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>Different tags can be combined using
logical operators &quot;and&quot; (&amp;), &quot;or&quot; (|), and
&quot;not&quot; (!), and parentheses. Some examples:</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>!out&amp;mod</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>!out&amp;(dig0|dig1)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>(!dig0&amp;!dig1)|mod1</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>mod1|mod2|mod3</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b><i>Warning</i></b>: Unless specified
otherwise by the user, by default iSanXoT eliminates outliers from the lower
level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle>iSanXoT automatically adds the tag “out” in
the third column of the relation table to label outliers, so that they are not
integrated. It is not thus recommended to use this tag for other purposes.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b><i>Note</i></b> that although the
discarded elements will not be included in calculations, the parameter Z will
be calculated and tabulated in the corresponding output (outStats) file.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>For further details see SanXoT wiki</p>

<p class=MsoListParagraphCxSpMiddle>(<a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a>).</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>FDR</i> is an FDR threshold other than the default value (0.01,
i.e. 1%) for outlier removal. If “0” is specified as the FDR value, then no outliers
will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Var(x)</i> sets a fixed value for the variance. The default
value (blank) means that the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first <i>Lower level-to-Higher level</i>
integration.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=136 src="index_files/image024.png"></p>

<p class=MsoCaption><a name="_Ref103114466">Figure </a>15. A sample task-table
displaying advanced parameters for the INTEGRATE module.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848006"></a><a name="_NORCOMBINE"></a><span
style='font-family:"Calibri",sans-serif'>NORCOMBINE</span></h3>

<p class=MsoNormal>The NORCOMBINE module combines technical or biological
replicates (<i><span
style='color:#BF8F00'>Figure 16</span></i>). For example, NORCOMBINE can be
used to merge the protein level data from 4 individual patients and 4 individual
controls into a patient- and a control level protein dataset, respectively.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=451 height=388 id="Picture 114" src="index_files/image025.png"></p>

<p class=MsoCaption><a name="_Ref103114484">Figure </a>16. Schematic
representation of the NORCOMBINE module used to combine technical or biological
replicates.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Experiment merging relies on the “Cardenio” program from the
SanXoT software package [1],
which is used to generate merged data files and relation tables that are later
integrated to the grouped level using “SanXoT” and “SanXoTSieve” (<i><span style='color:#BF8F00'>Figure 16</span></i>).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>NORCOMBINE requires the user to specify which <i>lowerNorm</i>
files contain the necessary data for the samples to be combined. These <i>lowerNorm</i>
files, previously generated by the INTEGRATE module, display the lower level
identifiers on the left, followed by the corresponding centred log<sub>2</sub>-ratio
values (i.e. the values obtained after subtracting the high level value) in the
second column, and either the integration statistical weight (in the case of <i>lowerNormV</i>)
or the variance (for <i>lowerNormW).</i> The SanXoT program “Cardenio” [1]
is then used to generate merged data files and relation tables that are later
integrated to the grouped level using “SanXoT” and “SanXoTSieve” (<i><span style='color:#BF8F00'>Figure 17</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=452 height=522 id="Picture 115" src="index_files/image026.png"></p>

<p class=MsoCaption><a name="_Ref103114585">Figure </a>17. The NORCOMBINE
module flowchart.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The default NORCOMBINE module task table shows the following
fields (<i><span style='color:#BF8F00'>Figure 18</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folders</i> indicates the names of the folder(s) containing
the lower level data (samples) to be combined.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level</i> indicates the type of elements to be combined (e.g.
peptides or proteins). </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Norm</i> specifies the normalization scheme to be used in the
integrations.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>lowerNorm</i> specifies the type of <i>lowerNorm</i> file (see
above) to be used.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> is the name of the folder where the grouped
level data and statistics are saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=467 height=89 src="index_files/image027.png"></p>

<p class=MsoCaption><a name="_Ref103114601">Figure </a>18. A sample task-table
in the NORCOMBINE module. In this case, the asterisk wildcard has been used to
select multiple sample folders.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The NORCOMBINE module accepts the following additional parameters
(<i><span style='color:#BF8F00'>Figure 19</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the
lower_level2higher_level Relation table will be integrated. Logical operators
can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>FDR</i> is an FDR threshold other than the default value
(0.01, i.e. 1%) for outlier removal. If “0” is specified as the FDR value, then
no outliers will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Var(x) sets a fixed value for the variance. The default value
(blank) means that the variance will be iteratively calculated based on the
Levenberg-Marquardt algorithm.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=72 src="index_files/image028.png"></p>

<p class=MsoCaption><a name="_Ref103114615">Figure </a>19. A sample task table
with advanced parameters in the NORCOMBINE module.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848007"></a><a name="_RATIOS"></a><span style='font-family:
"Calibri",sans-serif'>RATIOS</span></h3>

<p class=MsoNormal>This module prepares the data file and relation table
required as a first step in the calculation of a ratio defined by the user (e.g
KO vs WT). For that, the new log<sub>2</sub>-ratio is calculated as the
difference between numerator and denominator values, whereas the corresponding
statistical weight is assessed according to the method indicated by the user in
the V Method filed of the RATIOS task table (<i><span
style='color:#BF8F00'>Figure 20</span></i> and <i><span
style='color:#BF8F00'>Figure 21</span></i>): </p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>max</i> uses the maximum value between the numerator and
denominator statistical weight value. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>form</i> uses the value resulting from 1/(1/Vn + 1/Vd), where
Vn and Vd are the statistical weight value for the numerator and the
denominator, respectively.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>avg</i> uses the average value between the numerator and
denominator statistical weight value.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=581
height=199 id="Picture 17" src="index_files/image029.png"></p>

<p class=MsoCaption><a name="_Ref103114679">Figure </a>20. The RATIOS module flowchart.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The RATIOS module task table displays the following parameter
fields (<i><span style='color:#BF8F00'>Figure 21</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator column</i> specifies the name of the folder
containing the quantitative value to be used as a numerator for the new log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator column(s)</i> specifies the name of the
folder(s) containing the quantitative values to be used as a denominator for
the new log<sub>2</sub>-ratio calculation. The sample folders have to be
separated by comma.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level </i>designates the level (i.e. peptide, protein, gene or
category) at which the ratio is to be calculated.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates the name of the folder
where the resulting log<sub>2</sub>-ratio and statistical weight values will be
saved (e.g. KO_vs_WT).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=574 height=65 src="index_files/image030.png"></p>

<p class=MsoCaption><a name="_Ref103114687">Figure </a>21. A sample task-table in
the RATIOS module.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848008"></a><a name="_SBT"></a><span style='font-family:
"Calibri",sans-serif'>SBT</span></h3>

<p class=MsoNormal>This module is based on the Systems Biology Triangle (SBT)
algorithm [3], which performs an integration between the lower and the higher
levels using the variance previously obtained in an integration between the
lower and an intermediate level (<i><span
style='color:#BF8F00'>Figure 22</span></i>). Usually, the SBT module is applied
to carry out the protein-to-grand mean integration using the variance
associated with the protein-to-category integration.   </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=436 height=435 id="Picture 131" src="index_files/image031.png"></p>

<p class=MsoCaption><a name="_Ref103093737"></a><a name="_Ref103093681"><b>Figure
</b></a><b>22</b><b>. <a name="_Ref103093692">Schematic representation of the SBT
module.</a></b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters to be entered in the SBT module task
table are (<i><span
style='color:#BF8F00'>Figure 23</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name of the folder(s) where
the lower level data are located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level (e.g. protein).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Intermediate level</i> indicates the name of the intermediate level
(e.g. category).</p>

<p class=MsoNormal>By default, the higher level is the grand mean of the lower
level elements.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=292 height=66 src="index_files/image032.png"></p>

<p class=MsoCaption><a name="_Ref103094105">Figure </a>23. A sample task table in
the SBT module.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts the following additional parameters (<i><span style='color:#BF8F00'>Figure 24</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates an alternative folder to
store the resulting log<sub>2</sub>-ratio and statistical weight values other
than “Sample folder(s)”.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower-Higher level</i> and <i>Int(ermediate)-Higher level </i>specify
an alternative higher level other than the grand mean of the lower level elements.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate)</i> Tag and <i>Int(ermediate)-to-Hig(her)</i>
are the text label that indicates which elements from the lower level are
integrated into the intermediate level, and the intermediate level are
integrated into higher level. The tags must be specified in the third column of
the corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label “marked” is used as a Tag, only the lower level elements containing
the label marked in the third column of the lower_level2intermediate_level and
intermediate_level2higher_level Relation Tables will be integrated. Logical
operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate) FDR </i>and<i> Int(ermediate)-to-Hig(her)
FDR</i> determine an FDR threshold other than the default value (0.01, i.e. 1%)
for outlier removal in the lower level-to-intermediate level and intermediate
level-to-higher level integration, respectively. If “0” is specified as the FDR
value, then no outliers will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate) Var(x)</i> and <i>Int(ermediate)-to-Hig(her)
Var(x)</i> indicate the variance to be used in the lower level-to-intermediate
level and intermediate level-to-higher level integration, respectively, as an
alternative to the variance calculated in the lower level-to-intermediate level
integration.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=554 height=168 id="Picture 10" src="index_files/image033.png"></p>

<p class=MsoCaption><a name="_Ref103094086">Figure </a>24. A sample task table
with advanced parameters in the SBT module.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Toc199848009"></a><a
name="_Compound_modules"></a>Compound modules</h2>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The Compound modules perform a sequence of consecutive
integrations based on the WSPP statistical model [2]
and the SBT algorithm [3]. In addition, each module creates the initial level
and calibrate this initial level. The WSPP-SBT and WSPPG-SBT modules create and
calibrate the “scan” level; and the WPP-SBT and WPPG-SBT create and calibrate
the “peptide” level.</p>

<h3><a name="_Toc199848010"></a><a name="_WSPP-SBT_1"></a><span
style='font-family:"Calibri",sans-serif'>WSPP-SBT</span></h3>

<p class=MsoNormal>The WSPP-SBT module performs the following integrations:
scan-to-peptide, peptide-to-protein, protein-to-category, protein-to-proteinall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the protein-to-category integration, which
is applied to the protein-to-proteinall integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WSPP-SBT module are
(<i><span style='color:#BF8F00'>Figure 25</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Batch</i> is the column header that designates which elements
from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=542 height=227 src="index_files/image034.png"></p>

<p class=MsoCaption><a name="_Ref103096252">Figure </a>25. A sample task table
in the WSPP-SBT, WSPPG-SBT, WPP-SBT, and WPPG-SBT modules.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WSPP-SBT module accepts the following advanced
parameters (<i><span
style='color:#BF8F00'>Figure 26</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text
label that indicates which elements from the lower level (“p” and “c”) are
integrated into the higher level (“q” and “a”). The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.<span
class=MsoCommentReference><span style='font-size:8.0pt;line-height:107%'> </span></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p FDR</i>, <i>p&gt;q FDR</i>, <i>q&gt;c FDR</i> establish
an FDR threshold other than 0.01 (1%) for outlier removal in the integrations
scan-to-peptide, peptide-to-protein, and protein-to-category, respectively. If
FDR = 0 is selected, then the outliers are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p Var(x)</i>, <i>p&gt;q Var(x)</i>, <i>q&gt;c Var(x)</i>
set a fixed value for the variance in the integrations scan-to-peptide, peptide-to-protein,
and protein-to-category, respectively. By default, the variance will be
iteratively calculated based on the Levenberg-Marquardt algorithm in the first
lower level-to-higher level integration (<i><span
style='color:#BF8F00'>Figure 12</span></i>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=111 id="Picture 23" src="index_files/image035.png"></p>

<p class=MsoCaption><a name="_Ref103096290">Figure </a>26. Task-Table with
advanced parameters in the WSPP-SBT module.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848011"></a><a name="_WSPPG-SBT_1"></a><span
style='font-family:"Calibri",sans-serif'>WSPPG-SBT</span></h3>

<p class=MsoNormal>The WSPPG-SBT module performs the following integrations:
scan-to-peptide, peptide-to-protein, protein-to-gene, gene-to-category, gene-to-geneall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the gen-to-category integration, which is
applied to the gen-to-geneall integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WSPPG-SBT module are
(<i><span style='color:#BF8F00'>Figure 25</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Batch</i> is the column header that designates which elements
from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WSPPG-SBT module accepts the following advanced
parameters (<i><span
style='color:#BF8F00'>Figure 27</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>q&gt;g Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a
Tag</i> are the text label that indicates which elements from the lower level (“p”,
“q” and “c”) are integrated into the higher level (“q”, “g” and “a”). The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label “marked” is used as
a Tag, only the lower level elements containing the label marked in the third
column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p FDR</i>, <i>p&gt;q FDR</i>, <i>q&gt;g FDR</i>, <i>g&gt;c
FDR</i> establish an FDR threshold other than 0.01 (1%) for outlier removal in
the following integrations: scan-to-peptide, peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. If FDR = 0 is selected, then the outliers
are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p Var(x)</i>, <i>p&gt;q Var(x)</i>, <i>q&gt;g Var(x)</i>,
<i>g&gt;c Var(x)</i> set a fixed value for the variance in the integrations:
scan-to-peptide, peptide-to-protein, protein-to-gene, and gene-to-category,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (<i><span
style='color:#BF8F00'>Figure 12</span></i>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=66 id="Picture 24" src="index_files/image036.png"></p>

<p class=MsoCaption><a name="_Ref103096474">Figure </a>27. Task-table with
advanced parameters in the WSPPG-SBT module.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848012"></a><a name="_WPP-SBT_1"></a><span
style='font-family:"Calibri",sans-serif'>WPP-SBT</span></h3>

<p class=MsoNormal>The WPP-SBT module performs the integrations peptide-to-protein,
protein-to-category, protein-to-proteinall and category-to-categoryall. In
addition, the SBT algorithm is used to calculate the variance associated to the
protein-to-category integration, which is applied to the protein-to-proteinall
integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WPP-SBT module are (<i><span style='color:#BF8F00'>Figure 25</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Batch</i> is the column header that designates which elements
from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WPP-SBT module accepts the following advanced parameters
(<i><span style='color:#BF8F00'>Figure 28</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text
label that indicates which elements from the lower level (“p” and “c”) are
integrated into the higher level (“q” and “a”). The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation Table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q FDR</i>, <i>q&gt;c FDR</i> establish an FDR threshold
other than 0.01 (1%) for outlier removal in the peptide-to-protein and protein-to-category
integrations, respectively. If FDR = 0 is selected, then the outliers are not
discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Var(x)</i>, <i>q&gt;c Var(x)</i> set a fixed value for
the variance in the peptide-to-protein and protein-to-category integrations,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (<i><span
style='color:#BF8F00'>Figure 12</span></i>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=494 height=90 id="Picture 29" src="index_files/image037.png"></p>

<p class=MsoCaption><a name="_Ref103096698">Figure </a>28. Task-table with
advanced parameters in the WPP-SBT module.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848013"></a><a name="_WPPG-SBT_1"></a><span
style='font-family:"Calibri",sans-serif'>WPPG-SBT</span></h3>

<p class=MsoNormal>The WPPG-SBT module performs the integrations peptide-to-protein,
protein-to-gene, gene-to-category, gene-to-geneall, and category-to-categoryall.
In addition, the SBT algorithm is used to calculate the variance associated to
the gene-to-category integration, which is applied to the gene-to-geneall
integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WPPG-SBT module are
(<i><span style='color:#BF8F00'>Figure 25</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Batch</i> is the column header that designates which elements
from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WPPG-SBT module accepts the following advanced
parameters (<i><span
style='color:#BF8F00'>Figure 29</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>q&gt;g Tag, p&gt;a Tag, q&gt;a Tag,</i> <i>c&gt;a
Tag</i> are the text label that indicates which elements from the lower level (“p”,
“q” and “c”) are integrated into the higher level (“q”, “g” and “a”). The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label “marked” is used as
a Tag, only the lower level elements containing the label marked in the third
column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q FDR, q&gt;g FDR</i>, <i>g&gt;c FDR</i> establish an FDR
threshold other than 0.01 (1%) for outlier removal in the integrations peptide-to-protein,
protein-to-gene, and gene-to-category, respectively. If FDR = 0 is selected,
then the outliers are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Var(x)</i>, <i>q&gt;g Var(x), g&gt;c Var(x)</i> set a
fixed value for the variance in the integrations peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. By default, the variance will be
iteratively calculated based on the Levenberg-Marquardt algorithm in the first
lower level-to-higher level integration (<i><span
style='color:#BF8F00'>Figure 12</span></i>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=75 id="Picture 47" src="index_files/image038.png"></p>

<p class=MsoCaption><a name="_Ref103097509">Figure </a>29. Task-table with
advanced parameters in the WPPG-SBT module.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Toc199848014"></a><a
name="_Report_modules"></a><a name="_Reports_modules"></a>Reports modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848015"></a><a name="_REPORT"></a><span style='font-family:
"Calibri",sans-serif'>REPORT</span></h3>

<p class=MsoNormal>The REPORT module allows the collection of the statistical variables
(n, tags, Xinf, Vinf, Xsup, Vsup, Z, FDR, X’inf and Winf) from the different integrations
performed into result tables. </p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters showing in the REPORT task table are
(<i><span style='color:#BF8F00'>Figure 30</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name(s) of the folder(s)
where the values of the statistical variables to be retrieved are located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the starting level (i.e. peptide,
protein, or category) for the integration whose statistical variables are to be
reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the ending level for the
integration whose statistical variables are to be reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Reported vars</i> specifies which statistical variables will
be reported. The available variables are n, tags, Xinf, Vinf, Xsup, Vsup, Z,
FDR, X’inf and Winf.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output report</i> is the report filename (without extension).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=535 height=318 src="index_files/image039.png"></p>

<p class=MsoCaption><a name="_Ref103115700">Figure </a>30. A sample task table
in the REPORT module.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For instance, the first row of the task table shown in <i><span style='color:#BF8F00'>Figure 30</span></i> prompts the REPORT module to read the
variable “n” from the <i>scan2peptide_outStats.tsv</i> file that contains the statistical
outcome from the scan-to-peptide integration (the asterisk wildcard character
in <i>Sample folder(s)</i> causes REPORT to retrieve the “n” variable from
every sample). These “n” values are written to a report file named “Nscan_pep” that
is stored in the project “reports” folder.</p>

<p class=MsoNormal>The second row instructs the module to read the variables “Xinf”,
“Z” and “FDR” from the statistical outcome of the peptide-to-protein
integration (once again for every sample). These values are written to a report
file named “Nscan_Normpep_prot_XZ”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The REPORT module accepts the following advanced parameters
(<i><span style='color:#BF8F00'>Figure 31</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level names to show</i> allows the user to restrict the
elements to be written to the <i>Output report</i> to those from the <i>Lower
level</i> or the <i>Higher level</i>. Both levels are used by default.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Merge with report</i> designates the file whose <i>Reported
vars</i> will be incorporated into the <i>Output report</i> after intersection
with the latter file.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Add columns from relation table</i> appends <i>Lower level</i>
elements, extracted from the relation table designated, to the <i>Output report</i>.
It is possible to indicate multiple relation tables separated by a comma.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Show outliers</i>: checkbox parameter that shows or hides the
outliers. By default, outliers are not displayed (option unchecked).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Filter</i> allows to filter the data to be transferred to the <i>Output
report</i> based on the <i>Reported vars</i> (n, Z, FDR, etc.). For more details,
see <a href="#_Filter_in_REPORT">Filter</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=640 height=264 id="Picture 4" src="index_files/image040.png"></p>

<p class=MsoCaption><a name="_Ref103116151">Figure </a>31. A sample task-table
with advanced parameters in the REPORT module.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The reports indicated under <i>Output report</i> and <i>Merge
with report</i> are merged according to the column header that they share.
Thus, the REPORT task table shown in <i><span
style='color:#BF8F00'>Figure 32</span></i> will cause the module to incorporate
the number of scans per peptide, displayed in the report “Nscan_pep”, to the
report “Nscan_Normpep_prot_XZ”, as these two reports share the lower level
elements showing under the “peptide” header.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=600 height=156 src="index_files/image041.png"></p>

<p class=MsoCaption><a name="_Ref103116369">Figure </a>32. Report merging in
the REPORT module. The first task table row creates a report file (“Nscan_pep”)
with the (n)umber of scans per peptide. The second row creates a report file
called “Nscan_Normpep_prot_XZ” that contains, apart from the variables “Xinf”,
“Z”, and “FDR” coming from the peptide-to-protein integration, the (n)umber of scans
per peptide previously stored in the “Nscan_pep” report, as these two reports
share the lower level elements showing under the “peptide” header.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>In addition, it is possible to incorporate additional data
from one or more relation tables into the reports (<i><span
style='color:#BF8F00'>Figure 33</span></i>). When a given Relation Table is
indicated under <i>Add columns from relation table</i>, the REPORT module will
first attempt to incorporate to the <i>Output report</i> (“Npep_Quanprot_cat”
in the example) the elements related to the <i>Lower level</i> (“protein” in
this case) elements in the relation table. If the <i>Lower level</i> elements
are missing in the relation table, then REPORT will try to incorporate the
elements related to the Higher level (“category” in this case) instead. If
neither the <i>Lower level</i> nor the <i>Higher level</i> can be found in the
relation table, then no action is performed. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=573 height=101 id="Picture 27" src="index_files/image042.png"></p>

<p class=MsoCaption><a name="_Ref103116417">Figure </a>33. Adding data from
relation tables into the report files. The relation table “protein2gene”
contains a column with protein identifiers under the “protein” header and
another column with the corresponding gene name under the “gene” header,
whereas the relation table “protein2description” contains, apart from the
“protein” elements, a column with the corresponding protein description. The
first task table row will prompt REPORT to incorporate the gene names and
protein descriptions contained in the relation tables to the report file “Npep_Quanprot_cat”
report file, as “protein” is the <span style='font-style:normal'>Lower level</span>
they all three shares.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Finally, the report data can be filtered performing logical
operations with the <i>Reported vars</i> in the <i>Filter</i> field. For
instance, in the report task table displayed in <i><span
style='color:#BF8F00'>Figure 34</span></i>:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &lt;= 100, filters out from the report the
variables Z and FDR for the category-to-categoryall integration when the
(n)umber of proteins per category is greater than 100.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &gt;= 5 &amp; n_protein2category &lt;= 100
retrieves the variables Z and FDR for the category-to-categoryall integration
when the (n)umber of proteins per category is in the [5, 100] range.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>KO_vs_WT@FDR_category2categoryall &lt; 0.05, retrieves the variables
Z and FDR for the category-to-categoryall integration provided that the FDR corresponding
to the “KO_vs_WT” samples is less than 0.05.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=583 height=98 id="Picture 32" src="index_files/image043.png"></p>

<p class=MsoCaption><a name="_Ref103116468">Figure </a>34. Filtering the report
data.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The compound variables shown in the <i>Filter</i> field on <i><span style='color:#BF8F00'>Figure 34</span></i> follow the structure <i>Reported var</i>_<i>integration</i>,
like “n_protein2category”. Such filter applies to all samples. </p>

<p class=MsoNormal>However, the filter “KO_vs_WT@FDR_category2categoryall” is
applied based on the variable “FDR” from the category-to-categoryall
integration, but only to the “KO_vs_WT” sample. Moreover, the filter “WT1,WT2@FDR_category2categoryall”
is applied to the “WT1” and “WT2” samples.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848016"></a><a name="_SANSON"></a><span style='font-family:
"Calibri",sans-serif'>SANSON</span></h3>

<p class=MsoNormal>The SANSON module generates a similarity graph showing the
relationship between functional categories based on their protein components. </p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters to be provided for this module are (<i><span style='color:#BF8F00'>Figure 35</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the names of the folder(s)
where the lower level data file is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level elements
(“protein” in this case) to be used.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the name of the higher level (“category”
in this case) to which the lower level elements will be integrated.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder </i>designates a folder other than <i>Sample
folder(s)</i> where the results will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=383 height=65 src="index_files/image044.png"></p>

<p class=MsoCaption><a name="_Ref103116548">Figure </a>35. A sample task table in
the SANSON module.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The SANSON module accepts the following additional parameters
(<i><span style='color:#BF8F00'>Figure 36</span></i>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower norm</i> specifies the normalization scheme to be used
with the lower level elements. The default value is included the normalization
of lower level to all.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher norm</i> specifies the normalization scheme to be used
with the higher level elements. The default value is included the normalization
of higher level to all.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the
lower_level2higher_level Relation table will be integrated. Logical operators
can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.<i> </i></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Filter</i> allows to filter the data based on the FDR and
number of proteins. For more details, see <a href="#_Filter_in_SANSON">Filter</a>
in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=57 src="index_files/image045.png"></p>

<p class=MsoCaption><a name="_Ref103116564">Figure </a>36. A sample task-table
with advanced parameters in the SANSON module.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><a name="_Complete_Modules"></a><a name="_WSPP-SBT"></a><a
name="_WSPPG-SBT"></a><a name="_WPP-SBT"></a><a name="_WPPG-SBT"></a>&nbsp;</p>

<h2><a name="_Toc199848017"></a><a name="_Special_parameters"></a>Special
parameters</h2>

<p class=MsoNormal><a name="_Keywords_for_the"></a>&nbsp;</p>

<h3><a name="_Toc199848018"></a><a name="_Multiple_samples"></a>Multiple
samples</h3>

<p class=MsoNormal>The “Sample folder(s)” field of the different module task
tables admits multiple samples. For instance, let's consider the samples
created with the following LEVEL CREATOR task table:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=174 id="Picture 64" src="index_files/image046.png"></p>

<p class=MsoNormal>We can include multiple samples separated by a comma, for
example, in the INTEGRATE module task table: </p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=227 id="Picture 7" src="index_files/image047.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848019"></a><a name="_Asterisk_is_our"></a>Asterisk is our
jack of all trades</h3>

<p class=MsoNormal>The module task tables admit the usage of the asterisk
symbol as a wildcard character. Let’s once more consider the samples created
with the following LEVEL CREATOR task table:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=174 id="Picture 59" src="index_files/image046.png"></p>

<p class=MsoNormal>Each row calculates a ratio that is saved to the corresponding
<i>Output Sample folder</i>. Thus, the ratio of 113 to the mean of 113, 114, 115,
and 116 is saved to the “Jurkat_WT/WT_1” folder; the 114 to the mean of 113,
114, 115 ratios is saved to “Jurkat_WT/WT_2”, and so on. One way to create the
task table of the INTEGRATE module could be the following, where each row
represents an integration for a given sample:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=225 height=308 id="Picture 62" src="index_files/image048.png"></p>

<p class=MsoNormal>However, this task table can be simplified by applying the
asterisk wildcard. For instance, the task-table below allows to indicate multiple
sample folders, namely every folder starting with “Jurkat_WT/” or “Jurkat_KO/”.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=275 height=274 id="Picture 60" src="index_files/image049.png"></p>

<p class=MsoNormal>We can reduce this expression even more using just an asterisk:
the first row of the following task table performs the integrations <i>peptide-to-protein</i>,
<i>protein-to-category</i>, <i>peptide-to-peptideall</i>, <i>protein-to-proteinall</i>,
and <i>category-to-categoryall</i> in every sample folder defined with LEVEL
CREATOR.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=284 height=159 id="Picture 63" src="index_files/image050.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848020"></a><a name="_Multiple_samples_in"></a>Multiple
samples in the inputs and outputs</h3>

<p class=MsoNormal>In the cases we have multiple input samples separated by comma,
but we want to save the results in another output sample folder, we indicate
them in the same way giving the output folders separated by comma. It is
required to provide the same number of folders:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=236 id="Picture 13" src="index_files/image051.png"></p>

<p class=MsoNormal>In the same way happens with the asterisk character (jack of
all trades). In the “Output Sample folder(s)” we can add a suffix in the input
samples. The following task-table illustrate that the output sample folders
would be contain the “_New” suffix:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=487 height=158 id="Picture 30" src="index_files/image052.png"></p>

<p class=MsoNormal>In addition, we can rename the subfolder adding the new name
in the “Output Sample folder(s)” or add a new subfolder:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=399 height=181 id="Picture 33" src="index_files/image053.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848021"></a><a name="_More_params"></a>More params</h3>

<p class=MsoNormal>Some modules accept a column parameter in the Task-Table
called “More params”. This column allows you to provide advanced parameters for
the SanXoT programs [1].
The program descriptions are in the following wiki link:</p>

<p class=MsoNormal><a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The iSanXoT modules are composed by several programs of
SanXoT. For this reason, the “More params” of a module accepts the advanced
parameters of composed programs indicated by a name.</p>

<p class=MsoNormal>For example, </p>

<p class=MsoNormal style='margin-left:35.4pt'>INTEGRATE:</p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'>&quot;sanxot1&quot;:
&quot; -m 300 -g &quot;, &quot;sanxot2&quot;: &quot;-s --sweepdecimals=2.5&quot;</p>

<p class=MsoNormal>In the above example the first “sanxot” program that compose
the INTEGRATE module, receives the “-m 300 -g” as parameter, and the second
“sanxot” receives also the parameter &quot;-s --sweepdecimals=2.5&quot;.</p>

<p class=MsoNormal style='margin-left:35.4pt'>WSPP-SBT:</p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'>&quot;p2q_sanxot2&quot;:
&quot; -m 100 -s &quot;, &quot;q2a_sanxot1&quot;: &quot;-m 100&quot;</p>

<p class=MsoNormal>In this example, the WSPP-SBT module has multiple
integrations: scan-to-peptide, peptide-to-protein, etc. For more information,
see the WSPP-SBT section. Thus, the second “sanxot” program of peptide-to-protein
(q) integration will receive the parameter “-m 100 -s”, and then, the first
“sanxot” of protein (q)-to-proteinall (a) integration will receive the “-m
100”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>The program names for each Module </h4>

<p class=MsoNormal>INTEGRATE: sanxot1, sanxotsieve, sanxot2</p>

<p class=MsoNormal>NORCOMBINE: create_exp_tags, cardenio, sanxot1, sanxotsieve,
sanxot2 </p>

<p class=MsoNormal>SBT: l2i_sanxot1, l2i_sanxotsieve, l2i_sanxot2, i2h_sanxot1,
i2h_sanxotsieve, i2h_sanxot2, l2h_sanxot1, l2h_sanxotsieve, l2h_sanxot2</p>

<p class=MsoNormal>WSPP_SBT: level_creator, klibrate, s2p_sanxot1,
s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve, p2q_sanxot2,
q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1,
c2a_sanxotsieve, c2a_sanxot2</p>

<p class=MsoNormal>WSPPG_SBT: level_creator, klibrate, s2p_sanxot1,
s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve, p2q_sanxot2,
p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1, q2g_sanxotsieve,
q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2,
g2a_sanxot1, g2a_sanxotsieve, g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve,
c2a_sanxot2</p>

<p class=MsoNormal>WPP_SBT: level_creator, klibrate, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve,
c2a_sanxot2</p>

<p class=MsoNormal>WPPG_SBT: level_creator, klibrate, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1, q2g_sanxotsieve,
q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, g2a_sanxot1, g2a_sanxotsieve,
g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve, c2a_sanxot2</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848022"></a><a name="_Filter_param_(in"></a><a
name="_Filter_(for_REPORT"></a><a name="_Filter_in_REPORT"></a>Filter in REPORT
module</h3>

<p class=MsoNormal>The REPORT module module accepts a Filter parameter. This
parameter filters the data based on some variables depending on the module.</p>

<p class=MsoNormal>In the case of REPORT module, the filtered variables are the
Reported vars: n, Z, FDR, etc. For instance:</p>

<p class=MsoNormal><i>(FDR_category2category &lt; 0.05) &amp;
(n_protein2category &gt;=5) &amp; (n_protein2category &lt;= 100)</i></p>

<p class=MsoNormal><i>(FDR_category2category &lt; 0.05) &amp; (Z_protein2proteinall
&gt;= 2 | Z_protein2proteinall &lt;= -2)</i></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc199848023"></a><a name="_Filter_in_SANSON"></a>Filter in
SANSON module</h3>

<p class=MsoNormal>For the SANSON module, the filtered variables are FDR and
the related number (n_rel). For example:</p>

<p class=MsoNormal style='text-indent:35.4pt'>([FDR] &lt; 0.05) &amp; ([n_rel]
&gt;= 10) &amp; ([n_rel] &lt;= 100)</p>

<p class=MsoNormal>Different variables can be combined using the comparisons:
&gt;=, &lt;=, !=, &lt;&gt;, ==, &gt;, &lt;; and using logical operators “and”
(&amp;), “or” (|), and “not” (!). </p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk124849833"></a><a name="_Hlk154051585"></a><a
name="_Toc152521021"></a><a name="_Toc199848024"></a><a
name="_Sample_Workflows_with"></a><a name="_Adaptors"></a><a
name="_Input_Adaptor"></a>Sample Workflows with Application to Case Studies</h1>

</div>

<p class=MsoNormal>Below, we provide detailed descriptions of four sample
workflows that illustrate the capability of iSanXoT to statistically ascertain
changes in protein or peptide abundance across various biological contexts. It
is important to note that these workflows can be easily reused to process new
data (refer to the next section).</p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Hlk122708883"></a><a
name="_Toc152521022"></a><a name="_Toc199848025"></a><a
name="_Workflow_1:_One-step"></a><b><span style='font-size:20.0pt;line-height:
107%;font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow 1: One-step
quantification in a labeled experiment</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521023"></a><a
name="_Toc199848026"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>The identification and quantification data from
García-Marqués <i>et al</i>. [3] were used to illustrate this workflow. This
study characterizes the molecular alterations that take place over time when
vascular smooth muscle cells (VSMCs) are treated with angiotensin-II (AngII)
for 0, 2, 4, 6, 8, and 10 hours. Quantitative proteomics were performed using
isobaric iTRAQ 8-plex labeling. Workflow 1 analyzes a) changes in protein
abundance and b) alterations in functional categories produced by the
coordinated behavior of proteins at each of the specified times, in relation to
time 0. This is achieved in only one step using the compound module WSPP-SBT,
which automatically performs all the required tasks.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521024"></a><a
name="_Toc199848027"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and required input files for executing
this workflow can be downloaded from</p>

<p class=MsoNormal><a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP-SBT.zip">https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP-SBT.zip</a></p>

<p class=MsoNormal>Refer to the <i>Importing a Workflow Template</i> section
below for detailed instructions.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521025"></a><a
name="_Toc199848028"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>Workflow 1 requires the RELS CREATOR module, the WSPP-SBT
compound module, and the REPORT basic module (<i><span style='line-height:107%;
color:#BF8F00'>Figure 37</span></i>). The relation tables necessary for
performing the integrations are created by the RELS CREATOR module (<i><span style='line-height:107%;color:#BF8F00'>Figure 37</span></i><span
style='color:#BF8F00'>A</span>) from a table provided by the user. The WSPP-SBT
module performs a sequence of consecutive integrations based on the WSPP
statistical model [2] and the SBT algorithm [3] (<i><span
style='line-height:107%;color:#BF8F00'>Figure 37</span></i><span
style='color:#BF8F00'>B</span>). Finally, the REPORT module organizes the data
into tables containing the required information.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=655 height=288 id="Picture 20" src="index_files/image054.jpg"
alt="A diagram of a level calibration&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771130"></a><a name="_Ref127961610"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>37</span></i><i><span style='color:#BF8F00'>. Scheme of
workflow 1 (one-step quantification in a labeled experiment) showing module
components: RELS CREATOR (A) and WSPP-SBT and REPORT (B) </span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The WSPP-SBT module requires the
user to define the meaning of relative abundances, which iSanXoT consistently
expresses as log2ratios. In this case, the abundance data corresponds to the
intensities of iTRAQ reporters at the scan level, tabulated in the “ID-q” file
with the name of each reporter as a column header (see below for how these
tables are generated). The intensities of each scan at 0 h are in the “Abundance:
113” column and serve as a common reference to express abundance ratios; thus,
they are used as the denominator. The reporter intensities corresponding to
different time points serve as numerators for the ratios. The task table also
enables the user to assign an easily identifiable name to the folders where the
quantitative values of each sample are stored (<i><span style='line-height:
107%;color:#BF8F00'>Figure 38</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=532 height=179 id="Picture 38"
src="index_files/image055.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771182"></a><a name="_Ref129965819"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>38</span></i><i><span style='color:#BF8F00'>. The
WSPP-SBT task table for workflow 1.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The WSPP-SBT module initially
conducts a calibration process, assigning a statistical weight to each
log2ratio value at the scan level (<i><span
style='line-height:107%;color:#BF8F00'>Figure 37</span></i><span
style='color:#BF8F00'>B</span>), as described [2].
The statistical weight of each scan is the inverse of the estimated variance
associated with the log2 of intensity ratios [2].
Following the calibration of data at the scan level, the workflow proceeds with
integrations from <i>scan-to-peptide</i> and <i>peptide-to-protein</i>.</p>

<p class=MsoNormal style='text-indent:.5in'>At the protein level, the SBT
algorithm is applied for the detection of functional category changes
originating from the coordinated behavior of proteins (<i><span
style='line-height:107%;color:#BF8F00'>Figure 37</span></i><span
style='color:#BF8F00'>B</span>). The algorithm first calculates the variance of
the <i>protein-to-category</i> integration, providing an improved estimate of
the technical protein variance that is less influenced by biological changes [3].
This protein variance is then utilized to perform the <i>protein-to-grand</i>
mean integration (hereinafter referred to as <i>protein-to-proteinall</i>),
from which statistically significant abundance changes are detected. Finally,
the algorithm conducts the <i>category-to-grand</i> mean integration
(hereinafter referred to as <i>category-to-categoryall</i>), identifying
statistically significant category changes. All results from the integrations
performed by the WSPP-SBT module are saved for each sample in the <i>Output
Sample folder</i>, as indicated in the module task table (<i><span style='line-height:107%;color:#BF8F00'>Figure 38</span></i>).</p>

<p class=MsoNormal style='text-indent:.5in'>In every integration step, a
relation table (a text file) is required to link lower- to higher-level
elements. These relation tables can be automatically generated by the RELS
CREATOR module (<i><span style='line-height:107%;color:#BF8F00'>Figure 37</span></i><span
style='color:#BF8F00'>A</span>, upper) or provided by the user (<i><span style='line-height:107%;color:#BF8F00'>Figure 37</span></i><span
style='color:#BF8F00'>A</span>, lower). In this example (<i><span
style='line-height:107%;color:#BF8F00'>Figure 39</span></i>) the relation
tables linking scans to peptides and peptides to proteins are obtained from the
“ID-q” file by specifying the column names where they are located (in this
case, “Scan_Id”, “Pep_Id”, and “Master Protein Accessions”). The columns
“Master Protein Accessions” and “Master Protein Descriptions” in the “ID-q”
file contain the accession numbers and complete names of the proteins,
respectively. Consequently, a relation table, <i>protein2description</i>, is
also created, which can later be used to append the full name of the protein to
any of the created reports (see below).</p>

<p class=MsoNormal style='text-indent:.5in'>An example of the <i>peptide2protein</i>
relation table, linking the identified peptides to the proteins they originate
from, is shown <i><span style='line-height:107%;color:#BF8F00'>Figure 40</span></i><span
style='color:#BF8F00'>A</span>. The elements of the relation table <i>protein2category</i>
were extracted from a text file containing functional annotations for mouse
proteins, compiled from various protein function databases (<i><span style='line-height:107%;color:#BF8F00'>Figure 40</span></i><span
style='color:#BF8F00'>B</span>), as described by the authors [3].
It is important to note that relation tables are by default extracted from the
“ID-q” file. To use other text files, the absolute path with the location of
the text file must be indicated. The relation tables <i>protein2proteinall</i>
and <i>category2categoryall</i> guide the integration to a grand mean (a common
element called “[1]”). Although the integration <i>peptide2peptideall</i> is
not necessary in this workflow, it is included in this example since it may be
useful to inspect quantifications at the peptide level.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=638 height=163 id="Picture 1926671488"
src="index_files/image056.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref132484598"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>39</span></i><i><span
style='color:#BF8F00'>. The RELS CREATOR task table was specifically designed
for workflow 1.</span></i></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
style='font-size:14.0pt;line-height:107%'><img border=0 width=536 height=341
id="Picture 1876955742" src="index_files/image057.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref131431704"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>40</span></i><i><span
style='color:#BF8F00'>. An excerpt from the peptide2protein (A) and
protein2category (B) relation tables, illustrating the links between peptides
and proteins and proteins and categories, respectively.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>After the integrations are
executed, the REPORT module is employed to gather the specified statistical
variables from the <i>Output Sample folders</i> designated by the user and
organize them into tables (<i><span style='line-height:107%;color:#BF8F00'>Figure
41</span></i>). In this instance, the tabulation is focused on the results from
the samples (2h-AngII, 4h-AngII, 6h-AngII, 8h-AngII, and 10h-AngII).</p>

<p class=MsoNormal style='text-indent:.5in'>In this example the REPORT module
creates a protein table and a category table by performing the following steps:</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table named “Npep2prot”, which contains the count of
peptides used to quantify each protein.</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This involves extracting the number of elements (n) from the <i>peptide-to-protein</i>
integrations in the specified folders, representing the lower level (peptide)
used for the quantitation of the higher level (protein).</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table named “Npep2prot_Quantprot_filtered”, including
protein changes (Zqa) and their statistical significance (FDRqa).</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is achieved by extracting standardized log2 ratios (Z) and
False Discovery Rates (FDR) from the <i>protein-to-proteinall</i> integration
in the indicated folders, representing the lower level (protein). </p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add the count of peptides with which each protein is quantified
to the “Npep2prot_Quantprot_filtered” table.</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This step involves merging the previous table with the existing
“Npep2prot” table based on the common level (protein), excluding a specific
column (peptide), and eliminating duplicate entries.</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add an additional column to this table with the complete
description of the proteins.</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is accomplished by merging the previous table with the
relation table <i>protein2description</i> based on the common level in both
tables (protein).</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter the table to include only proteins with a statistically
significant abundance change (FDR &lt; 0.01).</p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
class=MsoHyperlink><span style='font-family:"Courier New";text-decoration:none'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span>This is
achieved by applying a condition based on the FDR to the results from the <i>protein2proteinall</i>
integration. For more detailed information, refer to the “Filter for report”
section in the iSanXoT documentation:</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.5in;line-height:normal'><a
href="https://cnic-proteomics.github.io/iSanXoT">https://cnic-proteomics.github.io/iSanXoT</a> 
</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table named “Nprot2cat”, which contains the count of
proteins used to quantify each category.</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>Extract the number of elements (n) from the <i>protein-to-category</i>
integrations in the specified folders, representing the lower level (protein)
used for the quantitation of the higher level (category).</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table named “Nprot2cat_Quantcat_filtered”, incorporating
category changes (Zca) and their statistical significance (FDRca).</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>Extract standardized log2 ratios (Z) and False Discovery Rates
(FDR) from the <i>category-to-categoryall</i> integration in the indicated
folders, representing the lower level (category).</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add to this table the count of proteins with which each category
is quantified.</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is achieved by merging the previous table with the existing
table “Nprot2cat” based on the common level in both tables (category),
excluding a specific column (protein), and eliminating duplicate entries.</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter the table to include only categories with a statistically
significant change (FDR &lt; 0.01).</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by applying a condition based on the FDR to the
results from the <i>category2categoryall</i> integration.</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table named “Npep2prot_Quanprot” containing the number
of peptides per protein, protein changes (Zqa), and their statistical
significance (FDRqa).</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done as previously explained, excluding the protein
descriptions and filters.</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table named “Nprot2cat_Quancat_Quanprot_filtered”, including
category changes (Zca) and their statistical significance (FDRca).</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>Extract standardized log2 ratios (Z) and False Discovery Rates
(FDR) from the <i>category-to-categoryall</i> integration in the indicated
folders, representing the lower level (category).</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add to this table the count of proteins per category, protein
changes (Zqa), and their statistical significance (FDRqa).</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is achieved by merging the previous table with the existing
tables “Nprot2cat” and “Npep2prot_Quantprot”.</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter the table to include only categories containing 5 or more
proteins or 100 or fewer proteins.</p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by applying a set of conditions in the <i>Filter</i>
column, joined with the &quot;&amp;&quot; operator.</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>Note
that the commands in the REPORT module, facilitating the construction of tables
essential for typical quantitative proteomics projects, are easily adaptable
and reusable for other projects.</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><i><img
border=0 width=717 height=165 id="Picture 1124067194"
src="index_files/image058.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></i></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref131431781"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>41</span></i><i><span
style='color:#BF8F00'>. The REPORT task table that was designed for workflow 1.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>In <i><span
style='color:#BF8F00'>Figure 43</span></i><i><span style='color:#BF8F00'>, </span></i>two
heat maps are presented, constructed from the protein and category tables
obtained using the REPORT module.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=253 height=357 id="Picture 266206403" src="index_files/image059.jpg"
alt="A comparison of a number of dna samples&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption>Figure 42. Relative abundance changes of proteins (Zqa, left)
and functional categories (Zca, right) are derived from the “Npep2prot_Quanprot_filtered”
and “Nprot2cat_Quancat_filtered” reports, respectively, generated by the REPORT
module in workflow 1 (Figure 41). Both report tables were sorted based on the
averages of Zqa and Zca, respectively. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>In <span
style='color:#BF8F00'>Figure 43</span><span lang=EN-GB> </span>illustrates
examples of functional categories displaying statistically significant changes
resulting from coordinated protein behavior. The data for these plots are
derived from the “Nprot2cat_Quancat_Quanprot_filtered” table generated by the
REPORT module. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=627 height=397 id="Picture 2138611462"
src="index_files/image060.jpg"
alt="A group of graphs with different colors&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref128391398">Figure </a>43. Examples of
time-dependent coordinated protein behavior in VSMCs treated with angiotensin-II,
revealed by the distribution of the standardized log2 ratio (Zqa) of protein
components in each category.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:14.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521026"></a><a
name="_Toc199848029"></a><a name="_Workflow_2:_Step-by-step"></a><b><span
style='font-size:20.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow 2: Step-by-step quantification and sample combination
in a labeled experiment</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521027"></a><a
name="_Toc199848030"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal style='page-break-after:avoid'>The workflow presented here
utilizes data from González-Amor <i>et al</i>. [5]
focusing on the contribution of interferon-stimulated gene 15 (ISG15) to
vascular damage associated with hypertension. The study employs knockout
mutants for the ISG15 gene, subjecting animals to AngII treatment or not. A
total of 16 samples from mouse aortic tissue represent four groups: four
WT-Control mice, four ISG15-KO mice, four WT+AngII mice, and four
ISG15-KO+AngII mice. The experiment, conducted in two isobaric iTRAQ 8-plex
batches, serves as a case study to guide the step-by-step creation of a workflow.
This workflow integrates quantitative results from individual samples to the
protein level, consolidates protein data across the four biological replicates
in each group, establishes ratios between conditions, and analyzes functional
category changes due to coordinated protein behavior using the SBT model.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=465 height=458 id="Picture 250499266"
src="index_files/image061.jpg"
alt="A diagram of a diagram&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771564"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>44</span></i><i><span
style='color:#BF8F00'>. Schematic representation of workflow 2, which involves
step-by-step quantification and sample combination in a labeled experiment. The
figure illustrates the key modules in the workflow, including RELS CREATOR (A)
and LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, RATIOS, SBT, and
REPORT (B) </span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521028"></a><a
name="_Toc199848031"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP_NORCOM_RATIOS_SBT.zip">https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP_NORCOM_RATIOS_SBT.zip</a></p>

<p class=MsoNormal>Refer to the <i>Importing a Workflow Template</i> section
below for detailed instructions.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521029"></a><a
name="_Toc199848032"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>Workflow 2 encompasses all six basic modules: LEVEL CREATOR,
LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, RATIOS, and SBT, along with the REPORT
module (<i><span style='line-height:107%;color:#BF8F00'>Figure 44</span></i>). The
task table in the starting module, LEVEL CREATOR, generates files at the scan
level containing log2 ratios and the corresponding sample folders (<i><span style='line-height:107%;color:#BF8F00'>Figure 45</span></i>). In this
example, similar to workflow 1, the name of each iTRAQ reporter served as a
column header in the “ID-q” file containing the intensities. Additionally, the “Experiment”
column indicates whether the intensities come from the first or second iTRAQ
8-plex batch. Each iTRAQ batch comprises two biological replicates from each of
the four groups. The average of reporter intensities from the two untreated WT
mice (reporters in the columns “113” and “117”) is used as an internal control
within each batch, serving as the denominator for the log2 ratios.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=623 height=327 id="Picture 50" src="index_files/image062.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771575"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>45</span></i><i><span
style='color:#BF8F00'>. The LEVEL CREATOR task table specifically designed for
workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>LEVEL CREATOR generates the <i>u_scan</i>
(uncalibrated scan) files, comprising scan identifiers (extracted from the “Scan_Id”
column in the “ID-q” table), log2-ratios at the scan level (<i>Xs</i>, as
defined in the task table), and uncalibrated weights (<i>Vs</i>, corresponding
to the intensities of the reporters in the Ratio numerator column) (<i><span style='line-height:107%;color:#BF8F00'>Figure 46</span></i>). These
uncalibrated weights (<i>Vs</i>) are indicative of quantification quality,
where a higher weight implies more accurate quantification. However, at this
stage, they are not associated with statistical variance.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=566 height=148 id="Picture 51" src="index_files/image063.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771591"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>46</span></i><i><span
style='color:#BF8F00'>. An excerpt from one of the u_scan files generated by
the LEVEL CREATOR module in workflow 2 is displayed, showing element
identifiers in the left column, log2 ratios in the center column, and
statistical weights in the right column.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The LEVEL CALIBRATOR module
calibrates the Vs weights by conducting a <i>u_scan-to-peptide</i> integration,
generating scan files with true, calibrated statistical weights, as defined in
the WSPP model (the inverse of the estimated individual scan variances) (<i><span style='color:#BF8F00'>Figure 47</span></i>, Top).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=379 height=144 id="Picture 320124786"
src="index_files/image064.jpg"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref132481392">Figure </a>47. The LEVEL CALIBRATOR
(Top) and INTEGRATE (Bottom) task tables for workflow 2.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Note that the LEVEL CALIBRATOR
automatically generates a plot to supervise the accuracy of calibrations (the
&quot;*_outGraph_VRank&quot; PNG file) in each sample folder. This plot shows
whether the model is able to predict experimental scan variances as a function
of the calibrated statistical weights (see <i><span
style='color:#BF8F00'>Figure 48</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=327 height=260 id="Picture 589007140"
src="index_files/image065.jpg"
alt="A graph with green and red dots&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref132481421">Figure </a>48. Automatically
generated graphs to supervise the accuracy of calibrations. These graphs
illustrate 1/MSD versus the rank of Vs (scan weight, which at this level
corresponds to reporter intensity). MSD represents the Mean Squared Deviation
of the scans versus the respective mean of the peptide to which they belong.
The scans are ordered by Vs, and the MSD is calculated in a sliding window of
200 scans [2].</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Similar to workflow 1, before
conducting the integrations, the relation tables need to be created using the
RELS CREATOR module. These tables exhibit a similar structure, as depicted in<i><span
style='color:#BF8F00'> </span></i><i><span
style='color:#BF8F00'>Figure 49</span></i>.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=629
height=112 id="Picture 93" src="index_files/image066.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref132481845">Figure </a>49. Illustrates the RELS
CREATOR task table for workflow 2.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The INTEGRATE module performs the <i>scan-to-peptide</i>,
<i>peptide-to-protein</i>, and <i>protein-to-protein</i> integrations based on
the module task table (<i><span
style='color:#BF8F00'>Figure 47</span></i>, Bottom). It's important to note
that, for consistency, all files created for each sample are automatically
stored in the folder specified in the task table of the LEVEL CREATOR module,
unless an alternative location is specified in the <i>Output Sample folder</i>
column.</p>

<p class=MsoNormal style='text-indent:.5in'>Note that the INTEGRATE module
automatically generates a plot to assess the accuracy of the GIA integration
model in each integration step. This is achieved by comparing the distribution
of Z values with that of the null hypothesis (standard normal distribution)
(refer to <i><span style='color:#BF8F00'>Figure 51</span></i>, left panels).
These graphs are then stored in the respective sample folders as “*_outGraph.png”
files.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>By default, iSanXoT removes
integration outliers. However, to avoid the removal of outlier elements in <i>the
protein-to-proteinall</i> integration, particularly as these represent
significantly altered proteins, a 0 FDR value was specified in the INTEGRATE
task table for this integration (refer to <span
style='color:#BF8F00'>Figure 47</span>, Bottom).</p>

<p class=MsoNormal style='text-indent:.5in'>Once protein levels are
established, workflow 2 utilizes the NORCOMBINE basic module (<i><span style='line-height:107%;color:#BF8F00'>Figure 44</span></i><span
style='color:#BF8F00'>B</span>) to integrate protein values from the four
biological replicates within each group. The resulting integrated protein
values per group are then stored in the folders WT-C, WT-AngII, ISG15-C, and
ISG15-AngII (<i><span
style='color:#BF8F00'>Figure 50</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=550 height=107 id="Picture 54" src="index_files/image067.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref132482832">Figure </a>50. The NORCOMBINE task
table for workflow 2.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The NORCOMBINE module integrates
biological replicates within sample groups using the GIA algorithm [3].
This algorithm models the distribution of protein values around the average,
considering error propagation theory and estimating a global variance for the
integration. The GIA algorithm operates under the assumption that individual
variances of all lower elements (proteins) are influenced by a global variance,
arising from biological variability within the same group. While this
assumption may not hold in all the cases, it can be easily checked by
inspecting the test distributions.</p>

<p class=MsoNormal style='text-indent:.5in'>Similar to the INTEGRATE module,
the NORCOMBINE module automatically generates graphs comparing the distribution
of the integrated Z variables with those of the standard normal distribution.
As depicted in <i><span
style='color:#BF8F00'>Figure 51</span></i> (right), the distribution of protein
Z values estimated by the model in the case of the ISG15-AngII group aligns
well with the null hypothesis. This agreement demonstrates that the assumption
of the model is a suitable approach for handling the biological variance of
samples within this group. Comparable results were obtained in the other three
groups (not shown).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=624 height=346 id="Picture 938156029"
src="index_files/image068.png"
alt="A graph of a function&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref152517792"></a><a name="_Ref133424017">Figure </a>51.
Distribution of the standardized log2 protein ratios (Zqa) from the four
individual ISG15-AngII VSMC samples (Left panel). This illustrates how the WSPP
model aligns well with the expected null distributions in the four cases. In
the right panel, the Figure shows the distribution from the integrated
ISG15-AngII sample group obtained with the NORCOMBINE module. This demonstrates
how the GIA assumption of a global biological variance is a valid approach to
address the biological variability within this group. Red: null hypothesis
(standard distribution); blue: experimental data.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Note also that the NORCOMBINE module
employs a weighted averaging technique from multiple samples. Through its good
fit to the null hypothesis (as illustrated in <i><span
style='color:#BF8F00'>Figure 51</span></i>), this approach enables accurate
control over outliers. This unique approach allows the integration of protein
values originating from unbalanced sample groups, distinct experiments, various
mass spectrometers, and even different labeling techniques (refer to, for
instance, [2]).</p>

<p class=MsoNormal style='text-indent:.5in'>The module's task table specifies
that samples were combined at the protein level using the <i>proteinall</i>
level for normalization. In this process, log2 protein ratios are initially
normalized by the grand mean before being integrated into an averaged protein
value. This normalization compensates for differences in protein load into each
iTRAQ channel. Importantly, it's worth noting that proteins could also be
integrated at other levels (such as organelles, subcellular compartments,
complexes, etc.) before being further integrated by NORCOMBINE, allowing for
various types of normalization. Finally, the <i>lowerNorm</i> column indicates
the file containing the normalized data, typically the <i>lowerNormV</i> files
previously generated by the INTEGRATE module. For more detailed information,
please refer to the iSanXoT documentation.</p>

<p class=MsoNormal>                The protein averages derived from the four
biological sample groups are then employed by the RATIOS basic module to
calculate two ratios: <i>WT-AngIIvsWT-C</i>, where wild-type AngII-treated
animals are compared to controls, and <i>ISG15-AngIIvsISG15-C</i>, where ISG15
AngII-treated animals are compared to ISG15 controls (<i><span
style='line-height:107%;color:#BF8F00'>Figure 52</span></i>). In the &quot;V
method&quot; column, users can specify the method used to assign a statistical
weight to the log2ratios, with the default being the &quot;max&quot; method
(for more details, please refer to the “RATIOS” module in the iSanXoT documentation:
<a href="https://cnic-proteomics.github.io/iSanXoT">https://cnic-proteomics.github.io/iSanXoT</a>).
</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=400 height=76 id="Picture 55" src="index_files/image069.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771774"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>52</span></i><i><span
style='color:#BF8F00'>. The RATIOS task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>                The final basic module executed in workflow
2 is the SBT (<i><span style='line-height:107%;color:#BF8F00'>Figure 53</span></i>),
which applies the SBT algorithm to the previously defined comparisons. The goal
is to detect changes in functional categories resulting from the coordinated
behavior of proteins, following the approach explained in workflow 1. The SBT
module offers greater flexibility by allowing triangle operations on any level,
not just proteins. In this case, the triangle is formed by the levels protein
and category (<i><span style='line-height:107%;color:#BF8F00'>Figure 52</span></i>)
and the corresponding grand mean.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=299 height=83 id="Picture 99" src="index_files/image070.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771804"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>53</span></i><i><span
style='color:#BF8F00'>. The SBT task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Finally, the REPORT module (<i><span style='line-height:107%;color:#BF8F00'>Figure 54</span></i>) is used, as in
Workflow 1, to generate tables with protein and category data. In this case,
additional features of the REPORT module are utilized. The table “Npep2prot” is
generated using an asterisk. This symbol serves as a wildcard character for
iSanXoT, indicating that the results from all samples containing
peptide-to-protein integration (i.e., ISG15-AngII-1, ISG15-AngII-2,
ISG15-AngII-3, ISG15-AngII-4, ISG15-AngII, ISG15-C-1, ISG15-C-2, ISG15-C-3,
ISG15-C-4, ISG15-C, and ISG15-AngIIvsISG15-C) are to be included in the table.</p>

<p class=MsoNormal style='text-indent:.5in'>However, the “Npep2prot_Quanprot_ISG15_filtered”
and “Npep2prot_Quanprot_WT_filtered” tables include protein changes (Zqa), the
statistical significance (FDRqa) of these changes, and the number of peptides
per protein only from the samples indicated in the <i>Sample folder(s)</i>
column. The report for the ISG15 samples is filtered by Zqa to display the most
extreme values (greater than 1 or less than -1) but only for the “ISG15-AngIIvsISG15-C”
sample. Additional filters for the minimum number of peptides per protein are
also applied to these tables. The tables containing category values are
filtered by Zca (greater than or equal to 2 or less than or equal to -2) and/or
by the number of proteins per category (between 5 and 100).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=702 height=176 id="Picture 104" src="index_files/image071.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771910"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>54</span></i><i><span
style='color:#BF8F00'>. The REPORT task table designed for workflow 2.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The tables generated by REPORT can
be used to generate heatmaps showing the most relevant protein abundance
changes (<i><span style='line-height:107%;color:#BF8F00'>Figure 55</span></i>).
As previously shown [5], iSanXoT analysis revealed a coordinated alteration of
proteins implicated in cardiovascular function, extracellular matrix and
remodeling, and vascular redox state in aortic tissue from AngII-infused
ISG15-KO mice (<i><span
style='color:#BF8F00'>Figure 56</span></i><i><span style='color:#BF8F00'>A</span></i>).
The coordinated protein behavior from some of the altered categories can be
analyzed in the sigmoid plots (<i><span
style='color:#BF8F00'>Figure 56</span></i><i><span style='color:#BF8F00'>B</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=477 height=442 id="Picture 15" src="index_files/image072.png"
alt="A screenshot of a diagram&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772030"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>55</span></i><i><span
style='color:#BF8F00'>. Differential abundance of functional proteins revealed
by workflow 2. The heatmap (A) for proteins (Zqa) is based on the “Npep2prot_Quanprot_ISG15_filtered”
REPORT table. The heatmap (B) displays the proteins (Zqa) for the WT samples
using the “Npep2prot_Quanprot_WT_filtered” REPORT table.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=707 height=406 id="Picture 1541420126"
src="index_files/image073.png"
alt="A screenshot of a graph&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref132485751">Figure </a>56. Functional category
changes arising from coordinated protein behavior. A) Bar graph for functional
categories (Zca) constructed from the “Nprot2cat_Quancat_filtered” REPORT table.
B) The distributions of the standardized log2 protein ratios (Zqa) are shown
for some of the functional categories that are significantly down-regulated
(Left) or up-regulated (Right). The data to create the sigmoid curves are taken
from the “Nprot2cat_Quancat_Quanprot_filtered” REPORT table. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b><span lang=EN-GB style='font-size:18.0pt;line-height:
107%'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521030"></a><a
name="_Toc199848033"></a><a name="_Workflow_3:_Quantification"></a><b><span
style='font-size:20.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow 3: Quantification of posttranslationally modified
peptides in a labeled experiment</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521031"></a><a
name="_Toc199848034"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>This workflow was employed to quantify reversibly oxidized
Cys peptides in mouse embryonic fibroblast (MEF) preparations subjected to
chemical oxidation with diamide. The experiment aimed to illustrate the
comparative performance of on-filter (FASILOX) and in-gel (GELSILOX) approaches
for studying the thiol redox proteome [4]. These techniques involved
differentially labeling Cys residues based on their oxidation state, resulting
in two distinct populations of reduced and oxidized Cys-containing peptides. MEF
samples were incubated with diamide (treated group) or PBS (control group), and
the resulting peptides were isobarically labeled with iTRAQ 8-plex (four
biological replicates per condition). The workflow is designed to detect
statistically significant abundance changes in peptides containing modified Cys
residues.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=537 height=224 id="Picture 1981727863"
src="index_files/image074.png"
alt="A diagram of a diagram of a diagram&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772176"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>57</span></i><i><span
style='color:#BF8F00'>. Scheme of workflow 3 (quantification of
posttranslationally modified peptides in a labeled experiment) showing module
components: RELS CREATOR (A) and LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE,
and REPORT (B).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521032"></a><a
name="_Toc199848035"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and the required input files for
executing this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP_PTM.zip">https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP_PTM.zip</a></p>

<p class=MsoNormal>Please refer to the <i>Importing a Workflow Template</i>
section below for detailed instructions. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521033"></a><a
name="_Toc199848036"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>Workflow 3 comprises the basic modules LEVEL CREATOR, LEVEL
CALIBRATOR, and INTEGRATE, as well as the RELS CREATOR and REPORT modules (<i><span style='line-height:107%;color:#BF8F00'>Figure 57</span></i>) and is very
similar to workflow 2. LEVEL CREATOR was used to design the ratios and to generate
the level files, sample folders and log2 ratios indicated in the corresponding
task table (<i><span style='line-height:107%;color:#BF8F00'>Figure 58</span></i>
and <i><span style='line-height:107%;color:#BF8F00'>Figure 59</span></i>).
LEVEL CALIBRATOR was used to calibrate statistical weights (<i><span style='line-height:107%;color:#BF8F00'>Figure 60</span></i>, top) and INTEGRATOR to
integrate from scan to peptide and from peptide to protein (<i><span style='line-height:107%;color:#BF8F00'>Figure 60</span></i>, bottom).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=622 height=224 id="Picture 243" src="index_files/image075.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772189"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>58</span></i><i><span
style='color:#BF8F00'>. The LEVEL CREATOR task table for workflow 3.</span></i></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=525 height=108 id="Picture 247" src="index_files/image076.png"
alt="A screenshot of a graph&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772226"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>59</span></i><i><span
style='color:#BF8F00'>. Excerpt from one of the </span></i><span
style='color:#BF8F00'>uscan<i> files generated by workflow 3 LEVEL CREATOR
module showing element identifiers (left column), log2 ratios (center column)
and statistical weights (right column).</i></span></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The only difference with workflow 2
lies in the INTEGRATE command used for the integration <i>peptide-to-protein</i>.
INTEGRATE can use a modified version of the GIA algorithm for the quantitative
analysis of posttranslational modifications (PTM) that includes a third column
containing <i>tags</i> in the relation tables, as described [7].
In this workflow the advanced option of INTEGRATE was activated to display the <i>Tag
</i>column, which is used to include only the peptides which are tagged in the
relation table with the text “<i>Not modified”</i> when calculating the protein
averages (<i><span style='line-height:107%;color:#BF8F00'>Figure 60</span></i>).
An example of tagged <i>peptide2protein</i> relation table is shown in <i><span style='color:#BF8F00'>Figure 61</span></i><i><span style='color:#BF8F00'>.</span></i>
Proteins are thus quantified using only peptides which are not modified in Cys.
However, although these Cys peptides do not contribute to protein averages,
they are assigned a <i>Zpq</i> value, which serves to evaluate whether they
deviate significantly from the expected distribution of peptides around their
protein averages [4]. If the deviation is statistically significant it can be
concluded that there is a change in abundance of the posttranslational
modification in relation to the protein it comes from. This philosophy can be
extended to any other kind of PTM.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=550 height=143 id="Picture 2140405750" src="index_files/image077.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772240"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>60</span></i><i><span
style='color:#BF8F00'>. The LEVEL CALIBRATOR (Top) and INTEGRATE (Bottom) task
tables for workflow 3.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=532 height=194 id="Picture 255" src="index_files/image078.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133425366">Figure </a>61. Excerpt from the peptide2protein
relation table used to integrate peptides to proteins. Note the presence of a
third column used to tag Cys-containing peptides, which will be excluded from
the calculation of protein averages in the peptide-to-protein integration.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>iSanXoT allows the automatic
generation of relation tables containing tags, which are extracted from the
“ID-q” table. To achieve this, RELS CREATOR employs a specific option (<i><span style='color:#BF8F00'>Figure 62</span></i>). In this case, this option instructs
RELS CREATOR to search the “ID-q” table for the column with the header <i>Modifications</i>
and to translate its content into the third column of the <i>peptide2protein</i>
relation table. In the “ID-q” table used in this instance, peptides containing
modified Cys residues were labeled as either “Reduced-Cys peptides” or
“Oxidized-Cys peptides” depending on the type of modification. RELS CREATOR
locates these tags in the relation table (<i><span
style='color:#BF8F00'>Figure 61</span></i> and <i><span
style='color:#BF8F00'>Figure 62</span></i>). iSanXoT allows the use of any tag
created by search engines or defined by the user, with the sole condition that
the tag indicated in the INTEGRATE command must match the tag in the third
column of the relation table (<i><span style='line-height:107%;color:#BF8F00'>Figure
60</span></i>, bottom).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=578 height=98 id="Picture 65" src="index_files/image079.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133427111">Figure </a>62. The RELS CREATOR
task table for workflow 3.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Finally, the REPORT module compiles
the statistical variables generated by the <i>peptide-to-protein</i>
integration for all the samples (<i>c1, c2, c3, c4, t1, t2, t3</i>, and <i>t4</i>),
as indicated by the asterisk (<i><span
style='color:#BF8F00'>Figure 63</span></i><i><span style='color:#BF8F00'>A</span></i>).
The REPORT commands mirror those employed in generating the protein tables in
workflow 1, with the distinction that peptide values are tabulated instead of
protein values, along with the inclusion of the number of scans per peptide
instead of the number of peptides per protein. The false discovery rate (FDR)
at the peptide level enables the detection of statistically significant changes
in posttranslational modifications (PTM). This REPORT also generates a second
filtered peptide table containing peptides with reduced Cys and the most
pronounced abundance changes. This table was utilized to create a heatmap (<i><span style='color:#BF8F00'>Figure 63</span></i><i><span style='color:#BF8F00'>B</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=652 height=681 id="Picture 12" src="index_files/image080.png"
alt="A screenshot of a graph&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133484449">Figure </a>63. (A) The task table
for Workflow 3 is in the REPORT module. (B) Relative abundance of
Cys-containing peptides in MEF samples is represented by peptide log2 ratios
expressed in units of standard deviation corrected by the protein mean (Zpq).
The data for the heatmap were derived from the “Nscan2pep_Quanpepprot_filtered”
report table. </p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Of significance, the peptides
integrated followed a standard distribution in all eight samples, as depicted
in blue in <i><span
style='color:#BF8F00'>Figure 64</span></i> for <i>t1</i>, <i>t2</i>, <i>c1</i>,
and <i>c2</i> samples. This evidences that the error distribution at the
peptide level could be accurately modeled using the GIA algorithm. In addition,
the treatment produced a generalized increase in the abundance of oxidized
Cys-containing peptides (orange curves), with concomitant decrease in the
abundance of reduced Cys-containing peptides (green curves). Consistently, the
opposite changes were observed in the controls.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=443 height=290 id="Imagen 40" src="index_files/image081.jpg"
alt="A graph of different colored lines&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133499855">Figure </a>64. Distribution of the
standardized variable at the peptide level (Zpq) in control MEF samples (t1, t2,
c1 and c2) for all the peptides quantitated (blue) and the oxidized (orange)
and reduced (green) Cys-containing peptide subpopulations. The theoretical normal
distribution N(0,1) is shown in red. Positive/negative Zpq values indicate
increased/decreased peptide abundance with respect to the average. These
sigmoidal curves were created from the “Nscan2pep_Quanpepprot” table generated
by REPORT.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b><span lang=EN-GB style='font-size:18.0pt;line-height:
107%'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521034"></a><a
name="_Toc199848037"></a><a name="_Workflow_4:_Label-free"></a><b><span
style='font-size:20.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow 4: Label-free quantification</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521035"></a><a
name="_Toc199848038"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>This workflow was employed to analyze quantitative data
obtained from a multicenter study conducted in Data-dependent Acquisition (DDA)
mode. The study utilized samples prepared exactly as described in the paper by
Navarro <i>et al</i>. [6]. Two hybrid proteome test samples were generated in
the study, comprising tryptic digests of human, yeast, and <i>Escherichia coli</i>
proteins mixed in two distinct proportions, as detailed in <i><span style='color:#BF8F00'>Table S1</span></i>. </p>

<p class=MsoNormal style='text-indent:.5in'>In this workflow, quadruplicate
peptide preparations from each sample underwent analysis by LC-MS/MS. Following
that, the MaxQuant [8]
software was employed for peptide identification and quantification. For
guidance on pre-processing data from MaxQuant and other software to adapt for
use with iSanXoT, please consult the section described below.</p>

<p class=MsoCaption style='page-break-after:avoid'><a name="_Ref153808960"></a><a
name="_Ref153808967">Table S</a>1. Proteome-hybrid samples A and B were
prepared, each containing known quantities of peptide digestions of HeLa,
Saccharomyces cerevisiae, and Escherichia coli. The samples were then mixed
according to the procedure outlined in Navarro et al., Nature Biotech 2016.</p>

<div align=center>

<table class=MsoTable15Grid5Dark border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=113 valign=top style='width:84.9pt;border:solid white 1.0pt;
  border-right:none;background:black;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='color:white'>&nbsp;</span></b></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:solid white 1.0pt;
  border-left:none;border-bottom:solid white 1.0pt;border-right:none;
  background:black;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><b><span style='color:white'>A</span></b></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:solid white 1.0pt;
  border-left:none;border-bottom:solid white 1.0pt;border-right:none;
  background:black;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><b><span style='color:white'>B</span></b></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border:solid white 1.0pt;
  border-left:none;background:black;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><b><span style='color:white'>FOLD B/A</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=113 valign=top style='width:84.9pt;border:solid white 1.0pt;
  border-top:none;background:black;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><i><span
  style='color:white'>E. coli</span></i></b></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#D9D9D9;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>20%</span></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#D9D9D9;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>5%</span></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#D9D9D9;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>0.25</span></p>
  </td>
 </tr>
 <tr>
  <td width=113 valign=top style='width:84.9pt;border:solid white 1.0pt;
  border-top:none;background:black;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><i><span
  style='color:white'>S. Cerevisiae</span></i></b></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#F2F2F2;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>15%</span></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#F2F2F2;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>30%</span></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#F2F2F2;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>2</span></p>
  </td>
 </tr>
 <tr>
  <td width=113 valign=top style='width:84.9pt;border:solid white 1.0pt;
  border-top:none;background:black;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><i><span
  style='color:white'>HeLa</span></i></b></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#D9D9D9;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>65%</span></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#D9D9D9;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal'><span style='color:black'>65%</span></p>
  </td>
  <td width=113 valign=top style='width:84.95pt;border-top:none;border-left:
  none;border-bottom:solid white 1.0pt;border-right:solid white 1.0pt;
  background:#D9D9D9;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
  line-height:normal;page-break-after:avoid'><span style='color:black'>1</span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=462 height=386 id="Picture 1177685409"
src="index_files/image082.jpg"
alt="A diagram of a protein source&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133509694">Figure </a>65. Scheme of workflow 4
(label-free quantification) showing module components: RELS CREATOR (A) and LEVEL
CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, and REPORT (B).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521036"></a><a
name="_Toc199848039"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and necessary input files for executing
this workflow can be downloaded from</p>

<p class=MsoNormal><a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WPP_LabelFree.zip">https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WPP_LabelFree.zip</a></p>

<p class=MsoNormal>Detailed instructions can be found in the <i>Importing a
Workflow Template</i> section below.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc152521037"></a><a
name="_Toc199848040"><b><span style='font-size:16.0pt;line-height:107%;
font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>Workflow 4 includes the basic iSanXoT modules: LEVEL
CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, and RATIOS, along with the
REPORT and RELS CREATOR modules (<i><span
style='color:#BF8F00'>Figure 65</span></i><i>)</i>. The starting module, LEVEL
CREATOR, generates the level files, sample folders and log2 ratios indicated in
the corresponding task (<i><span
style='color:#BF8F00'>Figure 66</span></i>) based on the quantitative data at
the peptide level obtained with MaxQuant for replicate A- and B-type samples.
In this example, we used the average of peptide intensities across all samples
as the denominator of the log2 ratio. However, in this case, the averages of
the four A-type and the four B-type samples are first calculated separately (as
indicated by the square brackets), and then the average of the two averaged
values is calculated (as indicated by the comma). This ensures that no log2
ratio is calculated when the four values are missing in either the A or the B
sample group. This module generates uncalibrated files at the peptide level (<i>u_peptide</i>)
(<span class=MsoIntenseEmphasis>Figure 67</span>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=625 height=261 id="Picture 44" src="index_files/image083.png"></p>

<p class=MsoCaption><a name="_Ref133509816">Figure </a>66. The LEVEL CREATOR
task table for workflow 4.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=606 height=132 id="Picture 72" src="index_files/image084.png"
alt="A screenshot of a graph&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133510232">Figure </a>67. Excerpt from one of
the u_peptide files generated by workflow 4 LEVEL CREATOR module showing
element identifiers (left column), log2 ratios (center column) and uncalibrated
statistical weights (right column).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The <i>u_peptide</i> level files
are subsequently calibrated using the LEVEL CALIBRATOR module through
integration to the protein level (<span
class=MsoIntenseEmphasis>Figure 68</span><i>, Top</i>), resulting in calibrated
<i>peptide</i> level files. The INTEGRATE module then executes <i>peptide-to-protein</i>
and <i>protein-to-proteinall</i> integrations as specified in the module task
table (<span class=MsoIntenseEmphasis>Figure 68</span>, Bottom).</p>

<p class=MsoNormal style='text-indent:.5in'>In this example, it is important to
note that the advanced option of INTEGRATE was activated to utilize the Tag
column, to indicate that only proteins containing the <i>Homo sapiens</i> tag
are employed in the <i>protein-to-proteinall</i> integrations (<span class=MsoIntenseEmphasis>Figure 68</span><i>, Bottom</i>). The decision to
restrict integration to human proteins serves two purposes: a) normalization is
performed using the grand mean of human proteins, unaffected by the presence of
yeast or E. coli proteins; b) estimation of the variance in the <i>protein-to-proteinall</i>
integration relies solely on human proteins, mitigating the impact of yeast and
E. coli proteins, which exhibit significant deviations from the mean. Note that
this procedure does not remove yeast or E. coli proteins from the normalized
files subsequently utilized by the NORCOMBINE module (as explained below). </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=536 height=154 id="Picture 799885626"
src="index_files/image085.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133510477">Figure </a>68. The LEVEL CALIBRATOR
(Top) and INTEGRATE (Bottom) task tables for workflow 4.</p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>In this workflow, the protein level
comprises numeric values derived from the “Protein group IDs” provided by <i>ID-q</i>,
generated through the <i>modificationSpecificPeptides.txt</i> file from
MaxQuant (refer to the <i>Preparing the ID-q file from MaxQuant output</i>
section). Similar to workflow 3, the <i>protein-to-proteinall</i> relation
table must include a third column that tags the species from which each protein
originates (see <span
class=MsoIntenseEmphasis>Figure 69</span>). The species names have been
extracted from the <i>ID-q</i> file. It is important to note that the tag
indicating human proteins aligns with the tag indicated in INTEGRATE (see <span class=MsoIntenseEmphasis>Figure 68</span><i>, Bottom</i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=299 height=203 id="Picture 73" src="index_files/image086.png"
alt="A table with numbers and text&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133510667">Figure </a>69. Excerpt from the protein2proteinall
workflow 4 relation table that illustrate the linkage between proteins and a
constant value representing the protein grand mean. The protein level is the
&quot;Protein group IDs&quot; obtained from ID-q, generated by MaxQuant. It is
noteworthy that a third column is employed to tag proteins with their
respective species, facilitating later species-specific integration in the
protein-to-proteinall context.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The <i>protein-to-proteinall</i>
relation table is automatically generated by the RELS CREATOR module, which
extracts information from the <i>ID-q</i> file created by the user. This file
includes the relationship between protein group identifiers (under the <i>Protein
group IDs</i> column header) and the corresponding species (under the <i>Species</i>
column header).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=574 height=79 id="Picture 36" src="index_files/image087.png"></p>

<p class=MsoCaption>Figure 70. The RELS CREATOR task table for workflow 4.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Next, the normalized data at the <i>protein</i>
level from the four replicates from each sample are combined into samples <i>A</i>
and <i>B</i>, respectively, using the NORCOMBINE basic module (<span class=MsoIntenseEmphasis>Figure 71</span>, Top). To compare these two samples,
new log2 ratios and statistical weights are calculated using the RATIOS basic
module (<span class=MsoIntenseEmphasis>Figure 71</span>, Bottom). Finally, a <i>protein-to-proteinall</i>
integration is carried out for the newly generated <i>B_vs_A</i> sample by the
module INTEGRATE (<span
class=MsoIntenseEmphasis>Figure 68</span>, Bottom), using again the <i>Homo
sapiens</i> tag.</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=560 height=136 id="Picture 9"
src="index_files/image088.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref153812677"></a><a name="_Ref133512128">Figure </a>71.
The NORCOMBINE (Top) and RATIOS (Bottom) task tables for workflow 4.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The REPORT module, employed in this
workflow, compiles data at the protein level alongside the number of peptides
per protein, like previous workflows (<span
class=MsoIntenseEmphasis>Figure 72</span>). In this specific instance, the
table encompasses Z and FDR, as well as the log2ratios of proteins from all
samples (<i>Xinf</i> from the <i>protein-to-proteinall</i> integration, also
denoted as <i>Xq</i>), the grand mean (<i>Xsup</i> from the <i>protein-to-proteinall</i>
integration, or <i>Xa</i>), the statistical weights (<i>Vinf</i> or <i>Vq</i>),
and the respective species (<i>tags</i>) for each protein. The grand mean,
utilized for log2-ratio normalization, along with the statistical weights, can
be employed in constructing plots, as illustrated in <span class=MsoIntenseEmphasis>Figure 73</span> (see below).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
style='font-size:14.0pt;line-height:107%'><img border=0 width=609 height=68
id="Picture 202067190" src="index_files/image089.png"></span></p>

<p class=MsoCaption><a name="_Ref133518992">Figure </a>72. The REPORT module
task table for workflow 4.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>It is noteworthy that variance
modelling, normalization, standardization and statistical weighting, according
to the GIA algorithm, are performed automatically, without data filtering,
pre-processing or missing value imputation [6], even in a situation where
numerous proteins have highly imbalanced data. Moreover, the sigmoid plots
automatically generated in each one of the integrations performed by INTEGRATE
clearly demonstrate that the GIA algorithm accurately predicts the distribution
of peptide quantifications around their proteins (<span class=MsoIntenseEmphasis>Figure 73</span><span class=MsoIntenseEmphasis>A</span>)
and of protein quantifications around the grand mean (<span class=MsoIntenseEmphasis>Figure 73</span><span class=MsoIntenseEmphasis>B</span>).
These results demonstrate that this statistical model is very suitable for the
analysis of label-free data.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=503 height=378 id="Picture 26" src="index_files/image090.jpg"
alt="A graph of a function&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133519177">Figure </a>73. Distribution of the
standardized variable at the peptide (Zpq) and protein (Zqa) levels for
label-free data analyzed with iSanXoT. A) Zpq distribution for the eight
individual A-type and B-type samples. B) Zqa distribution for the B-type vs
A-type comparison. Red: null hypothesis (standard distribution); blue:
experimental data.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'><a name="_Hlk121489265">The
combined statistics <i>B_vs_A</i> also shows how human, yeast and bacterial
proteins distribute approximately around the expected 0, 1, and -2 log2-values
(corresponding to 1-, 2- and 0.25-fold changes) (</a><span class=MsoIntenseEmphasis>Figure 74</span><span class=MsoIntenseEmphasis><span
style='color:windowtext;font-style:normal'>)</span></span>, and how protein
quantifications with higher statistical weights are more accurate. This plot
also confirms how iSanXoT provides highly accurate quantitative results in a fully
automated fashion.</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=548 height=435
id="Picture 1169663620" src="index_files/image091.jpg"
alt="A graph of different colored dots&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133523519">Figure </a>74. Quantification of
human (orange), yeast (grey) and bacterial (blue) proteins according to the
combined statistics B_vs_A. Shown are log2-ratios normalized by the grand mean
(Xq – Xa or Xinf - Xsup). This plot was generated from the table
“Npep2prot_Quanprot”.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>As a means to benchmark the
performance of iSanXoT for label-free data, we counted up how many proteins
exhibited statistically significant changes between the two preparations.
Proteins originating from E. coli and yeast were classified as true positives,
while those from <i>Homo sapiens</i> were labeled as false positives.
Subsequently, we computed key metrics, including the False Positive Rate (FPR),
True Positive Rate (TPR), and False Discovery Rate (FDR), based on these
classifications.</p>

<p class=MsoNormal style='text-indent:.5in'>The metrics were applied following two
distinct approaches. In the first approach, the integrated results of each
sample for each condition (a1, a2, a3, a4 and b1, b2, b3, b4) were considered separately
to obtain a standardized value for each protein (Zqa). Subsequently, a
conventional t-test was conducted to compare Zqa<sub> </sub>values between the
two samples and the <i>p-values</i> were adjusted for multiple hypothesis
testing (FDR &lt;= 0.05). The results are presented in<i><span
style='color:#BF8F00'> </span></i>Table
S2.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoCaption style='page-break-after:avoid'><a name="_Ref153809104">Table
S</a>2. Computation of False Positive Rate (FPR), True Positive Rate (TPR), and
False Discovery Rate (FDR) based on statistically significant proteins from the
integrated results of samples for each condition provided by iSanXoT and the
conventional t-test. The FPR, TPR, and FDR percentages were calculated from the
quantified proteins of iSanXoT and the identified proteins of the search engine.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=624 height=121 src="index_files/image092.png"></p>

<p class=MsoNormal style='text-indent:.5in'>Note that since iSanXoT does not
input missing values and the t-test cannot be calculated in cases when there
are fewer than two replicates, not all identified proteins could be quantified (<i><span style='color:#BF8F00'>Table S2</span></i>). In the subset of quantified
proteins, iSanXoT got TPRs near 90% while maintaining the FPR below 5%. </p>

<p class=MsoNormal style='text-indent:.5in'>In the second approach, we directly
calculated the statistically significant protein changes from the combined
statistics <i>B_vs_A</i>. This was done counting up how many Zqa values
significantly deviated from the expected N(0.1) distribution at FDR &lt;= 0.05.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoCaption style='page-break-after:avoid'><a name="_Ref153809172">Table
S</a>3. Computation of False Positive Rate (FPR), True Positive Rate (TPR), and
False Discovery Rate (FDR) based on significant changes determined by the iSanXoT
modules (NORCOMBINE, RATIO, INTEGRATE). The FPR, TPR, and FDR percentages were
calculated from the quantified proteins of iSanXoT and the identified proteins
of the search engine.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=624 height=119 id="Picture 2" src="index_files/image093.png"></p>

<p class=MsoNormal style='text-indent:.5in'>As shown in <i><span style='color:#BF8F00'>Table S3</span></i>, iSanXoT was able to quantify more
proteins than in the previous case. This was possible without performing
missing value imputation because the statistical model assigned a specific
variance to all the proteins integrated by NORCOMBINE, independently of the
number of replicates where the protein was quantified in each of the samples.
Note that all the protein variances were corrected by the variance detected by
the model at the time of averaging replicates.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The versatility of iSanXoT allows
for the creation of unlimited integrations depending on the desired levels. In
this specific experiment, where the proteins come from several species, we can integrate
the protein values to the species they belong to. This integration normalizes
the protein data within each species, producing standardized protein values
that describe the deviation from each species’ average. To perform this
integration, we have to define a protein2species relation table using the
RELS_CREATOR module (<i><span
style='color:#BF8F00'>Figure 75</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=524 height=107
src="index_files/image094.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref153808713">Figure </a>75. RELS_CREATOR module
for the protein-to-species workflow.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Again, we can follow two
approaches. In the first one each sample is separately integrated to species to
obtain a standardized Zqs value for each protein (note that in this case the <i>s</i>
subscript refers to the species level). A conventional t-test can then be
applied to the set of Zqs values (<i><span
style='color:#BF8F00'>Figure 76</span></i><i><span style='color:#BF8F00'>A</span></i>),
to detect whether there are significant differences between the two samples.
Here a statistically significant change would indicate that the protein
deviates from the rest of proteins of the same species. To construct these
integrations, we only need to indicate protein and species as the lower and
higher levels, respectively in the INTEGRATE module (<i><span style='color:#BF8F00'>Figure 76</span></i><i><span style='color:#BF8F00'>B</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=426 height=404
src="index_files/image095.jpg"
alt="A screenshot of a diagram&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref153810000">Figure </a>76. (A) Scheme of the
protein-to-species workflow (label-free quantification), integrating each
sample separately. (B) The INTEGRATE module that allows integration among the
levels described in (A). The outliers in the protein-to-species analysis are
not removed (set to 0 for FDR).</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>In the second approach, the protein
values in each sample are integrated using NORCOMBINE and the ratio between the
samples calculated using RATIO, as in <i><span
style='color:#BF8F00'>Figure 71</span></i>. But now the resulting level is
integrated to species (<i><span
style='color:#BF8F00'>Figure 77</span></i><i><span style='color:#BF8F00'>A</span></i>).
The statistically significant protein changes are then detected from the Zqa
values in the <i>B_vs_A</i> statistics, as before. To perform this, the
INTEGRATE task table requires a new <i>protein-to-species</i> integration
specifically for the <i>B_vs_A</i> combined statistic sample (<i><span style='color:#BF8F00'>Figure 77</span></i><i><span style='color:#BF8F00'>B</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=535 height=416
src="index_files/image096.jpg"
alt="A diagram of a diagram&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref153812489">Figure </a>77. Schematic
representation of the protein-to-species workflow, involving NORCOMBINE and
RATIO modules for processing protein values. (B) The INTEGRATE module
facilitating integration across the levels described in (A).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The REPORT module  utilized in the <i>protein-to-species</i>
workflows (<i><span
style='color:#BF8F00'>Figure 78</span></i>), resembles workflow 4 (<i><span style='color:#BF8F00'>Figure 72</span></i>), with the distinction of exclusively
incorporating the <i>protein-to-species</i> integration to derive the necessary
values for the analysis.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=638
height=84 src="index_files/image097.png"></p>

<p class=MsoCaption><a name="_Ref153816705">Figure </a>78. The REPORT module
task table for protein-to-species workflows.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>As mentioned earlier, the
coordinated behavior of proteins within each species can be analyzed by
inspecting the automatically-generated sigmoid plots (<i><span style='color:#BF8F00'>Figure 79</span></i><i><span style='color:#BF8F00'>A</span></i>),
clearly illustrating that the GIA algorithm accurately predicts the
distribution of protein quantifications. Similarly, the combined <i>B_vs_A </i>statistics,
when integrated to species, demonstrate how human, yeast, and E. coli proteins
distribute approximately around the expected log2-values of 0 (<i><span style='color:#BF8F00'>Figure 79</span></i><i><span style='color:#BF8F00'>B</span></i>).
</p>

<p class=MsoNormal style='page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=442 height=583
src="index_files/image098.jpg"
alt="A graph and diagram of a graph&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref153829903">Figure </a>79. (A) Distribution of
the standardized log2 protein ratios (Zqs) resulting from the
protein-to-species integration of B_vs_A. Red represents the null hypothesis
(standard distribution), and blue represents the experimental data. (B)
Quantification of human (orange), yeast (grey), and E. coli (blue) proteins
using the protein-to-species workflow with NORCOMBINE. The log2-ratios are
normalized by the respective species (Xq – Xs). The letters ‘q’ and ‘s’ denote
the protein and species, respectively.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The performance of iSanXoT in the
two approaches are summarized in <span
style='color:#BF8F00'>Table S4</span>. Here, the null hypothesis is that each
protein follows the same quantitative behaviour as the rest of proteins of the
same species. Hence, we can calculate a FPR per species. As explained above, iSanXoT
does not handle missing values as input, and hence the first approach  (<i><span style='color:#BF8F00'>Table S4</span></i><i><span style='color:#BF8F00'>A</span></i>)
quantifies less proteins than the second  (<i><span
style='color:#BF8F00'>Table S4</span></i><i><span style='color:#BF8F00'>B</span></i>).
Similarly, the second test is more sensitive to detect outliers. In both cases,
human and yeast proteins are quantified with low FPRs. In contrast, this
analysis highlights how E.coli proteins are more difficult to quantify due to
the large difference in abundance in the original samples.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoCaption style='page-break-after:avoid'><a name="_Ref153814637"></a><a
name="_Ref153814653">Table S</a>4. Computation of False Positive Rate (FPR),
based on the significant proteins obtained from the protein-to-species workflow
and the conventional t-test (A) and from the iSanXoT modules (NORCOMBINE,
RATIO, INTEGRATE) (B). The FPR values were calculated from the quantified
proteins of iSanXoT.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=551 height=337 id="Picture 1" src="index_files/image099.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h2><a name="_Toc199848041"><span lang=EN-GB>Workflow 5: PTM-compass</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc199848042"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>This novel integrative workflow automatically captures
multiple layers of PTM-related information, including variations in trypsin
efficiency, zonal changes, specific PTM alterations, and hypermodified regions.
This enables advanced control of artefacts and provides a coherent and
comprehensive interpretation of PTM data induced by mitochondrial heteroplasmy
in a mouse model [7]. The results reveal that heteroplasmy predominantly
affects cardiac tissue, inducing oxidative damage to proteins in the oxidative
phosphorylation system. These findings offer a molecular mechanism that
explains the structural and functional alterations observed in heart
mitochondria. Additionally, we identify significant PTM information previously
undetectable, including consistent detection of novel oxidative modifications
in Met and Cys residues directly from raw proteomics data.  </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc199848043"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and the required input files for
executing this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/PTM-compass.zip">https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/PTM-compass.zip</a></p>

<p class=MsoNormal>Please refer to the <i>Importing a Workflow Template</i>
section below for detailed instructions. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc199848044"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>The posttranslational modifications (PTM) workflow comprises
the basic modules LEVEL CREATOR, LEVEL CALIBRATOR, and INTEGRATE, along with
the REPORT and RELS CREATOR modules.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc199848045"></a><a name="_Workflow_6:_Single-Cell"></a><span
lang=EN-GB>Workflow 6: Single-Cell Proteomics</span></h2>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc199848046"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>We introduce a novel experimental and computational workflow
to study wild type and genetically modified single-cardiomyocyte proteomes.
Through an optimized isolation protocol and the integrative features of the
iSanXoT platform, we eliminate batch effects, reduce biases related to cell
size, extract quantitative data on subcellular compartments, and detect protein
changes within subcellular compartments.</p>

<p class=MsoNormal>This workflow improves the accuracy of data quantification
and supports clearer biological interpretation. Using this approach, we
demonstrate that Myc transcription factor overexpression reprograms adult
cardiomyocyte metabolism and gives rise to a subpopulation of pro-regenerative
cells. These advances lay the groundwork for systematic single-cell proteome
analysis, even in highly sensitive cell types such as cardiomyocytes. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc199848047"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and the required input files for
executing this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/SCP.zip">https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/SCP.zip</a></p>

<p class=MsoNormal>Please refer to the <i>Importing a Workflow Template</i>
section below for detailed instructions. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc199848048"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>The single-cell proteomics (SCP) workflow comprises the
basic modules LEVEL CREATOR, LEVEL CALIBRATOR - composed of two sub-modules, Combine
Calibrator and Level Calibrator - INTEGRATE, along with the REPORT and RELS
CREATOR modules.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc152521038"></a><a name="_Toc199848049">Importing a workflow
template</a></h1>

</div>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal>In this section, we will provide instructions to execute the
workflow examples and to import workflows that were previously created with
iSanXoT to be reused in other projects. We will use the first workflow
described in the previous section as an example. </p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>Start
by downloading the template for Workflow 1 and the input files from the iSanXoT
documentation</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>(<a
href="https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP-SBT.zip">https://raw.githubusercontent.com/CNIC-Proteomics/iSanXoT/master/docs/templates/2.1.0/WSPP-SBT.zip</a>).</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>Then,
extract the files included in the compressed archive to create a folder named
WSPP-SBT. Check that the WSPP-SBT folder has been created in your file system.
Proceed as follows:</p>

<p class=MsoNormal>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Open the
     iSanXoT application by double-clicking the application icon (<i><span style='color:#BF8F00'>Figure 80</span></i>).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=234 height=154 id="Picture 542053378"
src="index_files/image100.png"></p>

<p class=MsoCaption><a name="_Ref135132480">Figure </a>80. The iSanXoT startup
message.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Choose <i>New
     Project</i> from the <i>Project</i> menu (<i><span
     style='color:#BF8F00'>Figure 81</span></i>).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=429 height=250 id="Picture 573294088"
src="index_files/image101.png"></p>

<p class=MsoCaption><a name="_Ref135132530">Figure </a>81. Create New Project.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Provide a
     name of your choice for the project folder and indicate a path to locate
     this folder, then click the <i>Submit</i> button (<i><span style='color:#BF8F00'>Figure 81</span></i>).</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Choose <i>Import
     Workflow</i> from the <i>Project</i> menu (<i><span
     style='color:#BF8F00'>Figure 82</span></i><i>)</i> and select the folder WSPP-SBT
     created before (or any other iSanXoT project folder from which you want to
     import the workflow).</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=431 height=196 id="Picture 1970222289"
src="index_files/image102.png"></p>

<p class=MsoCaption><a name="_Ref135132621">Figure </a>82. Importing a
preexisting iSanXoT workflow to the newly-created project.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Inspect the
     WSPP-SBT task table (in the Compound modules tab), the RELS CREATOR task
     table (in the Relation tables tab) and the REPORT task table (in the
     Reports tab) to check that the tables indicated in Fig. S2, S3 and S5 have
     been correctly loaded. Note that if a different template is imported, only
     the corresponding task tables will be loaded.</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Now, click on
     <i>Choose identification file</i> and select “ID-q.tsv” in the WSPP-SBT
     folder (<i><span
     style='color:#BF8F00'>Figure 83</span></i><i>)</i>. Alternatively, select
     the desired identification/quantification table with which this workflow
     is to be executed. Section 3 below shows how to prepare the “ID-q” file
     based on the output from a variety of proteomics pipelines. Bear in mind
     that the tasks defined in the LEVEL CREATOR and RELS CREATOR modules have
     to match the samples and column names from the specific “ID-q” file used. </li>
</ul>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=373 height=321 id="Picture 3" src="index_files/image103.jpg"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref135132690">Figure </a>83. Choosing the
identification/quantification (ID-q) file for the newly-created project.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Select <i>Save
     Project</i> from the <i>Project </i>menu to save the changes, or directly
     press the <i>Save and Run</i> button to both save and execute the current
     workflow.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<b><span style='font-size:16.0pt;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span></b>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc152521039"></a><a name="_Toc199848050"></a><a
name="_Creating_the_identification/quantif"></a>Creating the
identification/quantification file from proteomics pipelines</h1>

</div>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>iSanXoT necessitates
an identification/quantification file in TSV format (<i>ID-q.tsv</i>), which
should include, at a minimum, the quantified features alongside their
corresponding quantitative values. Any TSV table can serve as an ID-q file,
given that quantitative values are organized with column headers. In this
arrangement, features (such as PSMs or peptides) are listed in rows, and their
quantitative values are presented in columns, with each column representing a
distinct sample. iSanXoT utilizes the column headers of the <i>ID-q</i> file to
extract essential information. </p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>Moreover,
when the <i>ID-q</i> file encompasses features quantified in more than one
experiment (e.g., different samples labeled with the same TMT-18plex tags), it
must include an additional column with the header “Batch” to specify the
experiment assignment of the features.  </p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>Finally,
iSanXoT requires additional information to generate the relation files
necessary for integrating quantified features into higher levels. This
information is typically found in the <i>ID-q</i> file. For example, iSanXoT
can utilize columns containing scan and peptide identifiers to construct the <i>scan2peptide</i>
relation table.</p>

<p class=MsoNormal style='text-indent:.5in'>The majority of proteomics software
tools generate tables that can be easily used for this purpose. In this
section, we will describe how to prepare the <i>ID-q</i> file based on the
output from the three most popular proteomics pipelines (<i><span style='color:#BF8F00'>Table S5</span></i>).</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoCaption style='page-break-after:avoid'><a name="_Ref153809262">Table
S</a>5. Output data from proteomics pipelines to be included in the <span
style='font-style:normal'>ID-q.tsv</span> file.</p>

<p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
line-height:normal'><img border=0 width=657 height=317 id="Picture 178556711"
src="index_files/image104.png"
alt="A table of data&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<h2><a name="_Toc152521040"></a><a name="_Toc199848051">Preparing the <i>ID-q</i>
file from Proteome Discoverer output</a></h2>

<p class=MsoNormal>In the case of Proteome Discoverer version 2.5 [9], the way that quantitative data are adapted for use with iSanXoT depends on whether
they originate from label-free or labelled experiments:</p>

<h3><a name="_Toc152521041"></a><a name="_Toc199848052">Label-free experiments</a></h3>

<p class=MsoNormal>In this scenario, quantitative data at the peptide level can
be adapted for use with iSanXoT from the <i>_PeptideGroups.txt</i> files
obtained when the <i>Processing</i> workflow node <i>Minora Feature Detector</i>
of Proteome Discoverer is employed. The following column headers of the <i>_PeptideGroups.txt</i>
files must be considered when preparing the <i>ID-q</i> file: </p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modifications</i>:
     Chemical or posttranslational modifications to the <i>Sequence </i>above;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Master Protein
     Accessions</i>: Accession code(s) for the protein(s) to which the peptide <i>Sequence</i>
     is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Abundance:
     FX: Sample Type: </i>Peptide intensity in the RAW file identified with <i>FX</i>
     and tagged as <i>Sample Type</i> in the Proteome Discoverer <i>Input Files</i>
     tab.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'><a
name="_Hlk135138073">The <i>peptide</i> level required for the <i>peptide-to-protein</i>
integration with iSanXoT can be obtained by merging the <i>Sequence</i> and <i>Modifications
fields </i>(see Section <i>Adapting the results from proteomics pipelines for
iSanXoT</i> below).</a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<h3><a name="_Toc152521042"></a><a name="_Toc199848053">Labelled experiments</a></h3>

<p class=MsoNormal>For labelled experiments (e.g., TMT- or iTRAQ-based),
quantitative data at the scan level can be adapted for use with iSanXoT from <a
name="OLE_LINK3">the <i>_PSMs.txt</i> </a>files generated when the <i>Processing</i>
workflow node <i>Reporter Ions Quantifier</i> of Proteome Discoverer is used.
The following column headers of the <i>_PSMs.txt</i> files must be considered
for preparing the <i>ID-q</i> file:  </p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Spectrum
     File</i>: Name of the RAW file where the PSM was identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>First Scan</i>:
     Spectrum (scan) number of the PSM in the RAW file;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modifications</i>:
     Chemical or posttranslational modifications to the <i>Sequence</i> above;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Master Protein
     Accessions</i>: Accession code(s) for the protein(s) to which the peptide <i>Sequence</i>
     is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Abundance:
     Quan Channel</i>:  Intensity of the reporter ion tagged as <i>Quan Channel</i>
     in the Proteome Discoverer <i>Samples</i> tab.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>For
the <i>scan to peptide</i> integration with iSanXoT, the <i>scan</i> level can
be obtained by merging the<i> Spectrum File</i> and <i>First</i> <i>Scan </i>fields,
and the <i>peptide</i> level by merging the <i>Sequence</i> and <i>Modifications</i>
fields  (see Section <i>Adapting the results from proteomics pipelines for
iSanXoT</i> below; make sure <i>Max. Number of Peptides Reported</i> = 1 was
selected in the <i>Input Data</i> section of the Proteome Discoverer <i>Processing</i>
node used).</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>&nbsp;</p>

<h2><a name="_Toc152521043"></a><a name="_Ref153276991"></a><a
name="_Ref153276996"></a><a name="_Toc199848054">Preparing the <i>ID-q</i> file
from MaxQuant output</a></h2>

<p class=MsoNormal>The way MaxQuant version 1.6.5.0 [8]
data are adapted for use with iSanXoT depends on whether they originate from
label-free or labelled proteomics experiments:</p>

<h3><a name="_Toc152521044"></a><a name="_Toc199848055">Label-free experiments</a></h3>

<p class=MsoNormal>In this case, the quantifications at the peptide level
required to prepare the <i>ID-q</i> file can be found in the <a name="OLE_LINK5"><i>modificationSpecificPeptides.txt</i>
</a>file, which is stored in the <i>“…combined/txt</i>” folder. The following
column headers of the <i>modificationSpecificPeptides.txt</i> file must be
considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modifications</i>:
     Chemical or posttranslational modifications to the <i>Sequence</i> above;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Proteins</i>:
     Identifier(s) of the protein(s) to which the peptide <i>Sequence</i> is
     ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Intensity
     Experiment</i>: Summed up extracted ion current of all isotopic clusters
     associated with the peptide Sequence identified across the raw files
     included in the <i>Experiment</i> as specified by the user in the MaxQuant
     <i>Raw data</i> tab.</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.25in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>The
<i>peptide</i> level required for the <i>peptide</i> to protein integration
with iSanXoT can be obtained by merging <i>Sequence</i> and <i>Modifications</i>
fields (see Section <i>Adapting the results from proteomics pipelines for
iSanXoT</i> below).</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc152521045"></a><a name="_Toc199848056">Labelled experiments</a></h3>

<p class=MsoNormal>When dealing with labelled experiments (e.g., iTRAQ- or
TMT-based), the necessary quantitative data at the scan level can be found in
the <i>modificationSpecificPeptides.txt</i> file, which is stored in the <i>“…combined/txt</i>”
folder. The following column headers of the <i>modificationSpecificPeptides.txt</i>
file must be considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Raw file</i>:
     Name of the RAW file where the PSM was identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Scan
     number</i>: Spectrum (scan) number of the PSM in the RAW file;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modified Sequence</i>:
     Amino acid sequence of the identified peptide including chemical or posttranslational
     modifications. This parameter is nonblank only when identification was
     successful.</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Proteins</i>:
     Identifier(s) of the protein(s) to which the peptide <i>Sequence</i> is
     ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Reporter
     intensity n</i>: Intensity of the reporter ion <i>n </i>as specified by
     the user in the MaxQuant <i>Group-specific parameters</i> tab.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>For
the <i>scan to peptide</i> integration with iSanXoT, the <i>scan</i> level can
be obtained by merging the<i> Raw File</i> and <i>Scan number </i>fields (see Section
<i>Adapting the results from proteomics pipelines for iSanXoT</i> below).</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124328462"></a><a name="_Toc152521046"></a><a
name="_Toc199848057">Preparing the <i>ID-q</i> file from FragPipe output</a></h2>

<p class=MsoNormal>The way that quantitative data from Fragpipe version 1.8.1 [10] are adapted for use with iSanXoT depends on whether they originate from label-free
or labelled experiments:</p>

<h3><a name="_Toc152521047"></a><a name="_Toc199848058">Label-free experiments</a></h3>

<p class=MsoNormal>FragPipe <i>Quant (MS1)</i> module stores the quantifications
at the peptide level necessary to prepare the <i>ID-q</i> file in a <i>combined_modified_peptide.tsv
</i>file. The following column headers of the <i>modificationSpecificPeptides.txt</i>
file must be considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modified Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Protein ID</i>:
     Identifier of the protein to which the <i>Modified</i> <i>Sequence</i> peptide
     is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Experiment
     Intensity</i>: Summed up intensity of the <i>Modified Sequence</i> peptide
     in the RAW files included in the <i>Experiment</i> as specified by the
     user in the FragPipe <i>Workflow</i> tab.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<h3><a name="_Toc152521048"></a><a name="_Toc199848059">Labelled experiments</a></h3>

<p class=MsoNormal>Fragpipe <i>Quant (Isobaric)</i> module generates a <i>psm.tsv</i>
output file that contains the quantitative data at the scan level obtained from
labelled experiments. The following column headers of the <i>psm.txt</i> file
must be considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Spectrum</i>:
     Spectrum (scan) identifier of the PSM in the XML file;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Spectrum
     File</i>: Name of the XML file where the PSM was identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modified Peptide</i>:
     Amino acid sequence of the identified peptide including chemical or
     posttranslational modifications;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Protein ID:
     </i>Identifier of the protein to which the <i>Modified Peptide</i> is
     ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Channel</i>: 
     Intensity of the reporter ion <i>Channel</i> as specified by the user in
     the FragPipe TMT-Integrator table of the <i>Quant (Isobaric)</i> module.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>The
<i>scan</i> level required for the later <i>scan to peptide</i> integration
with iSanXoT can be obtained by merging the<i> Spectrum</i> and <i>Spectrum
File </i>fields (see Section <i>Adapting the results from proteomics pipelines
for iSanXoT</i> below; make sure <i>Report top N = 1</i> was selected in the <i>Advanced
Output Options</i> of the FragPipe <i>MSFragger</i> module).</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc152521049"></a><a name="_Toc199848060">Adapting the results
from proteomics pipelines for iSanXoT</a></h2>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>iSanXoT requires an identification/quantification
tab-separated values file (<i>ID-q.tsv</i>) containing at least the identified
features along with their quantitative values (an batch identifier is required
if two or more batches are included). Users can either manually compose this <i>ID-q</i>
file (refer to the previous section for guidance on how to do this using data
from the four most popular proteomics pipelines) or have it prepared by the
iSanXoT <i>Input Adaptor</i>. The latter option is described in this section.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Run the
     iSanXoT application and create a new project or open an existing project.
     A new window will appear asking for the ID-q file (<i><span style='color:#BF8F00'>Figure 84</span></i>).</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>If you
     already have a suitable <i>ID-q</i> file, click the <i>Select
     User-Provided</i> option, and then <i>Choose identification file</i> to
     select the file (<i><span
     style='color:#BF8F00'>Figure 84</span></i>).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><b><span
style='font-size:16.0pt;line-height:107%'><img border=0 width=374 height=322
id="Picture 1846817758" src="index_files/image105.jpg"
alt="A screenshot of a computer&#10;&#10;Description automatically generated"></span></b></p>

<p class=MsoCaption><a name="_Ref135147559">Figure </a>84. Selecting an ID-q
file in the Input Adaptor main window.</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>If you do not
     have an <i>ID-q</i> file, click <i>Select Adaptor from proteomics
     pipelines</i>. This option will launch the iSanXoT adapter to import your
     quantitative data. The adapter has been tested with recent versions of
     MaxQuant, Trans-Proteomic Pipeline, FragPipe, and Proteome Discoverer. Click
     on <i>Choose folder + Add annots</i> to select the folder containing your
     quantitative data (<i><span
     style='color:#BF8F00'>Figure 85</span></i>). A three-panel window will
     pop-up (<i><span
     style='color:#BF8F00'>Figure 86</span></i><i>)</i>.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:.5in;text-align:center;text-indent:22.0pt;
page-break-after:avoid'><img border=0 width=502 height=151
id="Picture 1093422789" src="index_files/image106.png"></p>

<p class=MsoCaption><a name="_Ref135147782">Figure </a>85. Having the iSanXoT
Input Adaptor prepare the ID-q file.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:.5in;text-align:center;page-break-after:avoid'><img
border=0 width=385 height=448 id="Picture 86" src="index_files/image107.png"></p>

<p class=MsoCaption><a name="_Ref135147818">Figure </a>86. Adapting results
from a proteomics pipeline. In the top panel, several output files from
Proteome Discoverer have been selected. These PSMs.txt files, which contain
identification/quantification data, have been assigned an experiment name
(Jurkat) in the middle panel. The bottom panel has been used to create identifiers
by concatenating result file headers: ScanID (by concatenating Spectrum File,
First Scan and Charge) and pepID (by concatenating Sequence and Modifications).</p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><i><span style='color:#BF8F00'>&nbsp;</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The top panel
     displays the files included in the folder, allowing you to select one or
     more result files for consideration by the adapter. It's important to note
     that if several result files are chosen, they must have the same column
     headers.</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The middle
     panel is used to set the distribution of data items across experiments
     according to result filenames.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The bottom
     panel allows to create identifiers by concatenating result file headers.
     It is composed of two interfaces:</li>
 <ul style='margin-top:0in' type=circle>
  <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The left
      side lists the headers found in the result files, with selected header
      names being added to the right-side interface.</li>
  <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The
      right-side interface displays the selected header names used to generate
      the identifier, along with the user-provided identifier name.</li>
 </ul>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;text-indent:22.0pt'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Please note
     that the alphanumeric text that unambiguously identifies the items to be
     integrated is the only identifier that must be necessarily included in the
     <i>ID-q</i> file.</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Click the <i>Submit</i>
     button and the Input Adaptor will start generating the <i>ID-q.tsv</i>
     file.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc199848061"></a><a name="_References"></a><span lang=ES>References</span></h1>

</div>

<p class=MsoNormal><span lang=ES>&nbsp;</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span
lang=ES>[1] Trevisan-Herraz M, Bagwan N, Garcia-Marques F, Rodriguez JM, Jorge
I, et al. </span>SanXoT: a modular and versatile package for the quantitative
analysis of high-throughput proteomics experiments. <span lang=ES>Bioinformatics.
2019;35(9):1594-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[2]
Navarro P, Trevisan-Herraz M, Bonzon-Kulichenko E, Nunez E, Martinez-Acedo P,
et al. </span>General statistical framework for quantitative proteomics by
stable isotope labeling. <span lang=ES>J Proteome Res. 2014;13(3):1234-47.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[3]
Garcia-Marques F, Trevisan-Herraz M, Martinez-Martinez S, Camafeita E, Jorge I,
et al. </span>A Novel Systems-Biology Algorithm for the Analysis of Coordinated
Protein Responses Using Quantitative Proteomics. <span lang=ES>Mol Cell
Proteomics. 2016;15(5):1740-60.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[4]
Bonzon-Kulichenko E, Camafeita E, Lopez JA, Gomez-Serrano M, Jorge I, et al. </span>Improved
integrative analysis of the thiol redox proteome using filter-aided sample
preparation. <span lang=ES>J Proteomics. 2020;214:103624.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[5]
Gonzalez-Amor M, Garcia-Redondo AB, Jorge I, Zalba G, Becares M, et al. </span>Interferon
stimulated gene 15 pathway is a novel mediator of endothelial dysfunction and
aneurysms development in angiotensin II infused mice through increased
oxidative stress. Cardiovasc Res. 2021.</p>

<p class=EndNoteBibliography style='margin-bottom:0in'>[6] Navarro P, Kuharev
J, Gillet LC, Bernhardt OM, MacLean B, et al. A multicenter study benchmarks
software tools for label-free proteome quantification. <span lang=ES>Nat
Biotechnol. 2016;34(11):1130-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[7] Bagwan
N, Bonzon-Kulichenko E, Calvo E, Lechuga-Vieco AV, Michalakopoulos S, et al. </span>Comprehensive
Quantification of the Modified Proteome Reveals Oxidative Heart Damage in
Mitochondrial Heteroplasmy. Cell Rep. 2018;23(12):3685-97 e4.</p>

<p class=EndNoteBibliography style='margin-bottom:0in'>[8] Tyanova S, Temu T,
Cox J. The MaxQuant computational platform for mass spectrometry-based shotgun
proteomics. Nat Protoc. 2016;11(12):2301-19.</p>

<p class=EndNoteBibliography style='margin-bottom:0in'>[9] Orsburn BC. Proteome
Discoverer-A Community Enhanced Data Processing Suite for Protein Informatics.
Proteomes. 2021;9(1).</p>

<p class=EndNoteBibliography>[10] Kong AT, Leprevost FV, Avtonomov DM,
Mellacheruvu D, Nesvizhskii AI. MSFragger: ultrafast and comprehensive peptide
identification in mass spectrometry-based proteomics. Nat Methods.
2017;14(5):513-20.</p>

<p class=MsoNormal>&nbsp;</p>

</div>

</body>

</html>
