<!-- begin: add for the html documentation in iSanXoT -->
<template class="task-template" id="User_Guide">
	<html>
	<!-- end: add for the html documentation in iSanXoT -->
	
	<head>
	<base href="sections/help/"> <!-- add for the html documentation in iSanXoT -->

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Helvetica;
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:Courier;
	panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face
	{font-family:"Tms Rmn";
	panose-1:2 2 6 3 4 5 5 2 3 4;}
@font-face
	{font-family:Helv;
	panose-1:2 11 6 4 2 2 2 3 2 4;}
@font-face
	{font-family:"New York";
	panose-1:2 4 5 3 6 5 6 2 3 4;}
@font-face
	{font-family:System;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:Batang;
	panose-1:2 3 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:PMingLiU;
	panose-1:2 2 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;}
@font-face
	{font-family:Dotum;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimHei;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:MingLiU;
	panose-1:2 2 5 9 0 0 0 0 0 0;}
@font-face
	{font-family:Mincho;
	panose-1:2 2 6 9 4 3 5 8 3 5;}
@font-face
	{font-family:Gulim;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:Century;
	panose-1:2 4 6 4 5 5 5 2 3 4;}
@font-face
	{font-family:"Angsana New";
	panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face
	{font-family:"Cordia New";
	panose-1:2 11 3 4 2 2 2 2 2 4;}
@font-face
	{font-family:Mangal;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Latha;
	panose-1:2 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sylfaen;
	panose-1:1 10 5 2 5 3 6 3 3 3;}
@font-face
	{font-family:Vrinda;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Raavi;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Shruti;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sendnya;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Gautami;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Tunga;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Estrangelo Edessa";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Yu Gothic";
	panose-1:2 11 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:DengXian;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Palatino Linotype";
	panose-1:2 4 5 2 5 5 5 3 3 4;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"Arial Unicode MS";
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:"Segoe UI Emoji";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:Garamond;
	panose-1:2 2 4 4 3 3 1 1 8 3;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:center;
	line-height:107%;
	page-break-after:avoid;
	border:none;
	padding:0in;
	font-size:26.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:normal;
	font-style:italic;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:11.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:22.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-link:"Footnote Text Char";
	margin:0in;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-link:"Comment Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0in;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0in;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	text-align:center;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoFootnoteReference
	{vertical-align:super;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpFirst, li.MsoTitleCxSpFirst, div.MsoTitleCxSpFirst
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpMiddle, li.MsoTitleCxSpMiddle, div.MsoTitleCxSpMiddle
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpLast, li.MsoTitleCxSpLast, div.MsoTitleCxSpLast
	{mso-style-link:"Title Char";
	margin:0in;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
a:link, span.MsoHyperlink
	{font-family:"Calibri",sans-serif;
	color:#2E74B5;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
strong
	{font-family:"Times New Roman",serif;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-link:"Comment Subject Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Balloon Text Char";
	margin:0in;
	text-align:justify;
	font-size:9.0pt;
	font-family:"Segoe UI",sans-serif;}
p.MsoRMPane, li.MsoRMPane, div.MsoRMPane
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoQuote, li.MsoQuote, div.MsoQuote
	{mso-style-link:"Quote Char";
	margin-top:10.0pt;
	margin-right:.6in;
	margin-bottom:8.0pt;
	margin-left:.6in;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#404040;
	font-style:italic;}
span.MsoIntenseEmphasis
	{font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoIntenseReference
	{font-variant:small-caps;
	color:#4472C4;
	letter-spacing:.25pt;
	font-weight:bold;}
span.MsoBookTitle
	{letter-spacing:.25pt;
	font-weight:bold;
	font-style:italic;}
p.MsoTocHeading, li.MsoTocHeading, div.MsoTocHeading
	{margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-link:Title;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-link:"Footnote Text";}
p.lead, li.lead, div.lead
	{mso-style-name:lead;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.QuoteChar
	{mso-style-name:"Quote Char";
	mso-style-link:Quote;
	color:#404040;
	font-style:italic;}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-link:"Comment Text";}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-link:"Comment Subject";
	font-weight:bold;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-link:"Balloon Text";
	font-family:"Segoe UI",sans-serif;}
p.EndNoteBibliographyTitle, li.EndNoteBibliographyTitle, div.EndNoteBibliographyTitle
	{mso-style-name:"EndNote Bibliography Title";
	mso-style-link:"EndNote Bibliography Title Char";
	margin:0in;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyTitleChar
	{mso-style-name:"EndNote Bibliography Title Char";
	mso-style-link:"EndNote Bibliography Title";
	font-family:"Calibri",sans-serif;}
p.EndNoteBibliography, li.EndNoteBibliography, div.EndNoteBibliography
	{mso-style-name:"EndNote Bibliography";
	mso-style-link:"EndNote Bibliography Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyChar
	{mso-style-name:"EndNote Bibliography Char";
	mso-style-link:"EndNote Bibliography";
	font-family:"Calibri",sans-serif;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;
	font-family:"Calibri",sans-serif;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;
	font-family:"Calibri",sans-serif;}
span.UnresolvedMention1
	{mso-style-name:"Unresolved Mention1";
	font-family:"Times New Roman",serif;
	color:#605E5C;
	background:#E1DFDD;}
span.msoIns
	{mso-style-name:"";
	text-decoration:underline;
	color:teal;}
span.msoDel
	{mso-style-name:"";
	text-decoration:line-through;
	color:red;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:595.3pt 841.9pt;
	margin:70.85pt 85.05pt 70.85pt 85.05pt;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#2E74B5" vlink="#954F72" style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitle align=center style='text-align:center'><b><i><span
style='font-size:36.0pt;letter-spacing:.25pt'><img width=310 height=355
id="Picture 16" src="User_Guide_files/image001.png" alt=Molecule></span></i></b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitleCxSpFirst align=center style='text-align:center'><span
class=MsoBookTitle><span style='font-size:36.0pt'>User Guide:</span></span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
class=MsoBookTitle><span style='font-size:36.0pt'>Help and documentation</span></span></p>

<p class=MsoTitleCxSpLast align=center style='text-align:center'><span
class=MsoBookTitle>(1.2.8)</span></p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1>Table of contents</h1>

</div>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Introduction">Introduction</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_License">License</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Installation">Installation</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Download">Download</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Available_operating_systems">Available operating systems</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:"Courier New";text-decoration:none'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
class=MsoBookTitle><span style='line-height:107%;letter-spacing:0pt;font-weight:
normal;font-style:normal'><a href="#_Windows_distribution">Windows distribution</a></span></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_MacOS_distribution">MacOS distribution</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Linux_distribution">Linux distribution</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Symbol;letter-spacing:
0pt;font-weight:normal;font-style:normal'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Get_Started">Getting Started</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Modules">Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Basic_modules">Relation Tables Module</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_RELS_CREATOR">RELS CREATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Basic_modules">Basic Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_LEVEL_CREATOR">LEVEL CREATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_LEVEL_CALIBRATOR">LEVEL CALIBRATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_INTEGRATE">INTEGRATE</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_NORCOMBINE">NORCOMBINE</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_RATIOS">RATIOS</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a href="#_SBT">SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Compound_modules">Compound Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WSPP-SBT_1">WSPP-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WSPPG-SBT_1">WSPPG-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WPP-SBT_1">WPP-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Wingdings;text-decoration:none'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WPPG-SBT_1">WPPG-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Report_modules">Report Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_REPORT">REPORT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Wingdings;text-decoration:none'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_SANSON">SANSON</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Special_parameters">Special Parameters</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Multiple_samples">Multiple samples</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Wingdings;color:windowtext;
text-decoration:none'>§<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Asterisk_is_our">Asterisk is our jack of all trades</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
class=MsoHyperlink>Multiple samples </span><a href="#_Multiple_samples_in">Multiple
samples in the inputs and outputs</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_More_params">More params</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Filter_param_(in">Filter (for REPORT and SANSON)</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Sample_Workflows_with">Sample Workflows with
Application to Case Studies</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Workflow_1:_One-step">Workflow 1: One-step
quantification in a labeled experiment</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Workflow_2:_Step-by-step">Workflow 2: Step-by-step
quantification and sample combination in a labeled experiment</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Workflow_3:_Quantification">Workflow 3: Quantification
of posttranslationally modified peptides in a labeled experiment</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:"Courier New";text-decoration:none'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Workflow_4:_Label-free">Workflow 4: Label-free quantification</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Importing_a_workflow">Importing a workflow template</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Creating_the_identification/quantif">Creating the
identification/quantification file from proteomics pipelines</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Preparing_the_ID-q">Preparing the ID-q file from
Proteome Discoverer output</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Preparing_the_ID-q_1">Preparing the ID-q file from
MaxQuant output</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Preparing_the_ID-q_2">Preparing the ID-q file from
FragPipe output</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Adapting_the_results">Adapting the results from
proteomics pipelines for iSanXoT</a></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_References">References</a></p>

<p class=MsoNormal><span class=MsoHyperlink><span style='text-decoration:none'>&nbsp;</span></span></p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk118388234"></a><a name="_Hlk118304308"></a><a
name="_Introduction"></a>Introduction</h1>

</div>

<p class=MsoNormal><b><span lang=EN-GB>iSanXoT</span></b><span lang=EN-GB> is a
standalone application for statistical analysis of mass spectrometry-based
quantitative proteomics data.  iSanXoT builds upon SanXoT </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>, our previous publicly available
implementation of the weighted spectrum, peptide, and protein (WSPP)
statistical model </span><span
lang=EN-GB>[2]</span><span lang=EN-GB> using the Generic Integration Algorithm
(GIA) </span><span
lang=EN-GB>[3]</span><span lang=EN-GB>.</span></p>

<p class=MsoNormal><span lang=EN-GB>iSanXoT executes several kind of workflows
for quantitative high-throughput proteomics, <b>systems biology </b>and the <b>statistical
analysis</b>, <b>integration and comparison of experiments</b>.</span></p>

<p class=MsoNormal><span lang=EN-GB>iSanXoT was developed by the <b><i>Cardiovascular
Proteomics Lab</i></b>/<b><i>Proteomic Unit</i></b> at <b>The National Centre
for Cardiovascular Research</b> (CNIC, </span><a href="https://www.cnic.es"><span
lang=EN-GB style='color:#4472C4'>https://www.cnic.es</span></a><span
lang=EN-GB>).</span></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Download</span></h2>

<p class=MsoNormal><span lang=EN-GB>The multiple releases are available in the
&quot;release&quot; section, located in the following link:</span></p>

<p class=MsoNormal><a href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Installation</span></h2>

<h3><span lang=EN-GB>Available operating systems</span></h3>

<p class=MsoNormal><span lang=EN-GB>iSanXoT maintains the following operating
systems and architectures and may add additional ones in the future:</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-GB>Windows 10 Pro
(x64)</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-GB>MacOs High
Sierra (10.13.6)</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-GB>Ubuntu 20.04
(x64)</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, read the “Installation” section
in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Installation"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Installation</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Getting Started</span></h2>

<p class=MsoNormal><span lang=EN-GB>This chapter describes iSanXoT’s graphical
user interface and how to set up an analysis with iSanXoT.</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, read the “Getting Started”
section in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Getting_started"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Getting_started</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Modules</span></h2>

<p class=MsoNormal><span lang=EN-GB>The iSanXoT desktop application houses a
number of modules based on the SanXoT software package </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>. The information required to setup and
execute every module is provided in a task-table.   </span></p>

<p class=MsoNormal><span lang=EN-GB>There are four types of modules:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Relation tables is a module that creates the
relation tables used by the iSanXoT modules.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>The Basic modules call the individual scripts
included in the SanXoT software package </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>The Compound modules perform a sequence of
consecutive integrations based on the weighted spectrum, peptide and protein
(WSPP) statistical model </span><span
lang=EN-GB>[2]</span><span lang=EN-GB> and the systems-biology triangle (SBT)
algorithm </span><span
lang=EN-GB>[3]</span><span lang=EN-GB>.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Finally, there are two Reports: REPORT generates
report files displaying the quantitative results produced by the above Basic
and Composite modules when a workflow is executed; SANSON generates a
similarity graph showing relationships between functional categories on the
basis of the protein elements they share.</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, read the “Modules” section
in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Modules"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Modules</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Input Adaptor</span></h2>

<p class=MsoNormal><span lang=EN-GB>The iSanXoT Input Adaptor offers users the
possibility to either provide their own Identification/Quantification file,
which contains the identification and quantification data, or have the Input
Adaptor prepare this file from the results obtained using any of the mainstream
proteomics pipelines.</span></p>

<p class=MsoNormal><span lang=EN-GB>For further details, read the “Input Adaptor”
section in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Input_adaptor"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Input_adaptor</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Sample Workflows</span></h2>

<p class=MsoNormal>We describe in detail four sample workflows that illustrate
the capacity of iSanXoT to statistically ascertain abundance changes in both
multiplexed, isotopically labeled [3-5] and label-free [6]
proteomics experiments.</p>

<p class=MsoListParagraphCxSpFirst align=left style='text-align:left;
text-indent:-.25in'><span lang=EN-GB style='font-family:Symbol'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_1"><span
lang=EN-GB>Workflow 1</span></a><span lang=EN-GB>: One-step quantification in a
labeled experiment.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_2"><span
lang=EN-GB>Workflow 2</span></a><span lang=EN-GB>: Step-by-step quantification
and sample combination in a labeled experiment.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_3"><span
lang=EN-GB>Workflow 3</span></a><span lang=EN-GB>: Quantification of
posttranslationally modified peptides in a labeled experiment.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_4"><span
lang=EN-GB>Workflow 4</span></a><span lang=EN-GB>: Label-free quantification.</span></p>

<p class=MsoNormal><span lang=EN-GB>For further details, read each section in the
iSanXoT wiki. </span></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>More Documents</span></h2>

<p class=MsoNormal><span lang=EN-GB>User Guide (v1.2.8):</span></p>

<p class=MsoNormal><a name="_License"></a><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/user_guides/User_Guide_iSanXoT-1.2.8.pdf"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/user_guides/User_Guide_iSanXoT-1.2.8.pdf</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Hlk118382155"><span lang=EN-GB>License</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>This application is licensed under a <b>Creative
Commons Attribution-NonCommercial-NoDerivs 4.0 Unported License</b>.</span></p>

<p class=MsoNormal><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><span
lang=EN-GB style='color:#4472C4'>https://creativecommons.org/licenses/by-nc-nd/4.0/</span></a></p>

<h4>You are free to:</h4>

<p class=MsoNormal>Share - copy and redistribute the material in any medium or
format.</p>

<p class=MsoNormal>The licensor cannot revoke these freedoms as long as you
follow the license terms.</p>

<h3>Under the following terms</h3>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/License"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/License</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk118375025"></a><a name="_Installation"></a>Installation</h1>

</div>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3><a name="_Download"></a>Download</h3>

<p class=MsoNormal><span lang=EN-GB>The multiple releases are available in the
&quot;release&quot; section, located in the following link:</span></p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span lang=EN-GB
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB>&nbsp;</span></p>

<h3><a name="_Available_operating_systems"></a><span lang=EN-GB>Available
operating systems</span></h3>

<p class=MsoNormal><span lang=EN-GB>iSanXoT maintains the following operating systems
and architectures and may add additional ones in the future:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Windows 10 Pro (x64)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>MacOs High Sierra (10.13.6)</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Ubuntu 20.04 (x64)</span></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3><a name=windows-distribution></a><a name=installation-on-windows></a><a
name="_Installation_on_Windows"></a><a name="_Windows_distribution"></a>Windows
distribution</h3>

<p class=MsoNormal>The iSanXoT Windows distribution is packaged in a NSIS
Launcher (exe file).</p>

<p class=MsoNormal>Download the exe Launcher: iSanXoT_Launcher_1.X.X.win32-x64.exe</p>

<p class=MsoNormal>Double-click the Launcher file; the Installer window will
show up:</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 28" src="User_Guide_files/image002.jpg"></span></p>

<p class=MsoNormal>WARNING: Windows Defender SmartScreen might show a prompt suggesting
that you cancel the installation; in such case click “More info” and then
select the “Run anyway” option.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=332 height=140 id="Picture 43" src="User_Guide_files/image003.png"></p>

<p class=MsoNormal>Then, you can choose the iSanXoT installation folder:</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 25" src="User_Guide_files/image004.jpg"></span></p>

<p class=MsoNormal>Wait while iSanXoT is being installed.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 22" src="User_Guide_files/image005.png"
alt="Installing window"></span></p>

<p class=MsoNormal>Once the installation has been completed, you are ready to run.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 21" src="User_Guide_files/image006.png"
alt="Completing setup"></span></p>

<p class=MsoNormal><span style='font-family:"Segoe UI",sans-serif;color:#24292E'>&nbsp;</span></p>

<h3><a name=macos-distribution></a><a name="_MacOS_distribution"></a>MacOS
distribution</h3>

<p class=MsoNormal>The iSanXoT MacOS distribution is packaged in a DMG
container.</p>

<p class=MsoNormal>Download the DMG file: iSanXoT_Launcher_1.X.X.darwin-x64.dmg</p>

<p class=MsoNormal>Double-click the DMG file, then a Finder window will show up.
This window will usually display iSanXoT’s installer icon and a shortcut to the
Applications folder, together with some sort of linking arrow:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=416 height=309 id="Picture 19" src="User_Guide_files/image007.png"
alt="Installer window"></p>

<p class=MsoNormal>Simply drag the iSanXoT icon to your Applications folder...</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=439 height=315 id="Picture 18" src="User_Guide_files/image008.png"
alt="Installing window"></span></p>

<p class=MsoNormal>and you're done: the iSanXoT application is now installed.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name=linux-distribution></a><a name="_Linux_distribution"></a>Linux
distribution</h3>

<p class=MsoNormal>The iSanXoT Linux distribution is packaged in an <i>AppImage</i>.</p>

<p class=MsoNormal>Download the AppImage: iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=567 height=241 src="User_Guide_files/image009.png"></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:0in'>The AppImage file is
just the application's compressed image. When executed, the application is
mounted in a temporal folder. However, for the correct behaviour, the application
has to be extracted to the &quot;squashfs-root&quot; folder in the current
working directory using:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>&nbsp;</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in'>Then, launch the
iSanXoT application using:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in'>&nbsp;</p>

<p class=MsoListParagraphCxSpLast style='margin-left:0in;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun</span></em></p>

<em><span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><span
style='font-family:"Calibri",sans-serif;font-style:normal'>&nbsp;</span></em></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Get_Started"></a><em><span style='font-family:"Calibri Light",sans-serif;
font-style:normal'>Getting Started</span></em></h1>

</div>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>This chapter describes iSanXoT’s graphical user interface and
how to set up an analysis with iSanXoT.</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3>Opening the iSanXoT application</h3>

<p class=MsoNormal>To open the iSanXoT application:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Windows: from the Start menu choose <i>Programs &gt; iSanXoT</i>;
or double-click the iSanXoT desktop icon.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In MacOS: double-click the iSanXoT icon from the Applications
folder.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Linux: from the <i>AppImage</i> file.</p>

<p class=MsoListParagraphCxSpMiddle>The contents are extracted to the &quot;squashfs-root&quot;
directory in the current working directory using:</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.x.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle>Now you can launch the iSanXoT application:</p>

<p class=MsoListParagraphCxSpLast style='text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun [...]</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3>Installing required packages</h3>

<p class=MsoNormal>The first time iSanXoT is run a window will show up displaying
a progress bar to inform you about the percentage of packages that have been
installed. These packages contain the libraries required by iSanXoT’s backend, and
are installed the first time you open the application.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=482 height=317 src="User_Guide_files/image010.png"></p>

<p class=MsoCaption><b>Figure </b><b>1</b><b>. <a name="_Ref103093720">Installation
window.</a></b></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3>Closing the iSanXoT application</h3>

<p class=MsoNormal>WARNING: If valid changes were made to your project, make
sure to save it before quitting iSanXoT, as any changes will be lost otherwise
(you won’t be prompted for saving upon closing).  </p>

<p class=MsoNormal style='margin-left:.25in'>To close the iSanXoT application:</p>

<p class=MsoNormal style='margin-left:.25in'>In Windows and Linux: choose <i>Project
&gt; Exit</i>, or click the X in the upper right corner of the main iSanXoT window.</p>

<p class=MsoNormal style='margin-left:.25in'>In Mac: choose <i>iSanXoT (menu)
&gt; Exit</i>, or click the red X in the upper left corner of the main iSanXoT
window.</p>

<p class=MsoNormal>A dialog window will show up asking you to confirm the
application closing. Click “Yes” if you really want to quit iSanXoT. </p>

<p class=MsoNormal>&nbsp;</p>

<h3>iSanXoT Projects</h3>

<p class=MsoNormal>An iSanXoT project is primarily a container used to
structure the data coming from your input file(s) and your workflow. The input
file contains the identification and quantification data (for further details see
the <a href="#_Input_Adaptor">Input Adaptor</a> Section). These fully-customisable
workflows can perform quantitative proteomics analysis, systems biology
analysis, and comparison and merging of experimental data from technical or
biological replicates.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=251 height=240 src="User_Guide_files/image011.png"></p>

<p class=MsoCaption><a name="_Ref103114194"><b>Figure </b></a><b>2</b><b>. Project menu.</b></p>

<h4>Creating a new project</h4>

<p class=MsoNormal>Creating a project is the first step when conducting an
analysis with iSanXoT. By selecting <i>Project &gt; New Project</i> a window
will show up where you can provide a name for the project as well as select a
project folder where iSanXoT output files will be stored.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=536 height=199 src="User_Guide_files/image012.png"></p>

<p class=MsoCaption><b>Figure </b><b>3</b><b>. Window that creates a new
project.</b></p>

<h4>&nbsp;</h4>

<h4>Opening a project</h4>

<p class=MsoNormal>By selecting <i>Project &gt; Open Project</i> as a folder
selection dialog box shows up that allows the user to indicate the location of an
already existing project folder to be opened by iSanXoT.</p>

<p class=MsoNormal>&nbsp;</p>

<h3>iSanXoT Main Window</h3>

<p class=MsoNormal>The iSanXoT main window consists of an overhead menu, adaptor
and module tabs, and content and execution panels (<b>Figure 4</b>).</p>

<h4>Menu</h4>

<p class=MsoNormal>The Menu contains the following items:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Project: Allows operations related to projects and workflows.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Processes: Links to real-time display of the processes currently
in execution by iSanXoT (see <a href="#_Running_Processes">Running Processes</a>
Section).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Help: Houses the different Sections of iSanXoT help.</p>

<h4>Content tabs</h4>

<p class=MsoListParagraph style='text-indent:-.25in'>1.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Five tabs are displayed in iSanXoT’s project page. The Input tab
displays the Project folder, where iSanXoT output files are stored, as well as
the Identification file used in the project (see <a href="#_Adaptors">Input Adaptor</a>
Section). The remaining four tabs give access to iSanXoT modules: <a
href="#_Relation_tables_module">Relation Tables</a>, <a href="#_Basic_modules_1">Basic
Modules</a>, <a href="#_Compound_modules">Compound Modules</a>, and <a
href="#_Reports_modules">Report Modules</a>.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=570 height=337 id="Picture 132"
src="User_Guide_files/image013.png"></p>

<p class=MsoCaption><a name="_Ref103114143"></a><a name="_Ref103114150"><b>Figure
</b></a><b>4</b><b>. Main View of iSanXoT.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Content panel</h4>

<p class=MsoNormal>This panel houses the elements of the Input and Modules tabs,
which can be accessed through the sidebar menu showing on the left side of the panel.
A title and a brief description of the Input/Module element is provided, as
well as help icon linking to additional information on the specific Input
element or Module selected.</p>

<h4><a name="_Execution_panel"></a>Execution panel</h4>

<p class=MsoNormal>The execution panel, located on the bottom of the main
window, allows the user to indicate the number of processors to be used by
iSanXoT, 4 by default. The Start button launches the execution of the workflow
shaped by the Input elements and Modules.</p>

<p class=MsoNormal>&nbsp;</p>

<h2>Importing and Exporting Workflows</h2>

<p class=MsoNormal>A project is shaped by a workflow that instructs iSanXoT how
to process the data provided by the input file(s). While the whole project,
including workflow and data, can be saved as indicated below (see the <a
href="#_Saving_a_project">Saving a project</a> section), there is a way to
import and export just the workflow structure using iSanXoT menu (<b>Figure 2</b>).</p>

<h4>Import Workflow</h4>

<p class=MsoNormal>This option allows to import the task-tables of a workflow.
For that, you have to provide the folder where the workflow is saved.</p>

<h4>Export Workflow</h4>

<p class=MsoNormal>The export workflow saves the task-tables of a workflow in
the folder indicated by the user.</p>

<p class=MsoNormal>&nbsp;</p>

<h2>Executing a Project</h2>

<p class=MsoNormal>Once your project contains all the necessary input data and
workflow elements, you can execute the workflow by clicking “Save and Run” in
the “Execution panel” after indicating the number of processors to be allocated
for iSanXoT (see  <a href="#_Execution_panel">Execution panel</a> above). </p>

<p class=MsoNormal>Bear in mind that every time you click the “Save and Run”
button to execute a workflow, the project is first validated for consistency and
saved. To save a project without executing it, you must use the appropriate
menu item as explained in the next Section.</p>

<h4><a name="_Saving_a_project"></a>Saving a project</h4>

<p class=MsoNormal>The <i>Project &gt; Save Project</i> option saves your
project, which contains the input data and the workflow elements, to the
“Project folder”. The project files are saved in the “.isanxot” folder. WARNING:
Do not manipulate or delete the information stored in the “.isanxot” folder;
you risk losing your project.</p>

<p class=MsoNormal>Whenever iSanXoT is prompted to save a project, the corresponding
workflow is first validated for consistency and won’t be saved when failed. Neither
will the workflow execute if it has not been validated previously.</p>

<p class=MsoNormal>As well as the “.isanxot” folder, the following folders are
necessary to shape your project:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Exps, to store the files created by the Input Data adapters.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Jobs, to store the <i>sample folders</i> of your workflow.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Rels, to store the Relation Tables created by the RELS CREATOR module
(see below).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Reports, to store the Report files created by the REPORT module (see below).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Stats, to store statistical data.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Logs, to store the workflow execution log files.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Running_Processes"></a>Running Processes</h2>

<p class=MsoNormal>When workflow execution successfully starts, a new window shows
up displaying information about the processes currently running:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Project logs table shows project execution status. Several
project executions can be monitored here, and the user must click a row to have
the corresponding workflow logs displayed (see below).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Workflow logs table displays status for the jobs set up in the
workflow modules. If you click on a row you will see the trace log of the involved<span
class=MsoCommentReference><span style='font-size:8.0pt;line-height:107%'> j</span></span>obs
(unless the job status is “cached”).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=378 height=262 id="Picture 98"
src="User_Guide_files/image014.png"></p>

<p class=MsoCaption><b>Figure </b><b>5</b><b>. View of running processes.</b></p>

<p class=MsoNormal align=left style='text-align:left'>The running processes window
can be also reached from the menu by selecting the “Processes &gt; Main page” option.</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Modules"></a>Modules</h1>

</div>

<p class=MsoNormal>The iSanXoT desktop application houses a number of modules
based on the SanXoT software package [1].
The information required to setup and execute every module is provided in a task-table. 
 </p>

<p class=MsoNormal>There are four types of modules:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Relation tables is a module that creates the relation tables used
by the iSanXoT modules.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Basic modules call the individual scripts included in the SanXoT
software package [1].</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Compound modules perform a sequence of consecutive
integrations based on the weighted spectrum, peptide and protein (WSPP)
statistical model [2] and the systems-biology triangle (SBT) algorithm [3].</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Finally, there are two Reports: REPORT generates report files
displaying the quantitative results produced by the above Basic and Composite
modules when a workflow is executed; SANSON generates a similarity graph
showing relationships between functional categories on the basis of the protein
elements they share.</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Basic_modules"></a><a
name="_Relation_tables_module"></a>Relation tables module</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_RELS_CREATOR"></a><span style='font-family:"Calibri",sans-serif'>RELS
CREATOR</span></h3>

<p class=MsoNormal>This module generates relation tables (RT) from tab-separated
values (TSV) files. Relation tables, which are TSV files relating lower level
identifiers (e.g. peptides) to the corresponding higher level elements (e.g.
proteins), are required for module execution. For this reason, the naming convention
for the file indicated under “Relation Table to be created” in the RELS CREATOR
task table is <i>lower level + “2” + higher level</i>; e.g. whether an
integration is from the “peptide” level to the “protein” level, a relation
table called “peptide2protein” will be necessary.</p>

<p class=MsoNormal>The fields showing in the RELS CREATOR task table are (<b>Figure 6</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Relation Table to be created</i> specifies relation table
filenames. As commented above, the naming convention for these files is <i>lower
level + “2” + higher level</i>; e.g. whether an integration is from the
“peptide” level to the “protein” level, a relation table called
“peptide2protein” will be necessary.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of Lower level</i> is the column header that
designates which elements from the indicated file (see below) will be taken as
lower level elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of Higher level</i> is the column header that
designates which elements from the indicated file (see below) will be taken as higher
level elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of 3<sup>rd</sup> column</i> is the column header
that designates which elements from the indicated file (see below) will be
taken as third column elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Table from which RT is extracted is the full path name for the
TSV file to be used to build the relation tables. If the cell is empty, the
Input file (ID-q.tsv) is applied for iSanXoT workflow.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=163 src="User_Guide_files/image015.png"></p>

<p class=MsoCaption><a name="_Ref103114314"><b>Figure </b></a><b>6</b><b>.
A sample Task-Table in the example of RELS CREATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Basic_modules_1"></a>Basic
modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_LEVEL_CREATOR"></a><span style='font-family:"Calibri",sans-serif'>LEVEL
CREATOR</span></h3>

<p class=MsoNormal>This module creates levels, which are TSV files containing
identifiers, log<sub>2</sub>-ratio values and statistical weight values. The
data are extracted from the Identification/Quantification file (see <a
href="#_Adaptors">Input Adaptor</a> section).</p>

<p class=MsoNormal>The following fields are displayed in the LEVEL CREATOR task
table (<b>Figure 7</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used to create the level
indicated. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator column</i> specifies which column header from
the identification file designates the quantitative values to be used as a numerator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator column(s)</i> specifies which column header
from the identification file designates the quantitative values to be used as a
denominator for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level to be created</i> designates the level name.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates the name of the folder
where the level data file will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=581 height=174 src="User_Guide_files/image016.png"></p>

<p class=MsoCaption><a name="_Ref103114332"><b>Figure </b></a><b>7</b><b>.
A sample task-table in the LEVEL CREATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_LEVEL_CALIBRATOR"></a><span style='font-family:"Calibri",sans-serif'>LEVEL
CALIBRATOR</span></h3>

<p class=MsoNormal>This module calibrates the above-described levels using the
“Klibrate” program included in the SanXoT software package [1].
To perform the calibration, two parameters (weight constant and variance) are iteratively
calculated using the Levenberg-Marquardt algorithm (for more details see the information
about “Klibrate” in the SanXoT software package [1]).</p>

<p class=MsoNormal>The output calibrated level contains new statistic weight
values for the identifier and log<sub>2</sub>-ratio elements displayed in the
uncalibrated level data file. This is necessary for the levels to be used as
inputs to the INTEGRATE module.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The fields to be completed in this module are (<b>Figure
8</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name(s) of the folder(s) containing
the uncalibrated data file(s) that were previously generated by the LEVEL
CREATOR module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level for integration</i> indicates which lower level elements
are to be used in the integration carried out for the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level for integration</i> indicates which higher level
elements are to be used in the integration carried out for the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Name of calibrated level</i> is the name for the output data
file containing the new, calibrated statistical weight values.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> specifies the name of the folder
where the output data file containing the new, calibrated statistical weight
values will be saved. If the cell is empty, the output sample folder is the
given “Sample folder” (second column).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=193 id="Picture 5" src="User_Guide_files/image017.png"></p>

<p class=MsoCaption><a name="_Ref103114351"><b>Figure </b></a><b>8</b><b>.
A sample task table in the LEVEL CALIBRATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The LEVEL_CALIBRATOR module accepts the following additional
parameters (<b>Figure 9</b>): </p>

<p class=MsoNormal>To perform the calibration two parameters, have to be
calculated: the k (weight constant), and the variance.</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>K-constant </i>sets a forced value for the k-constant. Using
this parameter, the introduced value is forced as K-constant.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Var(x)</i> sets a forced value for the variance. Using this
parameter, the introduced value is forced as the variance.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=579
height=145 id="Picture 11" src="User_Guide_files/image018.png"></p>

<p class=MsoCaption style='text-align:justify'><a name="_Ref103114398"><b>Figure
</b></a><b>9</b><b>. A task-table displaying advanced parameters for the LEVEL
CALIBRATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_INTEGRATE"></a><span style='font-family:"Calibri",sans-serif'>INTEGRATE</span></h3>

<p class=MsoNormal>The INTEGRATE module performs statistical calculations based
on the WSPP model by iteratively applying the generic integration algorithm
(GIA) [3] on calibrated data files (<b>Figure 10</b>). </p>

<p class=MsoNormal>Integrations are carried out from lower level data to higher
level data (e.g. from the peptide level to the protein level and from the protein
level to the gene level).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=423 height=414 id="Picture 58"
src="User_Guide_files/image019.png"></p>

<p class=MsoCaption><a name="_Ref103096863"><b>Figure </b></a><b>10</b><b>. Schematic representation of the INTEGRATE module.</b> The integration is carried out
from any lower level to any higher level using the programs “SanXoT” and
“SanXoTSieve” and the generic integration algorithm (GIA).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>More in detail, the INTEGRATE module needs two TSV files as
inputs:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>A
data file containing three data columns: identifier (a text string that is used
to unambiguously identify the low level elements), quantitative value (log<sub>2</sub>-ratio
of the two measurements to be compared) and statistical weight (a parameter
that measures the accuracy of the quantitative value).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>A
relation table, which links the lower level identifiers to those in the higher
level. This file contains two columns: higher level identifiers on the left and
lower level identifiers on the right.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=582 height=524 id="Picture 41"
src="User_Guide_files/image020.png"></p>

<p class=MsoCaption><a name="_Ref103114442"><b>Figure </b></a><b>11</b><b>.
The INTEGRATE module flowchart.</b> A first integration is done with “SanXoT”
that calculates the variance; then “SanXoTSieve” removes outliers tagging them
in a new relation table; finally, a second integration is done with “SanXoT”
using the variance calculated.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For every integration, the SanXoT program calculates the general
variance using a robust iterative method. Then SanXoTSieve is used to tag
outlier elements [2] by assessing the probability that a lower level element be
a significant outlier of the standardized (i.e. N(0,1)) log<sub>2</sub>-ratio
distribution. The most extreme outliers are thus removed sequentially and the
integration repeated until all outliers below a user-defined false discovery
rate (FDR) threshold have been removed. Finally, a second integration is carried
out by SanXoT using the variance calculated in the first integration and
discarding the outliers tagged in the new relation table (<b>Figure
11</b>).</p>

<p class=MsoNormal>The output data files generated by INTEGRATE contain the
quantitative data for the higher level and can be used as inputs to other
modules; in addition, each integration generates several additional files which
contain information about the integration. For further details see SanXoT documentation
[1].</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The parameters to be provided in the INTEGRATE module task
table are (<b>Figure 12</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the names of the folder(s)
where the lower level data file is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level data
file to be used. This file contains three data columns: identifier,
quantitative value and statistical weight.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the name of the higher level to which
the lower level elements will be integrated.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=282 height=159 id="Picture 6" src="User_Guide_files/image021.png"></p>

<p class=MsoCaption><a name="_Ref103114454"><b>Figure </b></a><b>12</b><b>.
A sample task-table in the INTEGRATE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4><a name="_Advanced_parameters"></a>Advanced parameters</h4>

<p class=MsoNormal>The INTEGRATE module accepts the following additional parameters
(<b>Figure 13</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> is the name of the folder where the level
data and statistics are saved.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the
lower_level2higher_level Relation Table will be integrated. Logical operators
can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><i>Tag</i> is a parameter to distinguish
groups to perform the integration. For instance, if the user specifies “marked”
the elements containing the label “marked” in the third column of the “lower_level2higher_level”
Relation Table will be included in (or discarded from) the integration.</p>

<p class=MsoListParagraphCxSpMiddle>The tag can be used by inclusion, such as &quot;mod&quot;
or by exclusion, putting first the &quot;!&quot; symbol, such as &quot;!mod&quot;. 
Tags should be included in a third column of the relations file.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>Different tags can be combined using
logical operators &quot;and&quot; (&amp;), &quot;or&quot; (|), and
&quot;not&quot; (!), and parentheses. Some examples:</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>!out&amp;mod</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>!out&amp;(dig0|dig1)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>(!dig0&amp;!dig1)|mod1</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>mod1|mod2|mod3</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b><i>Warning</i></b>: Unless specified
otherwise by the user, by default iSanXoT eliminates outliers from the lower
level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle>iSanXoT automatically adds the tag “out” in
the third column of the relation table to label outliers, so that they are not
integrated. It is not thus recommended to use this tag for other purposes.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b><i>Note</i></b> that although the
discarded elements will not be included in calculations, the parameter Z will
be calculated and tabulated in the corresponding output (outStats) file.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>For further details see SanXoT wiki</p>

<p class=MsoListParagraphCxSpMiddle>(<a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a>).</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>FDR</i> is an FDR threshold other than the default value (0.01,
i.e. 1%) for outlier removal. If “0” is specified as the FDR value, then no outliers
will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Var(x)</i> sets a fixed value for the variance. The default
value (blank) means that the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first <i>Lower level-to-Higher level</i>
integration.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=136 src="User_Guide_files/image022.png"></p>

<p class=MsoCaption><a name="_Ref103114466"><b>Figure </b></a><b>13</b><b>.
A sample task-table displaying advanced parameters for the INTEGRATE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_NORCOMBINE"></a><span style='font-family:"Calibri",sans-serif'>NORCOMBINE</span></h3>

<p class=MsoNormal>The NORCOMBINE module combines technical or biological
replicates (<b>Figure 14</b>). For example, NORCOMBINE can be used to merge the
protein level data from 4 individual patients and 4 individual controls into a
patient- and a control level protein dataset, respectively.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=451 height=388 id="Picture 114"
src="User_Guide_files/image023.png"></p>

<p class=MsoCaption><a name="_Ref103114484"><b>Figure </b></a><b>14</b><b>. Schematic representation of the NORCOMBINE module used to combine technical or
biological replicates.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Experiment merging relies on the “Cardenio” program from the
SanXoT software package [1],
which is used to generate merged data files and relation tables that are later
integrated to the grouped level using “SanXoT” and “SanXoTSieve” (<b>Figure 14</b>).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>NORCOMBINE requires the user to specify which <i>lowerNorm</i>
files contain the necessary data for the samples to be combined. These <i>lowerNorm</i>
files, previously generated by the INTEGRATE module, display the lower level
identifiers on the left, followed by the corresponding centred log<sub>2</sub>-ratio
values (i.e. the values obtained after subtracting the high level value) in the
second column, and either the integration statistical weight (in the case of <i>lowerNormV</i>)
or the variance (for <i>lowerNormW).</i> The SanXoT program “Cardenio” [1]
is then used to generate merged data files and relation tables that are later
integrated to the grouped level using “SanXoT” and “SanXoTSieve” (<b>Figure 15</b>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=452 height=522 id="Picture 115"
src="User_Guide_files/image024.png"></p>

<p class=MsoCaption><a name="_Ref103114585"><b>Figure </b></a><b>15</b><b>.
The NORCOMBINE module flowchart.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The default NORCOMBINE module task table shows the following
fields (<b>Figure 16</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folders</i> indicates the names of the folder(s) containing
the lower level data (samples) to be combined.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level</i> indicates the type of elements to be combined (e.g.
peptides or proteins). </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Norm</i> specifies the normalization scheme to be used in the
integrations.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>lowerNorm</i> specifies the type of <i>lowerNorm</i> file (see
above) to be used.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> is the name of the folder where the grouped
level data and statistics are saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=467 height=89 src="User_Guide_files/image025.png"></p>

<p class=MsoCaption><a name="_Ref103114601"><b>Figure </b></a><b>16</b><b>.
A sample task-table in the NORCOMBINE module.</b> In this case, the asterisk
wildcard has been used to select multiple sample folders.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The NORCOMBINE module accepts the following additional parameters
(<b>Figure 17</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>FDR</i> is an FDR threshold other than the default value
(0.01, i.e. 1%) for outlier removal. If “0” is specified as the FDR value, then
no outliers will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Var(x) sets a fixed value for the variance. The default value
(blank) means that the variance will be iteratively calculated based on the
Levenberg-Marquardt algorithm.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=72 src="User_Guide_files/image026.png"></p>

<p class=MsoCaption><a name="_Ref103114615"><b>Figure </b></a><b>17</b><b>.
A sample task table with advanced parameters in the NORCOMBINE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_RATIOS"></a><span style='font-family:"Calibri",sans-serif'>RATIOS</span></h3>

<p class=MsoNormal>This module prepares the data file and relation table
required as a first step in the calculation of a ratio defined by the user (e.g
KO vs WT). For that, the new log<sub>2</sub>-ratio is calculated as the difference
between numerator and denominator values, whereas the corresponding statistical
weight is assessed according to the method indicated by the user in the V
Method filed of the RATIOS task table (<b>Figure 18</b> and <b>Figure
19</b>): </p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>max</i> uses the maximum value between the numerator and
denominator statistical weight value. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>form</i> uses the value resulting from 1/(1/Vn + 1/Vd), where
Vn and Vd are the statistical weight value for the numerator and the
denominator, respectively.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>avg</i> uses the average value between the numerator and
denominator statistical weight value.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=581
height=199 id="Picture 17" src="User_Guide_files/image027.png"></p>

<p class=MsoCaption><a name="_Ref103114679"><b>Figure </b></a><b>18</b><b>.
The RATIOS module flowchart.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The RATIOS module task table displays the following parameter
fields (<b>Figure 19</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator column</i> specifies the name of the folder
containing the quantitative value to be used as a numerator for the new log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator column(s)</i> specifies the name of the
folder(s) containing the quantitative values to be used as a denominator for
the new log<sub>2</sub>-ratio calculation. The sample folders have to be
separated by comma.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level </i>designates the level (i.e. peptide, protein, gene or
category) at which the ratio is to be calculated.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates the name of the folder
where the resulting log<sub>2</sub>-ratio and statistical weight values will be
saved (e.g. KO_vs_WT).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=574 height=65 src="User_Guide_files/image028.png"></p>

<p class=MsoCaption><a name="_Ref103114687"><b>Figure </b></a><b>19</b><b>.
A sample task-table in the RATIOS module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_SBT"></a><span style='font-family:"Calibri",sans-serif'>SBT</span></h3>

<p class=MsoNormal>This module is based on the Systems Biology Triangle (SBT)
algorithm [3], which performs an integration between the lower and the higher
levels using the variance previously obtained in an integration between the
lower and an intermediate level (<b>Figure 20</b>). Usually the SBT module is
applied to carry out the protein-to-grand mean integration using the variance
associated with the protein-to-category integration.   </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=436 height=435 id="Picture 131"
src="User_Guide_files/image029.png"></p>

<p class=MsoCaption><a name="_Ref103093737"></a><a name="_Ref103093681"><b>Figure
</b></a><b>20</b><b>. <a name="_Ref103093692">Schematic representation of the SBT
module.</a></b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters to be entered in the SBT module task
table are (<b>Figure 21</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name of the folder(s) where
the lower level data are located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level (e.g. protein).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Intermediate level</i> indicates the name of the intermediate level
(e.g. category).</p>

<p class=MsoNormal>By default, the higher level is the grand mean of the lower
level elements.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=292 height=66 src="User_Guide_files/image030.png"></p>

<p class=MsoCaption><a name="_Ref103094105"><b>Figure </b></a><b>21</b><b>.
A sample task table in the SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts the following additional parameters (<b>Figure 22</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates an alternative folder to
store the resulting log<sub>2</sub>-ratio and statistical weight values other
than “Sample folder(s)”.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower-Higher level</i> and <i>Int(ermediate)-Higher level </i>specify
an alternative higher level other than the grand mean of the lower level elements.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate)</i> Tag and <i>Int(ermediate)-to-Hig(her)</i>
are the text label that indicates which elements from the lower level are
integrated into the intermediate level, and the intermediate level are
integrated into higher level. The tags must be specified in the third column of
the corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label “marked” is used as a Tag, only the lower level elements containing
the label marked in the third column of the lower_level2intermediate_level and
intermediate_level2higher_level Relation Tables will be integrated. Logical
operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate) FDR </i>and<i> Int(ermediate)-to-Hig(her)
FDR</i> determine an FDR threshold other than the default value (0.01, i.e. 1%)
for outlier removal in the lower level-to-intermediate level and intermediate
level-to-higher level integration, respectively. If “0” is specified as the FDR
value, then no outliers will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate) Var(x)</i> and <i>Int(ermediate)-to-Hig(her)
Var(x)</i> indicate the variance to be used in the lower level-to-intermediate
level and intermediate level-to-higher level integration, respectively, as an
alternative to the variance calculated in the lower level-to-intermediate level
integration.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=554 height=168 id="Picture 10"
src="User_Guide_files/image031.png"></p>

<p class=MsoCaption><a name="_Ref103094086"><b>Figure </b></a><b>22</b><b>.
A sample task table with advanced parameters in the SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Compound_modules"></a>Compound
modules</h2>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The Compound modules perform a sequence of consecutive
integrations based on the WSPP statistical model [2]
and the SBT algorithm [3]. In addition, each module creates the initial level
and calibrate this initial level. The WSPP-SBT and WSPPG-SBT modules create and
calibrate the “scan” level; and the WPP-SBT and WPPG-SBT create and calibrate
the “peptide” level.</p>

<h3><a name="_WSPP-SBT_1"></a><span style='font-family:"Calibri",sans-serif'>WSPP-SBT</span></h3>

<p class=MsoNormal>The WSPP-SBT module performs the following integrations:
scan-to-peptide, peptide-to-protein, protein-to-category, protein-to-proteinall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the protein-to-category integration, which
is applied to the protein-to-proteinall integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WSPP-SBT module are
(<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=564 height=227 src="User_Guide_files/image032.png"></p>

<p class=MsoCaption><a name="_Ref103096252"><b>Figure </b></a><b>23</b><b>.
A sample task table in the WSPP-SBT, WSPPG-SBT, WPP-SBT, and WPPG-SBT modules.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WSPP-SBT module accepts the following advanced
parameters (<b>Figure 24</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text
label that indicates which elements from the lower level (“p” and “c”) are
integrated into the higher level (“q” and “a”). The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.<span
class=MsoCommentReference><span style='font-size:8.0pt;line-height:107%'> </span></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p FDR</i>, <i>p&gt;q FDR</i>, <i>q&gt;c FDR</i> establish
an FDR threshold other than 0.01 (1%) for outlier removal in the integrations
scan-to-peptide, peptide-to-protein, and protein-to-category, respectively. If
FDR = 0 is selected, then the outliers are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p Var(x)</i>, <i>p&gt;q Var(x)</i>, <i>q&gt;c Var(x)</i>
set a fixed value for the variance in the integrations scan-to-peptide, peptide-to-protein,
and protein-to-category, respectively. By default, the variance will be
iteratively calculated based on the Levenberg-Marquardt algorithm in the first
lower level-to-higher level integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=111 id="Picture 23"
src="User_Guide_files/image033.png"></p>

<p class=MsoCaption><a name="_Ref103096290"><b>Figure </b></a><b>24</b><b>. Task-Table with advanced parameters in the WSPP-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_WSPPG-SBT_1"></a><span style='font-family:"Calibri",sans-serif'>WSPPG-SBT</span></h3>

<p class=MsoNormal>The WSPPG-SBT module performs the following integrations:
scan-to-peptide, peptide-to-protein, protein-to-gene, gene-to-category, gene-to-geneall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the gen-to-category integration, which is
applied to the gen-to-geneall integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WSPPG-SBT module are
(<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WSPPG-SBT module accepts the following advanced
parameters (<b>Figure 25</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>q&gt;g Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a
Tag</i> are the text label that indicates which elements from the lower level (“p”,
“q” and “c”) are integrated into the higher level (“q”, “g” and “a”). The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label “marked” is used as
a Tag, only the lower level elements containing the label marked in the third
column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p FDR</i>, <i>p&gt;q FDR</i>, <i>q&gt;g FDR</i>, <i>g&gt;c
FDR</i> establish an FDR threshold other than 0.01 (1%) for outlier removal in
the following integrations: scan-to-peptide, peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. If FDR = 0 is selected, then the outliers
are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p Var(x)</i>, <i>p&gt;q Var(x)</i>, <i>q&gt;g Var(x)</i>,
<i>g&gt;c Var(x)</i> set a fixed value for the variance in the integrations:
scan-to-peptide, peptide-to-protein, protein-to-gene, and gene-to-category,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=66 id="Picture 24" src="User_Guide_files/image034.png"></p>

<p class=MsoCaption><a name="_Ref103096474"><b>Figure </b></a><b>25</b><b>. Task-table with advanced parameters in the WSPPG-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_WPP-SBT_1"></a><span style='font-family:"Calibri",sans-serif'>WPP-SBT</span></h3>

<p class=MsoNormal>The WPP-SBT module performs the integrations peptide-to-protein,
protein-to-category, protein-to-proteinall and category-to-categoryall. In
addition, the SBT algorithm is used to calculate the variance associated to the
protein-to-category integration, which is applied to the protein-to-proteinall
integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WPP-SBT module are (<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WPP-SBT module accepts the following advanced parameters
(<b>Figure 26</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text
label that indicates which elements from the lower level (“p” and “c”) are
integrated into the higher level (“q” and “a”). The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation Table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q FDR</i>, <i>q&gt;c FDR</i> establish an FDR threshold
other than 0.01 (1%) for outlier removal in the peptide-to-protein and protein-to-category
integrations, respectively. If FDR = 0 is selected, then the outliers are not
discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Var(x)</i>, <i>q&gt;c Var(x)</i> set a fixed value for
the variance in the peptide-to-protein and protein-to-category integrations,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=494 height=90 id="Picture 29" src="User_Guide_files/image035.png"></p>

<p class=MsoCaption><a name="_Ref103096698"><b>Figure </b></a><b>26</b><b>. Task-table with advanced parameters in the WPP-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_WPPG-SBT_1"></a><span style='font-family:"Calibri",sans-serif'>WPPG-SBT</span></h3>

<p class=MsoNormal>The WPPG-SBT module performs the integrations peptide-to-protein,
protein-to-gene, gene-to-category, gene-to-geneall, and category-to-categoryall.
In addition, the SBT algorithm is used to calculate the variance associated to
the gene-to-category integration, which is applied to the gene-to-geneall
integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WPPG-SBT module are
(<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WPPG-SBT module accepts the following advanced
parameters (<b>Figure 27</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>q&gt;g Tag, p&gt;a Tag, q&gt;a Tag,</i> <i>c&gt;a
Tag</i> are the text label that indicates which elements from the lower level (“p”,
“q” and “c”) are integrated into the higher level (“q”, “g” and “a”). The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label “marked” is used as
a Tag, only the lower level elements containing the label marked in the third
column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q FDR, q&gt;g FDR</i>, <i>g&gt;c FDR</i> establish an FDR
threshold other than 0.01 (1%) for outlier removal in the integrations peptide-to-protein,
protein-to-gene, and gene-to-category, respectively. If FDR = 0 is selected,
then the outliers are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Var(x)</i>, <i>q&gt;g Var(x), g&gt;c Var(x)</i> set a
fixed value for the variance in the integrations peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. By default, the variance will be
iteratively calculated based on the Levenberg-Marquardt algorithm in the first
lower level-to-higher level integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=75 id="Picture 47" src="User_Guide_files/image036.png"></p>

<p class=MsoCaption><a name="_Ref103097509"><b>Figure </b></a><b>27</b><b>. Task-table with advanced parameters in the WPPG-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Report_modules"></a><a
name="_Reports_modules"></a>Reports modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_REPORT"></a><span style='font-family:"Calibri",sans-serif'>REPORT</span></h3>

<p class=MsoNormal>The REPORT module allows the collection of the statistical variables
(n, tags, Xinf, Vinf, Xsup, Vsup, Z, and FDR) from the different integrations
performed into result tables. </p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters showing in the REPORT task table are
(<b>Figure 28</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name(s) of the folder(s)
where the values of the statistical variables to be retrieved are located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the starting level (i.e. peptide,
protein, or category) for the integration whose statistical variables are to be
reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the ending level for the
integration whose statistical variables are to be reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Reported vars</i> specifies which statistical variables will
be reported. The available variables are n, tags, Xinf, Vinf, Xsup, Vsup, Z,
FDR, X’inf and Winf.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output report</i> is the report filename (without extension).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=535 height=318 src="User_Guide_files/image037.png"></p>

<p class=MsoCaption><a name="_Ref103115700"><b>Figure </b></a><b>28</b><b>.
A sample task table in the REPORT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For instance, the first row of the task table shown in <b>Figure 28</b> prompts the REPORT module to read the variable “n” from the <i>scan2peptide_outStats.tsv</i>
file that contains the statistical outcome from the scan-to-peptide integration
(the asterisk wildcard character in <i>Sample folder(s)</i> causes REPORT to
retrieve the “n” variable from every sample). These “n” values are written to a
report file named “Nscan_pep” that is stored in the project “reports” folder.</p>

<p class=MsoNormal>The second row instructs the module to read the variables “Xinf”,
“Z” and “FDR” from the statistical outcome of the peptide-to-protein
integration (once again for every sample). These values are written to a report
file named “Nscan_Normpep_prot_XZ”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The REPORT module accepts the following advanced parameters
(<b>Figure 29</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level names to show</i> allows the user to restrict the
elements to be written to the <i>Output report</i> to those from the <i>Lower
level</i> or the <i>Higher level</i>. Both levels are used by default.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Merge with report</i> designates the file whose <i>Reported
vars</i> will be incorporated into the <i>Output report</i> after intersection
with the latter file.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Add columns from relation table</i> appends <i>Lower level</i>
elements, extracted from the relation table designated, to the <i>Output report</i>.
It is possible to indicate multiple relation tables separated by a comma.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Filter</i> allows to filter the data to be transferred to the <i>Output
report</i> based on the <i>Reported vars</i> (n, Z, FDR, etc.). For more details,
see <a href="#_Filter_(for_REPORT">Filter</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=261 id="Picture 4" src="User_Guide_files/image038.png"></p>

<p class=MsoCaption><a name="_Ref103116151"><b>Figure </b></a><b>29</b><b>.
A sample task-table with advanced parameters in the REPORT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The reports indicated under <i>Output report</i> and <i>Merge
with report</i> are merged according to the column header that they share.
Thus, the REPORT task table shown in <b>Figure 30</b> will cause the module to incorporate
the number of scans per peptide, displayed in the report “Nscan_pep”, to the
report “Nscan_Normpep_prot_XZ”, as these two reports share the lower level
elements showing under the “peptide” header.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=600 height=156 id="Picture 26"
src="User_Guide_files/image039.png"></p>

<p class=MsoCaption><a name="_Ref103116369"><b>Figure </b></a><b>30</b><b>.
Report merging in the REPORT module.</b> The first task table row creates a
report file (“Nscan_pep”) with the (n)umber of scans per peptide. The second
row creates a report file called “Nscan_Normpep_prot_XZ” that contains, apart
from the variables “Xinf”, “Z”, and “FDR” coming from the peptide-to-protein
integration, the (n)umber of scans per peptide previously stored in the “Nscan_pep”
report, as these two reports share the lower level elements showing under the
“peptide” header.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>In addition, it is possible to incorporate additional data
from one or more relation tables into the reports (<b>Figure 31</b>). When a
given Relation Table is indicated under <i>Add columns from relation table</i>,
the REPORT module will first attempt to incorporate to the <i>Output report</i>
(“Npep_Quanprot_cat” in the example) the elements related to the <i>Lower level</i>
(“protein” in this case) elements in the relation table. If the <i>Lower level</i>
elements are missing in the relation table, then REPORT will try to incorporate
the elements related to the Higher level (“category” in this case) instead. If
neither the <i>Lower level</i> nor the <i>Higher level</i> can be found in the
relation table, then no action is performed. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=573 height=101 id="Picture 27"
src="User_Guide_files/image040.png"></p>

<p class=MsoCaption><a name="_Ref103116417"><b>Figure </b></a><b>31</b><b>.
Adding data from relation tables into the report files. </b>The relation table “protein2gene”
contains a column with protein identifiers under the “protein” header and
another column with the corresponding gene name under the “gene” header,
whereas the relation table “protein2description” contains, apart from the “protein”
elements, a column with the corresponding protein description. The first task
table row will prompt REPORT to incorporate the gene names and protein
descriptions contained in the relation tables to the report file “Npep_Quanprot_cat”
report file, as “protein” is the <span style='font-style:normal'>Lower level</span>
they all three shares.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Finally, the report data can be filtered performing logical
operations with the <i>Reported vars</i> in the <i>Filter</i> field. For
instance, in the report task table displayed in <b>Figure 32</b>:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &lt;= 100, filters out from the report the
variables Z and FDR for the category-to-categoryall integration when the
(n)umber of proteins per category is greater than 100.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &gt;= 5 &amp; n_protein2category &lt;= 100
retrieves the variables Z and FDR for the category-to-categoryall integration
when the (n)umber of proteins per category is in the [5, 100] range.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>KO_vs_WT@FDR_category2categoryall &lt; 0.05, retrieves the variables
Z and FDR for the category-to-categoryall integration provided that the FDR corresponding
to the “KO_vs_WT” samples is less than 0.05.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=583 height=98 id="Picture 32" src="User_Guide_files/image041.png"></p>

<p class=MsoCaption><a name="_Ref103116468"><b>Figure </b></a><b>32</b><b>. Filtering the report data.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The compound variables shown in the <i>Filter</i> field on <b>Figure 32</b> follow the structure <i>Reported var</i>_<i>integration</i>, like “n_protein2category”.
Such filter applies to all samples. </p>

<p class=MsoNormal>However, the filter “KO_vs_WT@FDR_category2categoryall” is
applied based on the variable “FDR” from the category-to-categoryall
integration, but only to the “KO_vs_WT” sample. Moreover, the filter “WT1,WT2@FDR_category2categoryall”
is applied to the “WT1” and “WT2” samples.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_SANSON"></a><span style='font-family:"Calibri",sans-serif'>SANSON</span></h3>

<p class=MsoNormal>The SANSON module generates a similarity graph showing the
relationship between functional categories based on their protein components. </p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters to be provided for this module are (<b>Figure 33</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the names of the folder(s)
where the lower level data file is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level elements
(“protein” in this case) to be used.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the name of the higher level (“category”
in this case) to which the lower level elements will be integrated.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder </i>designates a folder other than <i>Sample
folder(s)</i> where the results will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=383 height=65 src="User_Guide_files/image042.png"></p>

<p class=MsoCaption><a name="_Ref103116548"><b>Figure </b></a><b>33</b><b>.
A sample task table in the SANSON module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The SANSON module accepts the following additional parameters
(<b>Figure 34</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower norm</i> specifies the normalization scheme to be used
with the lower level elements. The default value is included the normalization
of lower level to all.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher norm</i> specifies the normalization scheme to be used
with the higher level elements. The default value is included the normalization
of higher level to all.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the
lower_level2higher_level Relation table will be integrated. Logical operators
can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT eliminates
outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.<i> </i></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Filter</i> allows to filter the data based on the FDR and
number of proteins. For more details, see <a href="#_Filter_param_(in">Filter</a>
in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=57 id="Picture 2" src="User_Guide_files/image043.png"></p>

<p class=MsoCaption><a name="_Ref103116564"><b>Figure </b></a><b>34</b><b>.
A sample task-table with advanced parameters in the SANSON module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><a name="_Complete_Modules"></a><a name="_WSPP-SBT"></a><a
name="_WSPPG-SBT"></a><a name="_WPP-SBT"></a><a name="_WPPG-SBT"></a>&nbsp;</p>

<h2><a name="_Special_parameters"></a>Special parameters</h2>

<p class=MsoNormal><a name="_Keywords_for_the"></a>&nbsp;</p>

<h3><a name="_Multiple_samples"></a>Multiple samples</h3>

<p class=MsoNormal>The “Sample folder(s)” field of the different module task
tables admit multiple samples. For instance, let's consider the samples created
with the following LEVEL CREATOR task table:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=174 id="Picture 64" src="User_Guide_files/image044.png"></p>

<p class=MsoNormal>We can include multiple samples separated by a comma, for
example, in the INTEGRATE module task table: </p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=227 id="Picture 7" src="User_Guide_files/image045.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Asterisk_is_our"></a>Asterisk is our jack of all trades</h3>

<p class=MsoNormal>The module task tables admit the usage of the asterisk
symbol as a wildcard character. Let’s once more consider the samples created
with the following LEVEL CREATOR task table:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=174 id="Picture 59" src="User_Guide_files/image044.png"></p>

<p class=MsoNormal>Each row calculates a ratio that is saved to the corresponding
<i>Output Sample folder</i>. Thus, the ratio of 113 to the mean of 113, 114, 115,
and 116 is saved to the “Jurkat_WT/WT_1” folder; the 114 to the mean of 113,
114, 115 ratios is saved to “Jurkat_WT/WT_2”, and so on. One way to create the
task table of the INTEGRATE module could be the following, where each row
represents an integration for a given sample:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=225 height=308 id="Picture 62" src="User_Guide_files/image046.png"></p>

<p class=MsoNormal>However, this task table can be simplified applying the
asterisk wildcard. For instance, the task-table below allows to indicate multiple
sample folders, namely every folder starting with “Jurkat_WT/” or “Jurkat_KO/”.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=275 height=274 id="Picture 60" src="User_Guide_files/image047.png"></p>

<p class=MsoNormal>We can reduce this expression even more using just an asterisk:
the first row of the following task table performs the integrations peptide-to-protein,
protein-to-category, peptide-to-peptideall, protein-to-proteinall, and category-to-categoryall
in every sample folder defined with LEVEL CREATOR.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=284 height=159 id="Picture 63" src="User_Guide_files/image048.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Multiple_samples_in"></a>Multiple samples in the inputs and
outputs</h3>

<p class=MsoNormal>In the cases we have multiple input samples separated by
comma but we want to save the results in another output sample folder, we
indicate them in the same way giving the output folders separated by comma. It
is required to provide the same number of folders:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=236 id="Picture 13" src="User_Guide_files/image049.png"></p>

<p class=MsoNormal>In the same way happens with the asterisk character (jack of
all trades). In the “Output Sample folder(s)” we can add a suffix in the input
samples. The following task-table illustrate that the output sample folders
would be contain the “_New” suffix:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=487 height=158 id="Picture 30" src="User_Guide_files/image050.png"></p>

<p class=MsoNormal>In addition, we can rename the subfolder adding the new name
in the “Output Sample folder(s)” or add a new subfolder:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=399 height=181 id="Picture 33" src="User_Guide_files/image051.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_More_params"></a>More params</h3>

<p class=MsoNormal>Some modules accept a column parameter in the Task-Table
called “More params”. This column allows you to provide advanced parameters for
the SanXoT programs [1].
The program descriptions are in the following wiki link:</p>

<p class=MsoNormal><a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The iSanXoT module are composed by several programs of
SanXoT. For this reason, the “More params” of a module accepts the advanced
parameters of composed programs indicated by a name.</p>

<p class=MsoNormal>For example, </p>

<p class=MsoNormal style='margin-left:35.4pt'>INTEGRATE:</p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'>&quot;sanxot1&quot;:
&quot; -m 300 -g &quot;, &quot;sanxot2&quot;: &quot;-s --sweepdecimals=2.5&quot;</p>

<p class=MsoNormal>In above example the first “sanxot” program that compose the
INTEGRATE module, receives the “-m 300 -g” as parameter, and the second
“sanxot” receives also the parameter &quot;-s --sweepdecimals=2.5&quot;.</p>

<p class=MsoNormal style='margin-left:35.4pt'>WSPP-SBT:</p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'>&quot;p2q_sanxot2&quot;:
&quot; -m 100 -s &quot;, &quot;q2a_sanxot1&quot;: &quot;-m 100&quot;</p>

<p class=MsoNormal>In this example, the WSPP-SBT module has multiple
integrations: scan-to-peptide, peptide-to-protein, etc. For more information,
see the WSPP-SBT section. Thus, the second “sanxot” program of peptide-to-protein
(q) integration will receive the parameter “-m 100 -s”, and then, the first
“sanxot” of protein (q)-to-proteinall (a) integration will receive the “-m
100”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>The program names for each Module </h4>

<p class=MsoNormal>INTEGRATE: sanxot1, sanxotsieve, sanxot2</p>

<p class=MsoNormal>NORCOMBINE: create_exp_tags, cardenio, sanxot1, sanxotsieve,
sanxot2 </p>

<p class=MsoNormal>SBT: l2i_sanxot1, l2i_sanxotsieve, l2i_sanxot2, i2h_sanxot1,
i2h_sanxotsieve, i2h_sanxot2, l2h_sanxot1, l2h_sanxotsieve, l2h_sanxot2</p>

<p class=MsoNormal>WSPP_SBT: level_creator, klibrate, s2p_sanxot1,
s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve, p2q_sanxot2,
q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1,
c2a_sanxotsieve, c2a_sanxot2</p>

<p class=MsoNormal>WSPPG_SBT: level_creator, klibrate, s2p_sanxot1,
s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve, p2q_sanxot2,
p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1, q2g_sanxotsieve,
q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2,
g2a_sanxot1, g2a_sanxotsieve, g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve,
c2a_sanxot2</p>

<p class=MsoNormal>WPP_SBT: level_creator, klibrate, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve,
c2a_sanxot2</p>

<p class=MsoNormal>WPPG_SBT: level_creator, klibrate, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1, q2g_sanxotsieve,
q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, g2a_sanxot1, g2a_sanxotsieve,
g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve, c2a_sanxot2</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Filter_param_(in"></a><a name="_Filter_(for_REPORT"></a>Filter in
REPORT module</h3>

<p class=MsoNormal>The REPORT module module accepts a Filter parameter. This
parameter filters the data based on some variables depending on the module.</p>

<p class=MsoNormal>In the case of REPORT module, the filtered variables are the
Reported vars: n, Z, FDR, etc. For instance:</p>

<p class=MsoNormal><i>(FDR_category2category &lt; 0.05) &amp; (n_protein2category
&gt;=5) &amp; (n_protein2category &lt;= 100)</i></p>

<p class=MsoNormal><i>(FDR_category2category &lt; 0.05) &amp; (Z_protein2proteinall
&gt;= 2 | Z_protein2proteinall &lt;= -2)</i></p>

<p class=MsoNormal>&nbsp;</p>

<h3>Filter in SANSON module</h3>

<p class=MsoNormal>For the SANSON module, the filtered variables are FDR and
the related number (n_rel). For example:</p>

<p class=MsoNormal style='text-indent:35.4pt'>([FDR] &lt; 0.05) &amp; ([n_rel]
&gt;= 10) &amp; ([n_rel] &lt;= 100)</p>

<p class=MsoNormal>Different variables can be combined using the comparisons:
&gt;=, &lt;=, !=, &lt;&gt;, ==, &gt;, &lt;; and using logical operators “and”
(&amp;), “or” (|), and “not” (!). </p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk124849833"></a><a name="_Toc124328440"></a><a
name="_Sample_Workflows_with"></a><a name="_Adaptors"></a><a
name="_Input_Adaptor"></a>Sample Workflows with Application to Case Studies</h1>

</div>

<p class=MsoNormal>We describe below in detail four sample workflows that
illustrate the capacity of iSanXoT to statistically ascertain protein or
peptide abundance changes in a variety of biological contexts. Note that these
workflows may be easily reused to process new data (see next section).</p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Hlk122708883"></a><a
name="_Toc124328441"></a><a name="_Workflow_1:_One-step"></a><b><span
style='font-size:20.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow 1: One-step quantification in a labeled experiment</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328442"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>The identification and quantification data from
García-Marqués <i>et al</i>. [3] were used to illustrate this workflow. This
study characterizes the molecular alterations that take place along time when
vascular smooth muscle cells (VSMCs) are treated with angiotensin-II (AngII) for
0, 2, 4, 6, 8, and 10 h. Quantitative proteomics was performed using isobaric
iTRAQ 8-plex labeling. Workflow 1 analyzes a) protein abundance changes and b)
functional category alterations produced by the coordinated behaviour of
proteins at each one of the times, in relation to time 0. This is done using,
in only one step, the compound module WSPP-SBT, which performs automatically
all the required tasks.</p>

<p class=MsoNormal><a name="_Toc124328443">&nbsp;</a></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><b><span style='font-size:16.0pt;
line-height:107%;font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow
operation</span></b></p>

<p class=MsoNormal>Workflow 1 requires the RELS CREATOR module, the WSPP-SBT compound
module and the REPORT basic module (<i><span style='line-height:107%;
color:#BF8F00'>Figure 35</span></i>). The relation tables required to perform
the integrations are created by the RELS CREATOR module (<i><span style='line-height:107%;color:#BF8F00'>Figure 35</span></i><span
style='color:#BF8F00'>A</span>) from a table provided by the user. The WSPP-SBT
module performs a sequence of consecutive integrations based on the WSPP
statistical model [2] and the SBT algorithm [3] (<i><span
style='line-height:107%;color:#BF8F00'>Figure 35</span></i><span
style='color:#BF8F00'>B</span>). Finally, the REPORT module organizes the data
in tables containing the information needed.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=655 height=288 id="Picture 20"
src="User_Guide_files/image052.jpg"
alt="A diagram of a level calibration&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771130"></a><a name="_Ref127961610"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>35</span></i><i><span style='color:#BF8F00'>. Scheme of
workflow 1 (one-step quantification in a labeled experiment) showing module
components: RELS CREATOR (A) and WSPP-SBT and REPORT (B) </span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The WSPP-SBT module needs the user
to define the meaning of relative abundances, which iSanXoT always expresses as
log2ratios. In this case the abundance data corresponds to the intensities of
iTRAQ reporters at the scan level, which are tabulated in the “ID-q” file with
the name of each reporter as column header (see below how these tables are
generated). The intensities of each scan at 0 h are in column “Abundance: 113”
and are used as a common reference to express abundance ratios and are
therefore used as denominator. The reporter intensities corresponding to the
different time points are used as numerators of the ratios. The task table also
allows the user to put an easily identifiable name in the folders where the
quantitative values of each sample are stored (<i><span style='line-height:
107%;color:#BF8F00'>Figure 36</span></i>). </p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=556 height=179 id="Picture 38"
src="User_Guide_files/image053.png"
alt="A picture containing text, screenshot, font, number&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771182"></a><a name="_Ref129965819"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>36</span></i><i><span style='color:#BF8F00'>. The
WSPP-SBT task table for workflow 1.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The WSPP-SBT module first performs
a calibration to assign a statistical weight to each one of the log2ratio
values at the scan level (<i><span style='line-height:107%;color:#BF8F00'>Figure
35</span></i><span style='color:#BF8F00'>B</span>), as described [2].
The statistical weight of each scan is the inverse of the estimated variance
associated with the log2 of intensity ratios [2].
Once the data is calibrated at the scan level, the workflow performs the
integrations <i>scan-to-peptide</i> and <i>peptide-to-protein</i>. </p>

<p class=MsoNormal style='text-indent:.5in'>At the protein level the SBT
algorithm is then applied for the detection of functional category changes
originated by the coordinated behaviour of proteins (<i><span
style='line-height:107%;color:#BF8F00'>Figure 35</span></i><span
style='color:#BF8F00'>B</span>). The algorithm first calculates the variance of
the <i>protein-to-category</i> integration, which is an improved estimate of
the technical protein variance, since it is less influenced by biological
changes [3]. This protein variance is used to perform the <i>protein-to-grand
mean</i> integration (hereinafter referred to as <i>protein-to-proteinall</i>)
integration, from which statistically significant abundance changes are
detected. The algorithm finally performs the <i>category-to-grand mean</i> integration
(hereinafter referred to as <i>category-to-categoryall</i>), from which
statistically significant category changes are detected. All the results from
the integrations executed by the WSPP-SBT module are saved, per each sample, to
the <i>Output Sample folder </i>indicated in the module task table (<i><span style='line-height:107%;color:#BF8F00'>Figure 36</span></i>).</p>

<p class=MsoNormal>Every integration step needs a relation table (a text file) that
links lower- to higher-level elements. Relation tables can be automatically created
by the RELS CREATOR module (<i><span style='line-height:107%;color:#BF8F00'>Figure
35</span></i><span style='color:#BF8F00'>A</span>, upper) and can be provided
by the user (<i><span style='line-height:107%;color:#BF8F00'>Figure 35</span></i><span
style='color:#BF8F00'>A</span>, lower). In this example (<i><span
style='line-height:107%;color:#BF8F00'>Figure 37</span></i>) the relation
tables linking scan to peptides and peptides to proteins are obtained from the
“ID-q” file, just by indicating the name of the columns where they are located
(in this case “Scan_Id”, “Pep_Id” and “Master Protein Accessions”). In this
case the columns <i>Master Protein Accessions</i> and <i>Master Protein
Descriptions</i> in the “ID-q” file contain the accession numbers and the
complete name of the proteins, respectively, so that a relation table <i>protein2description</i>
is also created; this relation table may be later used to append the full name
of the protein to any of the created reports (see below). An example of the <i>peptide2protein</i>
relation table, linking the peptides identified to the proteins they come from
is showed in <i><span style='line-height:107%;color:#BF8F00'>Figure 38</span></i><span
style='color:#BF8F00'>A</span>. The elements of the relation table
protein2category were retrieved from a text file containing functional
annotations for mouse proteins gathered from several protein function databases
(<i><span style='line-height:107%;color:#BF8F00'>Figure 38</span></i><span
style='color:#BF8F00'>B</span>), as described by the authors [3].
Note that relation tables are by default extracted from the ID-q file; to use
other text files the absolute path with the location of the text file has to be
indicated. The relation tables <i>protein2proteinall </i>and <i>category2categoryall</i>
guide the integration to a grand mean (a common element called “[1]”). The
integration <i>peptide2peptideall</i> is not necessary in this workflow but is
included in this example since it may be useful to inspect quantifications at
peptide level.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=638 height=163 id="Picture 638197479"
src="User_Guide_files/image054.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref132484598"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>37</span></i><i><span
style='color:#BF8F00'>. The RELS CREATOR task table for workflow 1.</span></i></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
style='font-size:14.0pt;line-height:107%'><img border=0 width=536 height=341
id="Picture 1199129032" src="User_Guide_files/image055.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref131431704"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>38</span></i><i><span
style='color:#BF8F00'>. Excerpt from the peptide2protein (A) and protein2category
(B) relation tables that link peptides to proteins and proteins to categories,
respectively.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Once the integrations are
performed, the REPORT module is used to collect from the <i>Output sample
folders</i> stated by the user the statistical variables desired and to
organize them in tables (<i><span style='line-height:107%;color:#BF8F00'>Figure
39</span></i>). In this case,  the results from  the samples (2h-AngII,
4h-AngII, 6h-AngII, 8h-AngII, and 10h-AngII) are to be tabulated. </p>

<p class=MsoNormal style='text-indent:.5in'>In this example the REPORT module
creates a protein table and a category table by performing the following steps:</p>

<p class=MsoListParagraphCxSpFirst style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table called “Npep2prot&quot; containing the number of
peptides with which each protein is quantified. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by extracting from the <i>peptide-to-protein</i>
integrations in the indicated folders the number of elements (n) of the lower
level (peptide) used to quantitate the higher level (protein). </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table called “Npep2prot_Quantprot_filtered” containing
the protein changes Zqa and the statistical significance FDRqa of these
changes. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by extracting from the <i>protein-to-proteinall</i>
integration in the indicated folders the standardized log2 ratios (Z) and False
Discovery Rates (FDR) of the lower level (protein). </p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add to this table the number of peptides with which each protein
is quantified. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by merging the previous table with the existing
table “Npep2prot” according to the level common to the two tables (protein),
without including a specific column (peptide) and eliminating replicate
entries. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add to this table an additional column with the complete
description of the proteins. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by merging the previous table with the relation
table protein2description according to the level common to the two tables
(protein). </p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter the table so that only the proteins having a statistically
significant abundance change (FDR &lt; 0.01) are tabulated.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by applying in the Filter column a condition based
on the FDR to the results from the protein2proteinall integration. For more
detailed information, see the “Filter for report” in the iSanXoT wiki: <a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki">https://github.com/CNIC-Proteomics/iSanXoT/wiki</a>.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table called “Nprot2cat” containing the number of
proteins with which each category is quantified.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by extracting from the <i>protein-to-category</i>
integrations in the indicated folders the number of elements (n) of the lower
level (protein) used to quantitate the higher level (category).</p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table called “Nprot2cat_Quantcat_filtered” containing
the category changes Zca and the statistical significance FDRca of these
changes. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by extracting from the <i>category-to-categoryall</i>
integration in the indicated folders the standardized log2 ratios (Z) and False
Discovery Rates (FDR) of the lower level (category).</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add to this table the number of proteins with which each category
is quantified. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by merging the previous table with the existing
table “Nprot2cat” according to the level common to the two tables (category),
without including a specific column (protein) and eliminating replicate
entries.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter the table so that only the categories having a
statistically significant change (FDR &lt; 0.01) are tabulated. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by applying in the Filter column a condition based
on the FDR to the results from the category2categoryall integration. </p>

<p class=MsoListParagraphCxSpMiddle style='margin:0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table called “Npep2prot_Quanprot” containing the number
of peptides per protein, the protein changes Zqa and the statistical
significance FDRqa of these changes. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done as explained above, omitting the protein
descriptions and the filters. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Create a table called “Nprot2cat_Quancat_Quanprot_filtered” containing
the category changes Zca and the statistical significance FDRca of these
changes. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by extracting from the <i>category-to-categoryall</i>
integration in the indicated folders the standardized log2 ratios (Z) and False
Discovery Rates (FDR) of the lower level (category).</p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Add to this table the number of proteins per category, the
protein changes Zqa and the statistical significance FDRqa of these changes. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by merging the previous table with the existing
tables “Nprot2cat” and “Npep2prot_Quantprot”. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-bottom:0in;text-indent:-.25in;
line-height:normal'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Filter the table so that only the categories containing 5 or more
proteins or 100 or less proteins are tabulated. </p>

<p class=MsoListParagraphCxSpLast style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:1.0in;text-indent:-.25in;line-height:normal'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span>This is done by applying in the Filter column a set of conditions
joined with the “&amp;” operator. </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.5in;line-height:normal'>Note
that these commands in the REPORT module, which allow to construct tables
required in typical quantitative proteomics projects, are easily reusable for
other projects.</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><i><img
border=0 width=717 height=165 id="Picture 151791518"
src="User_Guide_files/image056.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></i></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref131431781"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>39</span></i><i><span
style='color:#BF8F00'>. The REPORT task table for workflow 1.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>In <i><span
style='color:#BF8F00'>Figure 41</span></i><i><span style='color:#BF8F00'> </span></i>we
show two heat maps constructed from the protein and category tables obtained
with the REPORT module.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=253 height=365 id="Picture 635905203" src="User_Guide_files/image057.jpg"
alt="A picture containing text, screenshot, colorfulness, child art&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption>Figure 40. Relative abundance changes of proteins (Zqa, Left)
and functional categories (Zca, Right) generated from the “Npep2prot_Quanprot_filtered”
and “Nprot2cat_Quancat_filtered” reports, respectively, obtained by the REPORT
module of workflow 1 (Figure 39). Both report tables were sorted by the average
of Zqa and Zca, respectively. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>In <span
style='color:#BF8F00'>Figure 41</span><span lang=EN-GB> </span>we show some
examples of functional categories showing statistically significant changes
produced by coordinated protein behaviour, plotted using the data in the “Nprot2cat_Quancat_Quanprot_filtered&quot;
table generated by the REPORT module. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=627 height=397 id="Picture 1322059248"
src="User_Guide_files/image058.jpg"
alt="A picture containing text, diagram, line, plan&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref128391398">Figure </a>41. Examples of
time-dependent coordinated protein behavior of VSMCs treated with
angiotensin-II as revealed by the distribution of the standardized log2 ratio (Zqa)
of protein components in each category.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328444"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from</p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP-SBT.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP-SBT.zip</span></a><u>.</u>
See the <i>Importing a workflow template</i> Section below for detailed
instructions.</p>

<span style='font-size:14.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328445"></a><a
name="_Workflow_2:_Step-by-step"></a><b><span style='font-size:20.0pt;
line-height:107%;font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow
2: Step-by-step quantification and sample combination in a labeled experiment</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328446"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal style='page-break-after:avoid'>The data from González-Amor <i>et
al</i>. [5] was used to illustrate this workflow. This study analyzes the
contribution of interferon-stimulated gene 15 (ISG15) to the vascular damage
associated with hypertension, using KO mutants for this gene and subjecting or
not the animals to AngII treatment. This experiment contained 16 samples from
mouse aortic tissue corresponding to four groups:  four WT-Control mice, four ISG15-KO
mice, four WT+AngII mice, and four ISG15-KO+AngII mice. The experiment was
performed using two isobaric iTRAQ  8-plex batches. This module illustrates how
to create, step by step, a workflow to integrate the quantitative results from
each sample to the protein level, to integrate protein data from the four
biological replicates in each group, to construct ratios between two
conditions, and to analyze functional category changes due to coordinated
protein behavior using the SBT model. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=465 height=458 id="Picture 490866323"
src="User_Guide_files/image059.jpg"
alt="A diagram of a diagram&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771564"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>42</span></i><i><span
style='color:#BF8F00'>. Scheme of workflow 2 (step-by-step quantification and
sample combination in a labeled experiment) showing module components: RELS
CREATOR (A) and LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, RATIOS,
SBT, and REPORT (B) </span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328447"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>Workflow 2 comprises all six basic modules: LEVEL CREATOR,
LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, RATIOS, and SBT as well as the REPORT
module (<i><span style='line-height:107%;color:#BF8F00'>Figure 42</span></i>).
The task table in the starting module, LEVEL CREATOR, generates the files at
the scan level containing the log2 ratios and the corresponding sample folders
(<i><span style='line-height:107%;color:#BF8F00'>Figure 43</span></i>). In this
example, as in workflow 1, the name of each iTRAQ reporter was used as column
header in the “ID-q” file containing the intensities. In addition, the column
Experiment indicates whether the intensities come from the first or second
iTRAQ 8-plex batch. Each iTRAQ batch contains two biological replicates from
each of the four groups and the average of reporter intensities from the two
untreated WT mice (reporters in the columns “113” and “117”) were used as
internal control within each batch, and therefore were used as denominator for
the log2ratios. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=620 height=324 id="Picture 50"
src="User_Guide_files/image060.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771575"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>43</span></i><i><span
style='color:#BF8F00'>. The LEVEL CREATOR task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>LEVEL CREATOR generates the <i>u_scan</i>
(<i>uncalibrated scan</i>) files, which contain the scan identifiers (taken
from the column “Scan_Id” in the “ID-q” table), the log2-ratios at the scan
level Xs (as defined in the task table) and the <i>uncalibrated</i> weights Vs
(which in the WSPP model are the intensities of the reporters in the Ratio
numerator column) (<i><span style='line-height:107%;color:#BF8F00'>Figure 44</span></i>).
The uncalibrated weights Vs are related to the quality of quantification (a
higher weight implicates a more accurate quantification), but they are not
associated to a statistical variance yet.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=566 height=148 id="Picture 51"
src="User_Guide_files/image061.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771591"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>44</span></i><i><span
style='color:#BF8F00'>. Excerpt from one of the </span></i><span
style='color:#BF8F00'>u_scan<i> files generated by workflow 2 LEVEL CREATOR
module showing element identifiers (left column), log2 ratios (center column),
and statistical weights (right column).</i></span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The LEVEL CALIBRATOR module
calibrates the Vs weights by performing a <i>u_scan-to-peptide integration </i>and
generates the <i>scan</i> (<i>calibrated scan</i>) files, which contain true,
calibrated statistical weights (defined in the WSPP model as the inverse of the
estimated individual scan variances) (<i><span
style='color:#BF8F00'>Figure 45</span></i><i><span
style='color:#BF8F00'><b>Error! Reference source not found.</b></span></i>, Top).
</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=379 height=144 id="Picture 1630667189"
src="User_Guide_files/image062.jpg"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoCaption><a name="_Ref132481392">Figure </a>45. The LEVEL CALIBRATOR
(Top) and INTEGRATE (Bottom) task tables for workflow 2.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Note that LEVEL CALIBRATOR
automatically generates a plot to supervise the accuracy of calibrations (“*_outGraph_VRank”
png file) in each sample folder, which show whether the model is able to
predict experimental scan variances as a function of the calibrated statistical
weights (<i><span style='color:#BF8F00'>Figure 46</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=327 height=260 id="Picture 2084979463"
src="User_Guide_files/image063.jpg"
alt="A picture containing text, diagram, line, plot&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref132481421">Figure </a>46. Automatically
generated graphs to supervise the accuracy of calibrations. These graphs
represent 1/MSD versus the rank of Vs (scan weight, which in this level
corresponds to reporter intensity). MSD is the Mean Squared Deviation of the
scans vs the respective mean of the peptide they belong to. The scans are
ordered by Vs and the MSD is calculated in a sliding window of 200 scans [2].</p>

<p class=MsoNormal style='page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>As in the case of workflow 1,
before performing the integrations, the relation tables have to be created with
the RELS CREATOR module, which have a similar structure (<i><span style='color:#BF8F00'>Figure 47</span></i>).</p>

<p class=MsoNormal align=left style='text-align:left;page-break-after:avoid'><img
border=0 width=629 height=112 id="Picture 93"
src="User_Guide_files/image064.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref132481845">Figure </a>47. The RELS CREATOR
task table for workflow 2.</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The INTEGRATE module performs the <i>scan-to-peptide</i>,
<i>peptide-to-protein</i> and <i>protein-to-proteinall</i> integrations
according to the module task table (<i><span
style='color:#BF8F00'>Figure 45</span></i><i><span
style='color:#BF8F00'><b>Error! Reference source not found.</b></span></i>, Bottom).
Note that for consistency all the files created for each sample are stored by
default in the folder indicated in the task table of LEVEL CREATOR module,
unless otherwise indicated in the Output Sample folder column.</p>

<p class=MsoNormal style='text-indent:.5in'>Note that the INTEGRATE module
automatically generates a plot to check the accuracy of the GIA integration
model in each one of the integration steps. This is done by comparing the
distribution of Z values with that of the null hypothesis (standard normal
distribution) (see <span
style='color:#BF8F00'><b>Error! Reference source not found.</b></span> below,
left panels). These graphs are stored in the corresponding sample folders (in
“*_outGraph.png” files). </p>

<p class=MsoNormal style='text-indent:.5in'>By default, iSanXoT removes
integration outliers. To prevent the removal of outlier elements in the <i>protein-to-proteinall</i>
integration, as these are just the proteins which are significantly altered, a <i>0</i>
<i>FDR</i> value was indicated in the INTEGRATE task table for this integration
(<span style='color:#BF8F00'>Figure 45</span><i><span
style='color:#BF8F00'><b>Error! Reference source not found.</b></span></i>, Bottom).</p>

<p class=MsoNormal style='text-indent:.5in'>Once protein levels are created,
workflow 2 uses the NORCOMBINE basic module (<i><span
style='line-height:107%;color:#BF8F00'>Figure 42</span></i><span
style='color:#BF8F00'>B</span>) to integrate protein values from the four biological
replicates in each group  to produce integrated protein values per group that
are stored in the folders  WT-C, WT-AngII, ISG15-C and ISG15-AngII (<i><span style='color:#BF8F00'>Figure 48</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=550 height=107 id="Picture 54"
src="User_Guide_files/image065.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoCaption><a name="_Ref132482832">Figure </a>48. The NORCOMBINE task
table for workflow 2.</p>

<p class=MsoCaption>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The NORCOMBINE module integrates
biological replicates within sample groups applying the GIA algorithm [3], which models the distribution of protein values around the average taking into
account error propagation theory and estimates a global variance for the
integration. The GIA algorithm assumes that the individual variances of all the
lower elements (proteins) are affected by a global variance (which in this case
arises from biological variability within the same group). While this
assumption may not hold in all the cases, it can be easily checked by
inspecting the test distributions. The NORCOMBINE module (like the INTEGRATE
module) automatically generates graphs comparing the distribution of the
integrated Z variables with those of the standard normal distribution. As shown
in <i><span style='color:#BF8F00'><b>Error! Reference source not found.</b></span></i>
right, the distribution of protein Z values estimated by the model in the case
of the ISG15-AngII group agree very well with the null hypothesis,
demonstrating that the assumption of the model is a good approach to treat the
biological variance of the samples within this group. Similar results were
obtained in the other three groups (not shown).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=624 height=346 id="Picture 2145529192"
src="User_Guide_files/image066.png"
alt="A picture containing line, diagram, plot, text&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133424017">Figure </a>49. Distribution of the
standardized log2 protein ratios (Zqa) from the four individual ISG15-AngII
VSMC samples (Left panel) showing how the WSPP model agrees well with the
expected null distributions in the four cases, and from the integrated
ISG15-AngII sample group obtained with the NORCOMBINE module (Right panel),
showing how the GIA assumption of a global biological variance is a good
approach to treat the biological variability of samples within this group. Red:
null hypothesis (standard distribution); blue: experimental data.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><i> </i></p>

<p class=MsoNormal style='text-indent:.5in'>Note also that the NORCOMBINE
module performs a weighted averaging from several samples and that from the
good fitting to the null hypothesis it allows an accurate control over outliers
(<i><span style='color:#BF8F00'>Figure 49</span></i>). This unique approach
allows the integration of protein values originating from unbalanced sample
groups, distinct experiments or mass spectrometers, and even different
labelling techniques (see for instance [2]).</p>

<p class=MsoNormal style='text-indent:.5in'>The module task table indicates
that samples were combined at the <i>protein</i> level using the <i>proteinall</i>
level for normalization. This means that log2 protein ratios are firstly
normalized by the grand mean before being integrated into an averaged protein
value; this compensates for differences in protein load into each iTRAQ
channel. Note that proteins could also be integrated to other levels (for
instance organules, subcellular compartments, complexes, …), before being
integrated by NORCOMBINE, allowing different kinds of normalizations. Finally,
the column lowerNorm indicates which is the file that contains the normalized data,
which usually are the <i>lowerNormV</i> files, previously generated by the
INTEGRATE module. Further details can be found in the iSanXoT documentation.</p>

<p class=MsoNormal>                The protein averages from the four biological
sample groups are then used by the RATIOS basic module to calculate two ratios:
<i>WT-AngIIvsWT-C</i>, where wild-type AngII-treated animals are compared to
controls; and <i>ISG15-AngIIvsISG15-C</i>, where ISG15 AngII-treated animals
are compared to ISG15 controls (<i><span style='line-height:107%;color:#BF8F00'>Figure
50</span></i>). The V method column allows the user to indicate the method used
to assign a statistical weight to the log2ratios, which by default is the
method called “max” (for further details see “RATIOS” module in the iSanXoT
wiki, <a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki">https://github.com/CNIC-Proteomics/iSanXoT/wiki</a>).
</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=400 height=76 id="Picture 55" src="User_Guide_files/image067.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771774"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>50</span></i><i><span
style='color:#BF8F00'>. The RATIOS task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>                The last basic module executed in workflow
2, SBT (<i><span style='line-height:107%;color:#BF8F00'>Figure 51</span></i>), applies
the SBT algorithm to the above defined comparisons for the detection of
functional category changes originated by the coordinated behaviour of proteins.
This is made as explained in workflow 1. The SBT module is more flexible since
it allows to perform the triangle operations to any kind of level, not only
proteins. In this case the triangle is made by the levels protein and category
(<i><span style='line-height:107%;color:#BF8F00'>Figure 50</span></i>) and the
corresponding grand mean.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=299 height=83 id="Picture 99" src="User_Guide_files/image068.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771804"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>51</span></i><i><span
style='color:#BF8F00'>. The SBT task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Finally, the REPORT module (<i><span style='line-height:107%;color:#BF8F00'>Figure 52</span></i>) is used as in
workflow 1 to generate tables with the protein and category data. In this case
additional features of the REPORT module are used. The table “Npep2prot&quot;
is generated using an asterisk. This symbol is used by iSanXoT as a wildcard
character to indicate that the results from all the samples containing the <i>peptide-to-protein</i>
integration (i.e. ISG15-AngII-1, ISG15-AngII-2, ISG15-AngII-3, ISG15-AngII-4, ISG15-AngII,
ISG15-C-1, ISG15-C-2, ISG15-C-3, ISG15-C-4, ISG15-C, and ISG15-AngIIvsISG15-C)
are to be included in the table. However, the “Npep2prot_Quanprot_ISG15_filtered”
and “Npep2prot_Quanprot_WT_filtered” tables include the protein changes (Zqa),
the statistical significance (FDRqa) of these changes, and the number of
peptides per protein only from the samples indicated in the <i>Sample folder(s)</i>
column. The report for the ISG15 samples is filtered by Zqa to show the most
extreme values (greater than 1 or less than -1) but only for the “ISG15-AngIIvsISG15-C”
sample. Additional filters for the minimum number of peptides per protein are
also used in these tables. The tables containing category values are filtered
by Zca (greater than or equal to 2 or less than or equal to -2) and/or by the
number of proteins per category (between 5 and 100). </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=702 height=176 id="Picture 104"
src="User_Guide_files/image069.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771910"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>52</span></i><i><span
style='color:#BF8F00'>. The REPORT task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The tables generated by REPORT can
be used to generate heatmaps showing the most relevant protein abundance
changes (<i><span style='line-height:107%;color:#BF8F00'>Figure 53</span></i>).
As previously shown [5], iSanXoT analysis revealed a coordinated alteration of
proteins implicated in cardiovascular function, extracellular matrix and
remodeling, and vascular redox state in aortic tissue from AngII-infused
ISG15-KO mice (<i><span
style='color:#BF8F00'>Figure 54</span></i><i><span style='color:#BF8F00'>A</span></i>).
The coordinated protein behavior from some of the altered categories can be
analyzed in the sigmoid plots (<i><span
style='color:#BF8F00'>Figure 54</span></i><i><span style='color:#BF8F00'>B</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=475 height=440 id="Picture 15"
src="User_Guide_files/image070.jpg"
alt="A picture containing text, parallel&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772030"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>53</span></i><i><span
style='color:#BF8F00'>. Differential abundance of functional proteins revealed
by workflow 2. The heatmap (A) for proteins (Zqa) is based on the “Npep2prot_Quanprot_ISG15_filtered”
REPORT table. The heatmap (B) displays the proteins (Zqa) for the WT samples
using the “Npep2prot_Quanprot_WT_filtered” REPORT table.</span></i></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=707 height=406 id="Picture 340217713"
src="User_Guide_files/image071.jpg"
alt="A screenshot of a graph&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoCaption><a name="_Ref132485751">Figure </a>54. Functional category
changes arising from coordinated protein behavior. A) Bar graph for functional
categories (Zca) constructed from the “Nprot2cat_Quancat_filtered” REPORT table.
B) The distributions of the standardized log2 protein ratios (Zqa) are shown
for some of the functional categories that are significantly down-regulated
(Left) or up-regulated (Right). The data to create the sigmoid curves are taken
from the “Nprot2cat_Quancat_Quanprot_filtered” REPORT table. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328448"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from <a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_NORCOM_RATIOS_SBT.zip">https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_NORCOM_RATIOS_SBT.zip</a><u>.</u>
See the <i>Importing a workflow template</i> Section below for detailed
instructions.<br clear=all style='page-break-before:always'>
</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328449"></a><a
name="_Workflow_3:_Quantification"></a><b><span style='font-size:20.0pt;
line-height:107%;font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow
3: Quantification of posttranslationally modified peptides in a labeled
experiment</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328450"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>This workflow was used to quantitate reversibly oxidized Cys
peptides in mouse embryonic fibroblast (MEF) preparations subjected to chemical
oxidation with diamide, an experiment that served to illustrate the comparative
performance of on-filter (FASILOX) and in-gel (GELSILOX) approaches to study
the thiol redox proteome [4]. These techniques introduced a differential label
on Cys residues, depending on their oxidation state, producing two separate
populations of reduced and oxidized Cys-containing peptides. MEF samples were
incubated with diamide (treated group), or PBS (control group) and the
resulting peptides were isobaricaly labeled with iTRAQ 8-plex (four biological
replicates per condition). The workflow detects statistically significant
abundance changes in peptides contanining modified Cys residues.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=537 height=225 id="Picture 404835365"
src="User_Guide_files/image072.jpg"
alt="A picture containing text, diagram, screenshot, line&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772176"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>55</span></i><i><span
style='color:#BF8F00'>. Scheme of workflow 3 (quantification of posttranslationally
modified peptides in a labeled experiment) showing module components: RELS
CREATOR (A) and LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE, and REPORT (B).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328451"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>Workflow 3 comprises the basic modules LEVEL CREATOR, LEVEL
CALIBRATOR, and INTEGRATE, as well as the RELS CREATOR and REPORT modules (<i><span style='line-height:107%;color:#BF8F00'>Figure 55</span></i>) and is very
similar to workflow 2. LEVEL CREATOR was used to design the ratios and to generate
the level files, sample folders and log2 ratios indicated in the corresponding
task table (<i><span style='line-height:107%;color:#BF8F00'>Figure 56</span></i>
and <i><span style='line-height:107%;color:#BF8F00'>Figure 57</span></i>).
LEVEL CALIBRATOR was used to calibrate statistical weights (<i><span style='line-height:107%;color:#BF8F00'>Figure 58</span></i>, top) and INTEGRATOR to
integrate from scan to peptide and from peptide to protein (<i><span style='line-height:107%;color:#BF8F00'>Figure 58</span></i>, bottom).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=557 height=118 id="Picture 243"
src="User_Guide_files/image073.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772189"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>56</span></i><i><span
style='color:#BF8F00'>. The LEVEL CREATOR task table for workflow 3.</span></i></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=527 height=108 id="Picture 247"
src="User_Guide_files/image074.png"
alt="A screenshot of a graph&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772226"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>57</span></i><i><span
style='color:#BF8F00'>. Excerpt from one of the </span></i><span
style='color:#BF8F00'>uscan<i> files generated by workflow 3 LEVEL CREATOR
module showing element identifiers (left column), log2 ratios (center column)
and statistical weights (right column).</i></span></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The only difference with workflow 2
lies in the INTEGRATE command used for the integration <i>peptide-to-protein</i>.
INTEGRATE can use a modified version of the GIA algorithm for the quantitative
analysis of post-translational modifications (PTM) that includes a third column
containing <i>tags</i> in the relation tables, as described [7].
In this workflow the advanced option of INTEGRATE was activated to display the <i>Tag
</i>column, which is used to include only the peptides which are tagged in the
relation table with the text <i>”Not modified”</i> when calculating the protein
averages (<i><span style='line-height:107%;color:#BF8F00'>Figure 58</span></i>).
An example of tagged <i>peptide2protein</i> relation table is shown in <i><span style='color:#BF8F00'>Figure 59</span></i><i><span style='color:#BF8F00'>.</span></i>
Proteins are thus quantified using only peptides which are not modified in Cys.
However, although these Cys peptides do not contribute to protein averages,
they are assigned a <i>Zpq</i> value, which serves to evaluate whether they
deviate significantly from the expected distribution of peptides around their
protein averages [4]. If the deviation is statistically significant it can be
concluded that there is a change in abundance of the posttranslational
modification in relation to the protein it comes from. This philosophy can be
extended to any other kind of PTM.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=548 height=143 id="Picture 192096234" src="User_Guide_files/image075.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772240"><i><span style='color:#BF8F00'>Figure
</span></i></a><i><span style='color:#BF8F00'>58</span></i><i><span
style='color:#BF8F00'>. The LEVEL CALIBRATOR (Top) and INTEGRATE (Bottom) task
tables for workflow 3.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=532 height=194 id="Picture 556561336"
src="User_Guide_files/image076.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref133425366">Figure </a>59. Excerpt from the peptide2protein
relation table used to integrate peptides to proteins. Note the presence of a
third column used to tag Cys-containing peptides, which will be excluded from
the calculation of protein averages in the peptide-to-protein integration.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>iSanXoT allows to automatically
generate relation tables containing tags, which are taken from the “ID-q”
table. For this end, RELS CREATOR uses a specific option (<i><span style='color:#BF8F00'>Figure 60</span></i>). In this particular case, this
option makes RELS CREATOR to look into the “ID-q” table for the column with the
header <i>Modifications</i> and translate its content into the third column of
the <i>peptide2protein</i> relation table. In the “ID-q” table used in this
case, the peptide containing modified Cys residues were labeled as “<i>Reduced-Cys
peptides</i>” and “<i>Oxidized-Cys peptides</i>” depending on the type of
modification. These tags are located in the relation table by  RELS CREATOR (<i><span style='color:#BF8F00'>Figure 59</span></i> and <i><span
style='color:#BF8F00'>Figure 60</span></i>). iSanXoT allows to use any tag
created by searching engines or defined by the user, with the only condition
that the tag indicated in the INTEGRATE command must match the tag in the third
column of the relation table (<i><span style='line-height:107%;color:#BF8F00'>Figure
58</span></i>, bottom).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=98 id="Picture 65" src="User_Guide_files/image077.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoCaption><a name="_Ref133427111">Figure </a>60. The RELS CREATOR
task table for workflow 3.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Finally, the REPORT module collects
the statistical variables generated by the <i>peptide-to-protein</i>
integration for all the samples (<i>c1, c2, c3, c4, t1, t2, t3,</i> and <i>t4</i>),
as prompted by the asterisk (<i><span
style='color:#BF8F00'>Figure 61</span></i><i><span style='color:#BF8F00'>A</span></i>).
The REPORT commands are similar to those used to generate the protein tables in
workflow 1, with the difference that peptide values are tabulated instead of
protein values, together with the number of scans per peptide, instead of the
number of peptides per protein. The FDR at the peptide level allows to detect
statistically significant changes in PTM. This REPORT also generates a second
filtered peptide table containing the peptides with reduced Cys with the most
extreme abundance changes. This table was used to generate a heatmap (<i><span style='color:#BF8F00'>Figure 61</span></i><i><span style='color:#BF8F00'>B</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=651 height=679 id="Picture 12"
src="User_Guide_files/image078.jpg"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoCaption><a name="_Ref133484449">Figure </a>61. (A) The task table
for Workflow 3 is in the REPORT module. (B) Relative abundance of
Cys-containing peptides in MEF samples measured by peptide log2 ratios
expressed in units of standard deviation corrected by the protein mean (<span
style='font-style:normal'>Zpq</span>). The data for the heatmap was generated
from the “Nscan2pep_Quanpepprot_filtered” report table. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Of note, the peptides included in
the integration followed a standard distribution in the eight samples, as shown
in blue on <i><span
style='color:#BF8F00'>Figure 62</span></i> for <i>t1</i>, <i>t2</i>, <i>c1</i>,
and <i>c2</i> samples. This evidences that the error distribution at the
peptide level could be accurately modeled using the GIA algorithm. In addition,
the treatment produced a generalized increase in the abundance of oxidized
Cys-containing peptides (orange curves), with concomitant decrease in the
abundance of reduced Cys-containing peptides (green curves). Consistently, the
opposite changes were observed in the controls.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=443 height=290 id="Imagen 40" src="User_Guide_files/image079.jpg"
alt="A picture containing text, diagram, line, plot&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133499855">Figure </a>62. Distribution of the
standardized variable at the peptide level (Zpq) in control MEF samples (t1, t2,
c1 and c2) for all the peptides quantitated (blue) and the oxidized (orange)
and reduced (green) Cys-containing peptide subpopulations. The theoretical normal
distribution N(0,1) is shown in red. Positive/negative Zpq values indicate
increased/decreased peptide abundance with respect to the average. These
sigmoidal curves were created from the “Nscan2pep_Quanpepprot” table generated
by REPORT.</p>

<p class=MsoCaption align=left style='text-align:left'>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328452"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_PTM.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_PTM.zip</span></a></p>

<p class=MsoNormal>See the <i>Importing a workflow template</i> Section below
for detailed instructions.</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b><span lang=EN-GB style='font-size:18.0pt;line-height:
107%'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328453"></a><a
name="_Workflow_4:_Label-free"></a><b><span style='font-size:20.0pt;line-height:
107%;font-family:"Calibri Light",sans-serif;color:#BF8F00'>Workflow 4:
Label-free quantification</span></b></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328454"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Experimental</span></b></a></p>

<p class=MsoNormal>This workflow was used to analyze quantitative data obtained
in a multicenter study aimed at evaluating different bioinformatics tools [6]. The authors generated two hybrid proteome test samples consisting of tryptic
digests of human, yeast and <i>Escherichia coli</i> proteins mixed in two
different defined proportions (13:3:4 for <i>A </i>samples and 13:6:1 for <i>B</i>
samples). Quadruplicate peptide samples were analyzed by LC-MS/MS, after which
peptide identification and quantification were carried out with several
software packages. In this example we show how to process the data obtained by MaxQuant
[8]
(see Section below to learn how data from MaxQuant and other softwares can be
adapted to be used with iSanXoT).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=461 height=385 id="Picture 1715712138"
src="User_Guide_files/image080.jpg"
alt="A picture containing text, diagram, number, font&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133509694">Figure </a>63. Scheme of workflow 4
(label-free quantification) showing module components: RELS CREATOR (A) and LEVEL
CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, and REPORT (B).</p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328455"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow operation</span></b></a></p>

<p class=MsoNormal>Workflow 4 includes the basic iSanXoT modules LEVEL CREATOR,
LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, and RATIOS along with the REPORT and
RELS CREATOR modules (<i><span
style='color:#BF8F00'>Figure 63</span></i><i>)</i>. The starting module, LEVEL
CREATOR, generates the level files, sample folders and log2 ratios indicated in
the corresponding task (<i><span
style='color:#BF8F00'>Figure 64</span></i>) based on the quantitative data at
the peptide level obtained with MaxQuant for replicate <i>A-</i> and <i>B</i>-type
samples. In this example we used as denominator of the log2ratio the average of
peptide intensities across all the samples. In this case, however, the averages
of the four <i>A</i>-type and the four <i>B</i>-type samples are first calculated
separately (as indicated by the square brackets), then the average of the two
averaged values is calculated (as indicated by the comma). This ensures that no
log2-ratio is calculated when the four values are missing in either the <i>A</i>
or the <i>B</i> sample group. This module generates uncalibrated files at the
peptide level (<i>u_peptide</i>) (<span
class=MsoIntenseEmphasis>Figure 65</span>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=595 height=237 id="Picture 44"
src="User_Guide_files/image081.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref133509816">Figure </a>64. The LEVEL CREATOR
task table for workflow 4.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=606 height=132 id="Picture 72"
src="User_Guide_files/image082.png"
alt="A screenshot of a graph&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoCaption><a name="_Ref133510232">Figure </a>65. Excerpt from one of
the u_peptide files generated by workflow 4 LEVEL CREATOR module showing
element identifiers (left column), log2 ratios (center column) and uncalibrated
statistical weights (right column).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The <i>u_peptide</i> level files
are then calibrated with the LEVEL CALIBRATOR module by performing an integration
to the protein level (<span
class=MsoIntenseEmphasis>Figure 66</span><i>, Top</i>), generating calibrated <i>peptide</i>
level files. The <i>peptide-to-protein</i> and <i>protein-to-proteinall</i>
integrations are then performed by the INTEGRATE module according to the module
task table (<span
class=MsoIntenseEmphasis>Figure 66</span>, Bottom). </p>

<p class=MsoNormal style='text-indent:.5in'>Note that in this example the
advanced option of INTEGRATE was activated to use the Tag column, so that only
the proteins containing the <i>Homo sapiens</i> tag are used in the
protein-to-proteinall integrations (<span
class=MsoIntenseEmphasis>Figure 66</span><i>, Bottom</i>). Restricting the
integration to human proteins serves for two purposes: a) the normalization is
done by the grand mean of the human proteins and is not affected by the
presence of yeast or <i>E. coli</i> proteins. b) only human proteins are used
to estimate the variance of the <i>protein-to-protein</i> all integration,
avoiding the effect of yeast and E. coli proteins, whose ratios have a large
deviation from the mean. Note that this procedure does not eliminate yeast or <i>E.
coli</i> proteins from the normalized files that are later used by the
NORCOMBINE module (see below). </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=536 height=154 id="Picture 627941333"
src="User_Guide_files/image083.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref133510477">Figure </a>66. The LEVEL CALIBRATOR
(Top) and INTEGRATE (Bottom) task tables for workflow 4.</p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>As in workflow 3, the <i>protein-to-proteinall</i>
relation table must contain a third column tagging the species from which each
protein comes from (<span
class=MsoIntenseEmphasis>Figure 67</span>). Note that the tag indicating the
human proteins matches the tag indicated in INTEGRATE (<span class=MsoIntenseEmphasis>Figure 66</span><i>, Bottom</i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=264 height=185 id="Picture 73"
src="User_Guide_files/image084.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with low confidence"></p>

<p class=MsoCaption><a name="_Ref133510667">Figure </a>67. Excerpt from the protein2proteinall
workflow 4 relation table that links proteins to a constant value representing
the protein grand mean. Note the use of a third column to tag proteins with
their corresponding species for later species-specific protein-to-proteinall
integration.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The <i>protein-to-proteinall</i>
relation table is automatically created by the RELS CREATOR module, which takes
this information from a file (HumanSaccEcoliPME12_divide_by_species.tsv)
created by the user. The file contains the relationship between protein
identifiers (<i>Protein_Id</i> column header) and the species (<i>Species</i>
column header) they come from.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=581 height=71 id="Picture 36" src="User_Guide_files/image085.png"></p>

<p class=MsoCaption>Figure 68. The RELS CREATOR task table for workflow 4.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Next, the normalized data at the <i>protein</i>
level from the four replicates from each sample are combined into samples <i>A</i>
and <i>B</i>, respectively, using the NORCOMBINE basic module (<span class=MsoIntenseEmphasis>Figure 69</span>, Top). To compare these two samples,
new log2 ratios and statistical weights are calculated using the RATIOS basic
module (<span class=MsoIntenseEmphasis>Figure 69</span>, Bottom). Finally, a <i>protein-to-proteinall</i>
integration is carried out for the newly-generated <i>B_vs_A</i> sample by the
module INTEGRATE (<span
class=MsoIntenseEmphasis>Figure 66</span>, Bottom), using again the <i>Homo
sapiens</i> tag.</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=560 height=136 id="Picture 9"
src="User_Guide_files/image086.png"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref133512128">Figure </a>69. The NORCOMBINE (Top)
and RATIOS (Bottom) task tables for workflow 4.</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The REPORT module used in this
workflow tabulates the data at protein level together with the number of
peptides per protein, as in previous workflows (<span
class=MsoIntenseEmphasis>Figure 70</span>). In this example, besides <i>Z</i>
and <i>FDR</i>, the table includes the log2ratios of proteins from all the
samples (<i>Xinf</i> from the <i>protein-to-proteinall </i>integration, to
which we also refer as <i>Xq</i>), the grand mean (<i>Xsup </i>from the <i>protein-to-proteinall
integration, </i>or <i>Xa</i>), the statistical weights (<i>Vinf</i> or <i>Vq</i>)
and the species each protein belongs to (<i>tags</i>). The grand mean, which is
used to normalize log2-ratios, and the statistical weights can be used to
construct plots such as in <span
class=MsoIntenseEmphasis>Figure 71</span> (see below).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
style='font-size:14.0pt;line-height:107%'><img border=0 width=609 height=68
id="Picture 26471980" src="User_Guide_files/image087.png"></span></p>

<p class=MsoCaption><a name="_Ref133518992">Figure </a>70. The REPORT module
task table for workflow 4.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>It is noteworthy that variance
modelling, normalization, standardization and statistical weighting, according
to the GIA algorithm, are performed automatically, without data filtering,
pre-processing or missing value imputation [6]<i><span
style='color:#BF8F00'><b>Error! Reference source not found.</b></span></i>,
even in a situation where numerous proteins have highly imbalanced data. Moreover,
the sigmoid plots automatically generated in each one of the integrations
performed by INTEGRATE clearly demonstrate that the GIA algorithm accurately
predicts the distribution of peptide quantifications around their proteins (<span class=MsoIntenseEmphasis>Figure 71</span><span class=MsoIntenseEmphasis>A</span>)
and of protein quantifications around the grand mean (<span class=MsoIntenseEmphasis>Figure 71</span><span class=MsoIntenseEmphasis>B</span>).
These results demonstrate that this statistical model is very suitable for the
analysis of label-free data.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=503 height=378 id="Picture 32061321"
src="User_Guide_files/image088.jpg"
alt="A picture containing text, diagram, line, plot&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133519177">Figure </a>71. Distribution of the
standardized variable at the peptide (Zpq) and protein (Zqa) levels for
label-free data analyzed with iSanXoT. A) Zpq distribution for the eight
individual A-type and B-type samples. B) Zqa distribution for the B-type vs
A-type comparison. Red: null hypothesis (standard distribution); blue:
experimental data.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'><a name="_Hlk121489265">The
combined statistics <i>B_vs_A</i> also shows how human, yeast and bacterial
proteins distribute around the expected 0, 1, and -2 log2- values
(corresponding to 1-, 2- and 0.25-fold changes) (</a><span class=MsoIntenseEmphasis>Figure 72</span><span class=MsoIntenseEmphasis>)</span>,
and how protein quantifications with higher statistical weights are more
accurate. This plot also confirms how iSanXoT provides highly accurate
quantitative results in a fully automated fashion.</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=480 height=413
id="Picture 771401894" src="User_Guide_files/image089.jpg"
alt="A picture containing text, screenshot, design&#10;&#10;Description automatically generated"></p>

<p class=MsoCaption><a name="_Ref133523519">Figure </a>72. Quantification of
human (orange), yeast (grey) and bacterial (blue) proteins according to the
combined statistics B_vs_A. Shown are log2-ratios normalized by the grand mean
(Xq – Xa or Xinf - Xsup). This plot was generated from the table
“Npep2prot_Quanprot”.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:2.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;page-break-after:avoid'><a name="_Toc124328456"><b><span
style='font-size:16.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>Workflow execution</span></b></a></p>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WPP_LabelFree.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WPP_LabelFree.zip</span></a></p>

<p class=MsoNormal>See the <i>Importing a workflow template</i> Section below
for detailed instructions.</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc124328457"></a><a name="_Importing_a_workflow"></a>Importing a
workflow template</h1>

</div>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal>In this section we will provide instructions to execute the
workflow examples and to import workflows that were previously created with
iSanXoT to be reused in other projects. We will use as example the first
workflow described in the previous section. </p>

<p class=MsoNormal>Start by downloading workflow 1 template and input files
from the iSanXoT wiki (<a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/workflows/WSPP-SBT.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/workflows/WSPP-SBT.zip</span></a>).
Then extract the files included in the compressed archive to create a folder
named WSPP-SBT. Check that the WSPP-SBT folder has been created in your file
system. Then proceed as follows:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Open the
     iSanXoT application by double-clicking the application icon (<i><span style='color:#BF8F00'>Figure 73</span></i>).</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=234 height=154 id="Picture 430598581"
src="User_Guide_files/image090.png"></p>

<p class=MsoCaption><a name="_Ref135132480">Figure </a>73. The iSanXoT startup
message.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Choose <i>New
     Project</i> from the <i>Project</i> menu (<i><span
     style='color:#BF8F00'>Figure 74</span></i>).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=429 height=250 id="Picture 954891610"
src="User_Guide_files/image091.png"></p>

<p class=MsoCaption><a name="_Ref135132530">Figure </a>74. Create New Project.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Provide a
     name of your choice for the project folder and indicate a path to locate
     this folder, then click the <i>Submit</i> button (<i><span style='color:#BF8F00'>Figure 74</span></i>).</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Choose <i>Import
     Workflow</i> from the <i>Project</i> menu (<i><span
     style='color:#BF8F00'>Figure 75</span></i><i>)</i> and select the folder WSPP-SBT
     created before (or any other iSanXoT project folder from which you want to
     import the workflow). </li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=431 height=196 id="Picture 1854767591"
src="User_Guide_files/image092.png"></p>

<p class=MsoCaption><a name="_Ref135132621">Figure </a>75. Importing a
preexisting iSanXoT workflow to the newly-created project.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Inspect the
     WSPP-SBT task table (in the Compound modules tab), the RELS CREATOR task
     table (in the Relation tables tab) and the REPORT task table (in the
     Reports tab) to check that the tables indicated in Fig. S2, S3 and S5 have
     been correctly loaded. Note that if a different template is imported, only
     the corresponding task tables will be loaded.</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Now click on <i>Choose
     identification file</i> and select “ID-q.tsv” in the WSPP-SBT folder (<i><span style='color:#BF8F00'>Figure 76</span></i><i>)</i>. Or, alternatively, select
     the desired identification/quantification table with which this workflow
     is to be executed. Section 3 below shows how to prepare the “ID-q” file
     based on the output from a variety of proteomics pipelines. Bear in mind
     that the tasks defined in the LEVEL CREATOR and RELS CREATOR modules have
     to match the samples and column names from the specific “ID-q” file used. </li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=421 height=362 id="Picture 3" src="User_Guide_files/image093.jpg"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></p>

<p class=MsoCaption><a name="_Ref135132690">Figure </a>76. Choosing the
identification/quantification (ID-q) file for the newly-created project.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Select <i>Save
     Project</i> from the <i>Project </i>menu to save the changes or directly
     press the <i>Save and Run</i> button to save and execute the current
     workflow.</li>
</ul>

<b><span style='font-size:16.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span></b>

<p class=MsoNormal><b><span style='font-size:16.0pt;line-height:107%'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc124328458"></a><a name="_Creating_the_identification/quantif"></a>Creating
the identification/quantification file from proteomics pipelines</h1>

</div>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:42.55pt;line-height:
normal'>iSanXoT requires an identification/quantification file in tsv format (<i>ID-q.tsv</i>)
containing at least the quantified features together with their quantitative
values. Any tsv table may be used as an <i>ID-q</i> file, provided that quantitative
values are arranged in a table with column headers, so that features (e.g. PSMs
or peptides) are arranged in rows and their quantitative values in columns,
where every column pertains to a different sample. The column headers of the <i>ID-q</i>
are used by iSanXoT to extract the necessary information. </p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:42.55pt;line-height:
normal'>In addition, when the <i>ID-q</i> file contains features quantitated in
more than one experiment (e.g. different samples labelled with the same TMT-18plex
tags), an additional column with the header <i>Experiment</i> must be included
indicating the experiment ascription of the features.  </p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:42.55pt;line-height:
normal'>Finally, iSanXoT also needs information to create the relation files
required to integrate the quantified features into higher levels. This
information is usually also present in the <i>ID-q</i> file. For instance,
iSanXoT can use the columns containing the scan and the peptide identifiers to
construct the <i>scan2peptide</i> relation table.</p>

<p class=MsoNormal style='text-indent:.5in'>The majority of proteomics software
tools generate tables that can be easily used for this purpose. In this Section
we shall describe how to prepare the <i>ID-q</i> file based on the output from
the three most popular proteomics pipelines (<i><span
style='color:#BF8F00'>Table S1</span></i>). </p>

<p class=MsoCaption><a name="_Ref135146243">Table S</a>1. Output data from
proteomics pipelines to be included in the <span style='font-style:normal'>ID-q.tsv</span>
file.</p>

<p class=MsoNormal align=center style='margin-bottom:0in;text-align:center;
line-height:normal'><img border=0 width=670 height=317 id="Picture 188195546"
src="User_Guide_files/image094.png"
alt="A picture containing text, screenshot, font, number&#10;&#10;Description automatically generated"></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<h2><a name="_Toc124328464"></a><a name="_Preparing_the_ID-q"></a>Preparing the
<i>ID-q</i> file from Proteome Discoverer output</h2>

<p class=MsoNormal>In the case of Proteome Discoverer version 2.5 [9], the way that quantitative data are adapted for use with iSanXoT depends on whether
they originate from label-free or labelled experiments:</p>

<h3>Label-free experiments</h3>

<p class=MsoNormal>In this case, quantitative data at the peptide level can be
adapted for use with iSanXot from the <a name="OLE_LINK1"></a><a
name="OLE_LINK2"><i>_PeptideGroups.txt</i> </a>files obtained when the <i>Processing</i>
workflow node <i>Minora Feature Detector</i> of Proteome Discoverer is used.
The following column headers of the <i>_PeptideGroups.txt </i>files<i> </i>must
be considered for preparing the <i>ID-q</i> file: </p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modifications</i>:
     Chemical or posttranslational modifications to the <i>Sequence </i>above;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Master Protein
     Accessions</i>: Accession code(s) for the protein(s) to which the peptide <i>Sequence</i>
     is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Abundance:
     FX: Sample Type: </i>Peptide intensity in the RAW file identified with <i>FX</i>
     and tagged as <i>Sample Type</i> in the Proteome Discoverer <i>Input Files</i>
     tab.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'><a
name="_Hlk135138073">The <i>peptide</i> level required for the <i>peptide to protein</i>
integration with iSanXoT can be obtained by merging the <i>Sequence</i> and <i>Modifications
fields </i>(see Section <i>Adapting the results from proteomics pipelines for
iSanXoT</i> below). </a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<h3>Labelled experiments</h3>

<p class=MsoNormal>For labelled experiments (e.g. TMT- or iTRAQ-based),
quantitative data at the scan level can be adapted for use with iSanXot from <a
name="OLE_LINK3">the <i>_PSMs.txt</i> </a>files generated when the <i>Processing</i>
workflow node <i>Reporter Ions Quantifier</i> of Proteome Discoverer is used.
The following column headers of the <i>_PSMs.txt</i> files must be considered
for preparing the <i>ID-q</i> file:  </p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Spectrum
     File</i>: Name of the RAW file where the PSM was identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>First Scan</i>:
     Spectrum (scan) number of the PSM in the RAW file;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modifications</i>:
     Chemical or posttranslational modifications to the <i>Sequence</i> above;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Master Protein
     Accessions</i>: Accession code(s) for the protein(s) to which the peptide <i>Sequence</i>
     is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Abundance:
     Quan Channel</i>:  Intensity of the reporter ion tagged as <i>Quan Channel</i>
     in the Proteome Discoverer <i>Samples</i> tab.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>For
the <i>scan to peptide</i> integration with iSanXoT, the <i>scan</i> level can
be obtained by merging the<i> Spectrum File</i> and <i>First</i> <i>Scan </i>fields,
and the <i>peptide</i> level  by merging the <i>Sequence</i> and <i>Modifications</i>
fields  (see Section <i>Adapting the results from proteomics pipelines for
iSanXoT</i> below; make sure <i>Max. Number of Peptides Reported</i> = 1 was
selected in the <i>Input Data</i> section of the Proteome Discoverer <i>Processing</i>
node used).</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>&nbsp;</p>

<h2><a name="_Toc124328459"></a><a name="_Preparing_the_ID-q_1"></a>Preparing
the <i>ID-q</i> file from MaxQuant output</h2>

<p class=MsoNormal>The way MaxQuant version 1.6.5.0 [8]
data are adapted for use with iSanXoT depends on whether they originate from
label-free or labelled proteomics experiments:</p>

<h3>Label-free experiments</h3>

<p class=MsoNormal>In this case, the quantifications at the peptide level
required to prepare the <i>ID-q</i> file can be found in the <a name="OLE_LINK5"><i>modificationSpecificPeptides.txt</i>
</a>file, which is stored in the <i>“…combined/txt</i>” folder. The following
column headers of the <i>modificationSpecificPeptides.txt</i> file must be
considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modifications</i>:
     Chemical or posttranslational modifications to the <i>Sequence</i> above;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Proteins</i>:
     Identifier(s) of the protein(s) to which the peptide <i>Sequence</i> is
     ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Intensity
     Experiment</i>: Summed up extracted ion current of all isotopic clusters
     associated with the peptide Sequence identified across the raw files
     included in the <i>Experiment</i> as specified by the user in the MaxQuant
     <i>Raw data</i> tab.</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.25in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>The
<i>peptide</i> level required for the <i>peptide</i> to protein integration
with iSanXoT can be obtained by merging <i>Sequence</i> and <i>Modifications</i>
fields (see Section <i>Adapting the results from proteomics pipelines for
iSanXoT</i> below).</p>

<p class=MsoNormal>&nbsp;</p>

<h3>Labelled experiments</h3>

<p class=MsoNormal>When dealing with labelled experiments (e.g. iTRAQ- or TMT-based),
the necessary quantitative data at the scan level can be found in the <i>msmsScans.txt</i>
file, which is stored in the <i>“…combined/txt</i>” folder. The following
column headers of the <i>modificationSpecificPeptides.txt</i> file must be
considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Raw file</i>:
     Name of the RAW file where the PSM was identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Scan
     number</i>: Spectrum (scan) number of the PSM in the RAW file;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modified Sequence</i>:
     Amino acid sequence of the identified peptide including chemical or posttranslational
     modifications. This parameter is nonblank only when identification was
     successful.</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Proteins</i>:
     Identifier(s) of the protein(s) to which the peptide <i>Sequence</i> is
     ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Reporter
     intensity n</i>: Intensity of the reporter ion <i>n </i>as specified by
     the user in the MaxQuant <i>Group-specific parameters</i> tab.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>For
the <i>scan to peptide</i> integration with iSanXoT, the <i>scan</i> level can
be obtained by merging the<i> Raw File</i> and <i>Scan number </i>fields (see Section
<i>Adapting the results from proteomics pipelines for iSanXoT</i> below).</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124328462"></a><a name="_Preparing_the_ID-q_2"></a>Preparing
the <i>ID-q</i> file from FragPipe output</h2>

<p class=MsoNormal>The way that quantitative data from Fragpipe version 1.8.1 [10]
are adapted for use with iSanXoT depends on whether they originate from
label-free or labelled experiments:</p>

<h3>Label-free experiments</h3>

<p class=MsoNormal>FragPipe <i>Quant (MS1)</i> module stores the quantifications
at the peptide level necessary to prepare the <i>ID-q</i> file in a <i>combined_modified_peptide.tsv
</i>file. The following column headers of the <i>modificationSpecificPeptides.txt</i>
file must be considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modified Sequence</i>:
     Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Protein ID</i>:
     Identifier of the protein to which the <i>Modified</i> <i>Sequence</i> peptide
     is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Experiment
     Intensity</i>: Summed up intensity of the <i>Modified Sequence</i> peptide
     in the RAW files included in the <i>Experiment</i> as specified by the
     user in the FragPipe <i>Workflow</i> tab.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<h3>Labelled experiments</h3>

<p class=MsoNormal>Fragpipe <i>Quant (Isobaric)</i> module generates a <i>psm.tsv</i>
output file that contains the quantitative data at the scan level obtained from
labelled experiments. The following column headers of the <i>psm.txt</i> file
must be considered for preparing the <i>ID-q</i> file:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Spectrum</i>:
     Spectrum (scan) identifier of the PSM in the XML file;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Spectrum
     File</i>: Name of the XML file where the PSM was identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Modified
     Peptide</i>: Amino acid sequence of the identified peptide including
     chemical or posttranslational modifications;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Protein ID:
     </i>Identifier of the protein to which the <i>Modified Peptide</i> is
     ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><i>Channel</i>: 
     Intensity of the reporter ion <i>Channel</i> as specified by the user in
     the FragPipe TMT-Integrator table of the <i>Quant (Isobaric)</i> module.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;text-indent:.25in;line-height:normal'>The
<i>scan</i> level required for the later <i>scan to peptide</i> integration
with iSanXoT can be obtained by merging the<i> Spectrum</i> and <i>Spectrum File
</i>fields (see Section <i>Adapting the results from proteomics pipelines for
iSanXoT</i> below; make sure <i>Report top N = 1</i> was selected in the <i>Advanced
Output Options</i> of the FragPipe <i>MSFragger</i> module).</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc124328465"></a><a name="_Adapting_the_results"></a>Adapting
the results from proteomics pipelines for iSanXoT</h1>

</div>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>iSanXoT requires an identification/quantification
tab-separated values file (<i>ID-q.tsv</i>) containing at least the identified
features together with their quantitative values (an experiment identifier is
required if two or more experiments are included). Users can either compose
this <i>ID-q</i> file manually (see the previous Section to learn how to do
that with data from the four most popular proteomics pipelines) or have it
prepared by the iSanXoT Input Adaptor. The latter option is described in this
Section.</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Run the
     iSanXoT application and create a new project or open an existing project.
     A new window will appear asking for the ID-q file (<i><span style='color:#BF8F00'>Figure 77</span></i>).</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>If you
     already have a suitable <i>ID-q</i> file, click the <i>Select
     User-Provided</i> option and then <i>Choose identification file</i> to
     select the file (<i><span
     style='color:#BF8F00'>Figure 77</span></i>).</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><b><span
style='font-size:16.0pt;line-height:107%'><img border=0 width=374 height=322
id="Picture 994047918" src="User_Guide_files/image095.jpg"
alt="A screenshot of a computer&#10;&#10;Description automatically generated with medium confidence"></span></b></p>

<p class=MsoCaption><a name="_Ref135147559">Figure </a>77. Selecting an ID-q
file in the Input Adaptor main window.</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>If you do not
     have an ID-q file, click <i>Select Adaptor from proteomics pipelines</i>.
     This option will launch the iSanXoT adapter to import your quantitative
     data. The adapter has been tested with recent versions of MaxQuant,
     Trans-Proteomic Pipeline, FragPipe, and Proteome Discoverer. Click on <i>Choose
     folder + Add anots</i> to select the folder containing your quantitative
     data (<i><span
     style='color:#BF8F00'>Figure 78</span></i>). A three-panel window will
     pop-up (<i><span
     style='color:#BF8F00'>Figure 79</span></i><i>)</i>.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:.5in;text-align:center;text-indent:22.0pt;
page-break-after:avoid'><img border=0 width=502 height=151
id="Picture 178886961" src="User_Guide_files/image096.png"></p>

<p class=MsoCaption><a name="_Ref135147782">Figure </a>78. Having the iSanXoT
Input Adaptor prepare the ID-q file.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:0in;margin-left:.5in;text-align:center;page-break-after:avoid'><img
border=0 width=385 height=448 id="Picture 86"
src="User_Guide_files/image097.png"></p>

<p class=MsoCaption><a name="_Ref135147818">Figure </a>79. Adapting results
from a proteomics pipeline. In the top panel several output files from Proteome
Discoverer have been selected. These PSMs.txt files, which contain
identification/quantification data, have been assigned an experiment name
(Jurkat) in the middle panel. The bottom panel has been used to create identifiers
by concatenating result file headers: ScanID (by concatenating Spectrum File,
First Scan and Charge) and pepID (by concatenating Sequence and Modifications).</p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><i><span style='color:#BF8F00'>&nbsp;</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The top panel
     displays the files included in the folder so that you can select one or
     more result files to be considered by the adapter. Please bear in mind
     that if several result files are selected, these must necessarily have the
     same column headers.</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The middle
     panel is used to set the distribution of data items across experiments
     according to result filenames.</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The bottom
     panel allows to create identifiers by concatenating result file headers.
     It is composed of two interfaces:</li>
 <ul style='margin-top:0in' type=circle>
  <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The headers
      found on the result files are listed on the left. Header names will be
      added to the interface on the right as they are selected;</li>
  <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>The
      interface on the right displays the header names selected to generate the
      identifier as well as the identifier name provided by the user.</li>
 </ul>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;text-indent:22.0pt'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Please note
     that the alphanumeric text that unambiguously identifies the items to be
     integrated is the only identifier that must be necessarily included in the
     <i>ID-q</i> file.</li>
 <li class=MsoNormal style='margin-bottom:0in;line-height:normal'>Click the <i>Submit</i>
     button and the Input Adaptor will start generating the <i>ID-q.tsv</i>
     file.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_References"></a><span lang=ES>References</span></h1>

</div>

<p class=MsoNormal><span lang=ES>&nbsp;</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span
lang=ES>[1] Trevisan-Herraz M, Bagwan N, Garcia-Marques F, Rodriguez JM, Jorge
I, Ezkurdia I, et al. </span>SanXoT: a modular and versatile package for the
quantitative analysis of high-throughput proteomics experiments. <span lang=ES>Bioinformatics.
2019;35(9):1594-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[2]
Navarro P, Trevisan-Herraz M, Bonzon-Kulichenko E, Nunez E, Martinez-Acedo P,
Perez-Hernandez D, et al. </span>General statistical framework for quantitative
proteomics by stable isotope labeling. <span lang=ES>J Proteome Res.
2014;13(3):1234-47.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[3]
Garcia-Marques F, Trevisan-Herraz M, Martinez-Martinez S, Camafeita E, Jorge I,
Lopez JA, et al. </span>A Novel Systems-Biology Algorithm for the Analysis of
Coordinated Protein Responses Using Quantitative Proteomics. <span lang=ES>Mol
Cell Proteomics. 2016;15(5):1740-60.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[4]
Bonzon-Kulichenko E, Camafeita E, Lopez JA, Gomez-Serrano M, Jorge I, Calvo E,
et al. </span>Improved integrative analysis of the thiol redox proteome using
filter-aided sample preparation. <span lang=ES>J Proteomics. 2020;214:103624.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[5]
Gonzalez-Amor M, Garcia-Redondo AB, Jorge I, Zalba G, Becares M, Ruiz-Rodriguez
MJ, et al. </span>Interferon stimulated gene 15 pathway is a novel mediator of
endothelial dysfunction and aneurysms development in angiotensin II infused
mice through increased oxidative stress. Cardiovasc Res. 2021.</p>

<p class=EndNoteBibliography style='margin-bottom:0in'>[6] Navarro P, Kuharev
J, Gillet LC, Bernhardt OM, MacLean B, Rost HL, et al. A multicenter study
benchmarks software tools for label-free proteome quantification. <span
lang=ES>Nat Biotechnol. 2016;34(11):1130-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in'><span lang=ES>[7] Bagwan
N, Bonzon-Kulichenko E, Calvo E, Lechuga-Vieco AV, Michalakopoulos S,
Trevisan-Herraz M, et al. </span>Comprehensive Quantification of the Modified
Proteome Reveals Oxidative Heart Damage in Mitochondrial Heteroplasmy. Cell
Rep. 2018;23(12):3685-97 e4.</p>

<p class=EndNoteBibliography style='margin-bottom:0in'>[8] Tyanova S, Temu T,
Cox J. The MaxQuant computational platform for mass spectrometry-based shotgun
proteomics. Nat Protoc. 2016;11(12):2301-19.</p>

<p class=EndNoteBibliography style='margin-bottom:0in'>[9] Orsburn BC. Proteome
Discoverer-A Community Enhanced Data Processing Suite for Protein Informatics.
Proteomes. 2021;9(1).</p>

<p class=EndNoteBibliography>[10] Kong AT, Leprevost FV, Avtonomov DM,
Mellacheruvu D, Nesvizhskii AI. MSFragger: ultrafast and comprehensive peptide
identification in mass spectrometry-based proteomics. Nat Methods.
2017;14(5):513-20.</p>

<p class=MsoNormal>&nbsp;</p>

</div>

</body>

</html>
</template>