<template class="task-template" id="User_Guide">
<html>

<head>
<base href="sections/help/">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Helvetica;
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:Courier;
	panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face
	{font-family:"Tms Rmn";
	panose-1:2 2 6 3 4 5 5 2 3 4;}
@font-face
	{font-family:Helv;
	panose-1:2 11 6 4 2 2 2 3 2 4;}
@font-face
	{font-family:"New York";
	panose-1:2 4 5 3 6 5 6 2 3 4;}
@font-face
	{font-family:System;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:Batang;
	panose-1:2 3 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:PMingLiU;
	panose-1:2 2 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;}
@font-face
	{font-family:Dotum;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimHei;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:MingLiU;
	panose-1:2 2 5 9 0 0 0 0 0 0;}
@font-face
	{font-family:Mincho;
	panose-1:2 2 6 9 4 3 5 8 3 5;}
@font-face
	{font-family:Gulim;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:Century;
	panose-1:2 4 6 4 5 5 5 2 3 4;}
@font-face
	{font-family:"Angsana New";
	panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face
	{font-family:"Cordia New";
	panose-1:2 11 3 4 2 2 2 2 2 4;}
@font-face
	{font-family:Mangal;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Latha;
	panose-1:2 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sylfaen;
	panose-1:1 10 5 2 5 3 6 3 3 3;}
@font-face
	{font-family:Vrinda;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Raavi;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Shruti;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sendnya;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Gautami;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Tunga;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Estrangelo Edessa";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Yu Gothic";
	panose-1:2 11 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:DengXian;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Palatino Linotype";
	panose-1:2 4 5 2 5 5 5 3 3 4;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"Arial Unicode MS";
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:"Segoe UI Emoji";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	margin-bottom:.0001pt;
	text-align:center;
	line-height:107%;
	page-break-after:avoid;
	border:none;
	padding:0in;
	font-size:26.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:normal;
	font-style:italic;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:11.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:22.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-link:"Footnote Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-link:"Comment Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	text-align:center;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoFootnoteReference
	{vertical-align:super;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-link:"Title Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpFirst, li.MsoTitleCxSpFirst, div.MsoTitleCxSpFirst
	{mso-style-link:"Title Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpMiddle, li.MsoTitleCxSpMiddle, div.MsoTitleCxSpMiddle
	{mso-style-link:"Title Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpLast, li.MsoTitleCxSpLast, div.MsoTitleCxSpLast
	{mso-style-link:"Title Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
a:link, span.MsoHyperlink
	{font-family:"Calibri",sans-serif;
	color:#2E74B5;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
strong
	{font-family:"Times New Roman",serif;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-link:"Comment Subject Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Balloon Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:9.0pt;
	font-family:"Segoe UI",sans-serif;}
p.MsoRMPane, li.MsoRMPane, div.MsoRMPane
	{margin:0in;
	margin-bottom:.0001pt;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoQuote, li.MsoQuote, div.MsoQuote
	{mso-style-link:"Quote Char";
	margin-top:10.0pt;
	margin-right:.6in;
	margin-bottom:8.0pt;
	margin-left:.6in;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#404040;
	font-style:italic;}
span.MsoIntenseReference
	{font-variant:small-caps;
	color:#4472C4;
	letter-spacing:.25pt;
	font-weight:bold;}
span.MsoBookTitle
	{letter-spacing:.25pt;
	font-weight:bold;
	font-style:italic;}
p.MsoTocHeading, li.MsoTocHeading, div.MsoTocHeading
	{margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	margin-bottom:.0001pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-link:Title;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-link:"Footnote Text";}
p.lead, li.lead, div.lead
	{mso-style-name:lead;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.QuoteChar
	{mso-style-name:"Quote Char";
	mso-style-link:Quote;
	color:#404040;
	font-style:italic;}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-link:"Comment Text";}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-link:"Comment Subject";
	font-weight:bold;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-link:"Balloon Text";
	font-family:"Segoe UI",sans-serif;}
p.EndNoteBibliographyTitle, li.EndNoteBibliographyTitle, div.EndNoteBibliographyTitle
	{mso-style-name:"EndNote Bibliography Title";
	mso-style-link:"EndNote Bibliography Title Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyTitleChar
	{mso-style-name:"EndNote Bibliography Title Char";
	mso-style-link:"EndNote Bibliography Title";
	font-family:"Calibri",sans-serif;}
p.EndNoteBibliography, li.EndNoteBibliography, div.EndNoteBibliography
	{mso-style-name:"EndNote Bibliography";
	mso-style-link:"EndNote Bibliography Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyChar
	{mso-style-name:"EndNote Bibliography Char";
	mso-style-link:"EndNote Bibliography";
	font-family:"Calibri",sans-serif;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;
	font-family:"Calibri",sans-serif;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;
	font-family:"Calibri",sans-serif;}
span.msoIns
	{mso-style-name:"";
	text-decoration:underline;
	color:teal;}
span.msoDel
	{mso-style-name:"";
	text-decoration:line-through;
	color:red;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:595.3pt 841.9pt;
	margin:70.85pt 85.05pt 70.85pt 85.05pt;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#2E74B5" vlink="#954F72">

<div class=WordSection1>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitle align=center style='text-align:center'><b><i><span
style='font-size:36.0pt;letter-spacing:.25pt'><img width=310 height=355
id="Picture 16" src="User_Guide_files/image001.png" alt=Molecule></span></i></b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoTitleCxSpFirst align=center style='text-align:center'><span
class=MsoBookTitle><span style='font-size:36.0pt'>User Guide:</span></span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
class=MsoBookTitle><span style='font-size:36.0pt'>Help and documentation</span></span></p>

<p class=MsoTitleCxSpLast align=center style='text-align:center'><span
class=MsoBookTitle>(1.2.7)</span></p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1>Table of contents</h1>

</div>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Introduction">Introduction</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_License">License</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Installation">Installation</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Download">Download</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Available_operating_systems">Available operating systems</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:"Courier New";text-decoration:none'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
class=MsoBookTitle><span style='line-height:107%;letter-spacing:0pt;font-weight:
normal;font-style:normal'><a href="#_Windows_distribution">Windows distribution</a></span></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_MacOS_distribution">MacOS distribution</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Linux_distribution">Linux distribution</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Symbol;letter-spacing:
0pt;font-weight:normal;font-style:normal'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Get_Started">Getting Started</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_Modules">Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Basic_modules">Relation Tables Module</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_RELS_CREATOR">RELS CREATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Basic_modules">Basic Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_LEVEL_CREATOR">LEVEL CREATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_LEVEL_CALIBRATOR">LEVEL CALIBRATOR</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_INTEGRATE">INTEGRATE</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_NORCOMBINE">NORCOMBINE</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_RATIOS">RATIOS</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a href="#_SBT">SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Compound_modules">Compound Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WSPP-SBT">WSPP-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WSPPG-SBT">WSPPG-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WPP-SBT">WPP-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Wingdings;text-decoration:none'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_WPPG-SBT">WPPG-SBT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Report_modules">Report Modules</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_REPORT">REPORT</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Wingdings;text-decoration:none'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_SANSON">SANSON</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Special_parameters">Special Parameters</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Keywords_for_the">Keywords for the Task-Tables</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Multiple_samples">Multiple samples</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Asterisk_is_our">Asterisk is our jack of all trades</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_More_params">More params</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-.25in'><span
class=MsoBookTitle><span style='line-height:107%;font-family:Wingdings;
letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><a
href="#_Filter_param_(in">Filter (for REPORT and SANSON)</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Sample_Workflows_with">Sample Workflows with
Application to Case Studies</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Workflow_1:_One-step">Workflow 1: One-step
quantification in a labeled experiment</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Workflow_2:_Step-by-step">Workflow 2: Step-by-step
quantification and sample combination in a labeled experiment</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Workflow_3:_Quantification">Workflow 3: Quantification
of posttranslationally modified peptides in a labeled experiment</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:"Courier New";text-decoration:none'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><a
href="#_Workflow_4:_Label-free">Workflow 4: Label-free quantification</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Executing_a_workflow">Executing a workflow template</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Creating_the_identification/quantif">Creating the
identification/quantification file from proteomics pipelines</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Preparing_the_ID-q">Preparing the ID-q file from
Proteome Discoverer output</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Preparing_the_ID-q_1">Preparing the ID-q file from
MaxQuant output</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Preparing_the_ID-q_2">Preparing the ID-q file from
Trans-Proteomic Pipeline output</a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><a href="#_Preparing_the_ID-q_3">Preparing the ID-q file from
FragPipe output</a></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="#_Adapting_the_results">Adapting the results from
proteomics pipelines for iSanXoT</a></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
class=MsoHyperlink><span style='font-family:Symbol;text-decoration:none'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><a href="#_References">References</a></p>

<p class=MsoNormal><span class=MsoHyperlink><span style='text-decoration:none'>&nbsp;</span></span></p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk118388234"></a><a name="_Hlk118304308"></a><a
name="_Introduction"></a>Introduction</h1>

</div>

<p class=MsoNormal><b><span lang=EN-GB>iSanXoT</span></b><span lang=EN-GB> is a
standalone application for statistical analysis of mass spectrometry-based
quantitative proteomics data.  iSanXoT builds upon SanXoT </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>, our previous publicly available
implementation of the weighted spectrum, peptide, and protein (WSPP)
statistical model </span><span
lang=EN-GB>[2]</span><span lang=EN-GB> using the Generic Integration Algorithm
(GIA) </span><span
lang=EN-GB>[3]</span><span lang=EN-GB>.</span></p>

<p class=MsoNormal><span lang=EN-GB>iSanXoT executes several kind of workflows
for quantitative high-throughput proteomics, <b>systems biology </b>and the <b>statistical
analysis</b>, <b>integration and comparison of experiments</b>.</span></p>

<p class=MsoNormal><span lang=EN-GB>iSanXoT was developed by the <b><i>Cardiovascular
Proteomics Lab</i></b>/<b><i>Proteomic Unit</i></b> at <b>The National Centre
for Cardiovascular Research</b> (CNIC, </span><a href="https://www.cnic.es"><span
lang=EN-GB style='color:#4472C4'>https://www.cnic.es</span></a><span
lang=EN-GB>).</span></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Download</span></h2>

<p class=MsoNormal><span lang=EN-GB>The multiple releases are available in the
&quot;release&quot; section, located in the following link:</span></p>

<p class=MsoNormal><a href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Installation</span></h2>

<h3><span lang=EN-GB>Available operating systems</span></h3>

<p class=MsoNormal><span lang=EN-GB>iSanXoT maintains the following operating
systems and architectures and may add additional ones in the future:</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-GB>Windows 10 Pro
(x64)</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-GB>MacOs High
Sierra (10.13.6)</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-GB>Ubuntu 20.04
(x64)</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, read the “Installation” section
in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Installation"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Installation</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Getting Started</span></h2>

<p class=MsoNormal><span lang=EN-GB>This chapter describes iSanXoT’s graphical
user interface and how to set up an analysis with iSanXoT.</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, read the “Getting Started”
section in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Getting_started"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Getting_started</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Modules</span></h2>

<p class=MsoNormal><span lang=EN-GB>The iSanXoT desktop application houses a
number of modules based on the SanXoT software package </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>. The information required to setup and
execute every module is provided in a task-table.   </span></p>

<p class=MsoNormal><span lang=EN-GB>There are four types of modules:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Relation tables is a module that creates the
relation tables used by the iSanXoT modules.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>The Basic modules call the individual scripts
included in the SanXoT software package </span><span
lang=EN-GB>[1]</span><span lang=EN-GB>.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>The Compound modules perform a sequence of
consecutive integrations based on the weighted spectrum, peptide and protein
(WSPP) statistical model </span><span
lang=EN-GB>[2]</span><span lang=EN-GB> and the systems-biology triangle (SBT)
algorithm </span><span
lang=EN-GB>[3]</span><span lang=EN-GB>.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Finally, there are two Reports: REPORT generates
report files displaying the quantitative results produced by the above Basic
and Composite modules when a workflow is executed; SANSON generates a
similarity graph showing relationships between functional categories on the
basis of the protein elements they share.</span></p>

<p class=MsoNormal><span lang=EN-GB>For more details, read the “Modules” section
in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Modules"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Modules</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Input Adaptor</span></h2>

<p class=MsoNormal><span lang=EN-GB>The iSanXoT Input Adaptor offers users the
possibility to either provide their own Identification/Quantification file,
which contains the identification and quantification data, or have the Input
Adaptor prepare this file from the results obtained using any of the mainstream
proteomics pipelines.</span></p>

<p class=MsoNormal><span lang=EN-GB>For further details, read the “Input
Adaptor” section in the iSanXoT wiki:</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Input_adaptor"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Input_adaptor</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>Sample Workflows</span></h2>

<p class=MsoNormal>We describe in detail four sample workflows that illustrate
the capacity of iSanXoT to statistically ascertain abundance changes in both
multiplexed, isotopically labeled [3-5] and label-free [6]
proteomics experiments.</p>

<p class=MsoListParagraphCxSpFirst align=left style='text-align:left;
text-indent:-.25in'><span lang=EN-GB style='font-family:Symbol'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_1"><span
lang=EN-GB>Workflow 1</span></a><span lang=EN-GB>: One-step quantification in a
labeled experiment ()</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_2"><span
lang=EN-GB>Workflow 2</span></a><span lang=EN-GB>: Step-by-step quantification
and sample combination in a labeled experiment.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_3"><span
lang=EN-GB>Workflow 3</span></a><span lang=EN-GB>: Quantification of
posttranslationally modified peptides in a labeled experiment.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_4"><span
lang=EN-GB>Workflow 4</span></a><span lang=EN-GB>: Label-free quantification.</span></p>

<p class=MsoNormal><span lang=EN-GB>For further details, read each section in the
iSanXoT wiki. </span></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><span lang=EN-GB>More Documents</span></h2>

<p class=MsoNormal><span lang=EN-GB>User Guide (v1.2.7):</span></p>

<p class=MsoNormal><a name="_License"></a><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/user_guides/User_Guide_iSanXoT-1.2.7.pdf"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/user_guides/User_Guide_iSanXoT-1.2.7.pdf</span></a></p>

<p class=MsoNormal><span lang=EN-GB>Presentations:</span></p>

<p class=MsoListParagraph style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>WSPP demo:</span></p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/WSPP_Demo_iSanXoT_1.0.X.pdf"><span
lang=EN-GB>https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/WSPP_Demo_iSanXoT_1.0.X.pdf</span></a></p>

<p class=MsoListParagraph style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Input Adaptor demo:</span></p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/Input_Adaptor_iSanXoT_1.0.X.pdf"><span
lang=EN-GB>https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/Input_Adaptor_iSanXoT_1.0.X.pdf</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h2><a name="_Hlk118382155"><span lang=EN-GB>License</span></a></h2>

<p class=MsoNormal><span lang=EN-GB>This application is licensed under a <b>Creative
Commons Attribution-NonCommercial-NoDerivs 4.0 Unported License</b>.</span></p>

<p class=MsoNormal><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><span
lang=EN-GB style='color:#4472C4'>https://creativecommons.org/licenses/by-nc-nd/4.0/</span></a></p>

<h4>You are free to:</h4>

<p class=MsoNormal>Share - copy and redistribute the material in any medium or
format.</p>

<p class=MsoNormal>The licensor cannot revoke these freedoms as long as you
follow the license terms.</p>

<h3>Under the following terms</h3>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/License"><span
lang=EN-GB style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/License</span></a></p>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk118375025"></a><a name="_Installation"></a>Installation</h1>

</div>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3><a name="_Download"></a>Download</h3>

<p class=MsoNormal><span lang=EN-GB>The multiple releases are available in the
&quot;release&quot; section, located in the following link:</span></p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span lang=EN-GB
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB>&nbsp;</span></p>

<h3><a name="_Available_operating_systems"></a><span lang=EN-GB>Available
operating systems</span></h3>

<p class=MsoNormal><span lang=EN-GB>iSanXoT maintains the following operating
systems and architectures and may add additional ones in the future:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Windows 10 Pro (x64)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
lang=EN-GB style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>MacOs High Sierra (10.13.6)</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span lang=EN-GB
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-GB>Ubuntu 20.04 (x64)</span></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3><a name=windows-distribution></a><a name=installation-on-windows></a><a
name="_Installation_on_Windows"></a><a name="_Windows_distribution"></a>Windows
distribution</h3>

<p class=MsoNormal>The iSanXoT Windows distribution is packaged in a NSIS
Launcher (exe file).</p>

<p class=MsoNormal>Download the exe Launcher: iSanXoT_Launcher_1.X.X.win32-x64.exe</p>

<p class=MsoNormal>Double-click the Launcher file; the Installer window will
show up:</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 28" src="User_Guide_files/image002.jpg"></span></p>

<p class=MsoNormal>WARNING: Windows Defender SmartScreen might show a prompt suggesting
that you cancel the installation; in such case click “More info” and then
select the “Run anyway” option.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=332 height=140 id="Picture 43" src="User_Guide_files/image003.png"></p>

<p class=MsoNormal>Then, you can choose the iSanXoT installation folder:</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 25" src="User_Guide_files/image004.jpg"></span></p>

<p class=MsoNormal>Wait while iSanXoT is being installed.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 22" src="User_Guide_files/image005.png"
alt="Installing window"></span></p>

<p class=MsoNormal>Once the installation has been completed, you are ready to run.</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 21" src="User_Guide_files/image006.png"
alt="Completing setup"></span></p>

<p class=MsoNormal><span style='font-family:"Segoe UI",sans-serif;color:#24292E'>&nbsp;</span></p>

<h3><a name=macos-distribution></a><a name="_MacOS_distribution"></a>MacOS
distribution</h3>

<p class=MsoNormal>The iSanXoT MacOS distribution is packaged in a DMG
container.</p>

<p class=MsoNormal>Download the DMG file: iSanXoT_Launcher_1.X.X.darwin-x64.dmg</p>

<p class=MsoNormal>Double-click the DMG file, then a Finder window will show up.
This window will usually display iSanXoT’s installer icon and a shortcut to the
Applications folder, together with some sort of linking arrow:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=416 height=309 id="Picture 19" src="User_Guide_files/image007.png"
alt="Installer window"></p>

<p class=MsoNormal>Simply drag the iSanXoT icon to your Applications folder...</p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=439 height=315 id="Picture 18" src="User_Guide_files/image008.png"
alt="Installing window"></span></p>

<p class=MsoNormal>and you're done: the iSanXoT application is now installed.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name=linux-distribution></a><a name="_Linux_distribution"></a>Linux
distribution</h3>

<p class=MsoNormal>The iSanXoT Linux distribution is packaged in an <i>AppImage</i>.</p>

<p class=MsoNormal>Download the AppImage: iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=567 height=241 src="User_Guide_files/image009.png"></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:0in'>The AppImage file is
just the application's compressed image. When executed, the application is
mounted in a temporal folder. However, for the correct behaviour, the application
has to be extracted to the &quot;squashfs-root&quot; folder in the current
working directory using:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>&nbsp;</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in'>Then, launch the
iSanXoT application using:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0in'>&nbsp;</p>

<p class=MsoListParagraphCxSpLast style='margin-left:0in;text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun</span></em></p>

<em><span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><span
style='font-family:"Calibri",sans-serif;font-style:normal'>&nbsp;</span></em></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Get_Started"></a><em><span style='font-family:"Calibri Light",sans-serif;
font-style:normal'>Getting Started</span></em></h1>

</div>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>This chapter describes iSanXoT’s graphical user interface and
how to set up an analysis with iSanXoT.</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3>Opening the iSanXoT application</h3>

<p class=MsoNormal>To open the iSanXoT application:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Windows: from the Start menu choose <i>Programs &gt; iSanXoT</i>;
or double-click the iSanXoT desktop icon.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In MacOS: double-click the iSanXoT icon from the Applications
folder.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>In Linux: from the <i>AppImage</i> file.</p>

<p class=MsoListParagraphCxSpMiddle>The contents are extracted to the &quot;squashfs-root&quot;
directory in the current working directory using:</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.x.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle>Now you can launch the iSanXoT application:</p>

<p class=MsoListParagraphCxSpLast style='text-indent:34.8pt'><em><span
style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun [...]</span></em></p>

<p class=MsoNormal><em><span style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3>Installing required packages</h3>

<p class=MsoNormal>The first time iSanXoT is run a window will show up displaying
a progress bar to inform you about the percentage of packages that have been
installed. These packages contain the libraries required by iSanXoT’s backend, and
are installed the first time you open the application.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=482 height=317 src="User_Guide_files/image010.png"></p>

<p class=MsoCaption><b>Figure </b><b>1</b><b>. <a name="_Ref103093720">Installation
window.</a></b></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3>Closing the iSanXoT application</h3>

<p class=MsoNormal>WARNING: If valid changes were made to your project, make
sure to save it before quitting iSanXoT, as any changes will be lost otherwise
(you won’t be prompted for saving upon closing).  </p>

<p class=MsoNormal style='margin-left:.25in'>To close the iSanXoT application:</p>

<p class=MsoNormal style='margin-left:.25in'>In Windows and Linux: choose <i>Project
&gt; Exit</i>, or click the X in the upper right corner of the main iSanXoT window.</p>

<p class=MsoNormal style='margin-left:.25in'>In Mac: choose <i>iSanXoT (menu)
&gt; Exit</i>, or click the red X in the upper left corner of the main iSanXoT
window.</p>

<p class=MsoNormal>A dialog window will show up asking you to confirm the
application closing. Click “Yes” if you really want to quit iSanXoT. </p>

<p class=MsoNormal>&nbsp;</p>

<h3>iSanXoT Projects</h3>

<p class=MsoNormal>An iSanXoT project is primarily a container used to
structure the data coming from your input file(s) and your workflow. The input
file contains the identification and quantification data (for further details see
the <a href="#_Input_Adaptor">Input Adaptor</a> Section). These fully-customisable
workflows can perform quantitative proteomics analysis, systems biology
analysis, and comparison and merging of experimental data from technical or
biological replicates.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=251 height=240 src="User_Guide_files/image011.png"></p>

<p class=MsoCaption><a name="_Ref103114194"><b>Figure </b></a><b>2</b><b>. Project menu.</b></p>

<h4>Creating a new project</h4>

<p class=MsoNormal>Creating a project is the first step when conducting an
analysis with iSanXoT. By selecting <i>Project &gt; New Project</i> a window
will show up where you can provide a name for the project as well as select a
project folder where iSanXoT output files will be stored.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=536 height=199 src="User_Guide_files/image012.png"></p>

<p class=MsoCaption><b>Figure </b><b>3</b><b>. Window that creates a new
project.</b></p>

<h4>&nbsp;</h4>

<h4>Opening a project</h4>

<p class=MsoNormal>By selecting <i>Project &gt; Open Project</i> as a folder
selection dialog box shows up that allows the user to indicate the location of an
already existing project folder to be opened by iSanXoT.</p>

<p class=MsoNormal>&nbsp;</p>

<h3>iSanXoT Main Window</h3>

<p class=MsoNormal>The iSanXoT main window consists of an overhead menu, adaptor
and module tabs, and content and execution panels (<b>Figure 4</b>).</p>

<h4>Menu</h4>

<p class=MsoNormal>The Menu contains the following items:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Project: Allows operations related to projects and workflows.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Processes: Links to real-time display of the processes currently
in execution by iSanXoT (see <a href="#_Running_Processes">Running Processes</a>
Section).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Help: Houses the different Sections of iSanXoT help.</p>

<h4>Content tabs</h4>

<p class=MsoListParagraph style='text-indent:-.25in'>1.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Five tabs are displayed in iSanXoT’s project page. The Input tab
displays the Project folder, where iSanXoT output files are stored, as well as
the Identification file used in the project (see <a href="#_Adaptors">Input Adaptor</a>
Section). The remaining four tabs give access to iSanXoT modules: <a
href="#_Relation_tables_module">Relation Tables</a>, <a href="#_Basic_modules_1">Basic
Modules</a>, <a href="#_Compound_modules">Compound Modules</a>, and <a
href="#_Reports_modules">Report Modules</a>.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=570 height=337 id="Picture 132"
src="User_Guide_files/image013.png"></p>

<p class=MsoCaption><a name="_Ref103114143"></a><a name="_Ref103114150"><b>Figure
</b></a><b>4</b><b>. Main View of iSanXoT.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Content panel</h4>

<p class=MsoNormal>This panel houses the elements of the Input and Modules tabs,
which can be accessed through the sidebar menu showing on the left side of the panel.
A title and a brief description of the Input/Module element is provided, as
well as help icon linking to additional information on the specific Input
element or Module selected.</p>

<h4><a name="_Execution_panel"></a>Execution panel</h4>

<p class=MsoNormal>The execution panel, located on the bottom of the main
window, allows the user to indicate the number of processors to be used by
iSanXoT, 4 by default. The Start button launches the execution of the workflow
shaped by the Input elements and Modules.</p>

<p class=MsoNormal>&nbsp;</p>

<h2>Importing and Exporting Workflows</h2>

<p class=MsoNormal>A project is shaped by a workflow that instructs iSanXoT how
to process the data provided by the input file(s). While the whole project,
including workflow and data, can be saved as indicated below (see the <a
href="#_Saving_a_project">Saving a project</a> section), there is a way to
import and export just the workflow structure using iSanXoT menu (<b>Figure
2</b>).</p>

<h4>Import Workflow</h4>

<p class=MsoNormal>This option allows to import the task-tables of a workflow.
For that, you have to provide the folder where the workflow is saved.</p>

<h4>Export Workflow</h4>

<p class=MsoNormal>The export workflow saves the task-tables of a workflow in
the folder indicated by the user.</p>

<p class=MsoNormal>&nbsp;</p>

<h2>Executing a Project</h2>

<p class=MsoNormal>Once your project contains all the necessary input data and
workflow elements, you can execute the workflow by clicking “Save and Run” in
the “Execution panel” after indicating the number of processors to be allocated
for iSanXoT (see  <a href="#_Execution_panel">Execution panel</a> above). </p>

<p class=MsoNormal>Bear in mind that every time you click the “Save and Run”
button to execute a workflow, the project is first validated for consistency and
saved. To save a project without executing it, you must use the appropriate
menu item as explained in the next Section.</p>

<h4><a name="_Saving_a_project"></a>Saving a project</h4>

<p class=MsoNormal>The <i>Project &gt; Save Project</i> option saves your
project, which contains the input data and the workflow elements, to the
“Project folder”. The project files are saved in the “.isanxot” folder. WARNING:
Do not manipulate or delete the information stored in the “.isanxot” folder;
you risk losing your project.</p>

<p class=MsoNormal>Whenever iSanXoT is prompted to save a project, the corresponding
workflow is first validated for consistency and won’t be saved when failed. Neither
will the workflow execute if it has not been validated previously.</p>

<p class=MsoNormal>As well as the “.isanxot” folder, the following folders are
necessary to shape your project:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Exps, to store the files created by the Input Data adapters.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Jobs, to store the <i>sample folders</i> of your workflow.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Rels, to store the Relation Tables created by the RELS CREATOR module
(see below).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Reports, to store the Report files created by the REPORT module (see below).</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Stats, to store statistical data.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Logs, to store the workflow execution log files.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Running_Processes"></a>Running Processes</h2>

<p class=MsoNormal>When workflow execution successfully starts, a new window shows
up displaying information about the processes currently running:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Project logs table shows project execution status. Several
project executions can be monitored here, and the user must click a row to have
the corresponding workflow logs displayed (see below).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Workflow logs table displays status for the jobs set up in the
workflow modules. If you click on a row you will see the trace log of the involved<span
class=MsoCommentReference><span style='font-size:8.0pt;line-height:107%'> j</span></span>obs
(unless the job status is “cached”).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=378 height=262 id="Picture 98"
src="User_Guide_files/image014.png"></p>

<p class=MsoCaption><b>Figure </b><b>5</b><b>. View of running processes.</b></p>

<p class=MsoNormal align=left style='text-align:left'>The running processes window
can be also reached from the menu by selecting the “Processes &gt; Main page” option.</p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Modules"></a>Modules</h1>

</div>

<p class=MsoNormal>The iSanXoT desktop application houses a number of modules
based on the SanXoT software package [1].
The information required to setup and execute every module is provided in a task-table. 
 </p>

<p class=MsoNormal>There are four types of modules:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Relation tables is a module that creates the relation tables used
by the iSanXoT modules.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Basic modules call the individual scripts included in the SanXoT
software package [1].</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Compound modules perform a sequence of consecutive
integrations based on the weighted spectrum, peptide and protein (WSPP)
statistical model [2] and the systems-biology triangle (SBT) algorithm [3].</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Finally, there are two Reports: REPORT generates report files
displaying the quantitative results produced by the above Basic and Composite
modules when a workflow is executed; SANSON generates a similarity graph
showing relationships between functional categories on the basis of the protein
elements they share.</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Basic_modules"></a><a
name="_Relation_tables_module"></a>Relation tables module</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_RELS_CREATOR"></a><span style='font-family:"Calibri",sans-serif'>RELS
CREATOR</span></h3>

<p class=MsoNormal>This module generates relation tables (RT) from tab-separated
values (TSV) files. Relation tables, which are TSV files relating lower level
identifiers (e.g. peptides) to the corresponding higher level elements (e.g.
proteins), are required for module execution. For this reason, the naming convention
for the file indicated under “Relation Table to be created” in the RELS CREATOR
task table is <i>lower level + “2” + higher level</i>; e.g. whether an
integration is from the “peptide” level to the “protein” level, a relation
table called “peptide2protein” will be necessary.</p>

<p class=MsoNormal>The fields showing in the RELS CREATOR task table are (<b>Figure 6</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Relation Table to be created</i> specifies relation table
filenames. As commented above, the naming convention for these files is <i>lower
level + “2” + higher level</i>; e.g. whether an integration is from the
“peptide” level to the “protein” level, a relation table called
“peptide2protein” will be necessary.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of Lower level</i> is the column header that
designates which elements from the indicated file (see below) will be taken as
lower level elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of Higher level</i> is the column header that
designates which elements from the indicated file (see below) will be taken as higher
level elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column name of 3<sup>rd</sup> column</i> is the column header
that designates which elements from the indicated file (see below) will be
taken as third column elements in the resulting relation table.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Table from which RT is extracted is the full path name for the
TSV file to be used to build the relation tables. If the cell is empty, the
Input file (ID-q.tsv) is applied for iSanXoT workflow.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=163 src="User_Guide_files/image015.png"></p>

<p class=MsoCaption><a name="_Ref103114314"><b>Figure </b></a><b>6</b><b>.
A sample Task-Table in the example of RELS CREATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Basic_modules_1"></a>Basic
modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_LEVEL_CREATOR"></a><span style='font-family:"Calibri",sans-serif'>LEVEL
CREATOR</span></h3>

<p class=MsoNormal>This module creates levels, which are TSV files containing
identifiers, log<sub>2</sub>-ratio values and statistical weight values. The
data are extracted from the Identification/Quantification file (see <a
href="#_Adaptors">Input Adaptor</a> section).</p>

<p class=MsoNormal>The following fields are displayed in the LEVEL CREATOR task
table (<b>Figure 7</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used to create the level
indicated. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator column</i> specifies which column header from
the identification file designates the quantitative values to be used as a numerator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator column(s)</i> specifies which column header
from the identification file designates the quantitative values to be used as a
denominator for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level to be created</i> designates the level name.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates the name of the folder
where the level data file will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=581 height=174 src="User_Guide_files/image016.png"></p>

<p class=MsoCaption><a name="_Ref103114332"><b>Figure </b></a><b>7</b><b>.
A sample task-table in the LEVEL CREATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_LEVEL_CALIBRATOR"></a><span style='font-family:"Calibri",sans-serif'>LEVEL
CALIBRATOR</span></h3>

<p class=MsoNormal>This module calibrates the above-described levels using the
“Klibrate” program included in the SanXoT software package [1].
To perform the calibration, two parameters (weight constant and variance) are iteratively
calculated using the Levenberg-Marquardt algorithm (for more details see the information
about “Klibrate” in the SanXoT software package [1]).</p>

<p class=MsoNormal>The output calibrated level contains new statistic weight
values for the identifier and log<sub>2</sub>-ratio elements displayed in the
uncalibrated level data file. This is necessary for the levels to be used as
inputs to the INTEGRATE module.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The fields to be completed in this module are (<b>Figure 8</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name(s) of the folder(s) containing
the uncalibrated data file(s) that were previously generated by the LEVEL
CREATOR module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level for integration</i> indicates which lower level elements
are to be used in the integration carried out for the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level for integration</i> indicates which higher level
elements are to be used in the integration carried out for the calibration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Name of calibrated level</i> is the name for the output data
file containing the new, calibrated statistical weight values.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> specifies the name of the folder
where the output data file containing the new, calibrated statistical weight
values will be saved. If the cell is empty, the output sample folder is the
given “Sample folder” (second column).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=193 id="Picture 5" src="User_Guide_files/image017.png"></p>

<p class=MsoCaption><a name="_Ref103114351"><b>Figure </b></a><b>8</b><b>.
A sample task table in the LEVEL CALIBRATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The LEVEL_CALIBRATOR module accepts the following additional
parameters (<b>Figure 9</b>): </p>

<p class=MsoNormal>To perform the calibration two parameters, have to be
calculated: the k (weight constant), and the variance.</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>K-constant </i>sets a forced value for the k-constant. Using
this parameter, the introduced value is forced as K-constant.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Var(x)</i> sets a forced value for the variance. Using this
parameter, the introduced value is forced as the variance.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=579
height=145 id="Picture 11" src="User_Guide_files/image018.png"></p>

<p class=MsoCaption style='text-align:justify'><a name="_Ref103114398"><b>Figure
</b></a><b>9</b><b>. A task-table displaying advanced parameters for the LEVEL
CALIBRATOR module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_INTEGRATE"></a><span style='font-family:"Calibri",sans-serif'>INTEGRATE</span></h3>

<p class=MsoNormal>The INTEGRATE module performs statistical calculations based
on the WSPP model by iteratively applying the generic integration algorithm
(GIA) [3] on calibrated data files (<b>Figure 10</b>). </p>

<p class=MsoNormal>Integrations are carried out from lower level data to higher
level data (e.g. from the peptide level to the protein level and from the protein
level to the gene level).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=423 height=414 id="Picture 58"
src="User_Guide_files/image019.png"></p>

<p class=MsoCaption><a name="_Ref103096863"><b>Figure </b></a><b>10</b><b>. Schematic representation of the INTEGRATE module.</b> The integration is carried out
from any lower level to any higher level using the programs “SanXoT” and
“SanXoTSieve” and the generic integration algorithm (GIA).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>More in detail, the INTEGRATE module needs two TSV files as
inputs:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>A
data file containing three data columns: identifier (a text string that is used
to unambiguously identify the low level elements), quantitative value (log<sub>2</sub>-ratio
of the two measurements to be compared) and statistical weight (a parameter
that measures the accuracy of the quantitative value).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>A
relation table, which links the lower level identifiers to those in the higher
level. This file contains two columns: higher level identifiers on the left and
lower level identifiers on the right.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=582 height=524 id="Picture 41"
src="User_Guide_files/image020.png"></p>

<p class=MsoCaption><a name="_Ref103114442"><b>Figure </b></a><b>11</b><b>.
The INTEGRATE module flowchart.</b> A first integration is done with “SanXoT”
that calculates the variance; then “SanXoTSieve” removes outliers tagging them
in a new relation table; finally, a second integration is done with “SanXoT”
using the variance calculated.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For every integration, the SanXoT program calculates the general
variance using a robust iterative method. Then SanXoTSieve is used to tag
outlier elements [2] by assessing the probability that a lower level element be
a significant outlier of the standardized (i.e. N(0,1)) log<sub>2</sub>-ratio
distribution. The most extreme outliers are thus removed sequentially and the
integration repeated until all outliers below a user-defined false discovery
rate (FDR) threshold have been removed. Finally, a second integration is carried
out by SanXoT using the variance calculated in the first integration and
discarding the outliers tagged in the new relation table (<b>Figure
11</b>).</p>

<p class=MsoNormal>The output data files generated by INTEGRATE contain the
quantitative data for the higher level and can be used as inputs to other
modules; in addition, each integration generates several additional files which
contain information about the integration. For further details see SanXoT documentation
[1].</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The parameters to be provided in the INTEGRATE module task
table are (<b>Figure 12</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the names of the folder(s)
where the lower level data file is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level data
file to be used. This file contains three data columns: identifier,
quantitative value and statistical weight.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the name of the higher level to which
the lower level elements will be integrated.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=282 height=159 id="Picture 6" src="User_Guide_files/image021.png"></p>

<p class=MsoCaption><a name="_Ref103114454"><b>Figure </b></a><b>12</b><b>.
A sample task-table in the INTEGRATE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4><a name="_Advanced_parameters"></a>Advanced parameters</h4>

<p class=MsoNormal>The INTEGRATE module accepts the following additional parameters
(<b>Figure 13</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> is the name of the folder where the level
data and statistics are saved.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the
lower_level2higher_level Relation Table will be integrated. Logical operators
can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><i>Tag</i> is a parameter to distinguish
groups to perform the integration. For instance, if the user specifies “marked”
the elements containing the label “marked” in the third column of the “lower_level2higher_level”
Relation Table will be included in (or discarded from) the integration.</p>

<p class=MsoListParagraphCxSpMiddle>The tag can be used by inclusion, such as &quot;mod&quot;
or by exclusion, putting first the &quot;!&quot; symbol, such as &quot;!mod&quot;. 
Tags should be included in a third column of the relations file.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>Different tags can be combined using
logical operators &quot;and&quot; (&amp;), &quot;or&quot; (|), and
&quot;not&quot; (!), and parentheses. Some examples:</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>!out&amp;mod</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>!out&amp;(dig0|dig1)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>(!dig0&amp;!dig1)|mod1</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>mod1|mod2|mod3</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b><i>Warning</i></b>: Unless specified
otherwise by the user, by default iSanXoT eliminates outliers from the lower
level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle>iSanXoT automatically adds the tag “out” in
the third column of the relation table to label outliers, so that they are not
integrated. It is not thus recommended to use this tag for other purposes.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b><i>Note</i></b> that although the
discarded elements will not be included in calculations, the parameter Z will
be calculated and tabulated in the corresponding output (outStats) file.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>For further details see SanXoT wiki</p>

<p class=MsoListParagraphCxSpMiddle>(<a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a>).</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>FDR</i> is an FDR threshold other than the default value (0.01,
i.e. 1%) for outlier removal. If “0” is specified as the FDR value, then no outliers
will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Var(x)</i> sets a fixed value for the variance. The default
value (blank) means that the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first <i>Lower level-to-Higher level</i>
integration.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=136 src="User_Guide_files/image022.png"></p>

<p class=MsoCaption><a name="_Ref103114466"><b>Figure </b></a><b>13</b><b>.
A sample task-table displaying advanced parameters for the INTEGRATE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_NORCOMBINE"></a><span style='font-family:"Calibri",sans-serif'>NORCOMBINE</span></h3>

<p class=MsoNormal>The NORCOMBINE module combines technical or biological
replicates (<b>Figure 14</b>). For example, NORCOMBINE can be used to merge the
protein level data from 4 individual patients and 4 individual controls into a
patient- and a control level protein dataset, respectively.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=451 height=388 id="Picture 114"
src="User_Guide_files/image023.png"></p>

<p class=MsoCaption><a name="_Ref103114484"><b>Figure </b></a><b>14</b><b>. Schematic representation of the NORCOMBINE module used to combine technical or
biological replicates.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Experiment merging relies on the “Cardenio” program from the
SanXoT software package [1],
which is used to generate merged data files and relation tables that are later
integrated to the grouped level using “SanXoT” and “SanXoTSieve” (<b>Figure 14</b>).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>NORCOMBINE requires the user to specify which <i>lowerNorm</i>
files contain the necessary data for the samples to be combined. These <i>lowerNorm</i>
files, previously generated by the INTEGRATE module, display the lower level
identifiers on the left, followed by the corresponding centred log<sub>2</sub>-ratio
values (i.e. the values obtained after subtracting the high level value) in the
second column, and either the integration statistical weight (in the case of <i>lowerNormV</i>)
or the variance (for <i>lowerNormW).</i> The SanXoT program “Cardenio” [1]
is then used to generate merged data files and relation tables that are later
integrated to the grouped level using “SanXoT” and “SanXoTSieve” (<b>Figure 15</b>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=452 height=522 id="Picture 115"
src="User_Guide_files/image024.png"></p>

<p class=MsoCaption><a name="_Ref103114585"><b>Figure </b></a><b>15</b><b>.
The NORCOMBINE module flowchart.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The default NORCOMBINE module task table shows the following
fields (<b>Figure 16</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folders</i> indicates the names of the folder(s) containing
the lower level data (samples) to be combined.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level</i> indicates the type of elements to be combined (e.g.
peptides or proteins). </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Norm</i> specifies the normalization scheme to be used in the
integrations.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>lowerNorm</i> specifies the type of <i>lowerNorm</i> file (see
above) to be used.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> is the name of the folder where the grouped
level data and statistics are saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=467 height=89 src="User_Guide_files/image025.png"></p>

<p class=MsoCaption><a name="_Ref103114601"><b>Figure </b></a><b>16</b><b>.
A sample task-table in the NORCOMBINE module.</b> In this case, the asterisk
wildcard has been used to select multiple sample folders.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The NORCOMBINE module accepts the following additional parameters
(<b>Figure 17</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>FDR</i> is an FDR threshold other than the default value
(0.01, i.e. 1%) for outlier removal. If “0” is specified as the FDR value, then
no outliers will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Var(x) sets a fixed value for the variance. The default value
(blank) means that the variance will be iteratively calculated based on the
Levenberg-Marquardt algorithm.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=72 src="User_Guide_files/image026.png"></p>

<p class=MsoCaption><a name="_Ref103114615"><b>Figure </b></a><b>17</b><b>.
A sample task table with advanced parameters in the NORCOMBINE module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_RATIOS"></a><span style='font-family:"Calibri",sans-serif'>RATIOS</span></h3>

<p class=MsoNormal>This module prepares the data file and relation table
required as a first step in the calculation of a ratio defined by the user (e.g
KO vs WT). For that, the new log<sub>2</sub>-ratio is calculated as the
difference between numerator and denominator values, whereas the corresponding
statistical weight is assessed according to the method indicated by the user in
the V Method filed of the RATIOS task table (<b>Figure 18</b> and <b>Figure 19</b>): </p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>max</i> uses the maximum value between the numerator and
denominator statistical weight value. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>form</i> uses the value resulting from 1/(1/Vn + 1/Vd), where
Vn and Vd are the statistical weight value for the numerator and the
denominator, respectively.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>avg</i> uses the average value between the numerator and
denominator statistical weight value.</p>

<p class=MsoNormal style='page-break-after:avoid'><img border=0 width=581
height=199 id="Picture 17" src="User_Guide_files/image027.png"></p>

<p class=MsoCaption><a name="_Ref103114679"><b>Figure </b></a><b>18</b><b>.
The RATIOS module flowchart.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The RATIOS module task table displays the following parameter
fields (<b>Figure 19</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator column</i> specifies the name of the folder
containing the quantitative value to be used as a numerator for the new log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator column(s)</i> specifies the name of the
folder(s) containing the quantitative values to be used as a denominator for
the new log<sub>2</sub>-ratio calculation. The sample folders have to be
separated by comma.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level </i>designates the level (i.e. peptide, protein, gene or
category) at which the ratio is to be calculated.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates the name of the folder
where the resulting log<sub>2</sub>-ratio and statistical weight values will be
saved (e.g. KO_vs_WT).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=574 height=65 src="User_Guide_files/image028.png"></p>

<p class=MsoCaption><a name="_Ref103114687"><b>Figure </b></a><b>19</b><b>.
A sample task-table in the RATIOS module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_SBT"></a><span style='font-family:"Calibri",sans-serif'>SBT</span></h3>

<p class=MsoNormal>This module is based on the Systems Biology Triangle (SBT)
algorithm [3], which performs an integration between the lower and the higher
levels using the variance previously obtained in an integration between the
lower and an intermediate level (<b>Figure 20</b>). Usually the SBT module is
applied to carry out the protein-to-grand mean integration using the variance
associated with the protein-to-category integration.   </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=436 height=435 id="Picture 131"
src="User_Guide_files/image029.png"></p>

<p class=MsoCaption><a name="_Ref103093737"></a><a name="_Ref103093681"><b>Figure
</b></a><b>20</b><b>. <a name="_Ref103093692">Schematic representation of the SBT
module.</a></b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters to be entered in the SBT module task
table are (<b>Figure 21</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name of the folder(s) where
the lower level data are located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level (e.g. protein).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Intermediate level</i> indicates the name of the intermediate level
(e.g. category).</p>

<p class=MsoNormal>By default, the higher level is the grand mean of the lower
level elements.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=292 height=66 src="User_Guide_files/image030.png"></p>

<p class=MsoCaption><a name="_Ref103094105"><b>Figure </b></a><b>21</b><b>.
A sample task table in the SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>This module accepts the following additional parameters (<b>Figure 22</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder</i> indicates an alternative folder to
store the resulting log<sub>2</sub>-ratio and statistical weight values other
than “Sample folder(s)”.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower-Higher level</i> and <i>Int(ermediate)-Higher level </i>specify
an alternative higher level other than the grand mean of the lower level elements.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate)</i> Tag and <i>Int(ermediate)-to-Hig(her)</i>
are the text label that indicates which elements from the lower level are
integrated into the intermediate level, and the intermediate level are
integrated into higher level. The tags must be specified in the third column of
the corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label “marked” is used as a Tag, only the lower level elements containing
the label marked in the third column of the lower_level2intermediate_level and
intermediate_level2higher_level Relation Tables will be integrated. Logical
operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module. </p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate) FDR </i>and<i> Int(ermediate)-to-Hig(her)
FDR</i> determine an FDR threshold other than the default value (0.01, i.e. 1%)
for outlier removal in the lower level-to-intermediate level and intermediate
level-to-higher level integration, respectively. If “0” is specified as the FDR
value, then no outliers will be discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Low(er)-to-Int(ermediate) Var(x)</i> and <i>Int(ermediate)-to-Hig(her)
Var(x)</i> indicate the variance to be used in the lower level-to-intermediate
level and intermediate level-to-higher level integration, respectively, as an
alternative to the variance calculated in the lower level-to-intermediate level
integration.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=554 height=168 id="Picture 10"
src="User_Guide_files/image031.png"></p>

<p class=MsoCaption><a name="_Ref103094086"><b>Figure </b></a><b>22</b><b>.
A sample task table with advanced parameters in the SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Compound_modules"></a>Compound
modules</h2>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The Compound modules perform a sequence of consecutive
integrations based on the WSPP statistical model [2]
and the SBT algorithm [3]. In addition, each module creates the initial level
and calibrate this initial level. The WSPP-SBT and WSPPG-SBT modules create and
calibrate the “scan” level; and the WPP-SBT and WPPG-SBT create and calibrate
the “peptide” level.</p>

<h3><span style='font-family:"Calibri",sans-serif'>WSPP-SBT</span></h3>

<p class=MsoNormal>The WSPP-SBT module performs the following integrations:
scan-to-peptide, peptide-to-protein, protein-to-category, protein-to-proteinall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the protein-to-category integration, which
is applied to the protein-to-proteinall integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WSPP-SBT module are
(<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=564 height=227 src="User_Guide_files/image032.png"></p>

<p class=MsoCaption><a name="_Ref103096252"><b>Figure </b></a><b>23</b><b>.
A sample task table in the WSPP-SBT, WSPPG-SBT, WPP-SBT, and WPPG-SBT modules.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WSPP-SBT module accepts the following advanced
parameters (<b>Figure 24</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text
label that indicates which elements from the lower level (“p” and “c”) are
integrated into the higher level (“q” and “a”). The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.<span
class=MsoCommentReference><span style='font-size:8.0pt;line-height:107%'> </span></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p FDR</i>, <i>p&gt;q FDR</i>, <i>q&gt;c FDR</i> establish
an FDR threshold other than 0.01 (1%) for outlier removal in the integrations
scan-to-peptide, peptide-to-protein, and protein-to-category, respectively. If
FDR = 0 is selected, then the outliers are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p Var(x)</i>, <i>p&gt;q Var(x)</i>, <i>q&gt;c Var(x)</i>
set a fixed value for the variance in the integrations scan-to-peptide, peptide-to-protein,
and protein-to-category, respectively. By default, the variance will be
iteratively calculated based on the Levenberg-Marquardt algorithm in the first
lower level-to-higher level integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=111 id="Picture 23"
src="User_Guide_files/image033.png"></p>

<p class=MsoCaption><a name="_Ref103096290"><b>Figure </b></a><b>24</b><b>. Task-Table with advanced parameters in the WSPP-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><span style='font-family:"Calibri",sans-serif'>WSPPG-SBT</span></h3>

<p class=MsoNormal>The WSPPG-SBT module performs the following integrations:
scan-to-peptide, peptide-to-protein, protein-to-gene, gene-to-category, gene-to-geneall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the gen-to-category integration, which is
applied to the gen-to-geneall integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WSPPG-SBT module are
(<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WSPPG-SBT module accepts the following advanced
parameters (<b>Figure 25</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>q&gt;g Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a
Tag</i> are the text label that indicates which elements from the lower level (“p”,
“q” and “c”) are integrated into the higher level (“q”, “g” and “a”). The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label “marked” is used as
a Tag, only the lower level elements containing the label marked in the third
column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p FDR</i>, <i>p&gt;q FDR</i>, <i>q&gt;g FDR</i>, <i>g&gt;c
FDR</i> establish an FDR threshold other than 0.01 (1%) for outlier removal in
the following integrations: scan-to-peptide, peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. If FDR = 0 is selected, then the outliers
are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>s&gt;p Var(x)</i>, <i>p&gt;q Var(x)</i>, <i>q&gt;g Var(x)</i>,
<i>g&gt;c Var(x)</i> set a fixed value for the variance in the integrations:
scan-to-peptide, peptide-to-protein, protein-to-gene, and gene-to-category,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=66 id="Picture 24" src="User_Guide_files/image034.png"></p>

<p class=MsoCaption><a name="_Ref103096474"><b>Figure </b></a><b>25</b><b>. Task-table with advanced parameters in the WSPPG-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><span style='font-family:"Calibri",sans-serif'>WPP-SBT</span></h3>

<p class=MsoNormal>The WPP-SBT module performs the integrations peptide-to-protein,
protein-to-category, protein-to-proteinall and category-to-categoryall. In
addition, the SBT algorithm is used to calculate the variance associated to the
protein-to-category integration, which is applied to the protein-to-proteinall
integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WPP-SBT module are (<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WPP-SBT module accepts the following advanced parameters
(<b>Figure 26</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text
label that indicates which elements from the lower level (“p” and “c”) are
integrated into the higher level (“q” and “a”). The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation Table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q FDR</i>, <i>q&gt;c FDR</i> establish an FDR threshold
other than 0.01 (1%) for outlier removal in the peptide-to-protein and protein-to-category
integrations, respectively. If FDR = 0 is selected, then the outliers are not
discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Var(x)</i>, <i>q&gt;c Var(x)</i> set a fixed value for
the variance in the peptide-to-protein and protein-to-category integrations,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=494 height=90 id="Picture 29" src="User_Guide_files/image035.png"></p>

<p class=MsoCaption><a name="_Ref103096698"><b>Figure </b></a><b>26</b><b>. Task-table with advanced parameters in the WPP-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h3><span style='font-family:"Calibri",sans-serif'>WPPG-SBT</span></h3>

<p class=MsoNormal>The WPPG-SBT module performs the integrations peptide-to-protein,
protein-to-gene, gene-to-category, gene-to-geneall, and category-to-categoryall.
In addition, the SBT algorithm is used to calculate the variance associated to
the gene-to-category integration, which is applied to the gene-to-geneall
integration.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters required by the WPPG-SBT module are
(<b>Figure 23</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Experiment</i> is the column header that designates which
elements from the Identification file will be used in the starting scan-to-peptide
integration.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Identifier column header</i> is the identification file column
header that unambiguously identifies the scans.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio numerator</i> specifies which identification file column
header designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Ratio denominator</i> specifies which identification file
column header designates the quantitative values to be used as a denominator
for the log<sub>2</sub>-ratio calculation.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Output Sample folder: indicates the name of the folder where the
resulting data files will be saved.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The WPPG-SBT module accepts the following advanced
parameters (<b>Figure 27</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Tag</i>, <i>q&gt;g Tag, p&gt;a Tag, q&gt;a Tag,</i> <i>c&gt;a
Tag</i> are the text label that indicates which elements from the lower level (“p”,
“q” and “c”) are integrated into the higher level (“q”, “g” and “a”). The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label “marked” is used as
a Tag, only the lower level elements containing the label marked in the third
column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q FDR, q&gt;g FDR</i>, <i>g&gt;c FDR</i> establish an FDR
threshold other than 0.01 (1%) for outlier removal in the integrations peptide-to-protein,
protein-to-gene, and gene-to-category, respectively. If FDR = 0 is selected,
then the outliers are not discarded.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>p&gt;q Var(x)</i>, <i>q&gt;g Var(x), g&gt;c Var(x)</i> set a
fixed value for the variance in the integrations peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. By default, the variance will be
iteratively calculated based on the Levenberg-Marquardt algorithm in the first
lower level-to-higher level integration (<b>Figure 10</b>).</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>More params</i> allows adding more parameters to the internal
programs of the module. For more details see <a href="#_More_params">More
params</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=75 id="Picture 47" src="User_Guide_files/image036.png"></p>

<p class=MsoCaption><a name="_Ref103097509"><b>Figure </b></a><b>27</b><b>. Task-table with advanced parameters in the WPPG-SBT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2 align=center style='text-align:center'><a name="_Report_modules"></a><a
name="_Reports_modules"></a>Reports modules</h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_REPORT"></a><span style='font-family:"Calibri",sans-serif'>REPORT</span></h3>

<p class=MsoNormal>The REPORT module allows the collection of the statistical variables
(n, tags, Xinf, Vinf, Xsup, Vsup, Z, and FDR) from the different integrations
performed into result tables. </p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters showing in the REPORT task table are
(<b>Figure 28</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the name(s) of the folder(s)
where the values of the statistical variables to be retrieved are located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the starting level (i.e. peptide,
protein, or category) for the integration whose statistical variables are to be
reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the ending level for the
integration whose statistical variables are to be reported.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Reported vars</i> specifies which statistical variables will
be reported. The available variables are n, tags, Xinf, Vinf, Xsup, Vsup, Z,
FDR, X’inf and Winf.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output report</i> is the report filename (without extension).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=535 height=318 src="User_Guide_files/image037.png"></p>

<p class=MsoCaption><a name="_Ref103115700"><b>Figure </b></a><b>28</b><b>.
A sample task table in the REPORT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For instance, the first row of the task table shown in <b>Figure 28</b> prompts the REPORT module to read the variable “n” from the <i>scan2peptide_outStats.tsv</i>
file that contains the statistical outcome from the scan-to-peptide integration
(the asterisk wildcard character in <i>Sample folder(s)</i> causes REPORT to
retrieve the “n” variable from every sample). These “n” values are written to a
report file named “Nscan_pep” that is stored in the project “reports” folder.</p>

<p class=MsoNormal>The second row instructs the module to read the variables “Xinf”,
“Z” and “FDR” from the statistical outcome of the peptide-to-protein
integration (once again for every sample). These values are written to a report
file named “Nscan_Normpep_prot_XZ”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The REPORT module accepts the following advanced parameters
(<b>Figure 29</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Level names to show</i> allows the user to restrict the
elements to be written to the <i>Output report</i> to those from the <i>Lower
level</i> or the <i>Higher level</i>. Both levels are used by default.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Merge with report</i> designates the file whose <i>Reported
vars</i> will be incorporated into the <i>Output report</i> after intersection
with the latter file.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Add columns from relation table</i> appends <i>Lower level</i>
elements, extracted from the relation table designated, to the <i>Output report</i>.
It is possible to indicate multiple relation tables separated by a comma.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Filter</i> allows to filter the data to be transferred to the <i>Output
report</i> based on the <i>Reported vars</i> (n, Z, FDR, etc.). For more details,
see <a href="#_Filter_(for_REPORT">Filter</a> in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=261 id="Picture 4" src="User_Guide_files/image038.png"></p>

<p class=MsoCaption><a name="_Ref103116151"><b>Figure </b></a><b>29</b><b>.
A sample task-table with advanced parameters in the REPORT module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The reports indicated under <i>Output report</i> and <i>Merge
with report</i> are merged according to the column header that they share.
Thus, the REPORT task table shown in <b>Figure 30</b> will cause the module to incorporate
the number of scans per peptide, displayed in the report “Nscan_pep”, to the
report “Nscan_Normpep_prot_XZ”, as these two reports share the lower level elements
showing under the “peptide” header.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=600 height=156 id="Picture 26"
src="User_Guide_files/image039.png"></p>

<p class=MsoCaption><a name="_Ref103116369"><b>Figure </b></a><b>30</b><b>.
Report merging in the REPORT module.</b> The first task table row creates a
report file (“Nscan_pep”) with the (n)umber of scans per peptide. The second
row creates a report file called “Nscan_Normpep_prot_XZ” that contains, apart
from the variables “Xinf”, “Z”, and “FDR” coming from the peptide-to-protein
integration, the (n)umber of scans per peptide previously stored in the “Nscan_pep”
report, as these two reports share the lower level elements showing under the
“peptide” header.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>In addition, it is possible to incorporate additional data
from one or more relation tables into the reports (<b>Figure 31</b>). When a
given Relation Table is indicated under <i>Add columns from relation table</i>,
the REPORT module will first attempt to incorporate to the <i>Output report</i>
(“Npep_Quanprot_cat” in the example) the elements related to the <i>Lower level</i>
(“protein” in this case) elements in the relation table. If the <i>Lower level</i>
elements are missing in the relation table, then REPORT will try to incorporate
the elements related to the Higher level (“category” in this case) instead. If
neither the <i>Lower level</i> nor the <i>Higher level</i> can be found in the
relation table, then no action is performed. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=573 height=101 id="Picture 27"
src="User_Guide_files/image040.png"></p>

<p class=MsoCaption><a name="_Ref103116417"><b>Figure </b></a><b>31</b><b>.
Adding data from relation tables into the report files. </b>The relation table “protein2gene”
contains a column with protein identifiers under the “protein” header and
another column with the corresponding gene name under the “gene” header,
whereas the relation table “protein2description” contains, apart from the
“protein” elements, a column with the corresponding protein description. The
first task table row will prompt REPORT to incorporate the gene names and
protein descriptions contained in the relation tables to the report file “Npep_Quanprot_cat”
report file, as “protein” is the <span style='font-style:normal'>Lower level</span>
they all three shares.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Finally, the report data can be filtered performing logical
operations with the <i>Reported vars</i> in the <i>Filter</i> field. For
instance, in the report task table displayed in <b>Figure 32</b>:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &lt;= 100, filters out from the report the
variables Z and FDR for the category-to-categoryall integration when the
(n)umber of proteins per category is greater than 100.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>n_protein2category &gt;= 5 &amp; n_protein2category &lt;= 100
retrieves the variables Z and FDR for the category-to-categoryall integration
when the (n)umber of proteins per category is in the [5, 100] range.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>KO_vs_WT@FDR_category2categoryall &lt; 0.05, retrieves the variables
Z and FDR for the category-to-categoryall integration provided that the FDR corresponding
to the “KO_vs_WT” samples is less than 0.05.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=583 height=98 id="Picture 32" src="User_Guide_files/image041.png"></p>

<p class=MsoCaption><a name="_Ref103116468"><b>Figure </b></a><b>32</b><b>. Filtering the report data.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The compound variables shown in the <i>Filter</i> field on <b>Figure 32</b> follow the structure <i>Reported var</i>_<i>integration</i>, like “n_protein2category”.
Such filter applies to all samples. </p>

<p class=MsoNormal>However, the filter “KO_vs_WT@FDR_category2categoryall” is
applied based on the variable “FDR” from the category-to-categoryall
integration, but only to the “KO_vs_WT” sample. Moreover, the filter “WT1,WT2@FDR_category2categoryall”
is applied to the “WT1” and “WT2” samples.</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_SANSON"></a><span style='font-family:"Calibri",sans-serif'>SANSON</span></h3>

<p class=MsoNormal>The SANSON module generates a similarity graph showing the
relationship between functional categories based on their protein components. </p>

<p class=MsoNormal>&nbsp;</p>

<h4>Standard parameters</h4>

<p class=MsoNormal>The standard parameters to be provided for this module are (<b>Figure 33</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Forced execution</i>: This checkbox field indicates whether to
force the execution or not.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Sample folder(s)</i> indicates the names of the folder(s)
where the lower level data file is located.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower level</i> indicates the name of the lower level elements
(“protein” in this case) to be used.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher level</i> indicates the name of the higher level (“category”
in this case) to which the lower level elements will be integrated.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Output Sample folder </i>designates a folder other than <i>Sample
folder(s)</i> where the results will be saved.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=383 height=65 src="User_Guide_files/image042.png"></p>

<p class=MsoCaption><a name="_Ref103116548"><b>Figure </b></a><b>33</b><b>.
A sample task table in the SANSON module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<h4>Advanced parameters</h4>

<p class=MsoNormal>The SANSON module accepts the following additional parameters
(<b>Figure 34</b>):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Lower norm</i> specifies the normalization scheme to be used
with the lower level elements. The default value is included the normalization
of lower level to all.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Higher norm</i> specifies the normalization scheme to be used
with the higher level elements. The default value is included the normalization
of higher level to all.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Tag</i> is a text label that indicates which elements from the
lower level are integrated into the higher level. The tags must be specified in
the third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the
lower_level2higher_level Relation table will be integrated. Logical operators
can also be used in the Tag field to make complex decisions.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-GB>By default, iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle>For further details, read the Advanced
Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a> module.<i> </i></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Filter</i> allows to filter the data based on the FDR and
number of proteins. For more details, see <a href="#_Filter_param_(in">Filter</a>
in the “Special Parameters” Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=57 id="Picture 2" src="User_Guide_files/image043.png"></p>

<p class=MsoCaption><a name="_Ref103116564"><b>Figure </b></a><b>34</b><b>.
A sample task-table with advanced parameters in the SANSON module.</b></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><a name="_Complete_Modules"></a><a name="_WSPP-SBT"></a><a
name="_WSPPG-SBT"></a><a name="_WPP-SBT"></a><a name="_WPPG-SBT"></a>&nbsp;</p>

<h2><a name="_Special_parameters"></a>Special parameters</h2>

<p class=MsoNormal><a name="_Keywords_for_the"></a>&nbsp;</p>

<h3><a name="_Multiple_samples"></a>Multiple samples</h3>

<p class=MsoNormal>The “Sample folder(s)” field of the different module task
tables admit multiple samples. For instance, let's consider the samples created
with the following LEVEL CREATOR task table:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=174 id="Picture 64" src="User_Guide_files/image044.png"></p>

<p class=MsoNormal>We can include multiple samples separated by a comma, for
example, in the INTEGRATE module task table: </p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=227 id="Picture 7" src="User_Guide_files/image045.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Asterisk_is_our"></a>Asterisk is our jack of all trades</h3>

<p class=MsoNormal>The module task tables admit the usage of the asterisk
symbol as a wildcard character. Let’s once more consider the samples created
with the following LEVEL CREATOR task table:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=174 id="Picture 59" src="User_Guide_files/image044.png"></p>

<p class=MsoNormal>Each row calculates a ratio that is saved to the corresponding
<i>Output Sample folder</i>. Thus, the ratio of 113 to the mean of 113, 114, 115,
and 116 is saved to the “Jurkat_WT/WT_1” folder; the 114 to the mean of 113,
114, 115 ratios is saved to “Jurkat_WT/WT_2”, and so on. One way to create the
task table of the INTEGRATE module could be the following, where each row
represents an integration for a given sample:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=225 height=308 id="Picture 62" src="User_Guide_files/image046.png"></p>

<p class=MsoNormal>However, this task table can be simplified applying the
asterisk wildcard. For instance, the task-table below allows to indicate multiple
sample folders, namely every folder starting with “Jurkat_WT/” or “Jurkat_KO/”.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=275 height=274 id="Picture 60" src="User_Guide_files/image047.png"></p>

<p class=MsoNormal>We can reduce this expression even more using just an asterisk:
the first row of the following task table performs the integrations peptide-to-protein,
protein-to-category, peptide-to-peptideall, protein-to-proteinall, and category-to-categoryall
in every sample folder defined with LEVEL CREATOR.</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=284 height=159 id="Picture 63" src="User_Guide_files/image048.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3>Multiple samples in the inputs and outputs</h3>

<p class=MsoNormal>In the cases we have multiple input samples separated by
comma but we want to save the results in another output sample folder, we
indicate them in the same way giving the output folders separated by comma. It
is required to provide the same number of folders:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=579 height=236 id="Picture 13" src="User_Guide_files/image049.png"></p>

<p class=MsoNormal>In the same way happens with the asterisk character (jack of
all trades). In the “Output Sample folder(s)” we can add a suffix in the input samples.
The following task-table illustrate that the output sample folders would be
contain the “_New” suffix:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=487 height=158 id="Picture 30" src="User_Guide_files/image050.png"></p>

<p class=MsoNormal>In addition, we can rename the subfolder adding the new name
in the “Output Sample folder(s)” or add a new subfolder:</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=399 height=181 id="Picture 33" src="User_Guide_files/image051.png"></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_More_params"></a>More params</h3>

<p class=MsoNormal>Some modules accept a column parameter in the Task-Table
called “More params”. This column allows you to provide advanced parameters for
the SanXoT programs [1].
The program descriptions are in the following wiki link:</p>

<p class=MsoNormal><a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The iSanXoT module are composed by several programs of
SanXoT. For this reason, the “More params” of a module accepts the advanced
parameters of composed programs indicated by a name.</p>

<p class=MsoNormal>For example, </p>

<p class=MsoNormal style='margin-left:35.4pt'>INTEGRATE:</p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'>&quot;sanxot1&quot;:
&quot; -m 300 -g &quot;, &quot;sanxot2&quot;: &quot;-s --sweepdecimals=2.5&quot;</p>

<p class=MsoNormal>In above example the first “sanxot” program that compose the
INTEGRATE module, receives the “-m 300 -g” as parameter, and the second
“sanxot” receives also the parameter &quot;-s --sweepdecimals=2.5&quot;.</p>

<p class=MsoNormal style='margin-left:35.4pt'>WSPP-SBT:</p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'>&quot;p2q_sanxot2&quot;:
&quot; -m 100 -s &quot;, &quot;q2a_sanxot1&quot;: &quot;-m 100&quot;</p>

<p class=MsoNormal>In this example, the WSPP-SBT module has multiple
integrations: scan-to-peptide, peptide-to-protein, etc. For more information,
see the WSPP-SBT section. Thus, the second “sanxot” program of peptide-to-protein
(q) integration will receive the parameter “-m 100 -s”, and then, the first
“sanxot” of protein (q)-to-proteinall (a) integration will receive the “-m
100”.</p>

<p class=MsoNormal>&nbsp;</p>

<h4>The program names for each Module </h4>

<p class=MsoNormal>INTEGRATE: sanxot1, sanxotsieve, sanxot2</p>

<p class=MsoNormal>NORCOMBINE: create_exp_tags, cardenio, sanxot1, sanxotsieve,
sanxot2 </p>

<p class=MsoNormal>SBT: l2i_sanxot1, l2i_sanxotsieve, l2i_sanxot2, i2h_sanxot1,
i2h_sanxotsieve, i2h_sanxot2, l2h_sanxot1, l2h_sanxotsieve, l2h_sanxot2</p>

<p class=MsoNormal>WSPP_SBT: level_creator, klibrate, s2p_sanxot1,
s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve, p2q_sanxot2,
q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1,
c2a_sanxotsieve, c2a_sanxot2</p>

<p class=MsoNormal>WSPPG_SBT: level_creator, klibrate, s2p_sanxot1,
s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve, p2q_sanxot2,
p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1, q2g_sanxotsieve,
q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2,
g2a_sanxot1, g2a_sanxotsieve, g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve,
c2a_sanxot2</p>

<p class=MsoNormal>WPP_SBT: level_creator, klibrate, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve,
c2a_sanxot2</p>

<p class=MsoNormal>WPPG_SBT: level_creator, klibrate, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1, q2g_sanxotsieve,
q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, g2a_sanxot1, g2a_sanxotsieve,
g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve, c2a_sanxot2</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Filter_param_(in"></a><a name="_Filter_(for_REPORT"></a>Filter (for
REPORT and SANSON)</h3>

<p class=MsoNormal>The REPORT module and SANSON module accept a Filter
parameter. This parameter filters the data based on some variables depending on
the module.</p>

<p class=MsoNormal>In the case of REPORT module, the filtered variables are the
Reported vars: n, Z, FDR, etc. For instance:</p>

<p class=MsoNormal>FDR_category2category &lt; 0.05) &amp; (n_protein2category
&gt;=5) &amp; (n_protein2category &lt;= 100)</p>

<p class=MsoNormal>For the SANSON module, the filtered variables are FDR and
the related number (n_rel). For example:</p>

<p class=MsoNormal style='text-indent:35.4pt'>([FDR] &lt; 0.05) &amp; ([n_rel]
&gt;= 10) &amp; ([n_rel] &lt;= 100)</p>

<p class=MsoNormal>Different variables can be combined using the comparisons:
&gt;=, &lt;=, !=, &lt;&gt;, ==, &gt;, &lt;; and using logical operators “and”
(&amp;), “or” (||), and “not” (!). </p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc124328440"></a><a name="_Sample_Workflows_with"></a>Sample
Workflows with Application to Case Studies</h1>

</div>

<p class=MsoNormal>We describe below in detail four sample workflows that
illustrate the capacity of iSanXoT to statistically ascertain abundance changes
in both multiplexed, isotopically labeled [3-5] and label-free [6]
proteomics experiments.</p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h2><a name="_Hlk122708883"></a><a name="_Toc124328441"></a><a
name="_Workflow_1:_One-step"></a>Workflow 1: One-step quantification in a
labeled experiment</h2>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h3><a name="_Toc124328442">Experimental</a></h3>

<p class=MsoNormal>The identification and quantification data from
García-Marqués <i>et al</i>. [3] were used to illustrate workflow 1
performance. This study characterizes the molecular alterations that take place
along time when vascular smooth muscle cells (VSMCs) are treated with
angiotensin-II (AngII). Cells were incubated with AngII for 0, 2, 4, 6, 8, and
10 h; after protein extraction and digestion, the resulting peptides were
isobarically labelled with iTRAQ 8-plex reagents, mixed appropriately and
analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS). Protein
identification was achieved by database searching against target and decoy
sequences using Proteome Discoverer 1.4 (Thermo Fisher Scientific; see Section
3 below to learn how these data are adapted to be used with iSanXoT). The false
discovery rate (FDR) for peptide identification was calculated using the
refined method [7].</p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h3><a name="_Toc124328443">Workflow operation</a></h3>

<p class=MsoNormal>Workflow 1 is made up of the compound module WSPP-SBT and
the basic module REPORT (<i><span style='line-height:107%;color:#BF8F00'>Figure
35</span></i>A). The WSPP-SBT performs a sequence of consecutive integrations
based on the WSPP statistical model [2] and the SBT algorithm [3].</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=655 height=304 id="Picture 20"
src="User_Guide_files/image052.jpg"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771130"></a><a name="_Ref122446667"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>35</span></i><i><span style='color:#BF8F00'>. Scheme of
workflow 1 (one-step quantification in a labeled experiment) showing module
components: WSPP-SBT and REPORT (A) and RELS CREATOR (B).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The WSPP-SBT module starts with the
creation of the log2 ratios at the scan (or spectrum) level. These log2 ratios
are calculated using the numerator and denominator values indicated in the
module task table (<i><span style='line-height:107%;color:#BF8F00'>Figure 36</span></i>)
and subsequently calibrated to account for the variability associated with
LC-MS/MS measurements, after which the <i>scan-to-peptide</i> and <i>peptide-to-protein</i>
integrations are carried out.</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=553 height=177 id="Picture 38"
src="User_Guide_files/image053.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771182"></a><a name="_Ref122448847"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>36</span></i><i><span style='color:#BF8F00'>. The
WSPP-SBT task table for workflow 1.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Then the SBT (systems biology
triangle) algorithm is applied for the detection of functional category changes
originated by the coordinated behaviour of proteins (<i><span style='line-height:
107%;color:#BF8F00'>Figure 35</span></i>A).  For that, the algorithm first
calculates the variance of the <i>protein-to-category</i> integration, which is
then applied to the <i>protein-to-protein grand mean</i> (hereinafter referred
to as <i>protein-to-proteinall</i>) integration, and finally performs the <i>category-to-category
grand mean</i> (hereinafter referred to as <i>category-to-categoryall</i>)
integration. All these integrations created by the WSPP-SBT module are saved in
the indicated <i>Output Sample folder</i>.</p>

<p class=MsoNormal style='text-indent:.5in'>Every integration step needs a
relation table that links lower- to higher-level elements. This text file is
created by the RELS CREATOR module (<i><span style='line-height:107%;
color:#BF8F00'>Figure 35</span></i>B and <i><span style='line-height:107%;
color:#BF8F00'>Figure 37</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=149 id="Picture 40"
src="User_Guide_files/image054.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771279"></a><a name="_Ref122622320"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>37</span></i><i><span style='color:#BF8F00'>. The RELS
CREATOR task table for workflow 1.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Thus, the <i>peptide2protein</i>
relation table (<i><span style='line-height:107%;color:#BF8F00'>Figure 38</span></i>A)
is a text file linking the peptides identified to the proteins they come from.
The elements of the <i>protein2category</i> relation table (<i><span style='line-height:107%;color:#BF8F00'>Figure 38</span></i>B) were retrieved from a text
file containing functional annotations for mouse proteins gathered from several
protein function databases (<i><span style='line-height:107%;color:#BF8F00'>Figure
37</span></i>), as described by the authors [3]. </p>

<p class=MsoNormal style='page-break-after:avoid'><span style='font-size:14.0pt;
line-height:107%'><img border=0 width=619 height=166 id="Picture 35"
src="User_Guide_files/image055.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771300"></a><a name="_Ref122622483"><i><span
style='color:#BF8F00'>Figure</span></i></a><i><span style='color:#BF8F00'> </span></i><i><span style='color:#BF8F00'>38</span></i><i><span style='color:#BF8F00'>. Excerpt from
the peptide2protein (A) and protein2category (B) relation tables that link
peptides to proteins and proteins to categories, respectively.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The REPORT module is used in this
workflow to collect the following statistical variables for all samples
(2h-AngII, 4h-AngII, 6h-AngII, 6h-Control, 8h-AngII, and10h-AngII), as prompted
by the asterisk (<i><span style='line-height:107%;color:#BF8F00'>Figure 39</span></i>).
The asterisk symbol is used in the task tables as a wildcard character. For
further details, consult the iSanXoT wiki (<a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki">https://github.com/CNIC-Proteomics/iSanXoT/wiki</a>).
The simple options used in this workflow are the following:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Number of peptides per protein (<i>n</i>), obtained from the <i>peptide-to-protein
     </i>integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Mean-corrected protein log2 ratios expressed in units of standard
     deviation (<i>Z</i>), calculated from the <i>protein-to-proteinall</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>False Discovery Rate (<i>FDR</i>) values indicating which <i>Z</i>
     values stand out as outliers in the <i>protein-to-proteinall</i>
     integration.</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Number of proteins per category (<i>n</i>), obtained from the <i>protein-to-category</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Mean-corrected category log2 ratios expressed in units of standard
     deviation (<i>Z</i>), calculated from the <i>category-to-categoryall</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>False Discovery Rate (<i>FDR</i>) values indicating which <i>Z</i>
     values stand out as outliers in the <i>category-to-categoryall</i>
     integration.</li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;margin-bottom:.0001pt;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><i><img
border=0 width=623 height=93 id="Picture 39" src="User_Guide_files/image056.png"></i></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771377"></a><a name="_Ref122622693"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>39</span></i><i><span style='color:#BF8F00'>. The REPORT
task table for workflow 1.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The following advanced options of
the REPORT module have been used in this workflow (<i><span style='line-height:
107%;color:#BF8F00'>Figure 39</span></i>):</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Merge with report</i> designates the report file whose
     variables will be incorporated into the current report on the basis of
     their element identifiers. Thus, the <i>Npep2prot_Quanprot</i> report is
     first populated with the Z values of the <i>protein-to-proteinall</i>
     integration, and then added the number of peptides per protein coming from
     the <i>Npep2prot</i> report (<i><span style='color:#BF8F00'>Figure 39</span></i>).
     This is possible because the <i>protein</i> level is present in both the <i>Npep2prot</i>
     and <i>Npep2prot_Quanprot</i> reports.</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Column headers to eliminate</i> restricts the elements to be
     written to the <i>Output reports Npep2prot_Quanprot </i>and <i>Nprot2cat_Quancat
     </i>to those from the protein and category level, respectively. This
     prevents data duplication upon merging with the <i>Npep2prot</i> and <i>Nprot2cat</i>
     reports, as these contain also the peptide and protein level, respectively
     (<i><span style='color:#BF8F00'>Figure 39</span></i>).</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Add columns from relation table</i> appends a column with the
     description of the protein to the <i>Npep2prot_Quanprot </i>report. Again,
     this is possible because the protein level is present in both the <i>Npep2prot_Quanprot
     </i>report and the <i>protein2description</i> relation table (<i><span style='color:#BF8F00'>Figure 39</span></i>).</li>
</ul>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=248 height=323 id="Picture 74" src="User_Guide_files/image057.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771474"></a><a name="_Ref122623033"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>40</span></i><i><span style='color:#BF8F00'>. Relative
abundance of proteins (Zq, Left) and functional categories (Zc, Right)
retrieved from the Npep2prot_Quanprot and Nprot2cat_Quancat reports, respectively,
generated with the REPORT module of Workflow1 (</span></i><i><span style='color:#BF8F00'>Figure 39</span></i><i><span style='color:#BF8F00'>).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>As previously shown with SanXoT
[2], iSanXoT workflow 1, based on the WSPP-SBT and the REPORT modules, reveals
a slight tendency to increased protein abundance over time. In addition, the
SBT module reveals that the categories found altered at 4, 8, and 10 h, but not
those at 2 h, were essentially the same that were altered at 6 h (<i><span style='line-height:107%;color:#BF8F00'>Figure 40</span></i>).</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328444">Workflow execution</a></h3>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from</p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP-SBT.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP-SBT.zip</span></a></p>

<p class=MsoNormal>See the <i>Executing a workflow template</i> Section below
for detailed instructions.</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:14.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h2><a name="_Toc124328445"></a><a name="_Workflow_2:_Step-by-step"></a>Workflow
2: Step-by-step quantification and sample combination in a labeled experiment</h2>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h3><a name="_Toc124328446">Experimental</a></h3>

<p class=MsoNormal style='page-break-after:avoid'>We have applied workflow 2 to
quantitative data obtained by González-Amor <i>et al</i>. [5]
in their study of the contribution of interferon-stimulated gene 15 (ISG15) to
vascular damage associated with hypertension. Protein extracts from mouse
aortic tissue (16 samples from eight individuals:</p>

<p class=MsoNormal style='page-break-after:avoid'>two WT-Control, two ISG15-KO,
two WT+AngII, and two ISG15-KO+AngII) were trypsin-digested, after which the
resulting peptides were isobarically labelled using iTRAQ 8-plex and
distributed in two labeled experiments that were analyzed by LC-MS/MS. Protein
identification </p>

<p class=MsoNormal>was achieved by database searching against target and decoy
sequences using Proteome Discoverer 2.1 (Thermo Fisher Scientific; see Section
3 below to learn how these data are adapted to be used with iSanXoT). The FDR
for peptide identification was calculated using the refined method [7].</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=466 height=458 id="Picture 78"
src="User_Guide_files/image058.jpg"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771564"></a><a name="_Ref122690540"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>41</span></i><i><span style='color:#BF8F00'>. Scheme of
workflow 2 (step-by-step quantification and sample combination in a labeled
experiment) showing module components: LEVEL CREATOR, LEVEL CALIBRATOR,
INTEGRATE, NORCOMBINE, RATIOS, SBT, and REPORT (A) and RELS CREATOR (B).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328447">Workflow operation</a></h3>

<p class=MsoNormal>Workflow 2 comprises all six basic modules: LEVEL CREATOR,
LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, RATIOS, and SBT plus the REPORT module
(<i><span style='line-height:107%;color:#BF8F00'>Figure 41</span></i>). The
starting module, LEVEL CREATOR, generates the level files, sample folders and
log2 ratios indicated in the corresponding task table (<i><span
style='line-height:107%;color:#BF8F00'>Figure 42</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=620 height=324 id="Picture 50"
src="User_Guide_files/image059.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771575"></a><a name="_Ref122690553"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>42</span></i><i><span style='color:#BF8F00'>. The LEVEL
CREATOR task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The output level files, <i>u_scan</i>
(<i>uncalibrated scan</i>), display element identifiers together with their
corresponding log2-ratio and statistical weight values (<i><span
style='line-height:107%;color:#BF8F00'>Figure 43</span></i>). This <i>uncalibrated</i>
weight is the inverse of the variance associated with the log2-ratio, and
therefore a higher weight corresponds to a more accurate quantification.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=566 height=148 id="Picture 51"
src="User_Guide_files/image060.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771591"></a><a name="_Ref122691254"><i><span
style='color:#BF8F00'>Figure</span></i></a><i><span style='color:#BF8F00'> </span></i><i><span style='color:#BF8F00'>43</span></i><i><span style='color:#BF8F00'>. Excerpt from
one of the u_scan files generated by workflow 2 LEVEL CREATOR module showing
element identifiers (left column), log2 ratios (center column), and statistical
weights (right column).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Prior to their integration into a
higher level, <i>u_scan</i> levels are calibrated to account for the
variability of the LC-MS/MS analysis with the LEVEL CALIBRATOR module, which
results in <i>scan</i> calibrated levels with recalculated statistical weights
(<i><span style='color:#BF8F00'>Figure 44</span></i>, Top). Then the INTEGRATE
module performs the <i>scan-to-peptide</i>, <i>peptide-to-protein</i> and <i>protein-to-proteinall</i>
integrations according to the module task table (<i><span
style='color:#BF8F00'>Figure 44</span></i>, Bottom). The protein outliers are
not discarded in this integration as prompted by the 0 value indicated for the
FDR parameter.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=379 height=144 id="Picture 87"
src="User_Guide_files/image061.jpg"></p>

<p class=MsoCaption><a name="_Ref124348788">Figure </a>44 . The LEVEL
CALIBRATOR (Top) and INTEGRATE (Bottom) task tables for workflow 2.</p>

<p class=MsoNormal style='page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Every integration requires a
specific relation table linking the corresponding lower- and higher-level
elements that is generated by RELS CREATOR (<i><span style='line-height:107%;
color:#BF8F00'>Figure 41</span></i>B and <i><span style='line-height:107%;
color:#BF8F00'>Figure 45</span></i>). Thus, the <i>peptide2protein </i>relation
table (<i><span style='line-height:107%;color:#BF8F00'>Figure 45</span></i>)
describes the relationships between the peptide level (<i>i.e.</i> the lower
level) and the protein level (<i>i.e.</i> the higher level). Note that the
elements of the <i>protein2category</i> relation table were retrieved from a
text file containing functional annotations from the Gene Ontology
knowledgebase [5].</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=575 height=115 id="Picture 93"
src="User_Guide_files/image062.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771687"></a><a name="_Ref122691348"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>45</span></i><i><span style='color:#BF8F00'>. The RELS
CREATOR task table for workflow 2.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Next, workflow 2 resorts to the
NORCOMBINE basic module (<i><span style='line-height:107%;color:#BF8F00'>Figure
41</span></i>A) to combine biological replicates, which results in the four new
samples indicated on <i><span style='line-height:107%;color:#BF8F00'>Figure 46</span></i>:
WT-C, WT-AngII, ISG15-C, and ISG15-AngII. The module task table indicates that
samples were combined at the <i>protein</i> level using the <i>proteinall</i>
level for normalization. NORCOMBINE also requires the user to specify which <i>lowerNorm</i>
files contain the necessary data for the samples to be combined. These <i>lowerNorm</i>
files, previously generated by the INTEGRATE module, contain lower-level
identifiers along with the corresponding centered log2-ratio values and either
the integration statistical weight (in the case of <i>lowerNormV</i>, as chosen
in this workflow) or the variance (for <i>lowerNormW</i>) (<i><span style='line-height:107%;color:#BF8F00'>Figure 46</span></i>). NORCOMBINE first generates
merged data files and relation tables that are later integrated to the
above-described four grouped levels.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=550 height=107 id="Picture 54"
src="User_Guide_files/image063.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771757"></a><a name="_Ref122699608"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>46</span></i><i><span style='color:#BF8F00'>. The
NORCOMBINE task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>                The four new samples (grouped levels or
sample groups) are then used by the RATIOS basic module to calculate two
ratios: <i>WT-AngIIvsWT-C</i>, where wild-type AngII-treated animals are
compared to controls; and <i>ISG15-AngIIvsISG15-C</i>, where ISG15
AngII-treated animals are compared to controls (<i><span style='line-height:
107%;color:#BF8F00'>Figure 47</span></i>). The module calculates the new log2
ratios as the difference between <i>Ratio numerator</i> and <i>Ratio
denominator</i> values, whereas the corresponding statistical weight is
assessed according to the <i>V Method</i> indicated in the task table (<i>max</i>
in this case, which means the maximum weight between the samples indicated
under <i>Ratio numerator</i> and <i>Ratio denominator</i>. The other two
choices are <i>avg</i>, the average of the two weights; and <i>form</i>, 1/(1/V<sub>n</sub>
+ 1/V<sub>d</sub>), where V<sub>n</sub> and V<sub>d</sub> are the statistical
weight of the samples indicated under<i> Ratio numerator</i> and <i>Ratio
denominator</i>, respectively).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=400 height=76 id="Picture 55" src="User_Guide_files/image064.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771774"></a><a name="_Ref122699630"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>47</span></i><i><span style='color:#BF8F00'>. The RATIOS
task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>                The last basic module executed in workflow
2, SBT (<i><span style='line-height:107%;color:#BF8F00'>Figure 48</span></i>),
relies on the systems biology triangle algorithm [3]
to perform an integration between the <i>Lower level</i> (<i>protein</i> in
this case) and the corresponding grand mean using the variance previously
obtained in the integration between the <i>Lower level</i> and an <i>Intermediate
level</i> (<i>category</i> in this case, <i><span style='line-height:107%;
color:#BF8F00'>Figure 48</span></i>). The way it is used here, the SBT module
reveals functional category changes originated by the coordinated behaviour of
their protein components. The integrations involved, <i>protein-to-category</i>,
<i>protein-to-proteinall</i>, and <i>category-to-categoryall</i>, require the
relation tables previously generated with RELS CREATOR (<i><span
style='line-height:107%;color:#BF8F00'>Figure 45</span></i>). As mentioned
above, the elements of the <i>protein2category</i> relation table were
retrieved from a text file containing functional annotations from the Gene
Ontology knowledgebase [3].</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=299 height=83 id="Picture 99" src="User_Guide_files/image065.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771804"></a><a name="_Ref122699654"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>48</span></i><i><span style='color:#BF8F00'>. The SBT
task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Finally, the REPORT module (<i><span style='line-height:107%;color:#BF8F00'>Figure 49</span></i>) is used in this
workflow to collect the following statistical variables for all samples, namely
the 16 individual samples generated with LEVEL CREATOR (<i>e.g.</i> <i>Ind/WT-C-1</i>
and <i>Ind/ISG15-AngII-1</i>, <i><span style='line-height:107%;color:#BF8F00'>Figure
42</span></i>), the four combined samples obtained with NORCOMBINE (<i>WT-C,
WT-AngII, ISG15-C</i>, and <i>ISG15-AngII</i>, <i><span style='line-height:
107%;color:#BF8F00'>Figure 46</span></i>), and the two ratio samples created by
RATIOS (<i>WT-AngIIvsWT-C</i> and ISG15-AngIIvsISG15-C, <i><span
style='line-height:107%;color:#BF8F00'>Figure 47</span></i>), as prompted by
the asterisk:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Number of peptides per protein (<i>n</i>), obtained from the <i>peptide-to-protein
     </i>integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Mean-corrected protein log2 ratios expressed in units of standard
     deviation (<i>Z</i>), calculated from the <i>protein-to-proteinall</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>False Discovery Rate (<i>FDR</i>) values indicating which <i>Z</i>
     values stand out as outliers in the <i>protein-to-proteinall</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Number of proteins per category (<i>n</i>), obtained from the <i>protein-to-category
     integration</i>;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Mean-corrected category log2 ratios expressed in units of standard
     deviation (<i>Z</i>), calculated from the <i>category-to-categoryall</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>False Discovery Rate (<i>FDR</i>) values indicating which <i>Z</i>
     values stand out as outliers in the <i>category-to-categoryall</i>
     integration.</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=568 height=106 id="Picture 104"
src="User_Guide_files/image066.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771910"></a><a name="_Ref122699736"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>49</span></i><i><span style='color:#BF8F00'>. The REPORT
task table for workflow 2.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The following advanced options of
the REPORT module have been used in this workflow (<i><span style='line-height:
107%;color:#BF8F00'>Figure 49</span></i>):</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Merge with report</i> designates the report file whose
     variables will be incorporated into the current report on the basis of
     their element identifiers. Thus, for example, the <i>Npep2prot_Quanprot</i>
     report is first populated with the <i>Z</i> values of the <i>protein-to-proteinall</i>
     integration, and then added the number of peptides per protein coming from
     the <i>Npep2prot</i> report (<i><span style='color:#BF8F00'>Figure 49</span></i>).
     This is possible because the <i>protein</i> level is present in both report
     files.</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Column headers to eliminate</i> restricts the elements to be
     written to the <i>Output reports Npep2prot_Quanprot </i>and <i>Nprot2cat_Quancat
     </i>to those from the protein and category level, respectively. This
     prevents data duplication upon merging with the <i>Npep2prot</i> and <i>Nprot2cat</i>
     reports, as these contain also the peptide and protein level, respectively
     (<i><span style='color:#BF8F00'>Figure 49</span></i>).</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>As previously shown with SanXoT [5], iSanXoT analysis revealed a coordinated alteration of proteins implicated in cardiovascular
function, extracellular matrix and remodeling, and vascular redox state in
aorta from AngII-infused ISG15-KO mice (<i><span style='line-height:107%;
color:#BF8F00'>Figure 50</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=528 height=239 id="Picture 15"
src="User_Guide_files/image067.jpg"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772030"></a><a name="_Ref122699849"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>50</span></i><i><span style='color:#BF8F00'>.
Differential abundance of functional categories (A) and proteins (B) revealed
by workflow 2 based on the mean-corrected log2-ratio expressed in units of
standard deviation at the category (Zc) and protein (Zq) level, respectively.</span></i></p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h3><a name="_Toc124328448">Workflow execution</a></h3>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_NORCOM_RATIOS_SBT.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_NORCOM_RATIOS_SBT.zip</span></a></p>

<p class=MsoNormal>See the <i>Executing a workflow template</i> Section below
for detailed instructions.</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b><span lang=EN-GB style='font-size:18.0pt;line-height:
107%'>&nbsp;</span></b></p>

<h2><a name="_Toc124328449"></a><a name="_Workflow_3:_Quantification"></a>Workflow
3: Quantification of posttranslationally modified peptides in a labeled
experiment</h2>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h3><a name="_Toc124328450">Experimental</a></h3>

<p class=MsoNormal>This workflow was used to quantitate Cys-oxidized peptides
in mouse embryonic fibroblast (MEF) preparations subjected to chemical
oxidation with diamide, an experiment that served to illustrate the comparative
performance of on-filter (FASILOX) and in-gel (GELSILOX) approaches to study
the thiol redox proteome [4]. MEF samples were incubated with diamide (treated
group) or PBS (control group) for 10 min (four biological replicates per
condition). The FASP filter was used to carry out the sequential alkylation and
reduction steps needed to separately label reduced and oxidized Cys residues,
after which the proteins were digested with trypsin and the resulting peptides
were isobaricaly labeled with iTRAQ 8-plex and analyzed by LC-MS/MS. Protein
identification was achieved with Proteome Discoverer 1.4 (Thermo Fisher
Scientific; see Section 3 below to learn how these data are adapted to be used
with iSanXoT). The FDR for peptide identification was calculated using the
refined method [7].</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=536 height=234 id="Picture 88"
src="User_Guide_files/image068.jpg"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772176"></a><a name="_Ref122700968"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>51</span></i><i><span style='color:#BF8F00'>. Scheme of
workflow 3 (quantification of posttranslationally modified peptides in a
labeled experiment) showing module components: LEVEL CREATOR, LEVEL CALIBRATOR,
INTEGRATE, and REPORT (A) and RELS CREATOR (B).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328451">Workflow operation</a></h3>

<p class=MsoNormal>Workflow 3 comprises the basic modules LEVEL CREATOR, LEVEL
CALIBRATOR, and INTEGRATE, as well as the RELS CREATOR and REPORT modules (<i><span style='line-height:107%;color:#BF8F00'>Figure 51</span></i>). The starting
module, LEVEL CREATOR, generates the level files, sample folders and log2
ratios indicated in the corresponding task table (<i><span style='line-height:
107%;color:#BF8F00'>Figure 52</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=557 height=118 id="Picture 243"
src="User_Guide_files/image069.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772189"></a><a name="_Ref122700980"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>52</span></i><i><span style='color:#BF8F00'>. The LEVEL
CREATOR task table for workflow 3.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The output level files, <i>uscan</i>
(<i>uncalibrated scan</i>), display element identifiers together with their
corresponding log2-ratio and statistical weight values (<i><span
style='line-height:107%;color:#BF8F00'>Figure 53</span></i>). This uncalibrated
weight is the inverse of the variance associated with the log2-ratio, and
therefore a higher weight corresponds to a more accurate quantification.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=527 height=108 id="Picture 247"
src="User_Guide_files/image070.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772226"></a><a name="_Ref122700992"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>53</span></i><i><span style='color:#BF8F00'>. Excerpt
from one of the uscan files generated by workflow 3 LEVEL CREATOR module
showing element identifiers (left column), log2 ratios (center column) and
statistical weights (right column).</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Prior to their integration into a
higher level, <i>uscan</i> levels are calibrated to account for the variability
of the LC-MS/MS analysis with the LEVEL CALIBRATOR module, which results in
scan calibrated levels with recalculated statistical weights (<i><span style='line-height:107%;color:#BF8F00'>Figure 54</span></i>, Top). Then the
INTEGRATE module performs the <i>scan-to-peptide</i> and <i>peptide-to-</i> <i>protein</i>
integrations according to the module task table (<i><span style='line-height:
107%;color:#BF8F00'>Figure 54</span></i>, Bottom). The latter integration will
provide the differential abundance of peptides (including posttranslationally
modified peptides) regardless of the differential abundance of proteins, and
the authors chose not to include the heterogeneous population of Cys-containing
peptides for the calculation of variance in the <i>peptide-to-protein</i>
integration [4].</p>

<p class=MsoNormal align=center style='text-align:center'><img border=0
width=545 height=153 id="Picture 89" src="User_Guide_files/image071.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772240"></a><a name="_Ref122701063"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>54</span></i><i><span style='color:#BF8F00'>. The LEVEL
CALIBRATOR (Top) and INTEGRATE (Bottom) task tables for workflow 3.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Thus, the <i>Tag</i> advanced
option of the INTEGRATE module task table displays <i>!C_Modification</i>,
which means that the <i>peptide</i> (lower level) elements marked with this tag
in the <i>peptide2protein</i> relation table (<i><span style='line-height:107%;
color:#BF8F00'>Figure 55</span></i>) will not be considered in the calculation
of the variance in the <i>peptide-to-protein</i> integration. Every integration
requires a specific relation table linking the corresponding lower- and
higher-level elements that is generated by RELS CREATOR (<i><span
style='line-height:107%;color:#BF8F00'>Figure 51</span></i>B and <i><span style='line-height:107%;color:#BF8F00'>Figure 55</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=579 height=98 id="Picture 65" src="User_Guide_files/image072.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772294"></a><a name="_Ref122701108"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>55</span></i><i><span style='color:#BF8F00'>. The RELS
CREATOR task table for workflow 3.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>In particular, the <i>peptide2protein</i>
relation table describes the relationships between the peptide level (<i>i.e</i>.
the lower level) and the protein level (<i>i.e</i>. the higher level), and in
this case includes a third column headed by <i>Modifications</i> where
Cys-containing peptides have been tagged with <i>C_Modification</i> (<i><span style='line-height:107%;color:#BF8F00'>Figure 56</span></i>). Accordingly,
Cys-containing peptides will not be considered for the calculation of variance
in the <i>peptide-to-protein</i> integration.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=535 height=193 id="Picture 255"
src="User_Guide_files/image073.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772399"></a><a name="_Ref122701294"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>56</span></i><i><span style='color:#BF8F00'>. Excerpt
from the peptide2protein relation table that links peptides to proteins. Note
the use of a third column to tag Cys-containing peptides, which will be later
excluded from the calculation of variance in the peptide-to-protein
integration.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Finally, the REPORT module is used
ion this workflow to collect the statistical variables generated by the <i>peptide-to-protein</i>
integration for all eight samples (<i>c1, c2, c3, c4, t1, t2, t3,</i> and <i>t4</i>),
as prompted by the asterisk (<i><span style='line-height:107%;color:#BF8F00'>Figure
57</span></i>A):</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Number of scans per peptide (<i>n</i>), obtained from the <i>scan-to-peptide</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Peptide log2 ratios expressed in units of standard deviation
     corrected by the protein mean (<i>Z</i>), calculated from the <i>peptide-to-protein</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>False Discovery Rate (<i>FDR</i>) values indicating which <i>Z</i>
     values stand out as outliers in the integration.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.25in'>The advanced options of the REPORT
module (<i><span style='line-height:107%;color:#BF8F00'>Figure 57</span></i>A):</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Merge with report</i> designates the merge of the number of
scans per peptide from the <i>Nscan2pep</i> report and the Z values and FDR of
the <i>peptide-to-protein</i> that will be saved in the <i>Nscan2pep_Quanpepprot
report</i>.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i>Column headers columns to eliminate</i> restricts the elements
to be written to the Output reports <i>Nscan2pep_Quanpepprot</i> to those from
the scan level.</p>

<p class=MsoNormal style='margin-left:.25in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'><i><span style='line-height:107%;
color:#BF8F00'>Figure 57</span></i>B shows the relative abundance of
Cys-containing peptides provided by the <i>peptide-to-protein</i> integration. </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=470 height=383 id="Picture 12"
src="User_Guide_files/image074.jpg"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772417"></a><a name="_Ref122701319"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>57</span></i><i><span style='color:#BF8F00'>. (A) The
REPORT module task table for workflow 3. (B) Relative abundance of
Cys-containing peptides in MEF samples measured by peptide log2 ratios
expressed in units of standard deviation corrected by the protein mean (Zpq).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328452">Workflow execution</a></h3>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_PTM.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_PTM.zip</span></a></p>

<p class=MsoNormal>See the <i>Executing a workflow template</i> Section below
for detailed instructions.</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b><span lang=EN-GB style='font-size:18.0pt;line-height:
107%'>&nbsp;</span></b></p>

<h2><a name="_Toc124328453"></a><a name="_Workflow_4:_Label-free"></a>Workflow
4: Label-free quantification</h2>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<h3><a name="_Toc124328454">Experimental</a></h3>

<p class=MsoNormal>This workflow has been applied to quantitative data obtained
in a multicenter study aimed at evaluating different bioinformatics tools [6]. The authors generated two hybrid proteome samples consisting of tryptic digests of
human, yeast and <i>Escherichia coli</i> proteins mixed in two different
defined proportions (13:3:4 for <i>a </i>samples and 13:6:1 for b samples).
Quadruplicate peptide samples were analyzed by LC-MS/MS, after which peptide
identification and quantification were carried out with MaxQuant [8],
among other software packages (see Section below to learn how these data are
adapted to be used with iSanXoT).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=480 height=371 id="Picture 90"
src="User_Guide_files/image075.jpg"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772496"></a><a name="_Ref122703200"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>58</span></i><i><span style='color:#BF8F00'>. Scheme of
workflow 4 (label-free quantification) showing module components: LEVEL CREATOR,
LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, and REPORT (A) and RELS CREATOR (B).</span></i></p>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h3><a name="_Toc124328455">Workflow operation</a></h3>

<p class=MsoNormal>Workflow 4 includes the basic iSanXoT modules LEVEL CREATOR,
LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, and RATIOS along with the REPORT and
RELS CREATOR modules (<i><span style='line-height:107%;color:#BF8F00'>Figure 58</span></i>).
The starting module, LEVEL CREATOR, generates the level files, sample folders
and log2 ratios indicated in the corresponding task table (<i><span style='line-height:107%;color:#BF8F00'>Figure 59</span></i>) based on the quantitative
data at the peptide level obtained with MaxQuant for replicate <i>a</i> and <i>b</i>
samples. To calculate the log2-ratio denominator, first the averages of the
four <i>a</i> and the four <i>b</i> samples are calculated separately (as
indicated by the square brackets), then the average of the two average values
is calculated (as indicated by the comma).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=595 height=237 id="Picture 44"
src="User_Guide_files/image076.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772513"></a><a name="_Ref122703216"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>59</span></i><i><span style='color:#BF8F00'>. The LEVEL
CREATOR task table for workflow 4.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>This ensures that no log2-ratio is
calculated when all four values are missing in either the <i>a</i> or the <i>b</i>
sample group. The eight <i>u_peptide</i> (uncalibrated peptide) output level
files display element identifiers together with their corresponding log2-ratio
and statistical weight values (<i><span style='line-height:107%;color:#BF8F00'>Figure
60</span></i>). </p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=606 height=132 id="Picture 72"
src="User_Guide_files/image077.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772529"></a><a name="_Ref122703265"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>60</span></i><i><span style='color:#BF8F00'>. Excerpt
from one of the u_peptide files generated by workflow 4 LEVEL CREATOR module
showing element identifiers (left column), log2 ratios (center column) and
statistical weights (right column).</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>This uncalibrated weight is the
inverse of the variance associated with the log2-ratio, and therefore a higher
weight corresponds to a more accurate quantification. Prior to their
integration into a higher level, <i>u_peptide</i> levels are calibrated with
the LEVEL CALIBRATOR module, which results in <i>peptide</i> calibrated levels
with recalculated statistical weights (<i><span style='line-height:107%;
color:#BF8F00'>Figure 61</span></i>, Top). Then the INTEGRATE module performs
the <i>peptide-to-protein</i> and <i>protein-to-proteinall</i> integrations
according to the module task table (<i><span style='line-height:107%;
color:#BF8F00'>Figure 61</span></i>, Bottom).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=536 height=154 id="Picture 91"
src="User_Guide_files/image078.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772550"></a><a name="_Ref122703295"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>61</span></i><i><span style='color:#BF8F00'>. The LEVEL
CALIBRATOR (Top) and INTEGRATE (Bottom) task tables for workflow 4.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Every integration requires a
specific relation table linking the corresponding lower- and higher-level
elements that is generated by RELS CREATOR (<i><span style='line-height:107%;
color:#BF8F00'>Figure 58</span></i>B and <i><span style='line-height:107%;
color:#BF8F00'>Figure 62</span></i>); for instance, the <i>peptide2protein </i>relation
table mentioned describes the relationships between the peptide level (<i>i.e.</i>
the lower level) and the protein level (<i>i.e.</i> the higher level).</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=581 height=76 id="Picture 36" src="User_Guide_files/image079.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772598"></a><a name="_Ref122703345"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>62</span></i><i><span style='color:#BF8F00'>. The RELS
CREATOR task table for workflow 4.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>As mentioned above, replicate <i>a</i>
and <i>b</i> samples contain different proportions of yeast and <i>Escherichia
coli</i> peptides (3:4 and 6:1,  respectively) and the same proportion of human
peptides; for this reason, only human proteins were considered in the
calculation of the variance for the <i>protein-to-proteinall</i> integrations,
as indicated by the <i>Tag</i> parameter used in the INTEGRATE task table (<i><span style='line-height:107%;color:#BF8F00'>Figure 61</span></i>, Bottom; note
that no outliers were removed in this integration as prompted by the <i>0</i>
value indicated for the <i>FDR </i>parameter). Accordingly, the <i>protein2proteinall</i>
relation table required by this integration includes a third column headed by <i>Species</i>
where proteins have been tagged with their corresponding species (<i><span style='line-height:107%;color:#BF8F00'>Figure 63</span></i>). This relation
table was built based on the file <i>HumanSaccEcoliPME12_...‌‌‌_species.tsv</i>.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=264 height=185 id="Picture 73"
src="User_Guide_files/image080.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772697"></a><a name="_Ref122703568"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>63</span></i><i><span style='color:#BF8F00'>. Excerpt
from the protein2proteinall workflow 4 relation table that links proteins to a
constant value representing the protein grand mean. Note the use of a third
column to tag proteins with their corresponding species for later
species-specific protein-to-proteinall integration.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Next, the four individual replicate
<i>a</i> (<i>a1, a2, a3,</i> and <i>a4</i>) and <i>b</i> (<i>b1, b2, b3,</i>
and <i>b4</i>) samples initially generated with LEVEL CREATOR (<i><span style='line-height:107%;color:#BF8F00'>Figure 59</span></i>) are combined
into samples <i>A</i> and <i>B</i>, respectively, using the NORCOMBINE basic
module (<i><span style='line-height:107%;color:#BF8F00'>Figure 64</span></i>,
Top). The module task table indicates that samples were combined at the <i>protein</i>
level using the <i>proteinall</i> level for normalization. NORCOMBINE also
requires the user to specify which <i>lowerNorm</i> files contain the necessary
data for the samples to be combined. These <i>lowerNorm</i> files, previously
generated by the INTEGRATE module (<i><span style='line-height:107%;color:#BF8F00'>Figure
61</span></i>, Bottom), contain lower-level identifiers along with the
corresponding centered log2-ratio values and either the integration statistical
weight (in the case of <i>lowerNormV</i>, as chosen in this workflow) or the
variance (for <i>lowerNormW</i>) (<i><span style='line-height:107%;color:#BF8F00'>Figure
64</span></i>, Top). NORCOMBINE first generates merged data files and relation
tables that are later integrated to the above-described combined samples <i>A</i>
and <i>B</i>.</p>

<p class=MsoNormal style='text-indent:.5in'>To compared these two samples,
first new log-2 ratios and statistical weights are calculated using the RATIOS
basic module (<i><span style='line-height:107%;color:#BF8F00'>Figure 64</span></i>,
Bottom), after which a <i>protein-to-proteinall</i> integration is carried out
for the newly-generated <i>B_vs_A</i> sample (<i><span style='line-height:107%;
color:#BF8F00'>Figure 61</span></i>, Bottom).</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=560 height=136 id="Picture 9"
src="User_Guide_files/image081.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772755"></a><a name="_Ref122703609"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>64</span></i><i><span style='color:#BF8F00'>. The
NORCOMBINE (Top) and RATIOS (Bottom) task tables for workflow 4.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>Finally, the REPORT module (<i><span style='line-height:107%;color:#BF8F00'>Figure 65</span></i>) is used in this
workflow to collect variables for all the samples involved (as prompted by the
asterisk), namely the eight individual samples generated with LEVEL CREATOR (<i>a1,
a2, a3,</i> <i>a4, b1, b2, b3, </i>and <i>b4</i>; <i><span style='line-height:
107%;color:#BF8F00'>Figure 59</span></i>), the combined samples <i>A</i> and <i>B
</i>obtained with NORCOMBINE (<i><span style='line-height:107%;color:#BF8F00'>Figure
64</span></i>, Top), and the <i>B_vs_A</i> sample generated by RATIOS (<i><span style='line-height:107%;color:#BF8F00'>Figure 64</span></i>, Bottom). The
following variables were considered:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Number of peptides per protein (<i>n</i>), obtained from the <i>peptide-to-protein
     </i>integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Protein log2 ratios (<i>Xinf</i>), calculated from the <i>protein-to-proteinall</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Protein grand mean (<i>Xsup</i>), calculated from the <i>protein-to-proteinall</i>
     integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>FDR</i> values indicating which <i>Z</i> values stand out as
     outliers in the <i>protein-to-proteinall</i> integration;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Species information (<i>tags</i>) showing in the <i>protein-to-proteinall</i>
     integration output file.</li>
</ul>

<p class=MsoNormal><span style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
style='font-size:14.0pt;line-height:107%'><img border=0 width=608 height=70
id="Picture 92" src="User_Guide_files/image082.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123768547"></a><a name="_Ref122703805"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>65</span></i><i><span style='color:#BF8F00'>. The REPORT module
task table for workflow 4.</span></i></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The following advanced options of
the REPORT module have been used in this workflow (<i><span style='line-height:
107%;color:#BF8F00'>Figure 65</span></i>):</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Merge with report</i> designates the report file whose
     variables will be incorporated into the current report on the basis of
     their element identifiers. Thus, the <i>Npep2prot_Quanprot</i> report is
     first populated with the <i>Xinf, Xsup, FDR</i> and <i>tags</i> values of
     the protein-to-proteinall integration, and then added the number of
     peptides per protein coming from the <i>Npep2prot report</i> (<i><span style='color:#BF8F00'>Figure 65</span></i>). This is possible because the <i>protein</i>
     level is present in both reports;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Column headers to eliminate</i> restricts the elements to be
     written to the corresponding <i>Output report</i> to those from the <i>protein</i>
     level (lower level; both lower and higher levels are used unless otherwise
     stated). This prevents data duplication when <i>Npep2prot_Quanprot</i> and
     <i>Npep2prot </i>reports are merged, as the latter contains also the
     peptide level.</li>
</ul>

<p class=MsoNormalCxSpMiddle style='margin-left:.5in'>&nbsp;</p>

<p class=MsoNormalCxSpMiddle style='margin-left:.5in'>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'><a name="_Hlk121489265">In
agreement with the results reported </a>[6], the combined statistics <i>B_vs_A</i>
analysis shows human, yeast and bacterial proteins normally distributed around
the expected 0, 1, and -2 mean-corrected log2-ratio (<i>X<sub>qa</sub> – grand mean</i>,
indeed, they are the X<sub>inf</sub> - X<sub>sup</sub>) values (corresponding
to 1, 2, and 0.25 fold change values) (<i><span style='line-height:107%;
color:#BF8F00'>Figure 66</span></i>).</p>

<p class=MsoNormal align=center style='text-align:center;text-indent:.5in;
page-break-after:avoid'><img border=0 width=453 height=384 id="Picture 94"
src="User_Guide_files/image083.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773060"></a><a name="_Ref122704192"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>66</span></i><i><span style='color:#BF8F00'>.
Quantification of human (orange), yeast (grey) and bacterial (blue) proteins
according to the combined statistics B_vs_A.</span></i></p>

<p class=MsoNormal style='text-indent:.5in'>&nbsp;</p>

<h3><a name="_Toc124328456">Workflow execution</a></h3>

<p class=MsoNormal>The workflow template and input files that are needed to
execute this workflow can be downloaded from </p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WPP_LabelFree.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WPP_LabelFree.zip</span></a></p>

<p class=MsoNormal>See the <i>Executing a workflow template</i> Section below
for detailed instructions.</p>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Toc124328457"></a><a name="_Executing_a_workflow"></a>Importing a
workflow template</h1>

</div>

<p class=MsoNormal><span lang=EN-GB>&nbsp;</span></p>

<p class=MsoNormal>To illustrate how to execute the workflows created with
iSanXoT, we shall make use of the sample workflows described in detail in the
previous Section. Start by downloading, for instance, workflow 1 template and
input files from the iSanXoT wiki (<a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/workflows/WSPP-SBT.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/workflows/WSPP-SBT.zip</span></a>).
Extract the files included in the compressed archive Then proceed as follows:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Open the iSanXoT application by double-clicking the application
     icon (<i><span style='color:#BF8F00'>Figure 67</span></i>).</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=234 height=154 id="Picture 95"
src="User_Guide_files/image084.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773105"></a><a name="_Ref122704913"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>67</span></i><i><span style='color:#BF8F00'>. The iSanXoT
startup message.</span></i></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Choose <i>New Project</i> from the <i>Project</i> menu (<i><span style='color:#BF8F00'>Figure 68</span></i>).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=429 height=250 id="Picture 96"
src="User_Guide_files/image085.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773114"></a><a name="_Ref122704926"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>68</span></i><i><span style='color:#BF8F00'>. Create New
Project.</span></i></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Provide a project name and a project folder, then click the <i>Submit</i>
     button (<i><span style='color:#BF8F00'>Figure 68</span></i>).</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>To import a workflow previously generated with iSanXoT, choose <i>Import
     Workflow</i> from the <i>Project</i> menu (<i><span style='color:#BF8F00'>Figure
     69</span></i>). Choose, for instance, workflow 1 by selecting the root
     folder where you stored the workflow files. </li>
</ul>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in;margin-bottom:.0001pt;line-height:normal'>&nbsp;</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=431 height=196 id="Picture 97"
src="User_Guide_files/image086.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773175"></a><a name="_Ref122704953"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>69</span></i><i><span style='color:#BF8F00'>. Importing a
preexisting iSanXoT workflow to the newly-created project.</span></i></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Now click on <i>Choose identification file</i> to provide an
     identification/quantification (<i>ID-q</i>) file (<i><span
     style='color:#BF8F00'>Figure 70</span></i>). The below section shows how
     to prepare this file based on the output from a variety of proteomics
     pipelines; but you can try the <i>ID-q.tsv</i> file included in the root
     folder where you stored workflow 1 files.</li>
</ul>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><img
border=0 width=421 height=362 id="Picture 3" src="User_Guide_files/image087.jpg"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773226"></a><a name="_Ref122704989"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>70</span></i><i><span style='color:#BF8F00'>. Choosing
the identification/quantification (ID-q) file for the newly-created project.</span></i></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Bear in mind that any path included in the RELS CREATOR module
     task table of the workflow imported will need be updated.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<b><span style='font-size:16.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span></b>

<p class=MsoNormal><b><span style='font-size:16.0pt;line-height:107%'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk124849730"></a><a name="_Toc124328458"></a><a
name="_Creating_the_identification/quantif"></a>Creating the
identification/quantification file from proteomics pipelines</h1>

</div>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>iSanXoT requires an identification/quantification file (<i>ID-q.tsv</i>)
containing identified features together with their quantitative values and the
experiment they originate from whether there is more than one experiment. This
tab-separated values <i>ID-q</i> file must fulfill:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Features (scan, peptide, protein, etc.) are unequivocally
     referenced using a unique identifier;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Quantitative values for the features are displayed across columns;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Experiments are specified in a column field headed by <i>Experiment</i>
     whether there is more than one experiment.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>In this Section we shall describe
how to prepare the <i>ID-q.tsv</i> file based on data obtained with the four
most popular proteomics pipelines.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoCaption style='page-break-after:avoid'>Table 1 . Column headers
from the tested proteomic pipelines that are needed for the input
idnetification/quantification file (ID-q.tsv). (*) Redundant proteins. You have
to select your main protein. (**) Provide the experiment names in the case of
multiple experiments.</p>

<div align=center>

<table class=MsoNormalTable border=0 cellspacing=0 cellpadding=0 width=662
 style='width:496.5pt;border-collapse:collapse'>
 <tr style='height:24.05pt'>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0in 1.5pt 0in 1.5pt;
  height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0in 1.5pt 0in 1.5pt;
  height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0in 1.5pt 0in 1.5pt;
  height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=411 colspan=5 valign=top style='width:308.1pt;border:solid windowtext 1.0pt;
  background:silver;padding:0in 1.5pt 0in 1.5pt;height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Id_q.txt
  column headers</span></i></b></p>
  </td>
 </tr>
 <tr style='height:23.35pt'>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0in 1.5pt 0in 1.5pt;
  height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0in 1.5pt 0in 1.5pt;
  height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0in 1.5pt 0in 1.5pt;
  height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=251 colspan=3 valign=top style='width:188.45pt;border:solid windowtext 1.0pt;
  border-top:none;background:gray;padding:0in 1.5pt 0in 1.5pt;height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>Features</span></b></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:none;border-right:solid windowtext 1.0pt;
  background:gray;padding:0in 1.5pt 0in 1.5pt;height:23.35pt'>
  <p class=MsoNormal align=center style='margin-top:12.0pt;margin-right:0in;
  margin-bottom:0in;margin-left:0in;margin-bottom:.0001pt;text-align:center;
  line-height:normal;text-autospace:none'><b><span style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:white'>Quantitation</span></b></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:none;border-right:solid windowtext 1.0pt;
  background:gray;padding:0in 1.5pt 0in 1.5pt;height:23.35pt'>
  <p class=MsoNormal align=center style='margin-top:12.0pt;margin-right:0in;
  margin-bottom:0in;margin-left:0in;margin-bottom:.0001pt;text-align:center;
  line-height:normal;text-autospace:none'><b><span style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:white'>Experiment</span></b></p>
  </td>
 </tr>
 <tr style='height:24.9pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>Proteomic
  pipeline</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>Experiment
  type</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>File type</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>Scan</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>Peptide</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>Protein</span></b></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>&nbsp;</span></b></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:gray;padding:0in 1.5pt 0in 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:white'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr style='height:30.95pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Proteome
  Discoverer (version 2.5)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>_PSMs.txt</span></i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum
  file + First Scan + Charge</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence
  + Modifications</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein
  Accessions (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Intensity</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:29.85pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>_PSMs.txt</span></i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum
  file + First Scan + Charge</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence
  + Modifications</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein
  Accessions (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Abundance:
  Channel identifier</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:22.65pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>MaxQuant
  (version 1.6.5.0)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>peptides.txt
  </span></i></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Id</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Leading
  razor protein</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Intensity
  sample</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:29.85pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>msmsScans.txt</span></i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Raw
  file + Scan number + Charge</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence
  + Modifications</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Proteins
  (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Reporter
  intensity sample</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:29.85pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Trans-Proteomics
  Pipeline (version 6.0.0)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>pepXML</span></i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'> file</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>proteins
  (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>precursor_intensity</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:23.1pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>pepXML</span></i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'> file</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>proteins
  (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Channel
  identifier</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:25.35pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Fragpipe
  (version 1.8.1)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>psm.tsv</span></i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Modified
  Peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Intensity</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0in 1.5pt 0in 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:26.25pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>psm.tsv</span></i><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Modified
  Peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Channel
  identifier</span></p>
  </td>
  <td width=72 valign=top style='width:.75in;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0in 1.5pt 0in 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoNormal align=center style='text-align:center'>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124328464"></a><a name="_Preparing_the_ID-q"></a>Preparing the
<i>ID-q</i> file from Proteome Discoverer output</h2>

<p class=MsoNormal>In the case of Proteome Discoverer version 2.5 [9],
the quantitative data required to prepare the <i>ID-q</i> file are extracted by
the <i>Processing</i> workflow nodes <i>Reporter Ions Quantifier</i> (for
labelled experiments) and <i>Minora Feature Detector</i> (for label-free
experiments). These data are written by the <i>Consensus</i> workflow to a text
file suffixed with <i>_PSMs.txt</i> displaying:</p>

<h3>Label-free experiments</h3>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>PSMs Peptide ID</i>: Unique identifier for each <i>Peptide</i>
     sequence identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Sequence</i>: Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Modifications</i>: Post-translational modifications contained
     within the sequence;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Protein Accessions</i>: Accession code(s) for the protein(s) to
     which the peptide sequence is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Intensity: </i>Precursor ion intensity (to be used in
     label-free experiments).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>The unequivocally identifier for a peptide has to be composed by the
union of <i>Sequence</i> and <i>Modifications</i> fields.</p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>&nbsp;</p>

<h3>Labelled experiments</h3>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Spectrum File</i>: Name of the RAW file the spectral MS/MS data
     was extracted from;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>First Scan</i>: RAW file derived scan number for the MS/MS
     spectrum;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Charge</i>: Charge state of the precursor ion;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>PSMs Peptide ID</i>: Unique identifier for each <i>Peptide</i>
     sequence identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Sequence</i>: Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Modifications</i>: Post-translational modifications contained
     within the sequence;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Protein Accessions</i>: Accession code(s) for the protein(s) to
     which the peptide sequence is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Abundance: Channel identifier</i>:  Reporter ion intensity (to
     be used in labelled experiments).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-indent:
.25in;line-height:normal'>In the case of multiple experiments, we recommend the
creation of a unique identifier for a Scan is composed by the union of the
fields: <i>Spectrum File</i>, <i>First </i>Scan and <i>Charge</i>; otherwise,
the <i>PSMs Peptide ID</i> field is enough for the Scan identification. The
unequivocally identifier for a peptide has to be composed by the union of <i>Sequence</i>
and <i>Modifications</i> fields.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124328459"></a><a name="_Preparing_the_ID-q_1"></a>Preparing
the <i>ID-q</i> file from MaxQuant output</h2>

<p class=MsoNormal>The way MaxQuant version 1.6.5.0 [8]
data are adapted for use with iSanXoT depends on whether they originate from
label-free or labelled proteomics experiments:</p>

<h3><a name="_Toc124328460">Label-free experiments</a></h3>

<p class=MsoNormal>In this case, the necessary data can be found in the <i>peptides.txt</i>
file, which is stored in the <i>“…combined/txt</i>” folder:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><a name="_Hlk120012892"><i>Id</i></a>: Unique, consecutive
     identifier for each peptide <i>Sequence</i> identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Sequence</i>: Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Leading razor protein</i>: Identifier of the top scoring
     protein to which the peptide <i>Sequence</i> is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Intensity + Experiment</i>: Summed up extracted ion current of
     all isotopic clusters associated with the peptide <i>Sequence</i>
     identified in the raw files making up <i>Experiment</i>.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328461">Labelled experiments</a></h3>

<p class=MsoNormal>When dealing with labelled experiments (e.g. iTRAQ- or
TMT-based), the necessary data can be found in the <i>msmsScans.txt</i> file,
which is stored in the <i>“…combined/txt</i>” folder:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Raw file</i>: Name of the RAW file from which the MS/MS data
     were extracted;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Scan number</i>: MS/MS spectrum number within a given RAW file;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Charge</i>: Charge state of the precursor ion;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Sequence</i>: Amino acid sequence of the identified peptide.
     This parameter is nonblank only when identification was successful.</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Proteins</i>: Identifier(s) of the protein(s) to which the
     peptide <i>Sequence</i> is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Reporter intensity + n</i>: Intensity of the reporter ion
     corresponding to sample <i>n</i>.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='text-indent:.5in'>The <i>msmsScans.txt</i> file lacks
the unique identifier that is required in the <i>ID-q</i> file. The user is
advised to create this unique identifier by concatenating <i>RAW file, Scan
number, </i>and<i> Charge</i>.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124328462"></a><a name="_Preparing_the_ID-q_2"></a>Preparing
the <i>ID-q</i> file from Trans-Proteomic Pipeline output</h2>

<p class=MsoNormal>Trans-Proteomics Pipeline version 6.0.0 [10]
comprises a number of tools (<i>e.g</i>. <i>ASAPRatio</i>, <i>XPRESS</i> and <i>Libra</i>)
that allow both label-free and isotopically labelled quantitative data to be
saved to a <i>pepXML</i> file. Using the <i>PepXML Viewer</i> application, this
file can be exported as a spreadsheet displaying:</p>

<h3>Label-free experiments</h3>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Spectrum</i>: Unique identifier for each <i>Peptide</i>
     sequence identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Peptide: </i>Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Protein: </i>Identifier of the protein to which the <i>Peptide</i>
     sequence is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Precursor_intensity: </i>Precursor ion intensity (to be used
     for label-free quantification).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>&nbsp;</p>

<h3>Labelled experiments</h3>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Spectrum</i>: Unique identifier for each <i>Peptide</i>
     sequence identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Peptide: </i>Amino acid sequence of the identified peptide;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Protein: </i>Identifier of the protein to which the <i>Peptide</i>
     sequence is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Channel identifier</i>:  Reporter ion intensity (to be used for
     isotopic labelling quantification).   </li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124328463"></a><a name="_Preparing_the_ID-q_3"></a>Preparing
the <i>ID-q</i> file from FragPipe output</h2>

<p class=MsoNormal>Fragpipe version 1.8.1 [11]
relies on the <i>Quant (MS1)</i> and <i>Quant (Isobaric)</i> modules to
generate a <i>psm.tsv</i> output file that contains the quantitative data
obtained from label-free and labelled experiments, respectively:</p>

<h3>Label-free experiments</h3>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Spectrum</i>: Unique identifier for each <i>Peptide</i>
     sequence identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Modified Peptide: Peptide sequence including modifications,
     modified residues;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Protein: </i>Identifier of the protein to which the <i>Peptide</i>
     sequence is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Intensity: </i>Precursor ion intensity (to be used in
     label-free experiments);</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Channel identifier</i>:  Reporter ion intensity (to be used in
     labelled experiments).</li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'>&nbsp;</p>

<h3>Labelled experiments</h3>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Spectrum</i>: Unique identifier for each <i>Peptide</i>
     sequence identified;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Modified Peptide: Peptide sequence including modifications,
     modified residues;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Protein: </i>Identifier of the protein to which the <i>Peptide</i>
     sequence is ascribed;</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Intensity: </i>Precursor ion intensity (to be used in
     label-free experiments);</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'><i>Channel identifier</i>:  Reporter ion intensity (to be used in
     labelled experiments).</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><a name="_Hlk124849833"></a><a name="_Toc124328465"></a><a
name="_Adapting_the_results"></a>Adapting the results from proteomics pipelines
for iSanXoT</h1>

</div>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><i><span style='color:#BF8F00'>&nbsp;</span></i></p>

<p class=MsoNormal>iSanXoT requires an identification/quantification
tab-separated values file (<i>ID-q.tsv</i>) containing identified features
together with their quantitative values and the experiment they originate from.
Users can either compose this <i>ID-q</i> file manually (see Section above to
learn how to do that with data from the four most popular proteomics pipelines)
or have it prepared by the iSanXoT Input Adaptor. The latter option is
described in this Section.</p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><b><span
style='font-size:16.0pt;line-height:107%'><img border=0 width=374 height=322
id="Picture 100" src="User_Guide_files/image088.jpg"></span></b></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773334"></a><a name="_Ref122706385"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>71</span></i><i><span style='color:#BF8F00'>. Selecting
an ID-q file in the Input Adaptor main window.</span></i></p>

<p class=MsoNormalCxSpMiddle style='margin-left:.5in'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Open the iSanXoT application, then see Section above to learn how
     to get to the <i>Input Adaptor </i>main<i> </i>window (<i><span style='color:#BF8F00'>Figure 71</span></i>).</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>If you already have an appropriate <i>ID-q</i> file, select the <i>Provide
     the Identification/Quantification file (ID-q) </i>option, then click on <i>Choose
     identification file</i> to select the file.</li>
</ul>

<p class=MsoNormalCxSpMiddle align=center style='margin-left:.5in;text-align:
center;text-indent:22.0pt;page-break-after:avoid'><img border=0 width=502
height=151 id="Picture 101" src="User_Guide_files/image089.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773359"></a><a name="_Ref122706414"><i><span
style='color:#BF8F00'>Figure </span></i></a><i><span
style='color:#BF8F00'>72</span></i><i><span style='color:#BF8F00'>. Having the
iSanXoT Input Adaptor prepare the ID-q file.</span></i></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>To have the iSanXoT Input Adaptor prepare the <i>ID-q</i> file
     from proteomics pipelines results (this works at least with MaxQuant,
     Trans-Proteomic Pipeline, FragPipe, and Proteome Discoverer), select the <i>Create
     adapted input file from proteomics pipelines results</i> option, then
     click on <i>Choose folder + Add anots</i> to select the folder containing
     the output files (<i><span style='color:#BF8F00'>Figure 72</span></i>). A
     three-panel window will pop-up (<i><span style='color:#BF8F00'>Figure 73</span></i>).</li>
</ul>

<p class=MsoNormalCxSpMiddle style='margin-left:.5in'>&nbsp;</p>

<p class=MsoNormalCxSpMiddle align=center style='margin-left:.5in;text-align:
center;page-break-after:avoid'><img border=0 width=385 height=448
id="Picture 86" src="User_Guide_files/image090.png"></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773412"></a><a name="_Ref123773368"></a><a
name="_Ref122706425"><i><span style='color:#BF8F00'>Figure </span></i></a><i><span style='color:#BF8F00'>73</span></i><i><span style='color:#BF8F00'>. Adapting
results from a proteomics pipeline. In the top panel several output files from
Proteome Discoverer have been selected. These </span></i><span
style='color:#BF8F00'>PSMs.txt<i> files, which contain
identification/quantification data, have been assigned an experiment name (</i>Jurkat<i>)
in the middle panel. The bottom panel has been used to create identifiers by
concatenating result file headers: </i>ScanID<i> (by concatenating </i>Spectrum
File, First Scan<i> and </i>Charge<i>) and </i>pepID<i> (by concatenating </i>Sequence<i>
and </i>Modifications<i>).</i></span></p>

<p class=MsoNormal>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>The top panel displays the files included in the folder so that
     you can select one or more result files to be considered by iSanXoT Input
     Adaptor. Please bear in mind that if several result files are selected,
     these must necessarily have the same column headers.</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>The middle panel is used to set the distribution of data items
     across experiments according to result filenames.</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>The bottom panel allows to create identifiers by concatenating
     result file headers. It is composed of two interfaces:</li>
 <ul style='margin-top:0in' type=circle>
  <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;
      line-height:normal'>The headers found on the result files are listed on
      the left. Header names will be added to the interface on the right as
      they are selected;</li>
  <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;
      line-height:normal'>The interface on the right displays the header names
      selected to generate the identifier as well as the identifier name
      provided by the user.</li>
 </ul>
</ul>

<p class=MsoNormalCxSpMiddle style='margin-left:.5in;text-indent:22.0pt'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Please note that the alphanumeric text that unambiguously
     identifies the items to be integrated is the only identifier that must be
     necessarily included in the <i>ID-q</i> file.</li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
     normal'>Click the <i>Submit</i> button and the Input Adaptor will start
     generating the <i>ID-q.tsv</i> file.</li>
</ul>

<p class=MsoNormal><a name="_Adaptors"></a><a name="_Input_Adaptor"></a>&nbsp;</p>

<span style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal>&nbsp;</p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0in 0in 0in'>

<h1><span lang=ES>References</span></h1>

</div>

<p class=MsoNormal><span lang=ES>&nbsp;</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'><span lang=ES>[1] Trevisan-Herraz M, Bagwan N, Garcia-Marques F, Rodriguez JM, Jorge
I, Ezkurdia I, et al. </span>SanXoT: a modular and versatile package for the
quantitative analysis of high-throughput proteomics experiments. <span lang=ES>Bioinformatics.
2019;35(9):1594-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'><span
lang=ES>[2] Navarro P, Trevisan-Herraz M, Bonzon-Kulichenko E, Nunez E,
Martinez-Acedo P, Perez-Hernandez D, et al. </span>General statistical
framework for quantitative proteomics by stable isotope labeling. <span
lang=ES>J Proteome Res. 2014;13(3):1234-47.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'><span
lang=ES>[3] Garcia-Marques F, Trevisan-Herraz M, Martinez-Martinez S, Camafeita
E, Jorge I, Lopez JA, et al. </span>A Novel Systems-Biology Algorithm for the
Analysis of Coordinated Protein Responses Using Quantitative Proteomics. <span
lang=ES>Mol Cell Proteomics. 2016;15(5):1740-60.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'><span
lang=ES>[4] Bonzon-Kulichenko E, Camafeita E, Lopez JA, Gomez-Serrano M, Jorge
I, Calvo E, et al. </span>Improved integrative analysis of the thiol redox
proteome using filter-aided sample preparation. <span lang=ES>J Proteomics.
2020;214:103624.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'><span
lang=ES>[5] Gonzalez-Amor M, Garcia-Redondo AB, Jorge I, Zalba G, Becares M,
Ruiz-Rodriguez MJ, et al. </span>Interferon stimulated gene 15 pathway is a
novel mediator of endothelial dysfunction and aneurysms development in
angiotensin II infused mice through increased oxidative stress. Cardiovasc Res.
2021.</p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'>[6]
Navarro P, Kuharev J, Gillet LC, Bernhardt OM, MacLean B, Rost HL, et al. A
multicenter study benchmarks software tools for label-free proteome
quantification. Nat Biotechnol. 2016;34(11):1130-6.</p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'>[7]
Navarro P, Vazquez J. A refined method to calculate false discovery rates for
peptide identification using decoy databases. J Proteome Res. 2009;8(4):1792-6.</p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'>[8]
Tyanova S, Temu T, Cox J. The MaxQuant computational platform for mass
spectrometry-based shotgun proteomics. Nat Protoc. 2016;11(12):2301-19.</p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'>[9]
Orsburn BC. Proteome Discoverer-A Community Enhanced Data Processing Suite for
Protein Informatics. Proteomes. 2021;9(1).</p>

<p class=EndNoteBibliography style='margin-bottom:0in;margin-bottom:.0001pt'>[10]
Deutsch EW, Mendoza L, Shteynberg D, Slagel J, Sun Z, Moritz RL.
Trans-Proteomic Pipeline, a standardized data processing pipeline for
large-scale reproducible proteomics informatics. Proteomics Clin Appl.
2015;9(7-8):745-54.</p>

<p class=EndNoteBibliography>[11] Kong AT, Leprevost FV, Avtonomov DM,
Mellacheruvu D, Nesvizhskii AI. MSFragger: ultrafast and comprehensive peptide
identification in mass spectrometry-based proteomics. Nat Methods.
2017;14(5):513-20.</p>

<p class=MsoNormal>&nbsp;</p>

</div>

</body>

</html>
</template>