<template class="task-template" id="User_Guide">
<html>

<head>
<base href="sections/help/">
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Helvetica;
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:Courier;
	panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face
	{font-family:"Tms Rmn";
	panose-1:2 2 6 3 4 5 5 2 3 4;}
@font-face
	{font-family:Helv;
	panose-1:2 11 6 4 2 2 2 3 2 4;}
@font-face
	{font-family:"New York";
	panose-1:2 4 5 3 6 5 6 2 3 4;}
@font-face
	{font-family:System;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:Batang;
	panose-1:2 3 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:PMingLiU;
	panose-1:2 1 6 1 0 1 1 1 1 1;}
@font-face
	{font-family:"MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;}
@font-face
	{font-family:Dotum;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimHei;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:MingLiU;
	panose-1:2 1 6 9 0 1 1 1 1 1;}
@font-face
	{font-family:Mincho;
	panose-1:2 2 6 9 4 3 5 8 3 5;}
@font-face
	{font-family:Gulim;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:Century;
	panose-1:2 4 6 4 5 5 5 2 3 4;}
@font-face
	{font-family:"Angsana New";
	panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face
	{font-family:"Cordia New";
	panose-1:2 11 3 4 2 2 2 2 2 4;}
@font-face
	{font-family:Mangal;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Latha;
	panose-1:2 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sylfaen;
	panose-1:1 10 5 2 5 3 6 3 3 3;}
@font-face
	{font-family:Vrinda;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Raavi;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Shruti;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sendnya;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Gautami;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Tunga;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Estrangelo Edessa";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Yu Gothic";
	panose-1:2 11 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:DengXian;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Palatino Linotype";
	panose-1:2 4 5 2 5 5 5 3 3 4;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"Arial Unicode MS";
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:"Segoe UI Emoji";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:center;
	line-height:107%;
	page-break-after:avoid;
	border:none;
	padding:0cm;
	font-size:26.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:normal;
	font-style:italic;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:0cm;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:11.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:5.0pt;
	margin-left:22.0pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-link:"Footnote Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-link:"Comment Text Char";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:10.0pt;
	margin-left:0cm;
	text-align:center;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.MsoFootnoteReference
	{vertical-align:super;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpFirst, li.MsoTitleCxSpFirst, div.MsoTitleCxSpFirst
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpMiddle, li.MsoTitleCxSpMiddle, div.MsoTitleCxSpMiddle
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpLast, li.MsoTitleCxSpLast, div.MsoTitleCxSpLast
	{mso-style-link:"Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:28.0pt;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
a:link, span.MsoHyperlink
	{font-family:"Calibri",sans-serif;
	color:#2E74B5;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
strong
	{font-family:"Times New Roman",serif;}
p
	{margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-link:"Comment Subject Char";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	font-size:9.0pt;
	font-family:"Segoe UI",sans-serif;}
p.MsoRMPane, li.MsoRMPane, div.MsoRMPane
	{margin:0cm;
	margin-bottom:.0001pt;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:36.0pt;
	text-align:justify;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoQuote, li.MsoQuote, div.MsoQuote
	{mso-style-link:"Quote Char";
	margin-top:10.0pt;
	margin-right:43.2pt;
	margin-bottom:8.0pt;
	margin-left:43.2pt;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:#404040;
	font-style:italic;}
span.MsoIntenseReference
	{font-variant:small-caps;
	color:#4472C4;
	letter-spacing:.25pt;
	font-weight:bold;}
span.MsoBookTitle
	{letter-spacing:.25pt;
	font-weight:bold;
	font-style:italic;}
p.MsoTocHeading, li.MsoTocHeading, div.MsoTocHeading
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	margin-bottom:.0001pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-weight:bold;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#BF8F00;
	font-style:italic;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-link:Title;
	font-family:"Calibri Light",sans-serif;
	letter-spacing:-.5pt;}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-link:"Footnote Text";}
p.lead, li.lead, div.lead
	{mso-style-name:lead;
	margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.QuoteChar
	{mso-style-name:"Quote Char";
	mso-style-link:Quote;
	color:#404040;
	font-style:italic;}
span.CommentTextChar
	{mso-style-name:"Comment Text Char";
	mso-style-link:"Comment Text";}
span.CommentSubjectChar
	{mso-style-name:"Comment Subject Char";
	mso-style-link:"Comment Subject";
	font-weight:bold;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-link:"Balloon Text";
	font-family:"Segoe UI",sans-serif;}
p.EndNoteBibliographyTitle, li.EndNoteBibliographyTitle, div.EndNoteBibliographyTitle
	{mso-style-name:"EndNote Bibliography Title";
	mso-style-link:"EndNote Bibliography Title Char";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:center;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyTitleChar
	{mso-style-name:"EndNote Bibliography Title Char";
	mso-style-link:"EndNote Bibliography Title";
	font-family:"Calibri",sans-serif;}
p.EndNoteBibliography, li.EndNoteBibliography, div.EndNoteBibliography
	{mso-style-name:"EndNote Bibliography";
	mso-style-link:"EndNote Bibliography Char";
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	text-align:justify;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.EndNoteBibliographyChar
	{mso-style-name:"EndNote Bibliography Char";
	mso-style-link:"EndNote Bibliography";
	font-family:"Calibri",sans-serif;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;
	font-family:"Calibri",sans-serif;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;
	font-family:"Calibri",sans-serif;}
span.msoIns
	{mso-style-name:"";
	text-decoration:underline;
	color:teal;}
span.msoDel
	{mso-style-name:"";
	text-decoration:line-through;
	color:red;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:595.3pt 841.9pt;
	margin:70.85pt 3.0cm 70.85pt 3.0cm;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>

</head>

<body lang=EN-GB link="#2E74B5" vlink="#954F72">

<div class=WordSection1>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoTitle align=center style='text-align:center'><b><i><span
lang=EN-US style='font-size:36.0pt;letter-spacing:.25pt'><img width=310
height=355 id="Picture 16" src="User_Guide_files/image001.png" alt=Molecule></span></i></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoTitleCxSpFirst align=center style='text-align:center'><span
class=MsoBookTitle><span lang=EN-US style='font-size:36.0pt'>User Guide:</span></span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
class=MsoBookTitle><span lang=EN-US style='font-size:36.0pt'>Help and
documentation</span></span></p>

<p class=MsoTitleCxSpLast align=center style='text-align:center'><span
class=MsoBookTitle><span lang=EN-US>(1.2.7)</span></span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><span lang=EN-US>Table of contents</span></h1>

</div>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Introduction">Introduction</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_License">License</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:Symbol;text-decoration:
none'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_Installation">Installation</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Download">Download</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Available_operating_systems">Available operating systems</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:"Courier New";
text-decoration:none'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;letter-spacing:
0pt;font-weight:normal;font-style:normal'><a href="#_Windows_distribution">Windows
distribution</a></span></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_MacOS_distribution">MacOS distribution</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Linux_distribution">Linux distribution</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:Symbol;
letter-spacing:0pt;font-weight:normal;font-style:normal'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_Get_Started">Getting Started</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:Symbol;text-decoration:
none'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_Modules">Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Basic_modules">Relation Tables Module</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_RELS_CREATOR">RELS CREATOR</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Basic_modules">Basic Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_LEVEL_CREATOR">LEVEL CREATOR</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_LEVEL_CALIBRATOR">LEVEL CALIBRATOR</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_INTEGRATE">INTEGRATE</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_NORCOMBINE">NORCOMBINE</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_RATIOS">RATIOS</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_SBT">SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Compound_modules">Compound Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WSPP-SBT">WSPP-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WSPPG-SBT">WSPPG-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WPP-SBT">WPP-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoHyperlink><span lang=EN-US style='font-family:Wingdings;
text-decoration:none'>§<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_WPPG-SBT">WPPG-SBT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Report_modules">Report Modules</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_REPORT">REPORT</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoHyperlink><span lang=EN-US style='font-family:Wingdings;
text-decoration:none'>§<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_SANSON">SANSON</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoBookTitle><span lang=EN-US style='line-height:107%;font-family:"Courier New";
letter-spacing:0pt;font-weight:normal;font-style:normal'>o<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Special_parameters">Special Parameters</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Keywords_for_the">Keywords for the Task-Tables</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Multiple_samples">Multiple samples</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Asterisk_is_our">Asterisk is our jack of all trades</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_More_params">More params</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:108.0pt;text-indent:
-18.0pt'><span class=MsoBookTitle><span lang=EN-US style='line-height:107%;
font-family:Wingdings;letter-spacing:0pt;font-weight:normal;font-style:normal'>§<span
style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><span
lang=EN-US><a href="#_Filter_param_(in">Filter (for REPORT and SANSON)</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Sample_Workflows_with">Sample
Workflows with Application to Case Studies</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Workflow_1:_One-step">Workflow 1:
One-step quantification in a labeled experiment</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Workflow_2:_Step-by-step">Workflow 2:
Step-by-step quantification and sample combination in a labeled experiment</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Workflow_3:_Quantification">Workflow
3: Quantification of posttranslationally modified peptides in a labeled
experiment</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:"Courier New";
text-decoration:none'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp; </span></span></span><span
lang=EN-US><a href="#_Workflow_4:_Label-free">Workflow 4: Label-free
quantification</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Executing_a_workflow">Executing a
workflow template</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Creating_the_identification/quantif">Creating
the identification/quantification file from proteomics pipelines</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Preparing_the_ID-q">Preparing the
ID-q file from Proteome Discoverer output</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Preparing_the_ID-q_1">Preparing the
ID-q file from MaxQuant output</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Preparing_the_ID-q_2">Preparing the
ID-q file from Trans-Proteomic Pipeline output</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:72.0pt;text-indent:-18.0pt'><span
lang=EN-US style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Preparing_the_ID-q_3">Preparing the
ID-q file from FragPipe output</a></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US><a href="#_Adapting_the_results">Adapting the
results from proteomics pipelines for iSanXoT</a></span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
class=MsoHyperlink><span lang=EN-US style='font-family:Symbol;text-decoration:
none'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><span lang=EN-US><a href="#_References">References</a></span></p>

<p class=MsoNormal><span class=MsoHyperlink><span lang=EN-US style='text-decoration:
none'>&nbsp;</span></span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Hlk118388234"></a><a name="_Hlk118304308"></a><a
name="_Introduction"></a><span lang=EN-US>Introduction</span></h1>

</div>

<p class=MsoNormal><b>iSanXoT</b> is a standalone application for statistical
analysis of mass spectrometry-based quantitative proteomics data.  iSanXoT
builds upon SanXoT [1],
our previous publicly available implementation of the weighted spectrum,
peptide, and protein (WSPP) statistical model [2]
using the Generic Integration Algorithm (GIA) [3].</p>

<p class=MsoNormal>iSanXoT executes several kind of workflows for quantitative
high-throughput proteomics, <b>systems biology </b>and the <b>statistical analysis</b>,
<b>integration and comparison of experiments</b>.</p>

<p class=MsoNormal>iSanXoT was developed by the <b><i>Cardiovascular Proteomics
Lab</i></b>/<b><i>Proteomic Unit</i></b> at <b>The National Centre for
Cardiovascular Research</b> (CNIC, <a href="https://www.cnic.es"><span
style='color:#4472C4'>https://www.cnic.es</span></a>).</p>

<p class=MsoNormal>&nbsp;</p>

<h2>Download</h2>

<p class=MsoNormal>The multiple releases are available in the
&quot;release&quot; section, located in the following link:</p>

<p class=MsoNormal><a href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<h2>Installation</h2>

<h3>Available operating systems</h3>

<p class=MsoNormal>iSanXoT maintains the following operating systems and
architectures and may add additional ones in the future:</p>

<p class=MsoNormal style='margin-left:35.4pt'>Windows 10 Pro (x64)</p>

<p class=MsoNormal style='margin-left:35.4pt'>MacOs High Sierra (10.13.6)</p>

<p class=MsoNormal style='margin-left:35.4pt'>Ubuntu 20.04 (x64)</p>

<p class=MsoNormal>For more details, read the “Installation” section in the
iSanXoT wiki:</p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Installation"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Installation</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<h2>Getting Started</h2>

<p class=MsoNormal>This chapter describes iSanXoT’s graphical user interface
and how to set up an analysis with iSanXoT.</p>

<p class=MsoNormal>For more details, read the “Getting Started” section in the
iSanXoT wiki:</p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Getting_started"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Getting_started</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<h2>Modules</h2>

<p class=MsoNormal>The iSanXoT desktop application houses a number of modules
based on the SanXoT software package [1].
The information required to setup and execute every module is provided in a
task-table.   </p>

<p class=MsoNormal>There are four types of modules:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Relation tables is a module that creates the relation tables used
by the iSanXoT modules.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Basic modules call the individual scripts included in the
SanXoT software package [1].</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>The Compound modules perform a sequence of consecutive
integrations based on the weighted spectrum, peptide and protein (WSPP)
statistical model [2] and the systems-biology triangle (SBT) algorithm [3].</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Finally, there are two Reports: REPORT generates report files
displaying the quantitative results produced by the above Basic and Composite
modules when a workflow is executed; SANSON generates a similarity graph
showing relationships between functional categories on the basis of the protein
elements they share.</p>

<p class=MsoNormal>For more details, read the “Modules” section in the iSanXoT
wiki:</p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Modules"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Modules</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<h2>Input Adaptor</h2>

<p class=MsoNormal>The iSanXoT Input Adaptor offers users the possibility to
either provide their own Identification/Quantification file, which contains the
identification and quantification data, or have the Input Adaptor prepare this
file from the results obtained using any of the mainstream proteomics
pipelines.</p>

<p class=MsoNormal>For further details, read the “Input Adaptor” section in the
iSanXoT wiki:</p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/Input_adaptor"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/Input_adaptor</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<h2>Sample Workflows</h2>

<p class=MsoNormal><span lang=EN-US>We describe in detail four sample workflows
that illustrate the capacity of iSanXoT to statistically ascertain abundance
changes in both multiplexed, isotopically labeled </span><span
lang=EN-US>[3-5]</span><span lang=EN-US> and label-free </span><span lang=EN-US>[6]</span><span lang=EN-US> proteomics experiments.</span></p>

<p class=MsoListParagraphCxSpFirst align=left style='text-align:left;
text-indent:-18.0pt'><span style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_1">Workflow
1</a>: One-step quantification in a labeled experiment ()</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_2">Workflow
2</a>: Step-by-step quantification and sample combination in a labeled
experiment.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_3">Workflow
3</a>: Quantification of posttranslationally modified peptides in a labeled
experiment.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><a href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/WKF_4">Workflow
4</a>: Label-free quantification.</p>

<p class=MsoNormal>For further details, read each section in the iSanXoT wiki. </p>

<p class=MsoNormal>&nbsp;</p>

<h2>More Documents</h2>

<p class=MsoNormal>User Guide (v1.2.7):</p>

<p class=MsoNormal><a name="_License"></a><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/user_guides/User_Guide_iSanXoT-1.2.7.pdf"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/user_guides/User_Guide_iSanXoT-1.2.7.pdf</span></a></p>

<p class=MsoNormal>Presentations:</p>

<p class=MsoListParagraph style='text-indent:-18.0pt'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>WSPP demo:</p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/WSPP_Demo_iSanXoT_1.0.X.pdf">https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/WSPP_Demo_iSanXoT_1.0.X.pdf</a></p>

<p class=MsoListParagraph style='text-indent:-18.0pt'><span style='font-family:
Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Input Adaptor demo:</p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/Input_Adaptor_iSanXoT_1.0.X.pdf">https://github.com/CNIC-Proteomics/iSanXoT/wiki/docs/presentations/Input_Adaptor_iSanXoT_1.0.X.pdf</a></p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Hlk118382155">License</a></h2>

<p class=MsoNormal>This application is licensed under a <b>Creative Commons
Attribution-NonCommercial-NoDerivs 4.0 Unported License</b>.</p>

<p class=MsoNormal><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><span
style='color:#4472C4'>https://creativecommons.org/licenses/by-nc-nd/4.0/</span></a></p>

<h4><span lang=EN-US>You are free to:</span></h4>

<p class=MsoNormal><span lang=EN-US>Share - copy and redistribute the material
in any medium or format.</span></p>

<p class=MsoNormal><span lang=EN-US>The licensor cannot revoke these freedoms
as long as you follow the license terms.</span></p>

<h3><span lang=EN-US>Under the following terms</span></h3>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/License"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/License</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Hlk118375025"></a><a name="_Installation"></a><span lang=EN-US>Installation</span></h1>

</div>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Download"></a><span lang=EN-US>Download</span></h3>

<p class=MsoNormal>The multiple releases are available in the
&quot;release&quot; section, located in the following link:</p>

<p class=MsoNormal style='text-indent:35.4pt'><a
href="https://github.com/CNIC-Proteomics/iSanXoT/releases"><span
style='color:#4472C4'>https://github.com/CNIC-Proteomics/iSanXoT/releases</span></a></p>

<p class=MsoNormal align=left style='text-align:left'>&nbsp;</p>

<h3><a name="_Available_operating_systems"></a>Available operating systems</h3>

<p class=MsoNormal>iSanXoT maintains the following operating systems and
architectures and may add additional ones in the future:</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Windows 10 Pro (x64)</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>MacOs High Sierra (10.13.6)</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Ubuntu 20.04 (x64)</p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<h3><a name=windows-distribution></a><a name=installation-on-windows></a><a
name="_Installation_on_Windows"></a><a name="_Windows_distribution"></a><span
lang=EN-US>Windows distribution</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT Windows distribution is
packaged in a NSIS Launcher (exe file).</span></p>

<p class=MsoNormal><span lang=EN-US>Download the exe Launcher: iSanXoT_Launcher_1.X.X.win32-x64.exe</span></p>

<p class=MsoNormal><span lang=EN-US>Double-click the Launcher file; the Installer
window will show up:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 28" src="User_Guide_files/image002.jpg"></span></p>

<p class=MsoNormal><span lang=EN-US>WARNING: Windows Defender SmartScreen might
show a prompt suggesting that you cancel the installation; in such case click
“More info” and then select the “Run anyway” option.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=332 height=140 id="Picture 43"
src="User_Guide_files/image003.png"></span></p>

<p class=MsoNormal><span lang=EN-US>Then, you can choose the iSanXoT
installation folder:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=399 height=310 id="Picture 25" src="User_Guide_files/image004.jpg"></span></p>

<p class=MsoNormal><span lang=EN-US>Wait while iSanXoT is being installed.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 22" src="User_Guide_files/image005.png"
alt="Installing window"></span></p>

<p class=MsoNormal><span lang=EN-US>Once the installation has been completed,
you are ready to run.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=406 height=316 id="Picture 21" src="User_Guide_files/image006.png"
alt="Completing setup"></span></p>

<p class=MsoNormal><span lang=EN-US style='font-family:"Segoe UI",sans-serif;
color:#24292E'>&nbsp;</span></p>

<h3><a name=macos-distribution></a><a name="_MacOS_distribution"></a><span
lang=EN-US>MacOS distribution</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT MacOS distribution is packaged
in a DMG container.</span></p>

<p class=MsoNormal><span lang=EN-US>Download the DMG file: iSanXoT_Launcher_1.X.X.darwin-x64.dmg</span></p>

<p class=MsoNormal><span lang=EN-US>Double-click the DMG file, then a Finder
window will show up. This window will usually display iSanXoT’s installer icon
and a shortcut to the Applications folder, together with some sort of linking arrow:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=416 height=309 id="Picture 19"
src="User_Guide_files/image007.png" alt="Installer window"></span></p>

<p class=MsoNormal><span lang=EN-US>Simply drag the iSanXoT icon to your
Applications folder...</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US
style='font-family:"Segoe UI",sans-serif;color:#24292E'><img border=0
width=439 height=315 id="Picture 18" src="User_Guide_files/image008.png"
alt="Installing window"></span></p>

<p class=MsoNormal><span lang=EN-US>and you're done: the iSanXoT application is
now installed.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name=linux-distribution></a><a name="_Linux_distribution"></a><span
lang=EN-US>Linux distribution</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT Linux distribution is packaged
in an <i>AppImage</i>.</span></p>

<p class=MsoNormal><span lang=EN-US>Download the AppImage: iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=567 height=241 src="User_Guide_files/image009.png"></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:0cm'><span lang=EN-US>The
AppImage file is just the application's compressed image. When executed, the application
is mounted in a temporal folder. However, for the correct behaviour, the application
has to be extracted to the &quot;squashfs-root&quot; folder in the current
working directory using:</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.X.X.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>&nbsp;</span></em></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm'><span lang=EN-US>Then,
launch the iSanXoT application using:</span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:0cm;text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun</span></em></p>

<em><span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif;
font-style:normal'><br clear=all style='page-break-before:always'>
</span></em>

<p class=MsoNormal align=left style='text-align:left'><em><span lang=EN-US
style='font-family:"Calibri",sans-serif;font-style:normal'>&nbsp;</span></em></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Get_Started"></a><em><span lang=EN-US style='font-family:"Calibri Light",sans-serif;
font-style:normal'>Getting Started</span></em></h1>

</div>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>This chapter describes iSanXoT’s graphical user interface and
how to set up an analysis with iSanXoT.</span></em></p>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3><span lang=EN-US>Opening the iSanXoT application</span></h3>

<p class=MsoNormal><span lang=EN-US>To open the iSanXoT application:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>In Windows: from the Start menu choose <i>Programs
&gt; iSanXoT</i>; or double-click the iSanXoT desktop icon.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>In MacOS: double-click the iSanXoT icon from the
Applications folder.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>In Linux: from the <i>AppImage</i> file.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>The contents are extracted
to the &quot;squashfs-root&quot; directory in the current working directory using:</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>./iSanXoT_Launcher_1.x.linux-x86_64.AppImage
--appimage-extract</span></em></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>Now you can launch the iSanXoT
application:</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:34.8pt'><em><span
lang=EN-US style='font-family:"Calibri",sans-serif'>squashfs-root/AppRun [...]</span></em></p>

<p class=MsoNormal><em><span lang=EN-US style='font-family:"Calibri",sans-serif;
font-style:normal'>&nbsp;</span></em></p>

<h3><span lang=EN-US>Installing required packages</span></h3>

<p class=MsoNormal><span lang=EN-US>The first time iSanXoT is run a window will
show up displaying a progress bar to inform you about the percentage of
packages that have been installed. These packages contain the libraries
required by iSanXoT’s backend, and are installed the first time you open the
application.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=482 height=317
src="User_Guide_files/image010.png"></span></p>

<p class=MsoCaption><b><span lang=EN-US>Figure </span></b><b><span
lang=EN-US>1</span></b><b><span lang=EN-US>. <a name="_Ref103093720">Installation
window.</a></span></b></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>Closing the iSanXoT application</span></h3>

<p class=MsoNormal><span lang=EN-US>WARNING: If valid changes were made to your
project, make sure to save it before quitting iSanXoT, as any changes will be
lost otherwise (you won’t be prompted for saving upon closing).  </span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>To close the
iSanXoT application:</span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>In Windows and
Linux: choose <i>Project &gt; Exit</i>, or click the X in the upper right
corner of the main iSanXoT window.</span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>In Mac: choose <i>iSanXoT
(menu) &gt; Exit</i>, or click the red X in the upper left corner of the main
iSanXoT window.</span></p>

<p class=MsoNormal><span lang=EN-US>A dialog window will show up asking you to
confirm the application closing. Click “Yes” if you really want to quit iSanXoT.
</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>iSanXoT Projects</span></h3>

<p class=MsoNormal><span lang=EN-US>An iSanXoT project is primarily a container
used to structure the data coming from your input file(s) and your workflow.
The input file contains the identification and quantification data (for further
details see the <a href="#_Input_Adaptor">Input Adaptor</a> Section). These fully-customisable
workflows can perform quantitative proteomics analysis, systems biology
analysis, and comparison and merging of experimental data from technical or
biological replicates.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=251 height=240
src="User_Guide_files/image011.png"></span></p>

<p class=MsoCaption><a name="_Ref103114194"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>2</span></b><b><span lang=EN-US>. Project menu.</span></b></p>

<h4><span lang=EN-US>Creating a new project</span></h4>

<p class=MsoNormal><span lang=EN-US>Creating a project is the first step when
conducting an analysis with iSanXoT. By selecting <i>Project &gt; New Project</i>
a window will show up where you can provide a name for the project as well as
select a project folder where iSanXoT output files will be stored.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=536 height=199
src="User_Guide_files/image012.png"></span></p>

<p class=MsoCaption><b><span lang=EN-US>Figure </span></b><b><span
lang=EN-US>3</span></b><b><span lang=EN-US>. Window that creates a new project.</span></b></p>

<h4><span lang=EN-US>&nbsp;</span></h4>

<h4><span lang=EN-US>Opening a project</span></h4>

<p class=MsoNormal><span lang=EN-US>By selecting <i>Project &gt; Open Project</i>
as a folder selection dialog box shows up that allows the user to indicate the
location of an already existing project folder to be opened by iSanXoT.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>iSanXoT Main Window</span></h3>

<p class=MsoNormal><span lang=EN-US>The iSanXoT main window consists of an
overhead menu, adaptor and module tabs, and content and execution panels (</span><span lang=EN-US><b>Figure 4</b></span><span lang=EN-US>).</span></p>

<h4><span lang=EN-US>Menu</span></h4>

<p class=MsoNormal><span lang=EN-US>The Menu contains the following items:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Project: Allows operations related to projects
and workflows.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Processes: Links to real-time display of the processes
currently in execution by iSanXoT (see <a href="#_Running_Processes">Running Processes</a>
Section).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Help: Houses the different Sections of iSanXoT
help.</span></p>

<h4><span lang=EN-US>Content tabs</span></h4>

<p class=MsoListParagraph style='text-indent:-18.0pt'><span lang=EN-US>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US>Five tabs are displayed in iSanXoT’s project page. The Input tab
displays the Project folder, where iSanXoT output files are stored, as well as
the Identification file used in the project (see <a href="#_Adaptors">Input Adaptor</a>
Section). The remaining four tabs give access to iSanXoT modules: <a
href="#_Relation_tables_module">Relation Tables</a>, <a href="#_Basic_modules_1">Basic
Modules</a>, <a href="#_Compound_modules">Compound Modules</a>, and <a
href="#_Reports_modules">Report Modules</a>.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=570 height=337 id="Picture 132"
src="User_Guide_files/image013.png"></span></p>

<p class=MsoCaption><a name="_Ref103114143"></a><a name="_Ref103114150"><b><span
lang=EN-US>Figure </span></b></a><b><span lang=EN-US>4</span></b><b><span
lang=EN-US>. Main View of iSanXoT.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Content panel</span></h4>

<p class=MsoNormal><span lang=EN-US>This panel houses the elements of the Input
and Modules tabs, which can be accessed through the sidebar menu showing on the
left side of the panel. A title and a brief description of the Input/Module
element is provided, as well as help icon linking to additional information on
the specific Input element or Module selected.</span></p>

<h4><a name="_Execution_panel"></a><span lang=EN-US>Execution panel</span></h4>

<p class=MsoNormal><span lang=EN-US>The execution panel, located on the bottom
of the main window, allows the user to indicate the number of processors to be
used by iSanXoT, 4 by default. The Start button launches the execution of the
workflow shaped by the Input elements and Modules.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><span lang=EN-US>Importing and Exporting Workflows</span></h2>

<p class=MsoNormal><span lang=EN-US>A project is shaped by a workflow that
instructs iSanXoT how to process the data provided by the input file(s). While
the whole project, including workflow and data, can be saved as indicated below
(see the <a href="#_Saving_a_project">Saving a project</a> section), there is a
way to import and export just the workflow structure using iSanXoT menu (</span><span lang=EN-US><b>Figure 2</b></span><span lang=EN-US>).</span></p>

<h4><span lang=EN-US>Import Workflow</span></h4>

<p class=MsoNormal><span lang=EN-US>This option allows to import the
task-tables of a workflow. For that, you have to provide the folder where the
workflow is saved.</span></p>

<h4><span lang=EN-US>Export Workflow</span></h4>

<p class=MsoNormal><span lang=EN-US>The export workflow saves the task-tables
of a workflow in the folder indicated by the user.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><span lang=EN-US>Executing a Project</span></h2>

<p class=MsoNormal><span lang=EN-US>Once your project contains all the
necessary input data and workflow elements, you can execute the workflow by
clicking “Save and Run” in the “Execution panel” after indicating the number of
processors to be allocated for iSanXoT (see  <a href="#_Execution_panel">Execution
panel</a> above). </span></p>

<p class=MsoNormal><span lang=EN-US>Bear in mind that every time you click the
“Save and Run” button to execute a workflow, the project is first validated for
consistency and saved. To save a project without executing it, you must use the
appropriate menu item as explained in the next Section.</span></p>

<h4><a name="_Saving_a_project"></a><span lang=EN-US>Saving a project</span></h4>

<p class=MsoNormal><span lang=EN-US>The <i>Project &gt; Save Project</i> option
saves your project, which contains the input data and the workflow elements, to
the “Project folder”. The project files are saved in the “.isanxot” folder. WARNING:
Do not manipulate or delete the information stored in the “.isanxot” folder;
you risk losing your project.</span></p>

<p class=MsoNormal><span lang=EN-US>Whenever iSanXoT is prompted to save a project,
the corresponding workflow is first validated for consistency and won’t be
saved when failed. Neither will the workflow execute if it has not been validated
previously.</span></p>

<p class=MsoNormal><span lang=EN-US>As well as the “.isanxot” folder, the
following folders are necessary to shape your project:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Exps, to store the files created by the Input
Data adapters.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Jobs, to store the <i>sample folders</i> of your
workflow.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Rels, to store the Relation Tables created by
the RELS CREATOR module (see below).</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Reports, to store the Report files created by
the REPORT module (see below).</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US>-<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Stats, to store statistical data.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Logs, to store the workflow execution log files.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><a name="_Running_Processes"></a><span lang=EN-US>Running Processes</span></h2>

<p class=MsoNormal><span lang=EN-US>When workflow execution successfully starts,
a new window shows up displaying information about the processes currently
running:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The Project logs table shows project execution status.
Several project executions can be monitored here, and the user must click a row
to have the corresponding workflow logs displayed (see below).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Workflow logs table displays status for the jobs
set up in the workflow modules. If you click on a row you will see the trace
log of the involved</span><span class=MsoCommentReference><span lang=EN-US
style='font-size:8.0pt;line-height:107%'> j</span></span><span lang=EN-US>obs (unless
the job status is “cached”).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=378 height=262 id="Picture 98"
src="User_Guide_files/image014.png"></span></p>

<p class=MsoCaption><b><span lang=EN-US>Figure </span></b><b><span
lang=EN-US>5</span></b><b><span lang=EN-US>. View of running processes.</span></b></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>The
running processes window can be also reached from the menu by selecting the “Processes
&gt; Main page” option.</span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span lang=EN-US
style='font-size:26.0pt;line-height:107%;font-family:"Calibri Light",sans-serif;
color:#BF8F00'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Modules"></a><span lang=EN-US>Modules</span></h1>

</div>

<p class=MsoNormal><span lang=EN-US>The iSanXoT desktop application houses a
number of modules based on the SanXoT software package </span><span
lang=EN-US>[1]</span><span lang=EN-US>. The information required to setup and
execute every module is provided in a task-table.   </span></p>

<p class=MsoNormal><span lang=EN-US>There are four types of modules:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Relation tables is a module that creates the
relation tables used by the iSanXoT modules.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The Basic modules call the individual scripts
included in the SanXoT software package </span><span
lang=EN-US>[1]</span><span lang=EN-US>.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>The Compound modules perform a sequence of
consecutive integrations based on the weighted spectrum, peptide and protein (WSPP)
statistical model </span><span lang=EN-US>[2]</span><span lang=EN-US> and the systems-biology
triangle (SBT) algorithm </span><span
lang=EN-US>[3]</span><span lang=EN-US>.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Finally, there are two Reports: REPORT generates
report files displaying the quantitative results produced by the above Basic
and Composite modules when a workflow is executed; SANSON generates a
similarity graph showing relationships between functional categories on the
basis of the protein elements they share.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Basic_modules"></a><a
name="_Relation_tables_module"></a><span lang=EN-US>Relation tables module</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_RELS_CREATOR"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>RELS
CREATOR</span></h3>

<p class=MsoNormal><span lang=EN-US>This module generates relation tables (RT) from
tab-separated values (TSV) files. Relation tables, which are TSV files relating
lower level identifiers (e.g. peptides) to the corresponding higher level
elements (e.g. proteins), are required for module execution. For this reason,
the naming convention for the file indicated under “Relation Table to be
created” in the RELS CREATOR task table is <i>lower level + “2” + higher level</i>;
e.g. whether an integration is from the “peptide” level to the “protein” level,
a relation table called “peptide2protein” will be necessary.</span></p>

<p class=MsoNormal><span lang=EN-US>The fields showing in the RELS CREATOR task
table are (</span><span lang=EN-US><b>Figure 6</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Relation Table to be created</span></i><span
lang=EN-US> specifies relation table filenames. As commented above, the naming
convention for these files is <i>lower level + “2” + higher level</i>; e.g.
whether an integration is from the “peptide” level to the “protein” level, a
relation table called “peptide2protein” will be necessary.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Column name of Lower level</span></i><span
lang=EN-US> is the column header that designates which elements from the
indicated file (see below) will be taken as lower level elements in the
resulting relation table.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Column name of Higher level</span></i><span
lang=EN-US> is the column header that designates which elements from the
indicated file (see below) will be taken as higher level elements in the
resulting relation table.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Column name of 3<sup>rd</sup> column</span></i><span
lang=EN-US> is the column header that designates which elements from the
indicated file (see below) will be taken as third column elements in the
resulting relation table.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Table from which RT is extracted is the full
path name for the TSV file to be used to build the relation tables. If the cell
is empty, the Input file (ID-q.tsv) is applied for iSanXoT workflow.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=163
src="User_Guide_files/image015.png"></span></p>

<p class=MsoCaption><a name="_Ref103114314"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>6</span></b><b><span lang=EN-US>. A sample Task-Table in the example
of RELS CREATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Basic_modules_1"></a><span
lang=EN-US>Basic modules</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_LEVEL_CREATOR"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>LEVEL
CREATOR</span></h3>

<p class=MsoNormal><span lang=EN-US>This module creates levels, which are TSV
files containing identifiers, log<sub>2</sub>-ratio values and statistical
weight values. The data are extracted from the Identification/Quantification
file (see <a href="#_Adaptors">Input Adaptor</a> section).</span></p>

<p class=MsoNormal><span lang=EN-US>The following fields are displayed in the
LEVEL CREATOR task table (</span><span lang=EN-US><b>Figure 7</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used to create the level indicated. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator column</span></i><span
lang=EN-US> specifies which column header from the identification file
designates the quantitative values to be used as a numerator for the log<sub>2</sub>-ratio
calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator column(s)</span></i><span
lang=EN-US> specifies which column header from the identification file
designates the quantitative values to be used as a denominator for the log<sub>2</sub>-ratio
calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level to be created</span></i><span
lang=EN-US> designates the level name.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> indicates the name of the folder where the level data file will be
saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=581 height=174
src="User_Guide_files/image016.png"></span></p>

<p class=MsoCaption><a name="_Ref103114332"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>7</span></b><b><span lang=EN-US>. A sample task-table in the LEVEL
CREATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_LEVEL_CALIBRATOR"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>LEVEL
CALIBRATOR</span></h3>

<p class=MsoNormal><span lang=EN-US>This module calibrates the above-described
levels using the “Klibrate” program included in the SanXoT software package </span><span
lang=EN-US>[1]</span><span lang=EN-US>. To perform the calibration, two parameters
(weight constant and variance) are iteratively calculated using the
Levenberg-Marquardt algorithm (for more details see the information about “Klibrate”
in the SanXoT software package </span><span
lang=EN-US>[1]</span><span lang=EN-US>).</span></p>

<p class=MsoNormal><span lang=EN-US>The output calibrated level contains new
statistic weight values for the identifier and log<sub>2</sub>-ratio elements displayed
in the uncalibrated level data file. This is necessary for the levels to be used
as inputs to the INTEGRATE module.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The fields to be completed in this module
are (</span><span lang=EN-US><b>Figure 8</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the name(s) of the folder(s) containing the uncalibrated data file(s)
that were previously generated by the LEVEL CREATOR module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level for integration</span></i><span
lang=EN-US> indicates which lower level elements are to be used in the integration
carried out for the calibration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level for integration</span></i><span
lang=EN-US> indicates which higher level elements are to be used in the
integration carried out for the calibration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Name of calibrated level</span></i><span
lang=EN-US> is the name for the output data file containing the new, calibrated
statistical weight values.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> specifies the name of the folder where the output data file
containing the new, calibrated statistical weight values will be saved. If the
cell is empty, the output sample folder is the given “Sample folder” (second
column).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=193 id="Picture 5"
src="User_Guide_files/image017.png"></span></p>

<p class=MsoCaption><a name="_Ref103114351"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>8</span></b><b><span lang=EN-US>. A sample task table in the LEVEL
CALIBRATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The LEVEL_CALIBRATOR module accepts the
following additional parameters (</span><span lang=EN-US><b>Figure 9</b></span><span
lang=EN-US>): </span></p>

<p class=MsoNormal><span lang=EN-US>To perform the calibration two parameters,
have to be calculated: the k (weight constant), and the variance.</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>K-constant </span></i><span lang=EN-US>sets a
forced value for the k-constant. Using this parameter, the introduced value is
forced as K-constant.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Var(x)</span></i><span lang=EN-US> sets a forced
value for the variance. Using this parameter, the introduced value is forced as
the variance.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US><img
border=0 width=579 height=145 id="Picture 11"
src="User_Guide_files/image018.png"></span></p>

<p class=MsoCaption style='text-align:justify'><a name="_Ref103114398"><b><span
lang=EN-US>Figure </span></b></a><b><span lang=EN-US>9</span></b><b><span
lang=EN-US>. A task-table displaying advanced parameters for the LEVEL
CALIBRATOR module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_INTEGRATE"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>INTEGRATE</span></h3>

<p class=MsoNormal><span lang=EN-US>The INTEGRATE module performs statistical
calculations based on the WSPP model by iteratively applying the generic
integration algorithm (GIA) </span><span
lang=EN-US>[3]</span><span lang=EN-US> on calibrated data files (</span><span lang=EN-US><b>Figure 10</b></span><span lang=EN-US>). </span></p>

<p class=MsoNormal><span lang=EN-US>Integrations are carried out from lower level
data to higher level data (e.g. from the peptide level to the protein level and
from the protein level to the gene level).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=423 height=414 id="Picture 58"
src="User_Guide_files/image019.png"></span></p>

<p class=MsoCaption><a name="_Ref103096863"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>10</span></b><b><span lang=EN-US>. Schematic representation of the
INTEGRATE module.</span></b><span lang=EN-US> The integration is carried out
from any lower level to any higher level using the programs “SanXoT” and
“SanXoTSieve” and the generic integration algorithm (GIA).</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>More in detail, the INTEGRATE module needs two
TSV files as inputs:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US>1.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>A data file containing three data columns:
identifier (a text string that is used to unambiguously identify the low level elements),
quantitative value (log<sub>2</sub>-ratio of the two measurements to be
compared) and statistical weight (a parameter that measures the accuracy of the
quantitative value).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
lang=EN-US>A relation table, which links the lower level identifiers to those
in the higher level. This file contains two columns: higher level identifiers
on the left and lower level identifiers on the right.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=582 height=524 id="Picture 41"
src="User_Guide_files/image020.png"></span></p>

<p class=MsoCaption><a name="_Ref103114442"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>11</span></b><b><span lang=EN-US>. The INTEGRATE module flowchart.</span></b><span
lang=EN-US> A first integration is done with “SanXoT” that calculates the
variance; then “SanXoTSieve” removes outliers tagging them in a new relation
table; finally, a second integration is done with “SanXoT” using the variance
calculated.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>For every integration, the SanXoT program calculates
the general variance using a robust iterative method. Then SanXoTSieve is used
to tag outlier elements </span><span
lang=EN-US>[2]</span><span lang=EN-US> by assessing the probability that a
lower level element be a significant outlier of the standardized (i.e. N(0,1))
log<sub>2</sub>-ratio distribution. The most extreme outliers are thus removed
sequentially and the integration repeated until all outliers below a
user-defined false discovery rate (FDR) threshold have been removed. Finally, a
second integration is carried out by SanXoT using the variance calculated in
the first integration and discarding the outliers tagged in the new relation
table (</span><span lang=EN-US><b>Figure 11</b></span><span lang=EN-US>).</span></p>

<p class=MsoNormal><span lang=EN-US>The output data files generated by
INTEGRATE contain the quantitative data for the higher level and can be used as
inputs to other modules; in addition, each integration generates several
additional files which contain information about the integration. For further
details see SanXoT documentation </span><span
lang=EN-US>[1]</span><span lang=EN-US>.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The parameters to be provided in the
INTEGRATE module task table are (</span><span lang=EN-US><b>Figure 12</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the names of the folder(s) where the lower level data file is
located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US> indicates
the name of the lower level data file to be used. This file contains three data
columns: identifier, quantitative value and statistical weight.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level</span></i><span lang=EN-US>
indicates the name of the higher level to which the lower level elements will
be integrated.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=282 height=159 id="Picture 6"
src="User_Guide_files/image021.png"></span></p>

<p class=MsoCaption><a name="_Ref103114454"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>12</span></b><b><span lang=EN-US>. A sample task-table in the
INTEGRATE module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><a name="_Advanced_parameters"></a><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The INTEGRATE module accepts the following additional
parameters (</span><span lang=EN-US><b>Figure 13</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> is the name of the folder where the level data and statistics are
saved.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Tag</span></i><span lang=EN-US> is a text
label that indicates which elements from the lower level are integrated into
the higher level. The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label “marked” is used as a Tag, only the lower level elements containing
the label marked in the third column of the lower_level2higher_level Relation Table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><i><span lang=EN-US>Tag</span></i><span
lang=EN-US> is a parameter to distinguish groups to perform the integration. For
instance, if the user specifies “marked” the elements containing the label
“marked” in the third column of the “lower_level2higher_level” Relation Table
will be included in (or discarded from) the integration.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>The tag can be used by
inclusion, such as &quot;mod&quot; or by exclusion, putting first the
&quot;!&quot; symbol, such as &quot;!mod&quot;.  Tags should be included in a
third column of the relations file.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>Different tags can be
combined using logical operators &quot;and&quot; (&amp;), &quot;or&quot; (|),
and &quot;not&quot; (!), and parentheses. Some examples:</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>!out&amp;mod</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>!out&amp;(dig0|dig1)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>(!dig0&amp;!dig1)|mod1</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>mod1|mod2|mod3</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:34.8pt'><span
lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><b><i><span lang=EN-US>Warning</span></i></b><span
lang=EN-US>: Unless specified otherwise by the user, by default iSanXoT
eliminates outliers from the lower level according to an FDR&lt;1% threshold.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>iSanXoT automatically adds
the tag “out” in the third column of the relation table to label outliers, so
that they are not integrated. It is not thus recommended to use this tag for
other purposes.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><b><i><span lang=EN-US>Note</span></i></b><span
lang=EN-US> that although the discarded elements will not be included in
calculations, the parameter Z will be calculated and tabulated in the
corresponding output (outStats) file.</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details see
SanXoT wiki</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>(<a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a>).</span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>&nbsp;</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>FDR</span></i><span lang=EN-US> is an FDR
threshold other than the default value (0.01, i.e. 1%) for outlier removal. If “0”
is specified as the FDR value, then no outliers will be discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Var(x)</span></i><span lang=EN-US> sets a
fixed value for the variance. The default value (blank) means that the variance
will be iteratively calculated based on the Levenberg-Marquardt algorithm in
the first <i>Lower level-to-Higher level</i> integration.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=136
src="User_Guide_files/image022.png"></span></p>

<p class=MsoCaption><a name="_Ref103114466"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>13</span></b><b><span lang=EN-US>. A sample task-table displaying
advanced parameters for the INTEGRATE module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_NORCOMBINE"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>NORCOMBINE</span></h3>

<p class=MsoNormal><span lang=EN-US>The NORCOMBINE module combines technical or
biological replicates (</span><span lang=EN-US><b>Figure 14</b></span><span
lang=EN-US>). For example, NORCOMBINE can be used to merge the protein level
data from 4 individual patients and 4 individual controls into a patient- and a
control level protein dataset, respectively.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=451 height=388 id="Picture 114"
src="User_Guide_files/image023.png"></span></p>

<p class=MsoCaption><a name="_Ref103114484"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>14</span></b><b><span lang=EN-US>. Schematic representation of the
NORCOMBINE module used to combine technical or biological replicates.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>Experiment merging relies on the “Cardenio”
program from the SanXoT software package </span><span
lang=EN-US>[1]</span><span lang=EN-US>, which is used to generate merged data
files and relation tables that are later integrated to the grouped level using
“SanXoT” and “SanXoTSieve” (</span><span lang=EN-US><b>Figure 14</b></span><span
lang=EN-US>).</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>NORCOMBINE requires the user to specify
which <i>lowerNorm</i> files contain the necessary data for the samples to be
combined. These <i>lowerNorm</i> files, previously generated by the INTEGRATE
module, display the lower level identifiers on the left, followed by the
corresponding centred log<sub>2</sub>-ratio values (i.e. the values obtained
after subtracting the high level value) in the second column, and either the integration
statistical weight (in the case of <i>lowerNormV</i>) or the variance (for <i>lowerNormW).</i>
The SanXoT program “Cardenio” </span><span
lang=EN-US>[1]</span><span lang=EN-US> is then used to generate merged data
files and relation tables that are later integrated to the grouped level using
“SanXoT” and “SanXoTSieve” (</span><span lang=EN-US><b>Figure 15</b></span><span
lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=452 height=522 id="Picture 115"
src="User_Guide_files/image024.png"></span></p>

<p class=MsoCaption><a name="_Ref103114585"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>15</span></b><b><span lang=EN-US>. The NORCOMBINE module flowchart.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The default NORCOMBINE module task table
shows the following fields (</span><span lang=EN-US><b>Figure 16</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folders</span></i><span lang=EN-US>
indicates the names of the folder(s) containing the lower level data (samples)
to be combined.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level</span></i><span lang=EN-US> indicates
the type of elements to be combined (e.g. peptides or proteins). </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Norm</span></i><span lang=EN-US> specifies
the normalization scheme to be used in the integrations.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>lowerNorm</span></i><span lang=EN-US> specifies
the type of <i>lowerNorm</i> file (see above) to be used.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> is the name of the folder where the grouped level data and
statistics are saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=467 height=89 src="User_Guide_files/image025.png"></span></p>

<p class=MsoCaption><a name="_Ref103114601"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>16</span></b><b><span lang=EN-US>. A sample task-table in the
NORCOMBINE module.</span></b><span lang=EN-US> In this case, the asterisk
wildcard has been used to select multiple sample folders.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The NORCOMBINE module accepts the following
additional parameters (</span><span lang=EN-US><b>Figure 17</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Tag</span></i><span lang=EN-US> is a text
label that indicates which elements from the lower level are integrated into
the higher level. The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label “marked” is used as a Tag, only the lower level elements containing
the label marked in the third column of the lower_level2higher_level Relation table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>FDR</span></i><span lang=EN-US> is an FDR
threshold other than the default value (0.01, i.e. 1%) for outlier removal. If
“0” is specified as the FDR value, then no outliers will be discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Var(x) sets a fixed value for the variance. The
default value (blank) means that the variance will be iteratively calculated
based on the Levenberg-Marquardt algorithm.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=72 src="User_Guide_files/image026.png"></span></p>

<p class=MsoCaption><a name="_Ref103114615"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>17</span></b><b><span lang=EN-US>. A sample task table with advanced
parameters in the NORCOMBINE module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_RATIOS"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>RATIOS</span></h3>

<p class=MsoNormal><span lang=EN-US>This module prepares the data file and
relation table required as a first step in the calculation of a ratio defined
by the user (e.g KO vs WT). For that, the new log<sub>2</sub>-ratio is
calculated as the difference between numerator and denominator values, whereas
the corresponding statistical weight is assessed according to the method indicated
by the user in the V Method filed of the RATIOS task table (</span><span lang=EN-US><b>Figure 18</b></span><span lang=EN-US> and </span><span
lang=EN-US><b>Figure 19</b></span><span lang=EN-US>): </span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>max</span></i><span lang=EN-US> uses the
maximum value between the numerator and denominator statistical weight value. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>form</span></i><span lang=EN-US> uses the value
resulting from 1/(1/Vn + 1/Vd), where Vn and Vd are the statistical weight
value for the numerator and the denominator, respectively.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>avg</span></i><span lang=EN-US> uses the
average value between the numerator and denominator statistical weight value.</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US><img
border=0 width=581 height=199 id="Picture 17"
src="User_Guide_files/image027.png"></span></p>

<p class=MsoCaption><a name="_Ref103114679"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>18</span></b><b><span lang=EN-US>. The RATIOS module flowchart.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The RATIOS module task table displays the
following parameter fields (</span><span lang=EN-US><b>Figure 19</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator column</span></i><span
lang=EN-US> specifies the name of the folder containing the quantitative value
to be used as a numerator for the new log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator column(s)</span></i><span
lang=EN-US> specifies the name of the folder(s) containing the quantitative
values to be used as a denominator for the new log<sub>2</sub>-ratio
calculation. The sample folders have to be separated by comma.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level </span></i><span lang=EN-US>designates
the level (i.e. peptide, protein, gene or category) at which the ratio is to be
calculated.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> indicates the name of the folder where the resulting log<sub>2</sub>-ratio
and statistical weight values will be saved (e.g. KO_vs_WT).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=574 height=65 src="User_Guide_files/image028.png"></span></p>

<p class=MsoCaption><a name="_Ref103114687"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>19</span></b><b><span lang=EN-US>. A sample task-table in the RATIOS
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_SBT"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>This module is based on the Systems Biology
Triangle (SBT) algorithm </span><span
lang=EN-US>[3]</span><span lang=EN-US>, which performs an integration between
the lower and the higher levels using the variance previously obtained in an
integration between the lower and an intermediate level (</span><span lang=EN-US><b>Figure 20</b></span><span lang=EN-US>). Usually the SBT module is
applied to carry out the protein-to-grand mean integration using the variance
associated with the protein-to-category integration.   </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=436 height=435 id="Picture 131"
src="User_Guide_files/image029.png"></span></p>

<p class=MsoCaption><a name="_Ref103093737"></a><a name="_Ref103093681"><b><span
lang=EN-US>Figure </span></b></a><b><span lang=EN-US>20</span></b><b><span
lang=EN-US>. <a name="_Ref103093692">Schematic representation of the SBT module.</a></span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters to be entered in
the SBT module task table are (</span><span lang=EN-US><b>Figure 21</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the name of the folder(s) where the lower level data are located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US>
indicates the name of the lower level (e.g. protein).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Intermediate level</span></i><span
lang=EN-US> indicates the name of the intermediate level (e.g. category).</span></p>

<p class=MsoNormal><span lang=EN-US>By default, the higher level is the grand
mean of the lower level elements.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=292 height=66 src="User_Guide_files/image030.png"></span></p>

<p class=MsoCaption><a name="_Ref103094105"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>21</span></b><b><span lang=EN-US>. A sample task table in the SBT
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>This module accepts the following additional
parameters (</span><span lang=EN-US><b>Figure 22</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder</span></i><span
lang=EN-US> indicates an alternative folder to store the resulting log<sub>2</sub>-ratio
and statistical weight values other than “Sample folder(s)”.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower-Higher level</span></i><span
lang=EN-US> and <i>Int(ermediate)-Higher level </i>specify an alternative
higher level other than the grand mean of the lower level elements.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Low(er)-to-Int(ermediate)</span></i><span
lang=EN-US> Tag and <i>Int(ermediate)-to-Hig(her)</i> are the text label that
indicates which elements from the lower level are integrated into the intermediate
level, and the intermediate level are integrated into higher level. The tags
must be specified in the third column of the corresponding Relation Table. This
allows the user to discard elements for integration without needing to
eliminate them from the Relation Table. Thus, if the label “marked” is used as
a Tag, only the lower level elements containing the label marked in the third
column of the lower_level2intermediate_level and
intermediate_level2higher_level Relation Tables will be integrated. Logical
operators can also be used in the Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Low(er)-to-Int(ermediate) FDR </span></i><span
lang=EN-US>and<i> Int(ermediate)-to-Hig(her) FDR</i> determine an FDR threshold
other than the default value (0.01, i.e. 1%) for outlier removal in the lower
level-to-intermediate level and intermediate level-to-higher level integration,
respectively. If “0” is specified as the FDR value, then no outliers will be
discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Low(er)-to-Int(ermediate) Var(x)</span></i><span
lang=EN-US> and <i>Int(ermediate)-to-Hig(her) Var(x)</i> indicate the variance
to be used in the lower level-to-intermediate level and intermediate level-to-higher
level integration, respectively, as an alternative to the variance calculated
in the lower level-to-intermediate level integration.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=554 height=168 id="Picture 10"
src="User_Guide_files/image031.png"></span></p>

<p class=MsoCaption><a name="_Ref103094086"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>22</span></b><b><span lang=EN-US>. A sample task table with advanced
parameters in the SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Compound_modules"></a><span
lang=EN-US>Compound modules</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The Compound modules perform a sequence of
consecutive integrations based on the WSPP statistical model </span><span lang=EN-US>[2]</span><span lang=EN-US> and the SBT algorithm </span><span lang=EN-US>[3]</span><span lang=EN-US>. In addition, each module creates the
initial level and calibrate this initial level. The WSPP-SBT and WSPPG-SBT
modules create and calibrate the “scan” level; and the WPP-SBT and WPPG-SBT
create and calibrate the “peptide” level.</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WSPP-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WSPP-SBT module performs the following
integrations: scan-to-peptide, peptide-to-protein, protein-to-category, protein-to-proteinall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the protein-to-category integration, which
is applied to the protein-to-proteinall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WSPP-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan-to-peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=564 height=227
src="User_Guide_files/image032.png"></span></p>

<p class=MsoCaption><a name="_Ref103096252"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>23</span></b><b><span lang=EN-US>. A sample task table in the
WSPP-SBT, WSPPG-SBT, WPP-SBT, and WPPG-SBT modules.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WSPP-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 24</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>p&gt;a
Tag</i>, <i>c&gt;a Tag</i> are the text label that indicates which elements
from the lower level (“p” and “c”) are integrated into the higher level (“q”
and “a”). The tags must be specified in the third column of the corresponding Relation
Table. This allows the user to discard elements for integration without needing
to eliminate them from the Relation Table. Thus, if the label “marked” is used
as a Tag, only the lower level elements containing the label marked in the
third column of the lower_level2higher_level Relation table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span><span class=MsoCommentReference><span lang=EN-US
style='font-size:8.0pt;line-height:107%'> </span></span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p FDR</span></i><span lang=EN-US>, <i>p&gt;q
FDR</i>, <i>q&gt;c FDR</i> establish an FDR threshold other than 0.01 (1%) for
outlier removal in the integrations scan-to-peptide, peptide-to-protein, and
protein-to-category, respectively. If FDR = 0 is selected, then the outliers
are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p Var(x)</span></i><span lang=EN-US>, <i>p&gt;q
Var(x)</i>, <i>q&gt;c Var(x)</i> set a fixed value for the variance in the
integrations scan-to-peptide, peptide-to-protein, and protein-to-category,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (</span><span lang=EN-US><b>Figure 10</b></span><span lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=111 id="Picture 23"
src="User_Guide_files/image033.png"></span></p>

<p class=MsoCaption><a name="_Ref103096290"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>24</span></b><b><span lang=EN-US>. Task-Table with advanced
parameters in the WSPP-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WSPPG-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WSPPG-SBT module performs the following
integrations: scan-to-peptide, peptide-to-protein, protein-to-gene, gene-to-category,
gene-to-geneall, and category-to-categoryall. In addition, the SBT algorithm is
used to calculate the variance associated to the gen-to-category integration,
which is applied to the gen-to-geneall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WSPPG-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan-to-peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WSPPG-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 25</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>q&gt;g
Tag</i>, <i>p&gt;a Tag</i>, <i>c&gt;a Tag</i> are the text label that indicates
which elements from the lower level (“p”, “q” and “c”) are integrated into the higher
level (“q”, “g” and “a”). The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label “marked” is used as a Tag, only the lower level elements containing
the label marked in the third column of the lower_level2higher_level Relation table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p FDR</span></i><span lang=EN-US>, <i>p&gt;q
FDR</i>, <i>q&gt;g FDR</i>, <i>g&gt;c FDR</i> establish an FDR threshold other
than 0.01 (1%) for outlier removal in the following integrations: scan-to-peptide,
peptide-to-protein, protein-to-gene, and gene-to-category, respectively. If FDR
= 0 is selected, then the outliers are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>s&gt;p Var(x)</span></i><span lang=EN-US>, <i>p&gt;q
Var(x)</i>, <i>q&gt;g Var(x)</i>, <i>g&gt;c Var(x)</i> set a fixed value for
the variance in the integrations: scan-to-peptide, peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. By default, the variance will be
iteratively calculated based on the Levenberg-Marquardt algorithm in the first
lower level-to-higher level integration (</span><span
lang=EN-US><b>Figure 10</b></span><span lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=66 id="Picture 24"
src="User_Guide_files/image034.png"></span></p>

<p class=MsoCaption><a name="_Ref103096474"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>25</span></b><b><span lang=EN-US>. Task-table with advanced
parameters in the WSPPG-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WPP-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WPP-SBT module performs the
integrations peptide-to-protein, protein-to-category, protein-to-proteinall and
category-to-categoryall. In addition, the SBT algorithm is used to calculate
the variance associated to the protein-to-category integration, which is
applied to the protein-to-proteinall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WPP-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan-to-peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WPP-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 26</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>p&gt;a
Tag</i>, <i>c&gt;a Tag</i> are the text label that indicates which elements
from the lower level (“p” and “c”) are integrated into the higher level (“q”
and “a”). The tags must be specified in the third column of the corresponding Relation
Table. This allows the user to discard elements for integration without needing
to eliminate them from the Relation Table. Thus, if the label “marked” is used
as a Tag, only the lower level elements containing the label marked in the
third column of the lower_level2higher_level Relation Table will be integrated.
Logical operators can also be used in the Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q FDR</span></i><span lang=EN-US>, <i>q&gt;c
FDR</i> establish an FDR threshold other than 0.01 (1%) for outlier removal in
the peptide-to-protein and protein-to-category integrations, respectively. If
FDR = 0 is selected, then the outliers are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Var(x)</span></i><span lang=EN-US>, <i>q&gt;c
Var(x)</i> set a fixed value for the variance in the peptide-to-protein and protein-to-category
integrations, respectively. By default, the variance will be iteratively
calculated based on the Levenberg-Marquardt algorithm in the first lower level-to-higher
level integration (</span><span lang=EN-US><b>Figure 10</b></span><span
lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=494 height=90 id="Picture 29"
src="User_Guide_files/image035.png"></span></p>

<p class=MsoCaption><a name="_Ref103096698"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>26</span></b><b><span lang=EN-US>. Task-table with advanced
parameters in the WPP-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US style='font-family:"Calibri",sans-serif'>WPPG-SBT</span></h3>

<p class=MsoNormal><span lang=EN-US>The WPPG-SBT module performs the
integrations peptide-to-protein, protein-to-gene, gene-to-category, gene-to-geneall,
and category-to-categoryall. In addition, the SBT algorithm is used to
calculate the variance associated to the gene-to-category integration, which is
applied to the gene-to-geneall integration.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters required by the
WPPG-SBT module are (</span><span lang=EN-US><b>Figure 23</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Experiment</span></i><span lang=EN-US> is the
column header that designates which elements from the Identification file will
be used in the starting scan-to-peptide integration.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Identifier column header</span></i><span
lang=EN-US> is the identification file column header that unambiguously
identifies the scans.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio numerator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a numerator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Ratio denominator</span></i><span lang=EN-US>
specifies which identification file column header designates the quantitative
values to be used as a denominator for the log<sub>2</sub>-ratio calculation.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>Output Sample folder: indicates the name of the
folder where the resulting data files will be saved.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The WPPG-SBT module accepts the following
advanced parameters (</span><span lang=EN-US><b>Figure 27</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Tag</span></i><span lang=EN-US>, <i>q&gt;g
Tag, p&gt;a Tag, q&gt;a Tag,</i> <i>c&gt;a Tag</i> are the text label that
indicates which elements from the lower level (“p”, “q” and “c”) are integrated
into the higher level (“q”, “g” and “a”). The tags must be specified in the
third column of the corresponding Relation Table. This allows the user to
discard elements for integration without needing to eliminate them from the Relation
Table. Thus, if the label “marked” is used as a Tag, only the lower level
elements containing the label marked in the third column of the lower_level2higher_level
Relation table will be integrated. Logical operators can also be used in the
Tag field to make complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q FDR, q&gt;g FDR</span></i><span
lang=EN-US>, <i>g&gt;c FDR</i> establish an FDR threshold other than 0.01 (1%)
for outlier removal in the integrations peptide-to-protein, protein-to-gene,
and gene-to-category, respectively. If FDR = 0 is selected, then the outliers
are not discarded.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>p&gt;q Var(x)</span></i><span lang=EN-US>, <i>q&gt;g
Var(x), g&gt;c Var(x)</i> set a fixed value for the variance in the
integrations peptide-to-protein, protein-to-gene, and gene-to-category,
respectively. By default, the variance will be iteratively calculated based on
the Levenberg-Marquardt algorithm in the first lower level-to-higher level
integration (</span><span lang=EN-US><b>Figure 10</b></span><span lang=EN-US>).</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>More params</span></i><span lang=EN-US> allows
adding more parameters to the internal programs of the module. For more details
see <a href="#_More_params">More params</a> in the “Special Parameters”
Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=75 id="Picture 47"
src="User_Guide_files/image036.png"></span></p>

<p class=MsoCaption><a name="_Ref103097509"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>27</span></b><b><span lang=EN-US>. Task-table with advanced
parameters in the WPPG-SBT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2 align=center style='text-align:center'><a name="_Report_modules"></a><a
name="_Reports_modules"></a><span lang=EN-US>Reports modules</span></h2>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_REPORT"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>REPORT</span></h3>

<p class=MsoNormal><span lang=EN-US>The REPORT module allows the collection of the
statistical variables (n, tags, Xinf, Vinf, Xsup, Vsup, Z, and FDR) from the
different integrations performed into result tables. </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters showing in the
REPORT task table are (</span><span lang=EN-US><b>Figure 28</b></span><span
lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US> indicates
the name(s) of the folder(s) where the values of the statistical variables to
be retrieved are located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US> indicates
the starting level (i.e. peptide, protein, or category) for the integration
whose statistical variables are to be reported.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level</span></i><span lang=EN-US> indicates
the ending level for the integration whose statistical variables are to be
reported.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Reported vars</span></i><span lang=EN-US>
specifies which statistical variables will be reported. The available variables
are n, tags, Xinf, Vinf, Xsup, Vsup, Z, FDR, X’inf and Winf.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output report</span></i><span lang=EN-US> is
the report filename (without extension).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=535 height=318
src="User_Guide_files/image037.png"></span></p>

<p class=MsoCaption><a name="_Ref103115700"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>28</span></b><b><span lang=EN-US>. A sample task table in the REPORT
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>For instance, the first row of the task table
shown in </span><span lang=EN-US><b>Figure 28</b></span><span lang=EN-US>
prompts the REPORT module to read the variable “n” from the <i>scan2peptide_outStats.tsv</i>
file that contains the statistical outcome from the scan-to-peptide integration
(the asterisk wildcard character in <i>Sample folder(s)</i> causes REPORT to
retrieve the “n” variable from every sample). These “n” values are written to a
report file named “Nscan_pep” that is stored in the project “reports” folder.</span></p>

<p class=MsoNormal><span lang=EN-US>The second row instructs the module to read
the variables “Xinf”, “Z” and “FDR” from the statistical outcome of the peptide-to-protein
integration (once again for every sample). These values are written to a report
file named “Nscan_Normpep_prot_XZ”.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The REPORT module accepts the following advanced
parameters (</span><span lang=EN-US><b>Figure 29</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Level names to show</span></i><span
lang=EN-US> allows the user to restrict the elements to be written to the <i>Output
report</i> to those from the <i>Lower level</i> or the <i>Higher level</i>. Both
levels are used by default.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Merge with report</span></i><span lang=EN-US>
designates the file whose <i>Reported vars</i> will be incorporated into the <i>Output
report</i> after intersection with the latter file.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Add columns from relation table</span></i><span
lang=EN-US> appends <i>Lower level</i> elements, extracted from the relation
table designated, to the <i>Output report</i>. It is possible to indicate multiple
relation tables separated by a comma.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Filter</span></i><span lang=EN-US> allows to filter
the data to be transferred to the <i>Output report</i> based on the <i>Reported
vars</i> (n, Z, FDR, etc.). For more details, see <a href="#_Filter_(for_REPORT">Filter</a>
in the “Special Parameters” Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=261 id="Picture 4"
src="User_Guide_files/image038.png"></span></p>

<p class=MsoCaption><a name="_Ref103116151"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>29</span></b><b><span lang=EN-US>. A sample task-table with advanced
parameters in the REPORT module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The reports indicated under <i>Output
report</i> and <i>Merge with report</i> are merged according to the column
header that they share. Thus, the REPORT task table shown in </span><span lang=EN-US><b>Figure 30</b></span><span lang=EN-US> will cause the module to incorporate
the number of scans per peptide, displayed in the report “Nscan_pep”, to the
report “Nscan_Normpep_prot_XZ”, as these two reports share the lower level elements
showing under the “peptide” header.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=600 height=156 id="Picture 26"
src="User_Guide_files/image039.png"></span></p>

<p class=MsoCaption><a name="_Ref103116369"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>30</span></b><b><span lang=EN-US>. Report merging in the REPORT
module.</span></b><span lang=EN-US> The first task table row creates a report file
(“Nscan_pep”) with the (n)umber of scans per peptide. The second row creates a
report file called “Nscan_Normpep_prot_XZ” that contains, apart from the
variables “Xinf”, “Z”, and “FDR” coming from the peptide-to-protein integration,
the (n)umber of scans per peptide previously stored in the “Nscan_pep” report,
as these two reports share the lower level elements showing under the “peptide”
header.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>In addition, it is possible to incorporate
additional data from one or more relation tables into the reports (</span><span lang=EN-US><b>Figure 31</b></span><span lang=EN-US>). When a given Relation Table
is indicated under <i>Add columns from relation table</i>, the REPORT module
will first attempt to incorporate to the <i>Output report</i>
(“Npep_Quanprot_cat” in the example) the elements related to the <i>Lower level</i>
(“protein” in this case) elements in the relation table. If the <i>Lower level</i>
elements are missing in the relation table, then REPORT will try to incorporate
the elements related to the Higher level (“category” in this case) instead. If
neither the <i>Lower level</i> nor the <i>Higher level</i> can be found in the
relation table, then no action is performed. </span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=573 height=101 id="Picture 27"
src="User_Guide_files/image040.png"></span></p>

<p class=MsoCaption><a name="_Ref103116417"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>31</span></b><b><span lang=EN-US>. Adding data from relation tables
into the report files. </span></b><span lang=EN-US>The relation table “protein2gene”
contains a column with protein identifiers under the “protein” header and
another column with the corresponding gene name under the “gene” header,
whereas the relation table “protein2description” contains, apart from the
“protein” elements, a column with the corresponding protein description. The
first task table row will prompt REPORT to incorporate the gene names and
protein descriptions contained in the relation tables to the report file “Npep_Quanprot_cat”
report file, as “protein” is the </span><span lang=EN-US style='font-style:
normal'>Lower level</span><span lang=EN-US> they all three shares.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>Finally, the report data can be filtered performing
logical operations with the <i>Reported vars</i> in the <i>Filter</i> field. For
instance, in the report task table displayed in </span><span
lang=EN-US><b>Figure 32</b></span><span lang=EN-US>:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>n_protein2category &lt;= 100, filters out from
the report the variables Z and FDR for the category-to-categoryall integration
when the (n)umber of proteins per category is greater than 100.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>n_protein2category &gt;= 5 &amp;
n_protein2category &lt;= 100 retrieves the variables Z and FDR for the category-to-categoryall
integration when the (n)umber of proteins per category is in the [5, 100] range.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US>KO_vs_WT@FDR_category2categoryall &lt; 0.05,
retrieves the variables Z and FDR for the category-to-categoryall integration provided
that the FDR corresponding to the “KO_vs_WT” samples is less than 0.05.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=583 height=98 id="Picture 32"
src="User_Guide_files/image041.png"></span></p>

<p class=MsoCaption><a name="_Ref103116468"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>32</span></b><b><span lang=EN-US>. Filtering the report data.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The compound variables shown in the <i>Filter</i>
field on </span><span lang=EN-US><b>Figure 32</b></span><span lang=EN-US> follow
the structure <i>Reported var</i>_<i>integration</i>, like “n_protein2category”.
Such filter applies to all samples. </span></p>

<p class=MsoNormal><span lang=EN-US>However, the filter “KO_vs_WT@FDR_category2categoryall”
is applied based on the variable “FDR” from the category-to-categoryall
integration, but only to the “KO_vs_WT” sample. Moreover, the filter “WT1,WT2@FDR_category2categoryall”
is applied to the “WT1” and “WT2” samples.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_SANSON"></a><span lang=EN-US style='font-family:"Calibri",sans-serif'>SANSON</span></h3>

<p class=MsoNormal><span lang=EN-US>The SANSON module generates a similarity
graph showing the relationship between functional categories based on their
protein components. </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Standard parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The standard parameters to be provided for this
module are (</span><span lang=EN-US><b>Figure 33</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Forced execution</span></i><span lang=EN-US>:
This checkbox field indicates whether to force the execution or not.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Sample folder(s)</span></i><span lang=EN-US>
indicates the names of the folder(s) where the lower level data file is
located.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower level</span></i><span lang=EN-US>
indicates the name of the lower level elements (“protein” in this case) to be
used.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher level</span></i><span lang=EN-US>
indicates the name of the higher level (“category” in this case) to which the
lower level elements will be integrated.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Output Sample folder </span></i><span
lang=EN-US>designates a folder other than <i>Sample folder(s)</i> where the
results will be saved.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=383 height=65 src="User_Guide_files/image042.png"></span></p>

<p class=MsoCaption><a name="_Ref103116548"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>33</span></b><b><span lang=EN-US>. A sample task table in the SANSON
module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>Advanced parameters</span></h4>

<p class=MsoNormal><span lang=EN-US>The SANSON module accepts the following additional
parameters (</span><span lang=EN-US><b>Figure 34</b></span><span lang=EN-US>):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Lower norm</span></i><span lang=EN-US>
specifies the normalization scheme to be used with the lower level elements.
The default value is included the normalization of lower level to all.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Higher norm</span></i><span lang=EN-US>
specifies the normalization scheme to be used with the higher level elements.
The default value is included the normalization of higher level to all.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Tag</span></i><span lang=EN-US> is a text
label that indicates which elements from the lower level are integrated into
the higher level. The tags must be specified in the third column of the
corresponding Relation Table. This allows the user to discard elements for
integration without needing to eliminate them from the Relation Table. Thus, if
the label “marked” is used as a Tag, only the lower level elements containing
the label marked in the third column of the lower_level2higher_level Relation table
will be integrated. Logical operators can also be used in the Tag field to make
complex decisions.</span></p>

<p class=MsoListParagraphCxSpMiddle>By default, iSanXoT eliminates outliers
from the lower level according to an FDR&lt;1% threshold.</p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US>For further details, read
the Advanced Parameters for the <a href="#_Advanced_parameters">INTEGRATE</a>
module.<i> </i></span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Filter</span></i><span lang=EN-US> allows to
filter the data based on the FDR and number of proteins. For more details, see <a
href="#_Filter_param_(in">Filter</a> in the “Special Parameters” Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=57 id="Picture 2"
src="User_Guide_files/image043.png"></span></p>

<p class=MsoCaption><a name="_Ref103116564"><b><span lang=EN-US>Figure </span></b></a><b><span lang=EN-US>34</span></b><b><span lang=EN-US>. A sample task-table with advanced
parameters in the SANSON module.</span></b></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><a name="_Complete_Modules"></a><a name="_WSPP-SBT"></a><a
name="_WSPPG-SBT"></a><a name="_WPP-SBT"></a><a name="_WPPG-SBT"></a><span
lang=EN-US>&nbsp;</span></p>

<h2><a name="_Special_parameters"></a><span lang=EN-US>Special parameters</span></h2>

<p class=MsoNormal><a name="_Keywords_for_the"></a><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Multiple_samples"></a><span lang=EN-US>Multiple samples</span></h3>

<p class=MsoNormal><span lang=EN-US>The “Sample folder(s)” field of the
different module task tables admit multiple samples. For instance, let's consider
the samples created with the following LEVEL CREATOR task table:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=579 height=174 id="Picture 64"
src="User_Guide_files/image044.png"></span></p>

<p class=MsoNormal><span lang=EN-US>We can include multiple samples separated
by a comma, for example, in the INTEGRATE module task table: </span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=579 height=227 id="Picture 7" src="User_Guide_files/image045.png"></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Asterisk_is_our"></a><span lang=EN-US>Asterisk is our jack of all
trades</span></h3>

<p class=MsoNormal><span lang=EN-US>The module task tables admit the usage of
the asterisk symbol as a wildcard character. Let’s once more consider the samples
created with the following LEVEL CREATOR task table:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=579 height=174 id="Picture 59"
src="User_Guide_files/image044.png"></span></p>

<p class=MsoNormal><span lang=EN-US>Each row calculates a ratio that is saved to
the corresponding <i>Output Sample folder</i>. Thus, the ratio of 113 to the
mean of 113, 114, 115, and 116 is saved to the “Jurkat_WT/WT_1” folder; the 114
to the mean of 113, 114, 115 ratios is saved to “Jurkat_WT/WT_2”, and so on.
One way to create the task table of the INTEGRATE module could be the
following, where each row represents an integration for a given sample:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=225 height=308 id="Picture 62"
src="User_Guide_files/image046.png"></span></p>

<p class=MsoNormal><span lang=EN-US>However, this task table can be simplified applying
the asterisk wildcard. For instance, the task-table below allows to indicate multiple
sample folders, namely every folder starting with “Jurkat_WT/” or “Jurkat_KO/”.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=275 height=274 id="Picture 60"
src="User_Guide_files/image047.png"></span></p>

<p class=MsoNormal><span lang=EN-US>We can reduce this expression even more
using just an asterisk: the first row of the following task table performs the
integrations peptide-to-protein, protein-to-category, peptide-to-peptideall,
protein-to-proteinall, and category-to-categoryall in every sample folder
defined with LEVEL CREATOR.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=284 height=159 id="Picture 63"
src="User_Guide_files/image048.png"></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>Multiple samples in the inputs and outputs</span></h3>

<p class=MsoNormal><span lang=EN-US>In the cases we have multiple input samples
separated by comma but we want to save the results in another output sample
folder, we indicate them in the same way giving the output folders separated by
comma. It is required to provide the same number of folders:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=579 height=236 id="Picture 13"
src="User_Guide_files/image049.png"></span></p>

<p class=MsoNormal><span lang=EN-US>In the same way happens with the asterisk
character (jack of all trades). In the “Output Sample folder(s)” we can add a
suffix in the input samples. The following task-table illustrate that the
output sample folders would be contain the “_New” suffix:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=487 height=158 id="Picture 30"
src="User_Guide_files/image050.png"></span></p>

<p class=MsoNormal><span lang=EN-US>In addition, we can rename the subfolder
adding the new name in the “Output Sample folder(s)” or add a new subfolder:</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=399 height=181 id="Picture 33"
src="User_Guide_files/image051.png"></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_More_params"></a><span lang=EN-US>More params</span></h3>

<p class=MsoNormal><span lang=EN-US>Some modules accept a column parameter in
the Task-Table called “More params”. This column allows you to provide advanced
parameters for the SanXoT programs </span><span
lang=EN-US>[1]</span><span lang=EN-US>. The program descriptions are in the
following wiki link:</span></p>

<p class=MsoNormal><span lang=EN-US><a
href="https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package">https://www.cnic.es/wiki/proteomica/index.php/SanXoT_software_package</a></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>The iSanXoT module are composed by several
programs of SanXoT. For this reason, the “More params” of a module accepts the
advanced parameters of composed programs indicated by a name.</span></p>

<p class=MsoNormal><span lang=EN-US>For example, </span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-US>INTEGRATE:</span></p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'><span
lang=EN-US>&quot;sanxot1&quot;: &quot; -m 300 -g &quot;, &quot;sanxot2&quot;:
&quot;-s --sweepdecimals=2.5&quot;</span></p>

<p class=MsoNormal><span lang=EN-US>In above example the first “sanxot” program
that compose the INTEGRATE module, receives the “-m 300 -g” as parameter, and
the second “sanxot” receives also the parameter &quot;-s
--sweepdecimals=2.5&quot;.</span></p>

<p class=MsoNormal style='margin-left:35.4pt'><span lang=EN-US>WSPP-SBT:</span></p>

<p class=MsoNormal style='margin-left:35.4pt;text-indent:35.4pt'><span
lang=EN-US>&quot;p2q_sanxot2&quot;: &quot; -m 100 -s &quot;,
&quot;q2a_sanxot1&quot;: &quot;-m 100&quot;</span></p>

<p class=MsoNormal><span lang=EN-US>In this example, the WSPP-SBT module has
multiple integrations: scan-to-peptide, peptide-to-protein, etc. For more
information, see the WSPP-SBT section. Thus, the second “sanxot” program of
peptide-to-protein (q) integration will receive the parameter “-m 100 -s”, and
then, the first “sanxot” of protein (q)-to-proteinall (a) integration will
receive the “-m 100”.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h4><span lang=EN-US>The program names for each Module </span></h4>

<p class=MsoNormal><span lang=EN-US>INTEGRATE: sanxot1, sanxotsieve, sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>NORCOMBINE: create_exp_tags, cardenio,
sanxot1, sanxotsieve, sanxot2 </span></p>

<p class=MsoNormal><span lang=EN-US>SBT: l2i_sanxot1, l2i_sanxotsieve,
l2i_sanxot2, i2h_sanxot1, i2h_sanxotsieve, i2h_sanxot2, l2h_sanxot1,
l2h_sanxotsieve, l2h_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WSPP_SBT: level_creator, klibrate,
s2p_sanxot1, s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1, p2a_sanxotsieve,
p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1,
c2a_sanxotsieve, c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WSPPG_SBT: level_creator, klibrate,
s2p_sanxot1, s2p_sanxotsieve, s2p_sanxot2, p2q_sanxot1, p2q_sanxotsieve,
p2q_sanxot2, p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1,
q2g_sanxotsieve, q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2,
p2a_sanxot1, p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve,
q2a_sanxot2, g2a_sanxot1, g2a_sanxotsieve, g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve,
c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WPP_SBT: level_creator, klibrate, p2q_sanxot1,
p2q_sanxotsieve, p2q_sanxot2, q2c_sanxot1, q2c_sanxotsieve, q2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, c2a_sanxot1,
c2a_sanxotsieve, c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>WPPG_SBT: level_creator, klibrate, p2q_sanxot1,
p2q_sanxotsieve, p2q_sanxot2, p2g_sanxot1, p2g_sanxotsieve, p2g_sanxot2, q2g_sanxot1,
q2g_sanxotsieve, q2g_sanxot2, g2c_sanxot1, g2c_sanxotsieve, g2c_sanxot2, p2a_sanxot1,
p2a_sanxotsieve, p2a_sanxot2, q2a_sanxot1, q2a_sanxotsieve, q2a_sanxot2, g2a_sanxot1,
g2a_sanxotsieve, g2a_sanxot2, c2a_sanxot1, c2a_sanxotsieve, c2a_sanxot2</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Filter_param_(in"></a><a name="_Filter_(for_REPORT"></a><span
lang=EN-US>Filter (for REPORT and SANSON)</span></h3>

<p class=MsoNormal><span lang=EN-US>The REPORT module and SANSON module accept
a Filter parameter. This parameter filters the data based on some variables
depending on the module.</span></p>

<p class=MsoNormal><span lang=EN-US>In the case of REPORT module, the filtered
variables are the Reported vars: n, Z, FDR, etc. For instance:</span></p>

<p class=MsoNormal><span lang=EN-US>FDR_category2category &lt; 0.05) &amp;
(n_protein2category &gt;=5) &amp; (n_protein2category &lt;= 100)</span></p>

<p class=MsoNormal><span lang=EN-US>For the SANSON module, the filtered
variables are FDR and the related number (n_rel). For example:</span></p>

<p class=MsoNormal style='text-indent:35.4pt'><span lang=EN-US>([FDR] &lt;
0.05) &amp; ([n_rel] &gt;= 10) &amp; ([n_rel] &lt;= 100)</span></p>

<p class=MsoNormal><span lang=EN-US>Different variables can be combined using
the comparisons: &gt;=, &lt;=, !=, &lt;&gt;, ==, &gt;, &lt;; and using logical
operators “and” (&amp;), “or” (||), and “not” (!). </span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Toc124328440"></a><a name="_Sample_Workflows_with"></a><span
lang=EN-US>Sample Workflows with Application to Case Studies</span></h1>

</div>

<p class=MsoNormal><span lang=EN-US>We describe below in detail four sample
workflows that illustrate the capacity of iSanXoT to statistically ascertain
abundance changes in both multiplexed, isotopically labeled </span><span lang=EN-US>[3-5]</span><span lang=EN-US> and label-free </span><span
lang=EN-US>[6]</span><span lang=EN-US> proteomics experiments.</span></p>

<p class=MsoNormal><span lang=EN-US style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h2><a name="_Hlk122708883"></a><a name="_Toc124328441"></a><a
name="_Workflow_1:_One-step"></a><span lang=EN-US>Workflow 1: One-step
quantification in a labeled experiment</span></h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328442"><span lang=EN-US>Experimental</span></a></h3>

<p class=MsoNormal><span lang=EN-US>The identification and quantification data
from García-Marqués <i>et al</i>. </span><span
lang=EN-US>[3]</span><span lang=EN-US> were used to illustrate workflow 1
performance. This study characterizes the molecular alterations that take place
along time when vascular smooth muscle cells (VSMCs) are treated with
angiotensin-II (AngII). Cells were incubated with AngII for 0, 2, 4, 6, 8, and
10 h; after protein extraction and digestion, the resulting peptides were
isobarically labelled with iTRAQ 8-plex reagents, mixed appropriately and
analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS). Protein
identification was achieved by database searching against target and decoy
sequences using Proteome Discoverer 1.4 (Thermo Fisher Scientific; see Section
3 below to learn how these data are adapted to be used with iSanXoT). The false
discovery rate (FDR) for peptide identification was calculated using the
refined method </span><span
lang=EN-US>[7]</span><span lang=EN-US>.</span></p>

<p class=MsoNormal><span lang=EN-US style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h3><a name="_Toc124328443"><span lang=EN-US>Workflow operation</span></a></h3>

<p class=MsoNormal><span lang=EN-US>Workflow 1 is made up of the compound
module WSPP-SBT and the basic module REPORT (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 35</span></i><span
lang=EN-US>A). The WSPP-SBT performs a sequence of consecutive integrations
based on the WSPP statistical model </span><span
lang=EN-US>[2]</span><span lang=EN-US> and the SBT algorithm </span><span lang=EN-US>[3]</span><span lang=EN-US>.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=655 height=304 id="Picture 20"
src="User_Guide_files/image052.jpg"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771130"></a><a name="_Ref122446667"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>35</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Scheme of workflow 1 (one-step quantification in a
labeled experiment) showing module components: WSPP-SBT and REPORT (A) and RELS
CREATOR (B).</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The WSPP-SBT
module starts with the creation of the log2 ratios at the scan (or spectrum) level.
These log2 ratios are calculated using the numerator and denominator values
indicated in the module task table (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 36</span></i><span
lang=EN-US>) and subsequently calibrated to account for the variability
associated with LC-MS/MS measurements, after which the <i>scan-to-peptide</i>
and <i>peptide-to-protein</i> integrations are carried out.</span></p>

<p class=MsoNormal align=center style='text-align:center;text-indent:36.0pt;
page-break-after:avoid'><span lang=EN-US><img border=0 width=553 height=177
id="Picture 38" src="User_Guide_files/image053.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771182"></a><a name="_Ref122448847"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>36</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The WSPP-SBT task table for workflow 1.</span></i></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Then the SBT
(systems biology triangle) algorithm is applied for the detection of functional
category changes originated by the coordinated behaviour of proteins (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 35</span></i><span
lang=EN-US>A).  For that, the algorithm first calculates the variance of the <i>protein-to-category</i>
integration, which is then applied to the <i>protein-to-protein grand mean</i>
(hereinafter referred to as <i>protein-to-proteinall</i>) integration, and
finally performs the <i>category-to-category grand mean</i> (hereinafter
referred to as <i>category-to-categoryall</i>) integration. All these
integrations created by the WSPP-SBT module are saved in the indicated <i>Output
Sample folder</i>.</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Every
integration step needs a relation table that links lower- to higher-level
elements. This text file is created by the RELS CREATOR module (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 35</span></i><span
lang=EN-US>B and </span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
37</span></i><span lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=149 id="Picture 40"
src="User_Guide_files/image054.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771279"></a><a name="_Ref122622320"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>37</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The RELS CREATOR task table for workflow 1.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Thus, the <i>peptide2protein</i>
relation table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
38</span></i><span lang=EN-US>A) is a text file linking the peptides identified
to the proteins they come from. The elements of the <i>protein2category</i>
relation table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
38</span></i><span lang=EN-US>B) were retrieved from a text file containing
functional annotations for mouse proteins gathered from several protein
function databases (</span><i><span lang=EN-US style='line-height:107%;
color:#BF8F00'>Figure 37</span></i><span lang=EN-US>), as described by the
authors </span><span
lang=EN-US>[3]</span><span lang=EN-US>. </span></p>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US
style='font-size:14.0pt;line-height:107%'><img border=0 width=619 height=166
id="Picture 35" src="User_Guide_files/image055.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771300"></a><a name="_Ref122622483"><i><span
lang=EN-US style='color:#BF8F00'>Figure</span></i></a><i><span lang=EN-US
style='color:#BF8F00'> </span></i><i><span
lang=EN-US style='color:#BF8F00'>38</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Excerpt from the peptide2protein (A) and
protein2category (B) relation tables that link peptides to proteins and
proteins to categories, respectively.</span></i></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The REPORT
module is used in this workflow to collect the following statistical variables
for all samples (2h-AngII, 4h-AngII, 6h-AngII, 6h-Control, 8h-AngII,
and10h-AngII), as prompted by the asterisk (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 39</span></i><span
lang=EN-US>). The asterisk symbol is used in the task tables as a wildcard
character. For further details, consult the iSanXoT wiki (</span><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki"><span lang=EN-US>https://github.com/CNIC-Proteomics/iSanXoT/wiki</span></a><span
lang=EN-US>). The simple options used in this workflow are the following:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Number of peptides per protein (<i>n</i>),
     obtained from the <i>peptide-to-protein </i>integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Mean-corrected protein log2 ratios expressed in
     units of standard deviation (<i>Z</i>), calculated from the <i>protein-to-proteinall</i>
     integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>False Discovery Rate (<i>FDR</i>) values
     indicating which <i>Z</i> values stand out as outliers in the <i>protein-to-proteinall</i>
     integration.</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Number of proteins per category (<i>n</i>),
     obtained from the <i>protein-to-category</i> integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Mean-corrected category log2 ratios expressed in
     units of standard deviation (<i>Z</i>), calculated from the <i>category-to-categoryall</i>
     integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>False Discovery Rate (<i>FDR</i>) values indicating
     which <i>Z</i> values stand out as outliers in the <i>category-to-categoryall</i>
     integration.</span></li>
</ul>

<p class=MsoNormal style='margin-top:0cm;margin-right:0cm;margin-bottom:0cm;
margin-left:36.0pt;margin-bottom:.0001pt;line-height:normal'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><i><span
lang=EN-US><img border=0 width=623 height=93 id="Picture 39"
src="User_Guide_files/image056.png"></span></i></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771377"></a><a name="_Ref122622693"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>39</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The REPORT task table for workflow 1.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The following
advanced options of the REPORT module have been used in this workflow (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 39</span></i><span
lang=EN-US>):</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Merge with report</span></i><span lang=EN-US>
     designates the report file whose variables will be incorporated into the
     current report on the basis of their element identifiers. Thus, the <i>Npep2prot_Quanprot</i>
     report is first populated with the Z values of the <i>protein-to-proteinall</i>
     integration, and then added the number of peptides per protein coming from
     the <i>Npep2prot</i> report (</span><i><span
     lang=EN-US style='color:#BF8F00'>Figure 39</span></i><span lang=EN-US>).
     This is possible because the <i>protein</i> level is present in both the <i>Npep2prot</i>
     and <i>Npep2prot_Quanprot</i> reports.</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Column headers to eliminate</span></i><span
     lang=EN-US> restricts the elements to be written to the <i>Output reports
     Npep2prot_Quanprot </i>and <i>Nprot2cat_Quancat </i>to those from the
     protein and category level, respectively. This prevents data duplication
     upon merging with the <i>Npep2prot</i> and <i>Nprot2cat</i> reports, as
     these contain also the peptide and protein level, respectively (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 39</span></i><span lang=EN-US>).</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Add columns from relation table</span></i><span
     lang=EN-US> appends a column with the description of the protein to the <i>Npep2prot_Quanprot
     </i>report. Again, this is possible because the protein level is present
     in both the <i>Npep2prot_Quanprot </i>report and the <i>protein2description</i>
     relation table (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 39</span></i><span
     lang=EN-US>).</span></li>
</ul>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=248 height=323 id="Picture 74"
src="User_Guide_files/image057.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771474"></a><a name="_Ref122623033"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>40</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Relative abundance of proteins (Zq, Left) and
functional categories (Zc, Right) retrieved from the Npep2prot_Quanprot and
Nprot2cat_Quancat reports, respectively, generated with the REPORT module of
Workflow1 (</span></i><i><span
lang=EN-US style='color:#BF8F00'>Figure 39</span></i><i><span lang=EN-US
style='color:#BF8F00'>).</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>As previously
shown with SanXoT [2], iSanXoT workflow 1, based on the WSPP-SBT and the REPORT
modules, reveals a slight tendency to increased protein abundance over time. In
addition, the SBT module reveals that the categories found altered at 4, 8, and
10 h, but not those at 2 h, were essentially the same that were altered at 6 h
(</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 40</span></i><span
lang=EN-US>).</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Toc124328444"><span lang=EN-US>Workflow execution</span></a></h3>

<p class=MsoNormal><span lang=EN-US>The workflow template and input files that
are needed to execute this workflow can be downloaded from</span></p>

<p class=MsoNormal><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP-SBT.zip"><span
lang=EN-US style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP-SBT.zip</span></a></p>

<p class=MsoNormal><span lang=EN-US>See the <i>Executing a workflow template</i>
Section below for detailed instructions.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:14.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><span lang=EN-US style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h2><a name="_Toc124328445"></a><a name="_Workflow_2:_Step-by-step"></a><span
lang=EN-US>Workflow 2: Step-by-step quantification and sample combination in a
labeled experiment</span></h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328446"><span lang=EN-US>Experimental</span></a></h3>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US>We have
applied workflow 2 to quantitative data obtained by González-Amor <i>et al</i>.
</span><span lang=EN-US>[5]</span><span lang=EN-US> in their study of the
contribution of interferon-stimulated gene 15 (ISG15) to vascular damage
associated with hypertension. Protein extracts from mouse aortic tissue (16
samples from eight individuals:</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US>two
WT-Control, two ISG15-KO, two WT+AngII, and two ISG15-KO+AngII) were
trypsin-digested, after which the resulting peptides were isobarically labelled
using iTRAQ 8-plex and distributed in two labeled experiments that were analyzed
by LC-MS/MS. Protein identification </span></p>

<p class=MsoNormal><span lang=EN-US>was achieved by database searching against
target and decoy sequences using Proteome Discoverer 2.1 (Thermo Fisher
Scientific; see Section 3 below to learn how these data are adapted to be used
with iSanXoT). The FDR for peptide identification was calculated using the
refined method </span><span
lang=EN-US>[7]</span><span lang=EN-US>.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=466 height=458 id="Picture 78"
src="User_Guide_files/image058.jpg"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771564"></a><a name="_Ref122690540"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>41</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Scheme of workflow 2 (step-by-step quantification and
sample combination in a labeled experiment) showing module components: LEVEL
CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, RATIOS, SBT, and REPORT (A)
and RELS CREATOR (B).</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Toc124328447"><span lang=EN-US>Workflow operation</span></a></h3>

<p class=MsoNormal><span lang=EN-US>Workflow 2 comprises all six basic modules:
LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, RATIOS, and SBT plus
the REPORT module (</span><i><span lang=EN-US style='line-height:107%;
color:#BF8F00'>Figure 41</span></i><span lang=EN-US>). The starting module,
LEVEL CREATOR, generates the level files, sample folders and log2 ratios
indicated in the corresponding task table (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 42</span></i><span
lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=620 height=324 id="Picture 50"
src="User_Guide_files/image059.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771575"></a><a name="_Ref122690553"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>42</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The LEVEL CREATOR task table for workflow 2.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The output level
files, <i>u_scan</i> (<i>uncalibrated scan</i>), display element identifiers
together with their corresponding log2-ratio and statistical weight values (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 43</span></i><span
lang=EN-US>). This <i>uncalibrated</i> weight is the inverse of the variance
associated with the log2-ratio, and therefore a higher weight corresponds to a
more accurate quantification.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=566 height=148 id="Picture 51"
src="User_Guide_files/image060.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771591"></a><a name="_Ref122691254"><i><span
lang=EN-US style='color:#BF8F00'>Figure</span></i></a><i><span lang=EN-US
style='color:#BF8F00'> </span></i><i><span
lang=EN-US style='color:#BF8F00'>43</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Excerpt from one of the u_scan files generated by
workflow 2 LEVEL CREATOR module showing element identifiers (left column), log2
ratios (center column), and statistical weights (right column).</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Prior to their
integration into a higher level, <i>u_scan</i> levels are calibrated to account
for the variability of the LC-MS/MS analysis with the LEVEL CALIBRATOR module,
which results in <i>scan</i> calibrated levels with recalculated statistical
weights (</span><i><span
lang=EN-US style='color:#BF8F00'>Figure 44</span></i><span lang=EN-US>, Top).
Then the INTEGRATE module performs the <i>scan-to-peptide</i>, <i>peptide-to-protein</i>
and <i>protein-to-proteinall</i> integrations according to the module task
table (</span><i><span
lang=EN-US style='color:#BF8F00'>Figure 44</span></i><span lang=EN-US>, Bottom).
The protein outliers are not discarded in this integration as prompted by the 0
value indicated for the FDR parameter.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=379 height=144 id="Picture 87"
src="User_Guide_files/image061.jpg"></span></p>

<p class=MsoCaption><a name="_Ref124348788"><span lang=EN-US>Figure </span></a><span lang=EN-US>44</span><span lang=EN-US> . The LEVEL CALIBRATOR (Top) and INTEGRATE
(Bottom) task tables for workflow 2.</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Every
integration requires a specific relation table linking the corresponding lower-
and higher-level elements that is generated by RELS CREATOR (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 41</span></i><span
lang=EN-US>B and </span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
45</span></i><span lang=EN-US>). Thus, the <i>peptide2protein </i>relation
table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
45</span></i><span lang=EN-US>) describes the relationships between the peptide
level (<i>i.e.</i> the lower level) and the protein level (<i>i.e.</i> the
higher level). Note that the elements of the <i>protein2category</i> relation
table were retrieved from a text file containing functional annotations from
the Gene Ontology knowledgebase </span><span
lang=EN-US>[5]</span><span lang=EN-US>.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=575 height=115 id="Picture 93"
src="User_Guide_files/image062.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771687"></a><a name="_Ref122691348"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>45</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The RELS CREATOR task table for workflow 2.</span></i></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Next, workflow 2
resorts to the NORCOMBINE basic module (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 41</span></i><span
lang=EN-US>A) to combine biological replicates, which results in the four new
samples indicated on </span><i><span lang=EN-US style='line-height:107%;
color:#BF8F00'>Figure 46</span></i><span lang=EN-US>: WT-C, WT-AngII, ISG15-C,
and ISG15-AngII. The module task table indicates that samples were combined at
the <i>protein</i> level using the <i>proteinall</i> level for normalization.
NORCOMBINE also requires the user to specify which <i>lowerNorm</i> files
contain the necessary data for the samples to be combined. These <i>lowerNorm</i>
files, previously generated by the INTEGRATE module, contain lower-level
identifiers along with the corresponding centered log2-ratio values and either
the integration statistical weight (in the case of <i>lowerNormV</i>, as chosen
in this workflow) or the variance (for <i>lowerNormW</i>) (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 46</span></i><span
lang=EN-US>). NORCOMBINE first generates merged data files and relation tables
that are later integrated to the above-described four grouped levels.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=550 height=107 id="Picture 54"
src="User_Guide_files/image063.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771757"></a><a name="_Ref122699608"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>46</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The NORCOMBINE task table for workflow 2.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>                The four new samples
(grouped levels or sample groups) are then used by the RATIOS basic module to
calculate two ratios: <i>WT-AngIIvsWT-C</i>, where wild-type AngII-treated
animals are compared to controls; and <i>ISG15-AngIIvsISG15-C</i>, where ISG15
AngII-treated animals are compared to controls (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 47</span></i><span
lang=EN-US>). The module calculates the new log2 ratios as the difference
between <i>Ratio numerator</i> and <i>Ratio denominator</i> values, whereas the
corresponding statistical weight is assessed according to the <i>V Method</i>
indicated in the task table (<i>max</i> in this case, which means the maximum
weight between the samples indicated under <i>Ratio numerator</i> and <i>Ratio
denominator</i>. The other two choices are <i>avg</i>, the average of the two
weights; and <i>form</i>, 1/(1/V<sub>n</sub> + 1/V<sub>d</sub>), where V<sub>n</sub>
and V<sub>d</sub> are the statistical weight of the samples indicated under<i>
Ratio numerator</i> and <i>Ratio denominator</i>, respectively).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=400 height=76 id="Picture 55"
src="User_Guide_files/image064.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771774"></a><a name="_Ref122699630"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>47</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The RATIOS task table for workflow 2.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>                The last basic module
executed in workflow 2, SBT (</span><i><span lang=EN-US style='line-height:
107%;color:#BF8F00'>Figure 48</span></i><span lang=EN-US>), relies on the
systems biology triangle algorithm </span><span
lang=EN-US>[3]</span><span lang=EN-US> to perform an integration between the <i>Lower
level</i> (<i>protein</i> in this case) and the corresponding grand mean using
the variance previously obtained in the integration between the <i>Lower level</i>
and an <i>Intermediate level</i> (<i>category</i> in this case, </span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 48</span></i><span
lang=EN-US>). The way it is used here, the SBT module reveals functional
category changes originated by the coordinated behaviour of their protein
components. The integrations involved, <i>protein-to-category</i>, <i>protein-to-proteinall</i>,
and <i>category-to-categoryall</i>, require the relation tables previously
generated with RELS CREATOR (</span><i><span lang=EN-US style='line-height:
107%;color:#BF8F00'>Figure 45</span></i><span lang=EN-US>). As mentioned above,
the elements of the <i>protein2category</i> relation table were retrieved from
a text file containing functional annotations from the Gene Ontology
knowledgebase [3].</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=299 height=83 id="Picture 99"
src="User_Guide_files/image065.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771804"></a><a name="_Ref122699654"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>48</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The SBT task table for workflow 2.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Finally, the
REPORT module (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
49</span></i><span lang=EN-US>) is used in this workflow to collect the
following statistical variables for all samples, namely the 16 individual
samples generated with LEVEL CREATOR (<i>e.g.</i> <i>Ind/WT-C-1</i> and <i>Ind/ISG15-AngII-1</i>,
</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 42</span></i><span
lang=EN-US>), the four combined samples obtained with NORCOMBINE (<i>WT-C,
WT-AngII, ISG15-C</i>, and <i>ISG15-AngII</i>, </span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 46</span></i><span
lang=EN-US>), and the two ratio samples created by RATIOS (<i>WT-AngIIvsWT-C</i>
and ISG15-AngIIvsISG15-C, </span><i><span lang=EN-US style='line-height:107%;
color:#BF8F00'>Figure 47</span></i><span lang=EN-US>), as prompted by the
asterisk:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Number of peptides per protein (<i>n</i>),
     obtained from the <i>peptide-to-protein </i>integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Mean-corrected protein log2 ratios expressed in
     units of standard deviation (<i>Z</i>), calculated from the <i>protein-to-proteinall</i>
     integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>False Discovery Rate (<i>FDR</i>) values
     indicating which <i>Z</i> values stand out as outliers in the <i>protein-to-proteinall</i>
     integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Number of proteins per category (<i>n</i>),
     obtained from the <i>protein-to-category integration</i>;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Mean-corrected category log2 ratios expressed in
     units of standard deviation (<i>Z</i>), calculated from the <i>category-to-categoryall</i>
     integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>False Discovery Rate (<i>FDR</i>) values
     indicating which <i>Z</i> values stand out as outliers in the <i>category-to-categoryall</i>
     integration.</span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=568 height=106 id="Picture 104"
src="User_Guide_files/image066.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123771910"></a><a name="_Ref122699736"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>49</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The REPORT task table for workflow 2.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The following advanced
options of the REPORT module have been used in this workflow (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 49</span></i><span
lang=EN-US>):</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Merge with report</span></i><span lang=EN-US>
     designates the report file whose variables will be incorporated into the
     current report on the basis of their element identifiers. Thus, for
     example, the <i>Npep2prot_Quanprot</i> report is first populated with the <i>Z</i>
     values of the <i>protein-to-proteinall</i> integration, and then added the
     number of peptides per protein coming from the <i>Npep2prot</i> report (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 49</span></i><span lang=EN-US>). This
     is possible because the <i>protein</i> level is present in both report
     files.</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Column headers to eliminate</span></i><span
     lang=EN-US> restricts the elements to be written to the <i>Output reports
     Npep2prot_Quanprot </i>and <i>Nprot2cat_Quancat </i>to those from the
     protein and category level, respectively. This prevents data duplication
     upon merging with the <i>Npep2prot</i> and <i>Nprot2cat</i> reports, as
     these contain also the peptide and protein level, respectively (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 49</span></i><span lang=EN-US>).</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>As previously
shown with SanXoT </span><span lang=EN-US>[5]</span><span lang=EN-US>, iSanXoT
analysis revealed a coordinated alteration of proteins implicated in cardiovascular
function, extracellular matrix and remodeling, and vascular redox state in
aorta from AngII-infused ISG15-KO mice (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 50</span></i><span
lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;text-indent:36.0pt;
page-break-after:avoid'><span lang=EN-US><img border=0 width=528 height=239
id="Picture 15" src="User_Guide_files/image067.jpg"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772030"></a><a name="_Ref122699849"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>50</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Differential abundance of functional categories (A) and
proteins (B) revealed by workflow 2 based on the mean-corrected log2-ratio
expressed in units of standard deviation at the category (Zc) and protein (Zq)
level, respectively.</span></i></p>

<p class=MsoNormal><span lang=EN-US style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h3><a name="_Toc124328448"><span lang=EN-US>Workflow execution</span></a></h3>

<p class=MsoNormal><span lang=EN-US>The workflow template and input files that
are needed to execute this workflow can be downloaded from </span></p>

<p class=MsoNormal><span lang=EN-US><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_NORCOM_RATIOS_SBT.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_NORCOM_RATIOS_SBT.zip</span></a></span></p>

<p class=MsoNormal><span lang=EN-US>See the <i>Executing a workflow template</i>
Section below for detailed instructions.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b><span style='font-size:18.0pt;line-height:107%'>&nbsp;</span></b></p>

<h2><a name="_Toc124328449"></a><a name="_Workflow_3:_Quantification"></a><span
lang=EN-US>Workflow 3: Quantification of posttranslationally modified peptides
in a labeled experiment</span></h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328450"><span lang=EN-US>Experimental</span></a></h3>

<p class=MsoNormal><span lang=EN-US>This workflow was used to quantitate
Cys-oxidized peptides in mouse embryonic fibroblast (MEF) preparations
subjected to chemical oxidation with diamide, an experiment that served to
illustrate the comparative performance of on-filter (FASILOX) and in-gel
(GELSILOX) approaches to study the thiol redox proteome </span><span lang=EN-US>[4]</span><span lang=EN-US>. MEF samples were incubated with diamide
(treated group) or PBS (control group) for 10 min (four biological replicates
per condition). The FASP filter was used to carry out the sequential alkylation
and reduction steps needed to separately label reduced and oxidized Cys
residues, after which the proteins were digested with trypsin and the resulting
peptides were isobaricaly labeled with iTRAQ 8-plex and analyzed by LC-MS/MS.
Protein identification was achieved with Proteome Discoverer 1.4 (Thermo Fisher
Scientific; see Section 3 below to learn how these data are adapted to be used
with iSanXoT). The FDR for peptide identification was calculated using the
refined method </span><span
lang=EN-US>[7]</span><span lang=EN-US>.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=536 height=234 id="Picture 88"
src="User_Guide_files/image068.jpg"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772176"></a><a name="_Ref122700968"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>51</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Scheme of workflow 3 (quantification of
posttranslationally modified peptides in a labeled experiment) showing module
components: LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE, and REPORT (A) and RELS
CREATOR (B).</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Toc124328451"><span lang=EN-US>Workflow operation</span></a></h3>

<p class=MsoNormal><span lang=EN-US>Workflow 3 comprises the basic modules
LEVEL CREATOR, LEVEL CALIBRATOR, and INTEGRATE, as well as the RELS CREATOR and
REPORT modules (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
51</span></i><span lang=EN-US>). The starting module, LEVEL CREATOR, generates
the level files, sample folders and log2 ratios indicated in the corresponding
task table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
52</span></i><span lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=557 height=118 id="Picture 243"
src="User_Guide_files/image069.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772189"></a><a name="_Ref122700980"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>52</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The LEVEL CREATOR task table for workflow 3.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The output level
files, <i>uscan</i> (<i>uncalibrated scan</i>), display element identifiers
together with their corresponding log2-ratio and statistical weight values (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 53</span></i><span
lang=EN-US>). This uncalibrated weight is the inverse of the variance
associated with the log2-ratio, and therefore a higher weight corresponds to a
more accurate quantification.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=527 height=108 id="Picture 247"
src="User_Guide_files/image070.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772226"></a><a name="_Ref122700992"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>53</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Excerpt from one of the uscan files generated by
workflow 3 LEVEL CREATOR module showing element identifiers (left column), log2
ratios (center column) and statistical weights (right column).</span></i></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Prior to their
integration into a higher level, <i>uscan</i> levels are calibrated to account
for the variability of the LC-MS/MS analysis with the LEVEL CALIBRATOR module,
which results in scan calibrated levels with recalculated statistical weights (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 54</span></i><span
lang=EN-US>, Top). Then the INTEGRATE module performs the <i>scan-to-peptide</i>
and <i>peptide-to-</i> <i>protein</i> integrations according to the module task
table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
54</span></i><span lang=EN-US>, Bottom). The latter integration will provide
the differential abundance of peptides (including posttranslationally modified
peptides) regardless of the differential abundance of proteins, and the authors
chose not to include the heterogeneous population of Cys-containing peptides
for the calculation of variance in the <i>peptide-to-protein</i> integration </span><span lang=EN-US>[4]</span><span lang=EN-US>.</span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US><img
border=0 width=545 height=153 id="Picture 89"
src="User_Guide_files/image071.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772240"></a><a name="_Ref122701063"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>54</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The LEVEL CALIBRATOR (Top) and INTEGRATE (Bottom) task tables
for workflow 3.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Thus, the <i>Tag</i>
advanced option of the INTEGRATE module task table displays <i>!C_Modification</i>,
which means that the <i>peptide</i> (lower level) elements marked with this tag
in the <i>peptide2protein</i> relation table (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 55</span></i><span
lang=EN-US>) will not be considered in the calculation of the variance in the <i>peptide-to-protein</i>
integration. Every integration requires a specific relation table linking the
corresponding lower- and higher-level elements that is generated by RELS
CREATOR (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
51</span></i><span lang=EN-US>B and </span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 55</span></i><span
lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=579 height=98 id="Picture 65"
src="User_Guide_files/image072.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772294"></a><a name="_Ref122701108"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>55</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The RELS CREATOR task table for workflow 3.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>In particular,
the <i>peptide2protein</i> relation table describes the relationships between
the peptide level (<i>i.e</i>. the lower level) and the protein level (<i>i.e</i>.
the higher level), and in this case includes a third column headed by <i>Modifications</i>
where Cys-containing peptides have been tagged with <i>C_Modification</i> (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 56</span></i><span
lang=EN-US>). Accordingly, Cys-containing peptides will not be considered for
the calculation of variance in the <i>peptide-to-protein</i> integration.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=535 height=193 id="Picture 255"
src="User_Guide_files/image073.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772399"></a><a name="_Ref122701294"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>56</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Excerpt from the peptide2protein relation table that
links peptides to proteins. Note the use of a third column to tag
Cys-containing peptides, which will be later excluded from the calculation of
variance in the peptide-to-protein integration.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Finally, the
REPORT module is used ion this workflow to collect the statistical variables
generated by the <i>peptide-to-protein</i> integration for all eight samples (<i>c1,
c2, c3, c4, t1, t2, t3,</i> and <i>t4</i>), as prompted by the asterisk (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 57</span></i><span
lang=EN-US>A):</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Number of scans per peptide (<i>n</i>), obtained
     from the <i>scan-to-peptide</i> integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Peptide log2 ratios expressed in units of
     standard deviation corrected by the protein mean (<i>Z</i>), calculated
     from the <i>peptide-to-protein</i> integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>False Discovery Rate (<i>FDR</i>) values
     indicating which <i>Z</i> values stand out as outliers in the integration.</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:18.0pt'><span lang=EN-US>The advanced
options of the REPORT module (</span><i><span lang=EN-US style='line-height:
107%;color:#BF8F00'>Figure 57</span></i><span lang=EN-US>A):</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt'><span
lang=EN-US style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Merge with report</span></i><span lang=EN-US>
designates the merge of the number of scans per peptide from the <i>Nscan2pep</i>
report and the Z values and FDR of the <i>peptide-to-protein</i> that will be
saved in the <i>Nscan2pep_Quanpepprot report</i>.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt'><span lang=EN-US
style='font-family:Symbol'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><i><span lang=EN-US>Column headers columns to eliminate</span></i><span
lang=EN-US> restricts the elements to be written to the Output reports <i>Nscan2pep_Quanpepprot</i>
to those from the scan level.</span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 57</span></i><span
lang=EN-US>B shows the relative abundance of Cys-containing peptides provided
by the <i>peptide-to-protein</i> integration. </span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=470 height=383 id="Picture 12"
src="User_Guide_files/image074.jpg"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772417"></a><a name="_Ref122701319"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>57</span></i><i><span lang=EN-US
style='color:#BF8F00'>. (A) The REPORT module task table for workflow 3. (B)
Relative abundance of Cys-containing peptides in MEF samples measured by
peptide log2 ratios expressed in units of standard deviation corrected by the
protein mean (Zpq).</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Toc124328452"><span lang=EN-US>Workflow execution</span></a></h3>

<p class=MsoNormal><span lang=EN-US>The workflow template and input files that
are needed to execute this workflow can be downloaded from </span></p>

<p class=MsoNormal><span lang=EN-US><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_PTM.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WSPP_PTM.zip</span></a></span></p>

<p class=MsoNormal><span lang=EN-US>See the <i>Executing a workflow template</i>
Section below for detailed instructions.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><b><span style='font-size:18.0pt;line-height:107%'>&nbsp;</span></b></p>

<h2><a name="_Toc124328453"></a><a name="_Workflow_4:_Label-free"></a><span
lang=EN-US>Workflow 4: Label-free quantification</span></h2>

<p class=MsoNormal>&nbsp;</p>

<h3><a name="_Toc124328454"><span lang=EN-US>Experimental</span></a></h3>

<p class=MsoNormal><span lang=EN-US>This workflow has been applied to
quantitative data obtained in a multicenter study aimed at evaluating different
bioinformatics tools </span><span
lang=EN-US>[6]</span><span lang=EN-US>. The authors generated two hybrid
proteome samples consisting of tryptic digests of human, yeast and <i>Escherichia
coli</i> proteins mixed in two different defined proportions (13:3:4 for <i>a </i>samples
and 13:6:1 for b samples). Quadruplicate peptide samples were analyzed by
LC-MS/MS, after which peptide identification and quantification were carried
out with MaxQuant </span><span
lang=EN-US>[8]</span><span lang=EN-US>, among other software packages (see
Section below to learn how these data are adapted to be used with iSanXoT).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=480 height=371 id="Picture 90"
src="User_Guide_files/image075.jpg"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772496"></a><a name="_Ref122703200"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>58</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Scheme of workflow 4 (label-free quantification)
showing module components: LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE,
NORCOMBINE, and REPORT (A) and RELS CREATOR (B).</span></i></p>

<p class=MsoNormal><span lang=EN-US style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<h3><a name="_Toc124328455"><span lang=EN-US>Workflow operation</span></a></h3>

<p class=MsoNormal><span lang=EN-US>Workflow 4 includes the basic iSanXoT
modules LEVEL CREATOR, LEVEL CALIBRATOR, INTEGRATE, NORCOMBINE, and RATIOS
along with the REPORT and RELS CREATOR modules (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 58</span></i><span
lang=EN-US>). The starting module, LEVEL CREATOR, generates the level files,
sample folders and log2 ratios indicated in the corresponding task table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 59</span></i><span
lang=EN-US>) based on the quantitative data at the peptide level obtained with
MaxQuant for replicate <i>a</i> and <i>b</i> samples. To calculate the
log2-ratio denominator, first the averages of the four <i>a</i> and the four <i>b</i>
samples are calculated separately (as indicated by the square brackets), then
the average of the two average values is calculated (as indicated by the
comma).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=595 height=237 id="Picture 44"
src="User_Guide_files/image076.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772513"></a><a name="_Ref122703216"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>59</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The LEVEL CREATOR task table for workflow 4.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>This ensures
that no log2-ratio is calculated when all four values are missing in either the
<i>a</i> or the <i>b</i> sample group. The eight <i>u_peptide</i> (uncalibrated
peptide) output level files display element identifiers together with their
corresponding log2-ratio and statistical weight values (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 60</span></i><span
lang=EN-US>). </span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=606 height=132 id="Picture 72"
src="User_Guide_files/image077.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772529"></a><a name="_Ref122703265"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>60</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Excerpt from one of the u_peptide files generated by
workflow 4 LEVEL CREATOR module showing element identifiers (left column), log2
ratios (center column) and statistical weights (right column).</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>This
uncalibrated weight is the inverse of the variance associated with the
log2-ratio, and therefore a higher weight corresponds to a more accurate
quantification. Prior to their integration into a higher level, <i>u_peptide</i>
levels are calibrated with the LEVEL CALIBRATOR module, which results in <i>peptide</i>
calibrated levels with recalculated statistical weights (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 61</span></i><span
lang=EN-US>, Top). Then the INTEGRATE module performs the <i>peptide-to-protein</i>
and <i>protein-to-proteinall</i> integrations according to the module task
table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
61</span></i><span lang=EN-US>, Bottom).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=536 height=154 id="Picture 91"
src="User_Guide_files/image078.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772550"></a><a name="_Ref122703295"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>61</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The LEVEL CALIBRATOR (Top) and INTEGRATE (Bottom) task
tables for workflow 4.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Every
integration requires a specific relation table linking the corresponding lower-
and higher-level elements that is generated by RELS CREATOR (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 58</span></i><span
lang=EN-US>B and </span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
62</span></i><span lang=EN-US>); for instance, the <i>peptide2protein </i>relation
table mentioned describes the relationships between the peptide level (<i>i.e.</i>
the lower level) and the protein level (<i>i.e.</i> the higher level).</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=581 height=76 id="Picture 36"
src="User_Guide_files/image079.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772598"></a><a name="_Ref122703345"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>62</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The RELS CREATOR task table for workflow 4.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>As mentioned
above, replicate <i>a</i> and <i>b</i> samples contain different proportions of
yeast and <i>Escherichia coli</i> peptides (3:4 and 6:1,  respectively) and the
same proportion of human peptides; for this reason, only human proteins were
considered in the calculation of the variance for the <i>protein-to-proteinall</i>
integrations, as indicated by the <i>Tag</i> parameter used in the INTEGRATE
task table (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
61</span></i><span lang=EN-US>, Bottom; note that no outliers were removed in
this integration as prompted by the <i>0</i> value indicated for the <i>FDR </i>parameter).
Accordingly, the <i>protein2proteinall</i> relation table required by this
integration includes a third column headed by <i>Species</i> where proteins
have been tagged with their corresponding species (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 63</span></i><span
lang=EN-US>). This relation table was built based on the file <i>HumanSaccEcoliPME12_...‌‌‌_species.tsv</i>.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=264 height=185 id="Picture 73"
src="User_Guide_files/image080.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772697"></a><a name="_Ref122703568"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>63</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Excerpt from the protein2proteinall workflow 4 relation
table that links proteins to a constant value representing the protein grand
mean. Note the use of a third column to tag proteins with their corresponding
species for later species-specific protein-to-proteinall integration.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Next, the four
individual replicate <i>a</i> (<i>a1, a2, a3,</i> and <i>a4</i>) and <i>b</i> (<i>b1,
b2, b3,</i> and <i>b4</i>) samples initially generated with LEVEL CREATOR (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 59</span></i><span
lang=EN-US>) are combined into samples <i>A</i> and <i>B</i>, respectively,
using the NORCOMBINE basic module (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 64</span></i><span
lang=EN-US>, Top). The module task table indicates that samples were combined
at the <i>protein</i> level using the <i>proteinall</i> level for
normalization. NORCOMBINE also requires the user to specify which <i>lowerNorm</i>
files contain the necessary data for the samples to be combined. These <i>lowerNorm</i>
files, previously generated by the INTEGRATE module (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 61</span></i><span lang=EN-US>,
Bottom), contain lower-level identifiers along with the corresponding centered
log2-ratio values and either the integration statistical weight (in the case of
<i>lowerNormV</i>, as chosen in this workflow) or the variance (for <i>lowerNormW</i>)
(</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 64</span></i><span
lang=EN-US>, Top). NORCOMBINE first generates merged data files and relation
tables that are later integrated to the above-described combined samples <i>A</i>
and <i>B</i>.</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>To compared
these two samples, first new log-2 ratios and statistical weights are
calculated using the RATIOS basic module (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 64</span></i><span
lang=EN-US>, Bottom), after which a <i>protein-to-proteinall</i> integration is
carried out for the newly-generated <i>B_vs_A</i> sample (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 61</span></i><span
lang=EN-US>, Bottom).</span></p>

<p class=MsoNormal align=center style='text-align:center;text-indent:36.0pt;
page-break-after:avoid'><span lang=EN-US><img border=0 width=560 height=136
id="Picture 9" src="User_Guide_files/image081.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123772755"></a><a name="_Ref122703609"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>64</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The NORCOMBINE (Top) and RATIOS (Bottom) task tables
for workflow 4.</span></i></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>Finally, the
REPORT module (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure
65</span></i><span lang=EN-US>) is used in this workflow to collect variables
for all the samples involved (as prompted by the asterisk), namely the eight
individual samples generated with LEVEL CREATOR (<i>a1, a2, a3,</i> <i>a4, b1,
b2, b3, </i>and <i>b4</i>; </span><i><span lang=EN-US style='line-height:107%;
color:#BF8F00'>Figure 59</span></i><span lang=EN-US>), the combined samples <i>A</i>
and <i>B </i>obtained with NORCOMBINE (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 64</span></i><span
lang=EN-US>, Top), and the <i>B_vs_A</i> sample generated by RATIOS (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 64</span></i><span
lang=EN-US>, Bottom). The following variables were considered:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Number of peptides per protein (<i>n</i>),
     obtained from the <i>peptide-to-protein </i>integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Protein log2 ratios (<i>Xinf</i>), calculated
     from the <i>protein-to-proteinall</i> integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Protein grand mean (<i>Xsup</i>), calculated from
     the <i>protein-to-proteinall</i> integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>FDR</span></i><span lang=EN-US> values
     indicating which <i>Z</i> values stand out as outliers in the <i>protein-to-proteinall</i>
     integration;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Species information (<i>tags</i>) showing in the <i>protein-to-proteinall</i>
     integration output file.</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US style='font-size:14.0pt;line-height:107%'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US style='font-size:14.0pt;line-height:107%'><img border=0 width=608
height=70 id="Picture 92" src="User_Guide_files/image082.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123768547"></a><a name="_Ref122703805"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>65</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The REPORT module task table for workflow 4.</span></i></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The following
advanced options of the REPORT module have been used in this workflow (</span><i><span lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 65</span></i><span
lang=EN-US>):</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Merge with report</span></i><span lang=EN-US>
     designates the report file whose variables will be incorporated into the
     current report on the basis of their element identifiers. Thus, the <i>Npep2prot_Quanprot</i>
     report is first populated with the <i>Xinf, Xsup, FDR</i> and <i>tags</i>
     values of the protein-to-proteinall integration, and then added the number
     of peptides per protein coming from the <i>Npep2prot report</i> (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 65</span></i><span lang=EN-US>). This
     is possible because the <i>protein</i> level is present in both reports;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Column headers to eliminate</span></i><span
     lang=EN-US> restricts the elements to be written to the corresponding <i>Output
     report</i> to those from the <i>protein</i> level (lower level; both lower
     and higher levels are used unless otherwise stated). This prevents data
     duplication when <i>Npep2prot_Quanprot</i> and <i>Npep2prot </i>reports
     are merged, as the latter contains also the peptide level.</span></li>
</ul>

<p class=MsoNormalCxSpMiddle style='margin-left:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormalCxSpMiddle style='margin-left:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><a name="_Hlk121489265"><span
lang=EN-US>In agreement with the results reported </span></a><span
lang=EN-US>[6]</span><span lang=EN-US>, the combined statistics <i>B_vs_A</i>
analysis shows human, yeast and bacterial proteins normally distributed around
the expected 0, 1, and -2 mean-corrected log2-ratio (<i>X<sub>qa</sub> – grand mean</i>,
indeed, they are the X<sub>inf</sub> - X<sub>sup</sub>) values (corresponding
to 1, 2, and 0.25 fold change values) (</span><i><span
lang=EN-US style='line-height:107%;color:#BF8F00'>Figure 66</span></i><span
lang=EN-US>).</span></p>

<p class=MsoNormal align=center style='text-align:center;text-indent:36.0pt;
page-break-after:avoid'><span lang=EN-US><img border=0 width=453 height=384
id="Picture 94" src="User_Guide_files/image083.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773060"></a><a name="_Ref122704192"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>66</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Quantification of human (orange), yeast (grey) and
bacterial (blue) proteins according to the combined statistics B_vs_A.</span></i></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Toc124328456"><span lang=EN-US>Workflow execution</span></a></h3>

<p class=MsoNormal><span lang=EN-US>The workflow template and input files that
are needed to execute this workflow can be downloaded from </span></p>

<p class=MsoNormal><span lang=EN-US><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WPP_LabelFree.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/cases/templates/WPP_LabelFree.zip</span></a></span></p>

<p class=MsoNormal><span lang=EN-US>See the <i>Executing a workflow template</i>
Section below for detailed instructions.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Toc124328457"></a><a name="_Executing_a_workflow"></a><span
lang=EN-US>Executing a workflow template</span></h1>

</div>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><span lang=EN-US>To illustrate how to execute the workflows
created with iSanXoT, we shall make use of the sample workflows described in
detail in the previous Section. Start by downloading, for instance, workflow 1
template and input files from the iSanXoT wiki (</span><span lang=EN-US><a
href="https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/workflows/WSPP-SBT.zip"><span
style='color:windowtext'>https://github.com/CNIC-Proteomics/iSanXoT/wiki/studies/workflows/WSPP-SBT.zip</span></a></span><span
lang=EN-US>). Extract the files included in the compressed archive Then proceed
as follows:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Open the iSanXoT application by double-clicking
     the application icon (</span><i><span lang=EN-US style='color:#BF8F00'>Figure
     67</span></i><span lang=EN-US>).</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=234 height=154 id="Picture 95"
src="User_Guide_files/image084.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773105"></a><a name="_Ref122704913"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>67</span></i><i><span lang=EN-US
style='color:#BF8F00'>. The iSanXoT startup message.</span></i></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Choose <i>New Project</i> from the <i>Project</i>
     menu (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 68</span></i><span
     lang=EN-US>).</span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=429 height=250 id="Picture 96"
src="User_Guide_files/image085.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773114"></a><a name="_Ref122704926"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>68</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Create New Project.</span></i></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Provide a project name and a project folder, then
     click the <i>Submit</i> button (</span><i><span
     lang=EN-US style='color:#BF8F00'>Figure 68</span></i><span lang=EN-US>).</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>To import a workflow previously generated with
     iSanXoT, choose <i>Import Workflow</i> from the <i>Project</i> menu (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 69</span></i><span lang=EN-US>).
     Choose, for instance, workflow 1 by selecting the root folder where you
     stored the workflow files. </span></li>
</ul>

<p class=MsoNormal style='margin-top:0cm;margin-right:0cm;margin-bottom:0cm;
margin-left:36.0pt;margin-bottom:.0001pt;line-height:normal'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=431 height=196 id="Picture 97"
src="User_Guide_files/image086.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773175"></a><a name="_Ref122704953"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>69</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Importing a preexisting iSanXoT workflow to the
newly-created project.</span></i></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Now click on <i>Choose identification file</i> to
     provide an identification/quantification (<i>ID-q</i>) file (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 70</span></i><span lang=EN-US>). The
     below section shows how to prepare this file based on the output from a variety
     of proteomics pipelines; but you can try the <i>ID-q.tsv</i> file included
     in the root folder where you stored workflow 1 files.</span></li>
</ul>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><span
lang=EN-US><img border=0 width=421 height=362 id="Picture 3"
src="User_Guide_files/image087.jpg"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773226"></a><a name="_Ref122704989"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>70</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Choosing the identification/quantification (ID-q) file
for the newly-created project.</span></i></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Bear in mind that any path included in the RELS
     CREATOR module task table of the workflow imported will need be updated.</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<b><span lang=EN-US style='font-size:16.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span></b>

<p class=MsoNormal><b><span lang=EN-US style='font-size:16.0pt;line-height:
107%'>&nbsp;</span></b></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Hlk124849730"></a><a name="_Toc124328458"></a><a
name="_Creating_the_identification/quantif"></a><span lang=EN-US>Creating the
identification/quantification file from proteomics pipelines</span></h1>

</div>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>iSanXoT requires an
identification/quantification file (<i>ID-q.tsv</i>) containing identified
features together with their quantitative values and the experiment they
originate from whether there is more than one experiment. This tab-separated
values <i>ID-q</i> file must fulfill:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Features (scan, peptide, protein, etc.) are unequivocally
     referenced using a unique identifier;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Quantitative values for the features are
     displayed across columns;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Experiments are specified in a column field
     headed by <i>Experiment</i> whether there is more than one experiment.</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>In this Section
we shall describe how to prepare the <i>ID-q.tsv</i> file based on data
obtained with the four most popular proteomics pipelines.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoCaption style='page-break-after:avoid'><span lang=EN-US>Table </span><span lang=EN-US>1</span><span lang=EN-US> . Column headers from the tested proteomic
pipelines that are needed for the input idnetification/quantification file
(ID-q.tsv). (*) Redundant proteins. You have to select your main protein. (**)
Provide the experiment names in the case of multiple experiments.</span></p>

<div align=center>

<table class=MsoNormalTable border=0 cellspacing=0 cellpadding=0 width=662
 style='width:496.5pt;border-collapse:collapse'>
 <tr style='height:24.05pt'>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0cm 1.5pt 0cm 1.5pt;
  height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0cm 1.5pt 0cm 1.5pt;
  height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0cm 1.5pt 0cm 1.5pt;
  height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=411 colspan=5 valign=top style='width:308.1pt;border:solid windowtext 1.0pt;
  background:silver;padding:0cm 1.5pt 0cm 1.5pt;height:24.05pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>Id_q.txt column headers</span></i></b></p>
  </td>
 </tr>
 <tr style='height:23.35pt'>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0cm 1.5pt 0cm 1.5pt;
  height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0cm 1.5pt 0cm 1.5pt;
  height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;background:white;padding:0cm 1.5pt 0cm 1.5pt;
  height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=251 colspan=3 valign=top style='width:188.45pt;border:solid windowtext 1.0pt;
  border-top:none;background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:23.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>Features</span></b></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:none;border-right:solid windowtext 1.0pt;
  background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:23.35pt'>
  <p class=MsoNormal align=center style='margin-top:12.0pt;margin-right:0cm;
  margin-bottom:0cm;margin-left:0cm;margin-bottom:.0001pt;text-align:center;
  line-height:normal;text-autospace:none'><b><span lang=EN-US style='font-size:
  8.0pt;font-family:"Times New Roman",serif;color:white'>Quantitation</span></b></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:solid windowtext 1.0pt;
  border-left:none;border-bottom:none;border-right:solid windowtext 1.0pt;
  background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:23.35pt'>
  <p class=MsoNormal align=center style='margin-top:12.0pt;margin-right:0cm;
  margin-bottom:0cm;margin-left:0cm;margin-bottom:.0001pt;text-align:center;
  line-height:normal;text-autospace:none'><b><span lang=EN-US style='font-size:
  8.0pt;font-family:"Times New Roman",serif;color:white'>Experiment</span></b></p>
  </td>
 </tr>
 <tr style='height:24.9pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>Proteomic pipeline</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>Experiment type</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>File type</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>Scan</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>Peptide</span></b></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-left:none;background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>Protein</span></b></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>&nbsp;</span></b></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:gray;padding:0cm 1.5pt 0cm 1.5pt;height:24.9pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><b><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:white'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr style='height:30.95pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Proteome
  Discoverer (version 2.5)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>_PSMs.txt</span></i><span lang=EN-US style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum
  file + First Scan + Charge</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence
  + Modifications</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein
  Accessions (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Intensity</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:30.95pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:29.85pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>_PSMs.txt</span></i><span lang=EN-US style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum
  file + First Scan + Charge</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence
  + Modifications</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein
  Accessions (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Abundance:
  Channel identifier</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:22.65pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>MaxQuant
  (version 1.6.5.0)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>peptides.txt </span></i></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Id</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Leading
  razor protein</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Intensity
  sample</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:22.65pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:29.85pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>msmsScans.txt</span></i><span lang=EN-US style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Raw
  file + Scan number + Charge</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Sequence
  + Modifications</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Proteins
  (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Reporter
  intensity sample</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:29.85pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Trans-Proteomics
  Pipeline (version 6.0.0)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>pepXML</span></i><span lang=EN-US style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:black'> file</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>proteins
  (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>precursor_intensity</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:29.85pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:23.1pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>pepXML</span></i><span lang=EN-US style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:black'> file</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>proteins
  (*)</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Channel
  identifier</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:23.1pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:25.35pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Fragpipe
  (version 1.8.1)</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>label
  free</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>psm.tsv</span></i><span lang=EN-US style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Modified
  Peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Intensity</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:#F2F2F2;padding:0cm 1.5pt 0cm 1.5pt;height:25.35pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
 <tr style='height:26.25pt'>
  <td width=84 valign=top style='width:62.8pt;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>isotopically
  labelled </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><i><span
  lang=EN-US style='font-size:8.0pt;font-family:"Times New Roman",serif;
  color:black'>psm.tsv</span></i><span lang=EN-US style='font-size:8.0pt;
  font-family:"Times New Roman",serif;color:black'> </span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Spectrum</span></p>
  </td>
  <td width=84 valign=top style='width:62.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Modified
  Peptide</span></p>
  </td>
  <td width=84 valign=top style='width:62.8pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Protein</span></p>
  </td>
  <td width=88 valign=top style='width:65.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Channel
  identifier</span></p>
  </td>
  <td width=72 valign=top style='width:54.0pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  background:white;padding:0cm 1.5pt 0cm 1.5pt;height:26.25pt'>
  <p class=MsoNormal align=center style='margin-bottom:0cm;margin-bottom:.0001pt;
  text-align:center;line-height:normal;text-autospace:none'><span lang=EN-US
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Experiment
  names (**)</span></p>
  </td>
 </tr>
</table>

</div>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><a name="_Toc124328464"></a><a name="_Preparing_the_ID-q"></a><span
lang=EN-US>Preparing the <i>ID-q</i> file from Proteome Discoverer output</span></h2>

<p class=MsoNormal><span lang=EN-US>In the case of Proteome Discoverer version
2.5 </span><span
lang=EN-US>[9]</span><span lang=EN-US>, the quantitative data required to
prepare the <i>ID-q</i> file are extracted by the <i>Processing</i> workflow
nodes <i>Reporter Ions Quantifier</i> (for labelled experiments) and <i>Minora
Feature Detector</i> (for label-free experiments). These data are written by
the <i>Consensus</i> workflow to a text file suffixed with <i>_PSMs.txt</i> displaying:</span></p>

<h3><span lang=EN-US>Label-free experiments</span></h3>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>PSMs Peptide ID</span></i><span lang=EN-US>:
     Unique identifier for each <i>Peptide</i> sequence identified;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Sequence</span></i><span lang=EN-US>: Amino
     acid sequence of the identified peptide;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Modifications</span></i><span lang=EN-US>:
     Post-translational modifications contained within the sequence;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Protein Accessions</span></i><span lang=EN-US>:
     Accession code(s) for the protein(s) to which the peptide sequence is
     ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Intensity: </span></i><span lang=EN-US>Precursor
     ion intensity (to be used in label-free experiments).</span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>The unequivocally identifier for a peptide has to be
composed by the union of <i>Sequence</i> and <i>Modifications</i> fields.</span></p>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>Labelled experiments</span></h3>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Spectrum File</span></i><span lang=EN-US>: Name
     of the RAW file the spectral MS/MS data was extracted from;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>First Scan</span></i><span lang=EN-US>: RAW
     file derived scan number for the MS/MS spectrum;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Charge</span></i><span lang=EN-US>: Charge
     state of the precursor ion;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>PSMs Peptide ID</span></i><span lang=EN-US>:
     Unique identifier for each <i>Peptide</i> sequence identified;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Sequence</span></i><span lang=EN-US>: Amino
     acid sequence of the identified peptide;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Modifications</span></i><span lang=EN-US>: Post-translational
     modifications contained within the sequence;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Protein Accessions</span></i><span lang=EN-US>:
     Accession code(s) for the protein(s) to which the peptide sequence is
     ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Abundance: Channel identifier</span></i><span
     lang=EN-US>:  Reporter ion intensity (to be used in labelled experiments).</span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;text-indent:
18.0pt;line-height:normal'><span lang=EN-US>In the case of multiple
experiments, we recommend the creation of a unique identifier for a Scan is
composed by the union of the fields: <i>Spectrum File</i>, <i>First </i>Scan
and <i>Charge</i>; otherwise, the <i>PSMs Peptide ID</i> field is enough for
the Scan identification. The unequivocally identifier for a peptide has to be
composed by the union of <i>Sequence</i> and <i>Modifications</i> fields.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><a name="_Toc124328459"></a><a name="_Preparing_the_ID-q_1"></a><span
lang=EN-US>Preparing the <i>ID-q</i> file from MaxQuant output</span></h2>

<p class=MsoNormal><span lang=EN-US>The way MaxQuant version 1.6.5.0 </span><span
lang=EN-US>[8]</span><span lang=EN-US> data are adapted for use with iSanXoT
depends on whether they originate from label-free or labelled proteomics
experiments:</span></p>

<h3><a name="_Toc124328460"><span lang=EN-US>Label-free experiments</span></a></h3>

<p class=MsoNormal><span lang=EN-US>In this case, the necessary data can be
found in the <i>peptides.txt</i> file, which is stored in the <i>“…combined/txt</i>”
folder:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><a name="_Hlk120012892"><i><span lang=EN-US>Id</span></i></a><span
     lang=EN-US>: Unique, consecutive identifier for each peptide <i>Sequence</i>
     identified;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Sequence</span></i><span lang=EN-US>: Amino
     acid sequence of the identified peptide;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Leading razor protein</span></i><span
     lang=EN-US>: Identifier of the top scoring protein to which the peptide <i>Sequence</i>
     is ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Intensity + Experiment</span></i><span
     lang=EN-US>: Summed up extracted ion current of all isotopic clusters
     associated with the peptide <i>Sequence</i> identified in the raw files
     making up <i>Experiment</i></span><span lang=EN-US>.</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h3><a name="_Toc124328461"><span lang=EN-US>Labelled experiments</span></a></h3>

<p class=MsoNormal><span lang=EN-US>When dealing with labelled experiments
(e.g. iTRAQ- or TMT-based), the necessary data can be found in the <i>msmsScans.txt</i>
file, which is stored in the <i>“…combined/txt</i>” folder:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Raw file</span></i><span lang=EN-US>: Name of
     the RAW file from which the MS/MS data were extracted;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Scan number</span></i><span lang=EN-US>: MS/MS
     spectrum number within a given RAW file;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Charge</span></i><span lang=EN-US>: Charge
     state of the precursor ion;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Sequence</span></i><span lang=EN-US>: Amino
     acid sequence of the identified peptide. This parameter is nonblank only
     when identification was successful.</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Proteins</span></i><span lang=EN-US>:
     Identifier(s) of the protein(s) to which the peptide <i>Sequence</i> is
     ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Reporter intensity + n</span></i><span
     lang=EN-US>: Intensity of the reporter ion corresponding to sample <i>n</i>.</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-US>The <i>msmsScans.txt</i>
file lacks the unique identifier that is required in the <i>ID-q</i> file. The
user is advised to create this unique identifier by concatenating <i>RAW file,
Scan number, </i>and<i> Charge</i>.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><a name="_Toc124328462"></a><a name="_Preparing_the_ID-q_2"></a><span
lang=EN-US>Preparing the <i>ID-q</i> file from Trans-Proteomic Pipeline output</span></h2>

<p class=MsoNormal><span lang=EN-US>Trans-Proteomics Pipeline version 6.0.0 </span><span
lang=EN-US>[10]</span><span lang=EN-US> comprises a number of tools (<i>e.g</i>.
<i>ASAPRatio</i>, <i>XPRESS</i> and <i>Libra</i>) that allow both label-free
and isotopically labelled quantitative data to be saved to a <i>pepXML</i>
file. Using the <i>PepXML Viewer</i> application, this file can be exported as
a spreadsheet displaying:</span></p>

<h3><span lang=EN-US>Label-free experiments</span></h3>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Spectrum</span></i><span lang=EN-US>: Unique
     identifier for each <i>Peptide</i> sequence identified;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Peptide: </span></i><span lang=EN-US>Amino
     acid sequence of the identified peptide;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Protein: </span></i><span lang=EN-US>Identifier
     of the protein to which the <i>Peptide</i> sequence is ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Precursor_intensity: </span></i><span
     lang=EN-US>Precursor ion intensity (to be used for label-free
     quantification).</span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>Labelled experiments</span></h3>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Spectrum</span></i><span lang=EN-US>: Unique
     identifier for each <i>Peptide</i> sequence identified;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Peptide: </span></i><span lang=EN-US>Amino
     acid sequence of the identified peptide;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Protein: </span></i><span lang=EN-US>Identifier
     of the protein to which the <i>Peptide</i> sequence is ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Channel identifier</span></i><span lang=EN-US>: 
     Reporter ion intensity (to be used for isotopic labelling quantification).  
     </span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<h2><a name="_Toc124328463"></a><a name="_Preparing_the_ID-q_3"></a><span
lang=EN-US>Preparing the <i>ID-q</i> file from FragPipe output</span></h2>

<p class=MsoNormal><span lang=EN-US>Fragpipe version 1.8.1 </span><span
lang=EN-US>[11]</span><span lang=EN-US> relies on the <i>Quant (MS1)</i> and <i>Quant
(Isobaric)</i> modules to generate a <i>psm.tsv</i> output file that contains
the quantitative data obtained from label-free and labelled experiments,
respectively:</span></p>

<h3><span lang=EN-US>Label-free experiments</span></h3>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Spectrum</span></i><span lang=EN-US>: Unique
     identifier for each <i>Peptide</i> sequence identified;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Modified Peptide: Peptide sequence including
     modifications, modified residues;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Protein: </span></i><span lang=EN-US>Identifier
     of the protein to which the <i>Peptide</i> sequence is ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Intensity: </span></i><span lang=EN-US>Precursor
     ion intensity (to be used in label-free experiments);</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Channel identifier</span></i><span lang=EN-US>: 
     Reporter ion intensity (to be used in labelled experiments).</span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
normal'><span lang=EN-US>&nbsp;</span></p>

<h3><span lang=EN-US>Labelled experiments</span></h3>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Spectrum</span></i><span lang=EN-US>: Unique
     identifier for each <i>Peptide</i> sequence identified;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Modified Peptide: Peptide sequence including
     modifications, modified residues;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Protein: </span></i><span lang=EN-US>Identifier
     of the protein to which the <i>Peptide</i> sequence is ascribed;</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Intensity: </span></i><span lang=EN-US>Precursor
     ion intensity (to be used in label-free experiments);</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><i><span lang=EN-US>Channel identifier</span></i><span lang=EN-US>: 
     Reporter ion intensity (to be used in labelled experiments).</span></li>
</ul>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><a name="_Hlk124849833"></a><a name="_Toc124328465"></a><a
name="_Adapting_the_results"></a><span lang=EN-US>Adapting the results from
proteomics pipelines for iSanXoT</span></h1>

</div>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><i><span lang=EN-US style='color:#BF8F00'>&nbsp;</span></i></p>

<p class=MsoNormal><span lang=EN-US>iSanXoT requires an
identification/quantification tab-separated values file (<i>ID-q.tsv</i>)
containing identified features together with their quantitative values and the
experiment they originate from. Users can either compose this <i>ID-q</i> file
manually (see Section above to learn how to do that with data from the four
most popular proteomics pipelines) or have it prepared by the iSanXoT Input
Adaptor. The latter option is described in this Section.</span></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><b><span
lang=EN-US style='font-size:16.0pt;line-height:107%'><img border=0 width=374
height=322 id="Picture 100" src="User_Guide_files/image088.jpg"></span></b></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773334"></a><a name="_Ref122706385"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>71</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Selecting an ID-q file in the Input Adaptor main
window.</span></i></p>

<p class=MsoNormalCxSpMiddle style='margin-left:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Open the iSanXoT application, then see Section above
     to learn how to get to the <i>Input Adaptor </i>main<i> </i>window (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 71</span></i><span lang=EN-US>).</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>If you already have an appropriate <i>ID-q</i>
     file, select the <i>Provide the Identification/Quantification file (ID-q) </i>option,
     then click on <i>Choose identification file</i> to select the file.</span></li>
</ul>

<p class=MsoNormalCxSpMiddle align=center style='margin-left:36.0pt;text-align:
center;text-indent:22.0pt;page-break-after:avoid'><span lang=EN-US><img
border=0 width=502 height=151 id="Picture 101"
src="User_Guide_files/image089.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773359"></a><a name="_Ref122706414"><i><span
lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span
lang=EN-US style='color:#BF8F00'>72</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Having the iSanXoT Input Adaptor prepare the ID-q file.</span></i></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>To have the iSanXoT Input Adaptor prepare the <i>ID-q</i>
     file from proteomics pipelines results (this works at least with MaxQuant,
     Trans-Proteomic Pipeline, FragPipe, and Proteome Discoverer), select the <i>Create
     adapted input file from proteomics pipelines results</i> option, then
     click on <i>Choose folder + Add anots</i> to select the folder containing
     the output files (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 72</span></i><span
     lang=EN-US>). A three-panel window will pop-up (</span><i><span lang=EN-US style='color:#BF8F00'>Figure 73</span></i><span lang=EN-US>).</span></li>
</ul>

<p class=MsoNormalCxSpMiddle style='margin-left:36.0pt'><span lang=EN-US>&nbsp;</span></p>

<p class=MsoNormalCxSpMiddle align=center style='margin-left:36.0pt;text-align:
center;page-break-after:avoid'><span lang=EN-US><img border=0 width=385
height=448 id="Picture 86" src="User_Guide_files/image090.png"></span></p>

<p class=MsoNormal align=center style='margin-bottom:10.0pt;text-align:center;
line-height:normal'><a name="_Ref123773412"></a><a name="_Ref123773368"></a><a
name="_Ref122706425"><i><span lang=EN-US style='color:#BF8F00'>Figure </span></i></a><i><span lang=EN-US style='color:#BF8F00'>73</span></i><i><span lang=EN-US
style='color:#BF8F00'>. Adapting results from a proteomics pipeline. In the top
panel several output files from Proteome Discoverer have been selected. These </span></i><span
lang=EN-US style='color:#BF8F00'>PSMs.txt<i> files, which contain
identification/quantification data, have been assigned an experiment name (</i>Jurkat<i>)
in the middle panel. The bottom panel has been used to create identifiers by
concatenating result file headers: </i>ScanID<i> (by concatenating </i>Spectrum
File, First Scan<i> and </i>Charge<i>) and </i>pepID<i> (by concatenating </i>Sequence<i>
and </i>Modifications<i>).</i></span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>The top panel displays the files included in the
     folder so that you can select one or more result files to be considered by
     iSanXoT Input Adaptor. Please bear in mind that if several result files
     are selected, these must necessarily have the same column headers.</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>The middle panel is used to set the distribution
     of data items across experiments according to result filenames.</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>The bottom panel allows to create identifiers by
     concatenating result file headers. It is composed of two interfaces:</span></li>
 <ul style='margin-top:0cm' type=circle>
  <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;
      line-height:normal'><span lang=EN-US>The headers found on the result
      files are listed on the left. Header names will be added to the interface
      on the right as they are selected;</span></li>
  <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;
      line-height:normal'><span lang=EN-US>The interface on the right displays
      the header names selected to generate the identifier as well as the
      identifier name provided by the user.</span></li>
 </ul>
</ul>

<p class=MsoNormalCxSpMiddle style='margin-left:36.0pt;text-indent:22.0pt'><span
lang=EN-US>&nbsp;</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Please note that the alphanumeric text that
     unambiguously identifies the items to be integrated is the only identifier
     that must be necessarily included in the <i>ID-q</i> file.</span></li>
 <li class=MsoNormal style='margin-bottom:0cm;margin-bottom:.0001pt;line-height:
     normal'><span lang=EN-US>Click the <i>Submit</i> button and the Input
     Adaptor will start generating the <i>ID-q.tsv</i> file.</span></li>
</ul>

<p class=MsoNormal><a name="_Adaptors"></a><a name="_Input_Adaptor"></a><span
lang=EN-US>&nbsp;</span></p>

<span lang=EN-US style='font-size:11.0pt;line-height:107%;font-family:"Calibri",sans-serif'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

<div style='border:none;border-top:solid windowtext 1.0pt;padding:1.0pt 0cm 0cm 0cm'>

<h1><span lang=ES>References</span></h1>

</div>

<p class=MsoNormal><span lang=ES>&nbsp;</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span lang=EN-US>[1] Trevisan-Herraz M, Bagwan N, Garcia-Marques F, Rodriguez JM,
Jorge I, Ezkurdia I, et al. SanXoT: a modular and versatile package for the
quantitative analysis of high-throughput proteomics experiments.
Bioinformatics. 2019;35(9):1594-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[2] Navarro P, Trevisan-Herraz M, Bonzon-Kulichenko E, Nunez E,
Martinez-Acedo P, Perez-Hernandez D, et al. General statistical framework for
quantitative proteomics by stable isotope labeling. J Proteome Res.
2014;13(3):1234-47.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[3] Garcia-Marques F, Trevisan-Herraz M, Martinez-Martinez S,
Camafeita E, Jorge I, Lopez JA, et al. A Novel Systems-Biology Algorithm for
the Analysis of Coordinated Protein Responses Using Quantitative Proteomics.
Mol Cell Proteomics. 2016;15(5):1740-60.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[4] Bonzon-Kulichenko E, Camafeita E, Lopez JA, Gomez-Serrano M,
Jorge I, Calvo E, et al. Improved integrative analysis of the thiol redox
proteome using filter-aided sample preparation. J Proteomics. 2020;214:103624.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[5] Gonzalez-Amor M, Garcia-Redondo AB, Jorge I, Zalba G, Becares M,
Ruiz-Rodriguez MJ, et al. Interferon stimulated gene 15 pathway is a novel
mediator of endothelial dysfunction and aneurysms development in angiotensin II
infused mice through increased oxidative stress. Cardiovasc Res. 2021.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[6] Navarro P, Kuharev J, Gillet LC, Bernhardt OM, MacLean B, Rost
HL, et al. A multicenter study benchmarks software tools for label-free
proteome quantification. Nat Biotechnol. 2016;34(11):1130-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[7] Navarro P, Vazquez J. A refined method to calculate false
discovery rates for peptide identification using decoy databases. J Proteome
Res. 2009;8(4):1792-6.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[8] Tyanova S, Temu T, Cox J. The MaxQuant computational platform
for mass spectrometry-based shotgun proteomics. Nat Protoc.
2016;11(12):2301-19.</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[9] Orsburn BC. Proteome Discoverer-A Community Enhanced Data
Processing Suite for Protein Informatics. Proteomes. 2021;9(1).</span></p>

<p class=EndNoteBibliography style='margin-bottom:0cm;margin-bottom:.0001pt'><span
lang=EN-US>[10] Deutsch EW, Mendoza L, Shteynberg D, Slagel J, Sun Z, Moritz
RL. Trans-Proteomic Pipeline, a standardized data processing pipeline for
large-scale reproducible proteomics informatics. Proteomics Clin Appl.
2015;9(7-8):745-54.</span></p>

<p class=EndNoteBibliography><span lang=EN-US>[11] Kong AT, Leprevost FV,
Avtonomov DM, Mellacheruvu D, Nesvizhskii AI. MSFragger: ultrafast and
comprehensive peptide identification in mass spectrometry-based proteomics. Nat
Methods. 2017;14(5):513-20.</span></p>

<p class=MsoNormal><span lang=EN-US>&nbsp;</span></p>

</div>

</body>

</html>
</template>
